

<!DOCTYPE html>


<html lang="en" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.19: https://docutils.sourceforge.io/" />

    <title>1. ColabFold Tutorial: Predicting Protein-Protein Complexes of TA Systems &#8212; Tutorials</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "light";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="../_static/styles/theme.css?digest=e353d410970836974a52" rel="stylesheet" />
<link href="../_static/styles/bootstrap.css?digest=e353d410970836974a52" rel="stylesheet" />
<link href="../_static/styles/pydata-sphinx-theme.css?digest=e353d410970836974a52" rel="stylesheet" />

  
  <link href="../_static/vendor/fontawesome/6.1.2/css/all.min.css?digest=e353d410970836974a52" rel="stylesheet" />
  <link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.1.2/webfonts/fa-solid-900.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.1.2/webfonts/fa-brands-400.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.1.2/webfonts/fa-regular-400.woff2" />

    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <link rel="stylesheet" href="../_static/styles/sphinx-book-theme.css?digest=14f4ca6b54d191a8c7657f6c759bf11a5fb86285" type="text/css" />
    <link rel="stylesheet" type="text/css" href="../_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/mystnb.4510f1fc1dee50b3e5859aac5469c37c29e427902b24a333a5f9fcb2f0b3ac41.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="../_static/design-style.4045f2051d55cab465a707391d5b2007.min.css" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../_static/scripts/bootstrap.js?digest=e353d410970836974a52" />
<link rel="preload" as="script" href="../_static/scripts/pydata-sphinx-theme.js?digest=e353d410970836974a52" />

    <script data-url_root="../" id="documentation_options" src="../_static/documentation_options.js"></script>
    <script src="../_static/jquery.js"></script>
    <script src="../_static/underscore.js"></script>
    <script src="../_static/_sphinx_javascript_frameworks_compat.js"></script>
    <script src="../_static/doctools.js"></script>
    <script src="../_static/sphinx_highlight.js"></script>
    <script src="../_static/clipboard.min.js"></script>
    <script src="../_static/copybutton.js"></script>
    <script src="../_static/scripts/sphinx-book-theme.js?digest=5a5c038af52cf7bc1a1ec88eea08e6366ee68824"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../_static/togglebutton.js"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="../_static/design-tabs.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="../_static/sphinx-thebe.js"></script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'ColabFold/Tutorial-Multimer';</script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="prev" title="1. ColabFold Tutorial: Predicting the structure of Bax protein" href="Tutorial-Monomer.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <a class="skip-link" href="#main-content">Skip to main content</a>
  
  <input type="checkbox"
          class="sidebar-toggle"
          name="__primary"
          id="__primary"/>
  <label class="overlay overlay-primary" for="__primary"></label>
  
  <input type="checkbox"
          class="sidebar-toggle"
          name="__secondary"
          id="__secondary"/>
  <label class="overlay overlay-secondary" for="__secondary"></label>
  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
<form class="bd-search d-flex align-items-center"
      action="../search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         id="search-input"
         placeholder="Search this book..."
         aria-label="Search this book..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form></div>
  </div>
  
    <nav class="bd-header navbar navbar-expand-lg bd-navbar">
    </nav>
  
  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      <div class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">
  

<a class="navbar-brand logo" href="../index.html">
  
  
  
  
    
    
      
    
    
    <img src="../_static/schrodinger_cat_white_bbg.png" class="logo__image only-light" alt="Logo image"/>
    <script>document.write(`<img src="../_static/schrodinger_cat_white_bbg.png" class="logo__image only-dark" alt="Logo image"/>`);</script>
  
  
</a></div>
        <div class="sidebar-primary-item"><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../index.html">
                    Tutorials
                </a>
            </li>
        </ul>
        <ul class="current nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="AlphaFold2.html">*Original ColabFold Notebook</a></li>

<li class="toctree-l1"><a class="reference internal" href="Tutorial-Monomer.html">1. <strong>ColabFold Tutorial:</strong> Predicting the structure of Bax protein</a></li>

<li class="toctree-l1 current active"><a class="current reference internal" href="#">1. <strong>ColabFold Tutorial:</strong> Predicting Protein-Protein Complexes of TA Systems</a></li>

</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
  </div>
  
  <div id="rtd-footer-container"></div>


      </div>
      
      <main id="main-content" class="bd-main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><label class="sidebar-toggle primary-toggle btn btn-sm" for="__primary" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</label></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">





<div class="dropdown dropdown-launch-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Launch interactive content">
    <i class="fas fa-rocket"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://colab.research.google.com/github/cc-ats/Tutorials/blob/master/./ColabFold/Tutorial-Multimer.ipynb" target="_blank"
   class="btn btn-sm dropdown-item"
   title="Launch onColab"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  
    <img src="../_static/images/logo_colab.png">
  </span>
<span class="btn__text-container">Colab</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-source-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Source repositories">
    <i class="fab fa-github"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://github.com/cc-ats/Tutorials" target="_blank"
   class="btn btn-sm btn-source-repository-button dropdown-item"
   title="Source repository"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="btn__text-container">Repository</span>
</a>
</li>
      
      
      
      
      <li><a href="https://github.com/cc-ats/Tutorials/issues/new?title=Issue%20on%20page%20%2FColabFold/Tutorial-Multimer.html&body=Your%20issue%20content%20here." target="_blank"
   class="btn btn-sm btn-source-issues-button dropdown-item"
   title="Open an issue"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="btn__text-container">Open issue</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="../_sources/ColabFold/Tutorial-Multimer.ipynb" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.ipynb</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>


<script>
document.write(`
  <button class="theme-switch-button btn btn-sm btn-outline-primary navbar-btn rounded-circle" title="light/dark" aria-label="light/dark" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="theme-switch" data-mode="light"><i class="fa-solid fa-sun"></i></span>
    <span class="theme-switch" data-mode="dark"><i class="fa-solid fa-moon"></i></span>
    <span class="theme-switch" data-mode="auto"><i class="fa-solid fa-circle-half-stroke"></i></span>
  </button>
`);
</script>

<script>
document.write(`
  <button class="btn btn-sm navbar-btn search-button search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass"></i>
  </button>
`);
</script>
<label class="sidebar-toggle secondary-toggle btn btn-sm" for="__secondary"title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</label>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>1. ColabFold Tutorial: Predicting Protein-Protein Complexes of TA Systems</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h1 nav-item toc-entry"><a class="reference internal nav-link" href="#">1. <strong>ColabFold Tutorial:</strong> Predicting Protein-Protein Complexes of TA Systems</a><ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#colabfold-v1-5-3-alphafold2-using-mmseqs2">ColabFold v1.5.3: AlphaFold2 using MMseqs2</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#news">News</a></li>
</ul>
</li>
</ul>
</li>
<li class="toc-h1 nav-item toc-entry"><a class="reference internal nav-link" href="#instructions-a-name-instructions-a">Instructions <a name="Instructions"></a></a></li>
</ul>

            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article" role="main">
                  
  <p><a href="https://colab.research.google.com/github/cc-ats/Tutorials/blob/main/ColabFold/Tutorial-Multimer.ipynb" target="_parent"><img alt="Open In Colab" src="https://colab.research.google.com/assets/colab-badge.svg" /></a></p>
<div class="cell docutils container">
<div class="cell_input docutils container">
<div class="highlight-ipython3 notranslate"><div class="highlight"><pre><span></span><span class="o">%%time</span>

<span class="c1"># @title **Install Python Packages** { run: &quot;auto&quot;, display-mode: &quot;form&quot; }</span>

<span class="kn">import</span> <span class="nn">os</span>
<span class="kn">from</span> <span class="nn">sys</span> <span class="kn">import</span> <span class="n">version_info</span>
<span class="n">PYTHON_VERSION</span> <span class="o">=</span> <span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">version_info</span><span class="o">.</span><span class="n">major</span><span class="si">}</span><span class="s2">.</span><span class="si">{</span><span class="n">version_info</span><span class="o">.</span><span class="n">minor</span><span class="si">}</span><span class="s2">&quot;</span>

<span class="k">if</span> <span class="ow">not</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">isfile</span><span class="p">(</span><span class="s2">&quot;COLABFOLD_READY&quot;</span><span class="p">):</span>
  <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;installing colabfold...&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;pip install -q --no-warn-conflicts &#39;colabfold[alphafold-minus-jax] @ git+https://github.com/sokrypton/ColabFold&#39;&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;pip install --upgrade dm-haiku&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;ln -s /usr/local/lib/python3.*/dist-packages/colabfold colabfold&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;ln -s /usr/local/lib/python3.*/dist-packages/alphafold alphafold&quot;</span><span class="p">)</span>
  <span class="c1"># patch for jax &gt; 0.3.25</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;sed -i &#39;s/weights = jax.nn.softmax(logits)/logits=jnp.clip(logits,-1e8,1e8);weights=jax.nn.softmax(logits)/g&#39; alphafold/model/modules.py&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;touch COLABFOLD_READY&quot;</span><span class="p">)</span>


<span class="k">if</span> <span class="ow">not</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">isfile</span><span class="p">(</span><span class="s2">&quot;CONDA_READY&quot;</span><span class="p">):</span>
  <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;installing conda...&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;wget -qnc https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;bash Miniforge3-Linux-x86_64.sh -bfp /usr/local&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;conda config --set auto_update_conda false&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;touch CONDA_READY&quot;</span><span class="p">)</span>


<span class="k">if</span> <span class="ow">not</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">isfile</span><span class="p">(</span><span class="s2">&quot;CONDA_READY&quot;</span><span class="p">):</span>
  <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;installing conda...&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;wget -qnc https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;bash Mambaforge-Linux-x86_64.sh -bfp /usr/local&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;mamba config --set auto_update_conda false&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;touch CONDA_READY&quot;</span><span class="p">)</span>


<span class="k">if</span> <span class="ow">not</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">isfile</span><span class="p">(</span><span class="s2">&quot;HH_READY&quot;</span><span class="p">)</span> <span class="ow">and</span> <span class="ow">not</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">isfile</span><span class="p">(</span><span class="s2">&quot;AMBER_READY&quot;</span><span class="p">):</span>
  <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;installing hhsuite and amber...&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;mamba install -y -c conda-forge -c bioconda kalign2=2.04 hhsuite=3.3.0 openmm=7.7.0 python=&#39;</span><span class="si">{</span><span class="n">PYTHON_VERSION</span><span class="si">}</span><span class="s2">&#39; pdbfixer&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;touch HH_READY&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;touch AMBER_READY&quot;</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
  <span class="k">if</span> <span class="ow">not</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">isfile</span><span class="p">(</span><span class="s2">&quot;HH_READY&quot;</span><span class="p">):</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;installing hhsuite...&quot;</span><span class="p">)</span>
    <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;mamba install -y -c conda-forge -c bioconda kalign2=2.04 hhsuite=3.3.0 python=&#39;</span><span class="si">{</span><span class="n">PYTHON_VERSION</span><span class="si">}</span><span class="s2">&#39;&quot;</span><span class="p">)</span>
    <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;touch HH_READY&quot;</span><span class="p">)</span>
  <span class="k">if</span> <span class="ow">not</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">isfile</span><span class="p">(</span><span class="s2">&quot;AMBER_READY&quot;</span><span class="p">):</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;installing amber...&quot;</span><span class="p">)</span>
    <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;mamba install -y -c conda-forge openmm=7.7.0 python=&#39;</span><span class="si">{</span><span class="n">PYTHON_VERSION</span><span class="si">}</span><span class="s2">&#39; pdbfixer&quot;</span><span class="p">)</span>
    <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;touch AMBER_READY&quot;</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
<section class="tex2jax_ignore mathjax_ignore" id="colabfold-tutorial-predicting-protein-protein-complexes-of-ta-systems">
<h1>1. <strong>ColabFold Tutorial:</strong> Predicting Protein-Protein Complexes of TA Systems<a class="headerlink" href="#colabfold-tutorial-predicting-protein-protein-complexes-of-ta-systems" title="Permalink to this heading">#</a></h1>
<p>Like the monomer example, we will use the default options set by Steinegger <em>et al.</em> which excludes Amber minimization and structure templates.</p>
<p>Predicting the structure of protein complexes can be investigated using the AlphaFold oligomers</p>
<p>There are subtle differences when running the AF-Multimer externsion:</p>
<ul class="simple">
<li><p>Model confidence metric is iPTM</p>
<ul>
<li><p>Interference predicted template modelling score</p></li>
</ul>
</li>
<li><p>Better for homomeric interfaces than for heteromeric interfaces (high conservation)</p></li>
<li><p>COM between chains are added to the loss function</p></li>
</ul>
<p>This example focuses on a toxin-antitoxin (TA) protein system, prevalent in bacterial systems. Toxins, or over-expressed proteins, become accumulated in the cell causing disturbing other cellular processes. Yet, their natural role aids in infections and apoptosis. Antitoxin, protein or RNA, can neutralize the toxin by forming stable complexes. How does the protein-protein interactions  maintianing a balance between the toxin and antitoxin.</p>
<p>We will predict the TA complex in <em>Vibrio cholerae</em>, specifically, VCParD1/2 (antitoxin) and VcParE (toxin)1/2.</p>
<p><strong>a. Background</strong></p>
<ul class="simple">
<li><p>TA systems are essential in maintaining cellular homeostasis</p></li>
<li><p>Low sequence similariy to maintain specificity towards cognate receptor</p></li>
<li><p>ParDE was the first TA system to be identified as a stabilizing element upon plasmisd loss</p></li>
<li><p>Potential in biotechnological applications</p></li>
</ul>
<p><strong>b. ParE attenuates Toxicity Protecting Cells</strong></p>
<ul class="simple">
<li><p>ParE can inhibit DNA gyrase supercoiling</p></li>
<li><p>Protects from anti-gyrase antibiotitcs</p></li>
<li><p>At the same time ParD is degraded</p></li>
<li><p>C-terminal domain is key to formation of PaParDE complexes</p></li>
</ul>
<p>The C-terminal residues of ParD was found to be crucial in formation and stabilizatino of toxin-antitoxin complexes. This resulted in protection from proteases and attenuates toxicitiy from antibiotics in bacterial systems.</p>
<p>Interestingly, structure-based studies have resolved several oligomeric TA complexes. Despite low sequence similarity, their structure is highly conserved. Letâ€™s investivate VcParDE and later compare it with PaParDE.</p>
<div class="cell docutils container">
<div class="cell_input docutils container">
<div class="highlight-ipython3 notranslate"><div class="highlight"><pre><span></span><span class="c1">#@title # **2. Notebook Setup** { display-mode: &quot;form&quot; }</span>

<span class="kn">from</span> <span class="nn">google.colab</span> <span class="kn">import</span> <span class="n">files</span>
<span class="kn">import</span> <span class="nn">os</span>
<span class="kn">import</span> <span class="nn">re</span>
<span class="kn">import</span> <span class="nn">hashlib</span>
<span class="kn">import</span> <span class="nn">random</span>

<span class="kn">from</span> <span class="nn">sys</span> <span class="kn">import</span> <span class="n">version_info</span>
<span class="n">python_version</span> <span class="o">=</span> <span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">version_info</span><span class="o">.</span><span class="n">major</span><span class="si">}</span><span class="s2">.</span><span class="si">{</span><span class="n">version_info</span><span class="o">.</span><span class="n">minor</span><span class="si">}</span><span class="s2">&quot;</span>

<span class="k">def</span> <span class="nf">add_hash</span><span class="p">(</span><span class="n">x</span><span class="p">,</span><span class="n">y</span><span class="p">):</span>
  <span class="k">return</span> <span class="n">x</span><span class="o">+</span><span class="s2">&quot;_&quot;</span><span class="o">+</span><span class="n">hashlib</span><span class="o">.</span><span class="n">sha1</span><span class="p">(</span><span class="n">y</span><span class="o">.</span><span class="n">encode</span><span class="p">())</span><span class="o">.</span><span class="n">hexdigest</span><span class="p">()[:</span><span class="mi">5</span><span class="p">]</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown ##a. Protein Sequence for Structure Prediction</span>
<span class="c1">#@markdown For bax (human), the amino acid sequence was obtained from</span>
<span class="c1">#@markdown UniProt ([Q07812](https://www.uniprot.org/uniprotkb/Q07812/entry))</span>
<span class="n">query_sequence</span> <span class="o">=</span> <span class="s1">&#39;MHTLTANDAKRNFGELLLSAQREPVIISKNSKNTVVVMSIKDFEELEAMKLDYLKHCFESAQKDLDSGKTVDGATFLNTL:MQNKQYKLSQLAQEHLLKIKHYTIENFAEAQWQKYKSTLLSGFQTLADNPGLGKSCEDIYQNGFYFPVGKHMAYYTKEANFILIVAVLGQSQLPQKHLKQSRFVS:MAKNTSITLGEHFDGFITSQIQSGRYGSASEVIRSALRLLENQETKLQSLRQLLIEGEQSGDADYDLDSFINELDSENI:MKPFNLTVAAKADLRDIALFTQRRWGKEQRNVYLKQFDDSFWLLAENPDIGKSCDEIREGYRKFPQGSHVIFYQQTGSQQIRVIRILHKSMDVNPIFGA &#39;</span> <span class="c1">#@param {type:&quot;string&quot;}</span>
<span class="c1">#@markdown **Note:** For modeling complexes, use `:` to specify inter-protein chainbreaks</span>
<span class="c1">#@markdown (supports homo- and hetro-oligomers).</span>
<span class="c1">#@markdown For example, **PI...SK:PI...SK** for a homodimer.</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown ## b. Job Name (also name of saving directory)</span>
<span class="n">jobname</span> <span class="o">=</span> <span class="s1">&#39;&#39;</span> <span class="c1">#@param {type:&quot;string&quot;}</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown  ## c. Number of predicted models to relax using Amber</span>
<span class="n">num_relax</span> <span class="o">=</span> <span class="mi">5</span> <span class="c1">#@param [0, 1, 5] {type:&quot;raw&quot;}</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown ## **d. Option for Structure Template**</span>
<span class="c1">#@markdown  Search for PDB structure (xyz-coordinates)</span>
<span class="n">template_mode</span> <span class="o">=</span> <span class="s2">&quot;none&quot;</span> <span class="c1">#@param [&quot;none&quot;, &quot;pdb100&quot;,&quot;custom&quot;]</span>
<span class="c1">#@markdown `none` = no template information is used. \</span>
<span class="c1">#@markdown `pdb100` = detect templates in pdb100 (see [notes](#pdb100)). \</span>
<span class="c1">#@markdown `custom` - upload and search own templates (PDB or mmCIF format, see [notes](#custom_templates))</span>

<span class="n">use_amber</span> <span class="o">=</span> <span class="n">num_relax</span> <span class="o">&gt;</span> <span class="mi">0</span>

<span class="n">query_sequence</span> <span class="o">=</span> <span class="s2">&quot;&quot;</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">query_sequence</span><span class="o">.</span><span class="n">split</span><span class="p">())</span> <span class="c1"># remove whitespaces</span>

<span class="n">basejobname</span> <span class="o">=</span> <span class="s2">&quot;&quot;</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">jobname</span><span class="o">.</span><span class="n">split</span><span class="p">())</span>
<span class="n">basejobname</span> <span class="o">=</span> <span class="n">re</span><span class="o">.</span><span class="n">sub</span><span class="p">(</span><span class="sa">r</span><span class="s1">&#39;\W+&#39;</span><span class="p">,</span> <span class="s1">&#39;&#39;</span><span class="p">,</span> <span class="n">basejobname</span><span class="p">)</span>
<span class="n">jobname</span> <span class="o">=</span> <span class="n">add_hash</span><span class="p">(</span><span class="n">basejobname</span><span class="p">,</span> <span class="n">query_sequence</span><span class="p">)</span>

<span class="c1"># check if directory with jobname exists</span>
<span class="k">def</span> <span class="nf">check</span><span class="p">(</span><span class="n">folder</span><span class="p">):</span>
  <span class="k">if</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">folder</span><span class="p">):</span>
    <span class="k">return</span> <span class="kc">False</span>
  <span class="k">else</span><span class="p">:</span>
    <span class="k">return</span> <span class="kc">True</span>
<span class="k">if</span> <span class="ow">not</span> <span class="n">check</span><span class="p">(</span><span class="n">jobname</span><span class="p">):</span>
  <span class="n">n</span> <span class="o">=</span> <span class="mi">0</span>
  <span class="k">while</span> <span class="ow">not</span> <span class="n">check</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">_</span><span class="si">{</span><span class="n">n</span><span class="si">}</span><span class="s2">&quot;</span><span class="p">):</span> <span class="n">n</span> <span class="o">+=</span> <span class="mi">1</span>
  <span class="n">jobname</span> <span class="o">=</span> <span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">_</span><span class="si">{</span><span class="n">n</span><span class="si">}</span><span class="s2">&quot;</span>

<span class="c1"># make directory to save results</span>
<span class="n">os</span><span class="o">.</span><span class="n">makedirs</span><span class="p">(</span><span class="n">jobname</span><span class="p">,</span> <span class="n">exist_ok</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>

<span class="c1"># save queries</span>
<span class="n">queries_path</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">jobname</span><span class="p">,</span> <span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.csv&quot;</span><span class="p">)</span>
<span class="k">with</span> <span class="nb">open</span><span class="p">(</span><span class="n">queries_path</span><span class="p">,</span> <span class="s2">&quot;w&quot;</span><span class="p">)</span> <span class="k">as</span> <span class="n">text_file</span><span class="p">:</span>
  <span class="n">text_file</span><span class="o">.</span><span class="n">write</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;id,sequence</span><span class="se">\n</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">,</span><span class="si">{</span><span class="n">query_sequence</span><span class="si">}</span><span class="s2">&quot;</span><span class="p">)</span>

<span class="k">if</span> <span class="n">template_mode</span> <span class="o">==</span> <span class="s2">&quot;pdb100&quot;</span><span class="p">:</span>
  <span class="n">use_templates</span> <span class="o">=</span> <span class="kc">True</span>
  <span class="n">custom_template_path</span> <span class="o">=</span> <span class="kc">None</span>
<span class="k">elif</span> <span class="n">template_mode</span> <span class="o">==</span> <span class="s2">&quot;custom&quot;</span><span class="p">:</span>
  <span class="n">custom_template_path</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">jobname</span><span class="p">,</span><span class="sa">f</span><span class="s2">&quot;template&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">makedirs</span><span class="p">(</span><span class="n">custom_template_path</span><span class="p">,</span> <span class="n">exist_ok</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
  <span class="n">uploaded</span> <span class="o">=</span> <span class="n">files</span><span class="o">.</span><span class="n">upload</span><span class="p">()</span>
  <span class="n">use_templates</span> <span class="o">=</span> <span class="kc">True</span>
  <span class="k">for</span> <span class="n">fn</span> <span class="ow">in</span> <span class="n">uploaded</span><span class="o">.</span><span class="n">keys</span><span class="p">():</span>
    <span class="n">os</span><span class="o">.</span><span class="n">rename</span><span class="p">(</span><span class="n">fn</span><span class="p">,</span><span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">custom_template_path</span><span class="p">,</span><span class="n">fn</span><span class="p">))</span>
<span class="k">else</span><span class="p">:</span>
  <span class="n">custom_template_path</span> <span class="o">=</span> <span class="kc">None</span>
  <span class="n">use_templates</span> <span class="o">=</span> <span class="kc">False</span>
</pre></div>
</div>
</div>
</div>
<div class="cell docutils container">
<div class="cell_input docutils container">
<div class="highlight-ipython3 notranslate"><div class="highlight"><pre><span></span><span class="c1">#@title # **3. Multiple Sequence Alignment (MSA)**</span>

<span class="c1">#@markdown ## a. MSA options</span>
<span class="n">msa_mode</span> <span class="o">=</span> <span class="s2">&quot;mmseqs2_uniref_env&quot;</span> <span class="c1">#@param [&quot;mmseqs2_uniref_env&quot;, &quot;mmseqs2_uniref&quot;,&quot;single_sequence&quot;,&quot;custom&quot;]</span>
<span class="c1">#@markdown - pairing mode</span>
<span class="c1">#@markdown - single sequence</span>
<span class="c1">#@markdown - custom MSA upload</span>

<span class="c1">#@markdown \</span>
<span class="n">pair_mode</span> <span class="o">=</span> <span class="s2">&quot;unpaired_paired&quot;</span> <span class="c1">#@param [&quot;unpaired_paired&quot;,&quot;unpaired&quot;,&quot;paired&quot;] {type:&quot;string&quot;}</span>
<span class="c1">#@markdown - &quot;unpaired_paired&quot; - pair sequences from same species + unpaired MSA</span>
<span class="c1">#@markdown - &quot;unpaired&quot; - seperate MSA for each chain</span>
<span class="c1">#@markdown - &quot;paired&quot; - only use paired sequences</span>

<span class="c1"># decide which a3m to use</span>
<span class="k">if</span> <span class="s2">&quot;mmseqs2&quot;</span> <span class="ow">in</span> <span class="n">msa_mode</span><span class="p">:</span>
  <span class="n">a3m_file</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">jobname</span><span class="p">,</span><span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.a3m&quot;</span><span class="p">)</span>

<span class="k">elif</span> <span class="n">msa_mode</span> <span class="o">==</span> <span class="s2">&quot;custom&quot;</span><span class="p">:</span>
  <span class="n">a3m_file</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">jobname</span><span class="p">,</span><span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.custom.a3m&quot;</span><span class="p">)</span>
  <span class="k">if</span> <span class="ow">not</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">isfile</span><span class="p">(</span><span class="n">a3m_file</span><span class="p">):</span>
    <span class="n">custom_msa_dict</span> <span class="o">=</span> <span class="n">files</span><span class="o">.</span><span class="n">upload</span><span class="p">()</span>
    <span class="n">custom_msa</span> <span class="o">=</span> <span class="nb">list</span><span class="p">(</span><span class="n">custom_msa_dict</span><span class="o">.</span><span class="n">keys</span><span class="p">())[</span><span class="mi">0</span><span class="p">]</span>
    <span class="n">header</span> <span class="o">=</span> <span class="mi">0</span>
    <span class="kn">import</span> <span class="nn">fileinput</span>
    <span class="k">for</span> <span class="n">line</span> <span class="ow">in</span> <span class="n">fileinput</span><span class="o">.</span><span class="n">FileInput</span><span class="p">(</span><span class="n">custom_msa</span><span class="p">,</span><span class="n">inplace</span><span class="o">=</span><span class="mi">1</span><span class="p">):</span>
      <span class="k">if</span> <span class="n">line</span><span class="o">.</span><span class="n">startswith</span><span class="p">(</span><span class="s2">&quot;&gt;&quot;</span><span class="p">):</span>
         <span class="n">header</span> <span class="o">=</span> <span class="n">header</span> <span class="o">+</span> <span class="mi">1</span>
      <span class="k">if</span> <span class="ow">not</span> <span class="n">line</span><span class="o">.</span><span class="n">rstrip</span><span class="p">():</span>
        <span class="k">continue</span>
      <span class="k">if</span> <span class="n">line</span><span class="o">.</span><span class="n">startswith</span><span class="p">(</span><span class="s2">&quot;&gt;&quot;</span><span class="p">)</span> <span class="o">==</span> <span class="kc">False</span> <span class="ow">and</span> <span class="n">header</span> <span class="o">==</span> <span class="mi">1</span><span class="p">:</span>
         <span class="n">query_sequence</span> <span class="o">=</span> <span class="n">line</span><span class="o">.</span><span class="n">rstrip</span><span class="p">()</span>
      <span class="nb">print</span><span class="p">(</span><span class="n">line</span><span class="p">,</span> <span class="n">end</span><span class="o">=</span><span class="s1">&#39;&#39;</span><span class="p">)</span>

    <span class="n">os</span><span class="o">.</span><span class="n">rename</span><span class="p">(</span><span class="n">custom_msa</span><span class="p">,</span> <span class="n">a3m_file</span><span class="p">)</span>
    <span class="n">queries_path</span><span class="o">=</span><span class="n">a3m_file</span>
    <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;moving </span><span class="si">{</span><span class="n">custom_msa</span><span class="si">}</span><span class="s2"> to </span><span class="si">{</span><span class="n">a3m_file</span><span class="si">}</span><span class="s2">&quot;</span><span class="p">)</span>

<span class="k">else</span><span class="p">:</span>
  <span class="n">a3m_file</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">jobname</span><span class="p">,</span><span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.single_sequence.a3m&quot;</span><span class="p">)</span>
  <span class="k">with</span> <span class="nb">open</span><span class="p">(</span><span class="n">a3m_file</span><span class="p">,</span> <span class="s2">&quot;w&quot;</span><span class="p">)</span> <span class="k">as</span> <span class="n">text_file</span><span class="p">:</span>
    <span class="n">text_file</span><span class="o">.</span><span class="n">write</span><span class="p">(</span><span class="s2">&quot;&gt;1</span><span class="se">\n</span><span class="si">%s</span><span class="s2">&quot;</span> <span class="o">%</span> <span class="n">query_sequence</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
<div class="cell docutils container">
<div class="cell_input docutils container">
<div class="highlight-ipython3 notranslate"><div class="highlight"><pre><span></span><span class="c1"># @title # **4. Advanced Settings** { run: &quot;auto&quot;, display-mode: &quot;form&quot; }</span>

<span class="c1">#@markdown ## a. ML Model</span>
<span class="n">model_type</span> <span class="o">=</span> <span class="s2">&quot;auto&quot;</span> <span class="c1">#@param [&quot;auto&quot;, &quot;alphafold2_ptm&quot;, &quot;alphafold2_multimer_v1&quot;, &quot;alphafold2_multimer_v2&quot;, &quot;alphafold2_multimer_v3&quot;]</span>
<span class="c1">#@markdown **Note:** If `auto` selected, will use `alphafold2_ptm` for monomer prediction</span>
<span class="c1">#@markdown and `alphafold2_multimer_v3` for complex prediction.</span>
<span class="c1">#@markdown Any of the `mode_types` can be used (regardless if input is monomer or complex).</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown ## b. Re-Train with Predicted Structures</span>
<span class="n">num_recycles</span> <span class="o">=</span> <span class="s2">&quot;0&quot;</span> <span class="c1">#@param [&quot;auto&quot;, &quot;0&quot;, &quot;1&quot;, &quot;3&quot;, &quot;6&quot;, &quot;12&quot;, &quot;24&quot;, &quot;48&quot;]</span>
<span class="c1">#@markdown **Note:**</span>
<span class="c1">#@markdown - If `auto` selected, will use `num_recycles=20`,</span>
<span class="c1">#@markdown - If `model_type=alphafold2_multimer_v3`, else `num_recycles=3`</span>

<span class="c1">#@markdown \</span>
<span class="n">recycle_early_stop_tolerance</span> <span class="o">=</span> <span class="s2">&quot;auto&quot;</span> <span class="c1">#@param [&quot;auto&quot;, &quot;0.0&quot;, &quot;0.5&quot;, &quot;1.0&quot;]</span>
<span class="c1">#@markdown - if `auto` selected, will use `tol=0.5`</span>
<span class="c1">#@markdown - If `model_type=alphafold2_multimer_v3`,  else `tol=0.0`.</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown ## c. Amber Relaxation</span>
<span class="n">relax_max_iterations</span> <span class="o">=</span> <span class="mi">200</span> <span class="c1">#@param [0, 200, 2000] {type:&quot;raw&quot;}</span>
<span class="c1">#@markdown - max amber relax iterations</span>
<span class="c1">#@markdown -  `0` = unlimited (AlphaFold2 default, can take very long)</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown ## d. MSA Pairing</span>
<span class="n">pairing_strategy</span> <span class="o">=</span> <span class="s2">&quot;greedy&quot;</span> <span class="c1">#@param [&quot;greedy&quot;, &quot;complete&quot;] {type:&quot;string&quot;}</span>
<span class="c1">#@markdown - `greedy` = pair any taxonomically matching subsets,</span>
<span class="c1">#@markdown - `complete` = all sequences have to match in one line.</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown ## e. Sample settings</span>
<span class="c1">#@markdown -  enable dropouts and increase number of seeds to sample predictions from uncertainty of the model.</span>
<span class="c1">#@markdown -  decrease `max_msa` to increase uncertainity</span>
<span class="n">max_msa</span> <span class="o">=</span> <span class="s2">&quot;auto&quot;</span> <span class="c1">#@param [&quot;auto&quot;, &quot;512:1024&quot;, &quot;256:512&quot;, &quot;64:128&quot;, &quot;32:64&quot;, &quot;16:32&quot;]</span>
<span class="n">num_seeds</span> <span class="o">=</span> <span class="mi">1</span> <span class="c1">#@param [1,2,4,8,16] {type:&quot;raw&quot;}</span>
<span class="n">use_dropout</span> <span class="o">=</span> <span class="kc">False</span> <span class="c1">#@param {type:&quot;boolean&quot;}</span>

<span class="n">num_recycles</span> <span class="o">=</span> <span class="kc">None</span> <span class="k">if</span> <span class="n">num_recycles</span> <span class="o">==</span> <span class="s2">&quot;auto&quot;</span> <span class="k">else</span> <span class="nb">int</span><span class="p">(</span><span class="n">num_recycles</span><span class="p">)</span>
<span class="n">recycle_early_stop_tolerance</span> <span class="o">=</span> <span class="kc">None</span> <span class="k">if</span> <span class="n">recycle_early_stop_tolerance</span> <span class="o">==</span> <span class="s2">&quot;auto&quot;</span> <span class="k">else</span> <span class="nb">float</span><span class="p">(</span><span class="n">recycle_early_stop_tolerance</span><span class="p">)</span>
<span class="k">if</span> <span class="n">max_msa</span> <span class="o">==</span> <span class="s2">&quot;auto&quot;</span><span class="p">:</span> <span class="n">max_msa</span> <span class="o">=</span> <span class="kc">None</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown # **5. Save settings**</span>
<span class="c1">#@markdown This will ask for your credentials, to save the outputs.</span>
<span class="n">save_all</span> <span class="o">=</span> <span class="kc">False</span> <span class="c1">#@param {type:&quot;boolean&quot;}</span>
<span class="n">save_recycles</span> <span class="o">=</span> <span class="kc">False</span> <span class="c1">#@param {type:&quot;boolean&quot;}</span>
<span class="n">save_to_google_drive</span> <span class="o">=</span> <span class="kc">False</span> <span class="c1">#@param {type:&quot;boolean&quot;}</span>
<span class="c1">#@markdown -  if the save_to_google_drive option was selected, the result zip will be uploaded to your Google Drive</span>
<span class="n">dpi</span> <span class="o">=</span> <span class="mi">200</span> <span class="c1">#@param {type:&quot;integer&quot;}</span>
<span class="c1">#@markdown - set dpi for image resolution</span>

<span class="k">if</span> <span class="n">save_to_google_drive</span><span class="p">:</span>
  <span class="kn">from</span> <span class="nn">pydrive.drive</span> <span class="kn">import</span> <span class="n">GoogleDrive</span>
  <span class="kn">from</span> <span class="nn">pydrive.auth</span> <span class="kn">import</span> <span class="n">GoogleAuth</span>
  <span class="kn">from</span> <span class="nn">google.colab</span> <span class="kn">import</span> <span class="n">auth</span>
  <span class="kn">from</span> <span class="nn">oauth2client.client</span> <span class="kn">import</span> <span class="n">GoogleCredentials</span>
  <span class="n">auth</span><span class="o">.</span><span class="n">authenticate_user</span><span class="p">()</span>
  <span class="n">gauth</span> <span class="o">=</span> <span class="n">GoogleAuth</span><span class="p">()</span>
  <span class="n">gauth</span><span class="o">.</span><span class="n">credentials</span> <span class="o">=</span> <span class="n">GoogleCredentials</span><span class="o">.</span><span class="n">get_application_default</span><span class="p">()</span>
  <span class="n">drive</span> <span class="o">=</span> <span class="n">GoogleDrive</span><span class="p">(</span><span class="n">gauth</span><span class="p">)</span>
  <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;You are logged into Google Drive and are good to go!&quot;</span><span class="p">)</span>

<span class="c1">#@markdown Don&#39;t forget to hit `Runtime` -&gt; `Run all` after updating the form.</span>
</pre></div>
</div>
</div>
</div>
<div class="cell docutils container">
<div class="cell_input docutils container">
<div class="highlight-ipython3 notranslate"><div class="highlight"><pre><span></span><span class="c1">#@title # **6. Run Prediction** { display-mode: &quot;form&quot; }</span>

<span class="n">display_images</span> <span class="o">=</span> <span class="kc">True</span> <span class="c1">#@param {type:&quot;boolean&quot;}</span>


<span class="kn">import</span> <span class="nn">sys</span>
<span class="kn">import</span> <span class="nn">warnings</span>
<span class="n">warnings</span><span class="o">.</span><span class="n">simplefilter</span><span class="p">(</span><span class="n">action</span><span class="o">=</span><span class="s1">&#39;ignore&#39;</span><span class="p">,</span> <span class="n">category</span><span class="o">=</span><span class="ne">FutureWarning</span><span class="p">)</span>
<span class="kn">from</span> <span class="nn">Bio</span> <span class="kn">import</span> <span class="n">BiopythonDeprecationWarning</span>
<span class="n">warnings</span><span class="o">.</span><span class="n">simplefilter</span><span class="p">(</span><span class="n">action</span><span class="o">=</span><span class="s1">&#39;ignore&#39;</span><span class="p">,</span> <span class="n">category</span><span class="o">=</span><span class="n">BiopythonDeprecationWarning</span><span class="p">)</span>
<span class="kn">from</span> <span class="nn">pathlib</span> <span class="kn">import</span> <span class="n">Path</span>
<span class="kn">from</span> <span class="nn">colabfold.download</span> <span class="kn">import</span> <span class="n">download_alphafold_params</span><span class="p">,</span> <span class="n">default_data_dir</span>
<span class="kn">from</span> <span class="nn">colabfold.utils</span> <span class="kn">import</span> <span class="n">setup_logging</span>
<span class="kn">from</span> <span class="nn">colabfold.batch</span> <span class="kn">import</span> <span class="n">get_queries</span><span class="p">,</span> <span class="n">run</span><span class="p">,</span> <span class="n">set_model_type</span>
<span class="kn">from</span> <span class="nn">colabfold.plot</span> <span class="kn">import</span> <span class="n">plot_msa_v2</span>

<span class="kn">import</span> <span class="nn">os</span>
<span class="kn">import</span> <span class="nn">numpy</span> <span class="k">as</span> <span class="nn">np</span>
<span class="k">try</span><span class="p">:</span>
  <span class="n">K80_chk</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">popen</span><span class="p">(</span><span class="s1">&#39;nvidia-smi | grep &quot;Tesla K80&quot; | wc -l&#39;</span><span class="p">)</span><span class="o">.</span><span class="n">read</span><span class="p">()</span>
<span class="k">except</span><span class="p">:</span>
  <span class="n">K80_chk</span> <span class="o">=</span> <span class="s2">&quot;0&quot;</span>
  <span class="k">pass</span>
<span class="k">if</span> <span class="s2">&quot;1&quot;</span> <span class="ow">in</span> <span class="n">K80_chk</span><span class="p">:</span>
  <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;WARNING: found GPU Tesla K80: limited to total length &lt; 1000&quot;</span><span class="p">)</span>
  <span class="k">if</span> <span class="s2">&quot;TF_FORCE_UNIFIED_MEMORY&quot;</span> <span class="ow">in</span> <span class="n">os</span><span class="o">.</span><span class="n">environ</span><span class="p">:</span>
    <span class="k">del</span> <span class="n">os</span><span class="o">.</span><span class="n">environ</span><span class="p">[</span><span class="s2">&quot;TF_FORCE_UNIFIED_MEMORY&quot;</span><span class="p">]</span>
  <span class="k">if</span> <span class="s2">&quot;XLA_PYTHON_CLIENT_MEM_FRACTION&quot;</span> <span class="ow">in</span> <span class="n">os</span><span class="o">.</span><span class="n">environ</span><span class="p">:</span>
    <span class="k">del</span> <span class="n">os</span><span class="o">.</span><span class="n">environ</span><span class="p">[</span><span class="s2">&quot;XLA_PYTHON_CLIENT_MEM_FRACTION&quot;</span><span class="p">]</span>

<span class="kn">from</span> <span class="nn">colabfold.colabfold</span> <span class="kn">import</span> <span class="n">plot_protein</span>
<span class="kn">from</span> <span class="nn">pathlib</span> <span class="kn">import</span> <span class="n">Path</span>
<span class="kn">import</span> <span class="nn">matplotlib.pyplot</span> <span class="k">as</span> <span class="nn">plt</span>

<span class="c1"># For some reason we need that to get pdbfixer to import</span>
<span class="k">if</span> <span class="n">use_amber</span> <span class="ow">and</span> <span class="sa">f</span><span class="s2">&quot;/usr/local/lib/python</span><span class="si">{</span><span class="n">python_version</span><span class="si">}</span><span class="s2">/site-packages/&quot;</span> <span class="ow">not</span> <span class="ow">in</span> <span class="n">sys</span><span class="o">.</span><span class="n">path</span><span class="p">:</span>
    <span class="n">sys</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">insert</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span> <span class="sa">f</span><span class="s2">&quot;/usr/local/lib/python</span><span class="si">{</span><span class="n">python_version</span><span class="si">}</span><span class="s2">/site-packages/&quot;</span><span class="p">)</span>

<span class="k">def</span> <span class="nf">input_features_callback</span><span class="p">(</span><span class="n">input_features</span><span class="p">):</span>
  <span class="k">if</span> <span class="n">display_images</span><span class="p">:</span>
    <span class="n">plot_msa_v2</span><span class="p">(</span><span class="n">input_features</span><span class="p">)</span>
    <span class="n">plt</span><span class="o">.</span><span class="n">show</span><span class="p">()</span>
    <span class="n">plt</span><span class="o">.</span><span class="n">close</span><span class="p">()</span>

<span class="k">def</span> <span class="nf">prediction_callback</span><span class="p">(</span><span class="n">protein_obj</span><span class="p">,</span> <span class="n">length</span><span class="p">,</span>
                        <span class="n">prediction_result</span><span class="p">,</span> <span class="n">input_features</span><span class="p">,</span> <span class="n">mode</span><span class="p">):</span>
  <span class="n">model_name</span><span class="p">,</span> <span class="n">relaxed</span> <span class="o">=</span> <span class="n">mode</span>
  <span class="k">if</span> <span class="ow">not</span> <span class="n">relaxed</span><span class="p">:</span>
    <span class="k">if</span> <span class="n">display_images</span><span class="p">:</span>
      <span class="n">fig</span> <span class="o">=</span> <span class="n">plot_protein</span><span class="p">(</span><span class="n">protein_obj</span><span class="p">,</span> <span class="n">Ls</span><span class="o">=</span><span class="n">length</span><span class="p">,</span> <span class="n">dpi</span><span class="o">=</span><span class="mi">150</span><span class="p">)</span>
      <span class="n">plt</span><span class="o">.</span><span class="n">show</span><span class="p">()</span>
      <span class="n">plt</span><span class="o">.</span><span class="n">close</span><span class="p">()</span>

<span class="n">result_dir</span> <span class="o">=</span> <span class="n">jobname</span>
<span class="n">log_filename</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">jobname</span><span class="p">,</span><span class="s2">&quot;log.txt&quot;</span><span class="p">)</span>
<span class="n">setup_logging</span><span class="p">(</span><span class="n">Path</span><span class="p">(</span><span class="n">log_filename</span><span class="p">))</span>

<span class="n">queries</span><span class="p">,</span> <span class="n">is_complex</span> <span class="o">=</span> <span class="n">get_queries</span><span class="p">(</span><span class="n">queries_path</span><span class="p">)</span>
<span class="n">model_type</span> <span class="o">=</span> <span class="n">set_model_type</span><span class="p">(</span><span class="n">is_complex</span><span class="p">,</span> <span class="n">model_type</span><span class="p">)</span>

<span class="k">if</span> <span class="s2">&quot;multimer&quot;</span> <span class="ow">in</span> <span class="n">model_type</span> <span class="ow">and</span> <span class="n">max_msa</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span><span class="p">:</span>
  <span class="n">use_cluster_profile</span> <span class="o">=</span> <span class="kc">False</span>
<span class="k">else</span><span class="p">:</span>
  <span class="n">use_cluster_profile</span> <span class="o">=</span> <span class="kc">True</span>

<span class="n">download_alphafold_params</span><span class="p">(</span><span class="n">model_type</span><span class="p">,</span> <span class="n">Path</span><span class="p">(</span><span class="s2">&quot;.&quot;</span><span class="p">))</span>
<span class="n">results</span> <span class="o">=</span> <span class="n">run</span><span class="p">(</span>
    <span class="n">queries</span><span class="o">=</span><span class="n">queries</span><span class="p">,</span>
    <span class="n">result_dir</span><span class="o">=</span><span class="n">result_dir</span><span class="p">,</span>
    <span class="n">use_templates</span><span class="o">=</span><span class="n">use_templates</span><span class="p">,</span>
    <span class="n">custom_template_path</span><span class="o">=</span><span class="n">custom_template_path</span><span class="p">,</span>
    <span class="n">num_relax</span><span class="o">=</span><span class="n">num_relax</span><span class="p">,</span>
    <span class="n">msa_mode</span><span class="o">=</span><span class="n">msa_mode</span><span class="p">,</span>
    <span class="n">model_type</span><span class="o">=</span><span class="n">model_type</span><span class="p">,</span>
    <span class="n">num_models</span><span class="o">=</span><span class="mi">5</span><span class="p">,</span>
    <span class="n">num_recycles</span><span class="o">=</span><span class="n">num_recycles</span><span class="p">,</span>
    <span class="n">relax_max_iterations</span><span class="o">=</span><span class="n">relax_max_iterations</span><span class="p">,</span>
    <span class="n">recycle_early_stop_tolerance</span><span class="o">=</span><span class="n">recycle_early_stop_tolerance</span><span class="p">,</span>
    <span class="n">num_seeds</span><span class="o">=</span><span class="n">num_seeds</span><span class="p">,</span>
    <span class="n">use_dropout</span><span class="o">=</span><span class="n">use_dropout</span><span class="p">,</span>
    <span class="n">model_order</span><span class="o">=</span><span class="p">[</span><span class="mi">1</span><span class="p">,</span><span class="mi">2</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span><span class="mi">4</span><span class="p">,</span><span class="mi">5</span><span class="p">],</span>
    <span class="n">is_complex</span><span class="o">=</span><span class="n">is_complex</span><span class="p">,</span>
    <span class="n">data_dir</span><span class="o">=</span><span class="n">Path</span><span class="p">(</span><span class="s2">&quot;.&quot;</span><span class="p">),</span>
    <span class="n">keep_existing_results</span><span class="o">=</span><span class="kc">False</span><span class="p">,</span>
    <span class="n">rank_by</span><span class="o">=</span><span class="s2">&quot;auto&quot;</span><span class="p">,</span>
    <span class="n">pair_mode</span><span class="o">=</span><span class="n">pair_mode</span><span class="p">,</span>
    <span class="n">pairing_strategy</span><span class="o">=</span><span class="n">pairing_strategy</span><span class="p">,</span>
    <span class="n">stop_at_score</span><span class="o">=</span><span class="nb">float</span><span class="p">(</span><span class="mi">100</span><span class="p">),</span>
    <span class="n">prediction_callback</span><span class="o">=</span><span class="n">prediction_callback</span><span class="p">,</span>
    <span class="n">dpi</span><span class="o">=</span><span class="n">dpi</span><span class="p">,</span>
    <span class="n">zip_results</span><span class="o">=</span><span class="kc">False</span><span class="p">,</span>
    <span class="n">save_all</span><span class="o">=</span><span class="n">save_all</span><span class="p">,</span>
    <span class="n">max_msa</span><span class="o">=</span><span class="n">max_msa</span><span class="p">,</span>
    <span class="n">use_cluster_profile</span><span class="o">=</span><span class="n">use_cluster_profile</span><span class="p">,</span>
    <span class="n">input_features_callback</span><span class="o">=</span><span class="n">input_features_callback</span><span class="p">,</span>
    <span class="n">save_recycles</span><span class="o">=</span><span class="n">save_recycles</span><span class="p">,</span>
    <span class="n">user_agent</span><span class="o">=</span><span class="s2">&quot;colabfold/google-colab-main&quot;</span><span class="p">,</span>
<span class="p">)</span>
<span class="n">results_zip</span> <span class="o">=</span> <span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.result.zip&quot;</span>
<span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;zip -r </span><span class="si">{</span><span class="n">results_zip</span><span class="si">}</span><span class="s2"> </span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">&quot;</span><span class="p">)</span>
</pre></div>
</div>
</div>
<div class="cell_output docutils container">
<div class="output stream highlight-myst-ansi notranslate"><div class="highlight"><pre><span></span>2023-12-08 03:34:55,968 alphafold2_multimer_v3_model_2_seed_000 recycle=18 pLDDT=85.8 pTM=0.479 ipTM=0.372 tol=0.629
2023-12-08 03:35:09,602 alphafold2_multimer_v3_model_2_seed_000 recycle=19 pLDDT=85.5 pTM=0.478 ipTM=0.374 tol=1.08
</pre></div>
</div>
</div>
</div>
<div class="cell docutils container">
<div class="cell_input docutils container">
<div class="highlight-ipython3 notranslate"><div class="highlight"><pre><span></span><span class="c1"># @title # **7. Visualize 3D Structure of ColabFold Prediction** { run: &quot;auto&quot;, display-mode: &quot;form&quot; }</span>

<span class="c1">#@markdown Rank number/score for the **monomer model is the average pLDDT**,</span>
<span class="c1">#@markdown where a ranking of 1 correlates better local structure confidence.</span>
<span class="c1">#@markdown An acceptable model prediction should average $\ge 85$.</span>
<span class="c1">#@markdown The number of ranked models was set in step `6. Run Prediction` (step 6, line 75.).</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown Here, you can change the value for `rank_num` to visualize the 5 predictions.</span>

<span class="kn">import</span> <span class="nn">py3Dmol</span>
<span class="kn">import</span> <span class="nn">glob</span>
<span class="kn">import</span> <span class="nn">matplotlib.pyplot</span> <span class="k">as</span> <span class="nn">plt</span>
<span class="kn">from</span> <span class="nn">colabfold.colabfold</span> <span class="kn">import</span> <span class="n">plot_plddt_legend</span>
<span class="kn">from</span> <span class="nn">colabfold.colabfold</span> <span class="kn">import</span> <span class="n">pymol_color_list</span><span class="p">,</span> <span class="n">alphabet_list</span>

<span class="n">rank_num</span> <span class="o">=</span> <span class="mi">1</span> <span class="c1">#@param [&quot;1&quot;, &quot;2&quot;, &quot;3&quot;, &quot;4&quot;, &quot;5&quot;] {type:&quot;raw&quot;}</span>
<span class="n">color</span> <span class="o">=</span> <span class="s2">&quot;rainbow&quot;</span> <span class="c1">#@param [&quot;chain&quot;, &quot;lDDT&quot;, &quot;rainbow&quot;]</span>
<span class="n">show_sidechains</span> <span class="o">=</span> <span class="kc">False</span> <span class="c1">#@param {type:&quot;boolean&quot;}</span>
<span class="n">show_mainchains</span> <span class="o">=</span> <span class="kc">False</span> <span class="c1">#@param {type:&quot;boolean&quot;}</span>

<span class="n">tag</span> <span class="o">=</span> <span class="n">results</span><span class="p">[</span><span class="s2">&quot;rank&quot;</span><span class="p">][</span><span class="mi">0</span><span class="p">][</span><span class="n">rank_num</span> <span class="o">-</span> <span class="mi">1</span><span class="p">]</span>
<span class="n">jobname_prefix</span> <span class="o">=</span> <span class="s2">&quot;.custom&quot;</span> <span class="k">if</span> <span class="n">msa_mode</span> <span class="o">==</span> <span class="s2">&quot;custom&quot;</span> <span class="k">else</span> <span class="s2">&quot;&quot;</span>
<span class="n">pdb_filename</span> <span class="o">=</span> <span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">/</span><span class="si">{</span><span class="n">jobname</span><span class="si">}{</span><span class="n">jobname_prefix</span><span class="si">}</span><span class="s2">_unrelaxed_</span><span class="si">{</span><span class="n">tag</span><span class="si">}</span><span class="s2">.pdb&quot;</span>
<span class="n">pdb_file</span> <span class="o">=</span> <span class="n">glob</span><span class="o">.</span><span class="n">glob</span><span class="p">(</span><span class="n">pdb_filename</span><span class="p">)</span>

<span class="k">def</span> <span class="nf">show_pdb</span><span class="p">(</span><span class="n">rank_num</span><span class="o">=</span><span class="mi">1</span><span class="p">,</span> <span class="n">show_sidechains</span><span class="o">=</span><span class="kc">False</span><span class="p">,</span> <span class="n">show_mainchains</span><span class="o">=</span><span class="kc">False</span><span class="p">,</span> <span class="n">color</span><span class="o">=</span><span class="s2">&quot;lDDT&quot;</span><span class="p">):</span>
  <span class="n">model_name</span> <span class="o">=</span> <span class="sa">f</span><span class="s2">&quot;rank_</span><span class="si">{</span><span class="n">rank_num</span><span class="si">}</span><span class="s2">&quot;</span>
  <span class="n">view</span> <span class="o">=</span> <span class="n">py3Dmol</span><span class="o">.</span><span class="n">view</span><span class="p">(</span><span class="n">js</span><span class="o">=</span><span class="s1">&#39;https://3dmol.org/build/3Dmol.js&#39;</span><span class="p">,)</span>
  <span class="n">view</span><span class="o">.</span><span class="n">addModel</span><span class="p">(</span><span class="nb">open</span><span class="p">(</span><span class="n">pdb_file</span><span class="p">[</span><span class="mi">0</span><span class="p">],</span><span class="s1">&#39;r&#39;</span><span class="p">)</span><span class="o">.</span><span class="n">read</span><span class="p">(),</span><span class="s1">&#39;pdb&#39;</span><span class="p">)</span>


  <span class="k">if</span> <span class="n">color</span> <span class="o">==</span> <span class="s2">&quot;lDDT&quot;</span><span class="p">:</span>
    <span class="n">view</span><span class="o">.</span><span class="n">setStyle</span><span class="p">({</span><span class="s1">&#39;cartoon&#39;</span><span class="p">:</span> <span class="p">{</span><span class="s1">&#39;colorscheme&#39;</span><span class="p">:</span> <span class="p">{</span><span class="s1">&#39;prop&#39;</span><span class="p">:</span><span class="s1">&#39;b&#39;</span><span class="p">,</span><span class="s1">&#39;gradient&#39;</span><span class="p">:</span> <span class="s1">&#39;roygb&#39;</span><span class="p">,</span><span class="s1">&#39;min&#39;</span><span class="p">:</span><span class="mi">50</span><span class="p">,</span><span class="s1">&#39;max&#39;</span><span class="p">:</span><span class="mi">90</span><span class="p">}}})</span>
  <span class="k">elif</span> <span class="n">color</span> <span class="o">==</span> <span class="s2">&quot;rainbow&quot;</span><span class="p">:</span>
    <span class="n">view</span><span class="o">.</span><span class="n">setStyle</span><span class="p">({</span><span class="s1">&#39;cartoon&#39;</span><span class="p">:</span> <span class="p">{</span><span class="s1">&#39;color&#39;</span><span class="p">:</span><span class="s1">&#39;spectrum&#39;</span><span class="p">}})</span>
  <span class="k">elif</span> <span class="n">color</span> <span class="o">==</span> <span class="s2">&quot;chain&quot;</span><span class="p">:</span>
    <span class="n">chains</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">queries</span><span class="p">[</span><span class="mi">0</span><span class="p">][</span><span class="mi">1</span><span class="p">])</span> <span class="o">+</span> <span class="mi">1</span> <span class="k">if</span> <span class="n">is_complex</span> <span class="k">else</span> <span class="mi">1</span>
    <span class="k">for</span> <span class="n">n</span><span class="p">,</span><span class="n">chain</span><span class="p">,</span><span class="n">color</span> <span class="ow">in</span> <span class="nb">zip</span><span class="p">(</span><span class="nb">range</span><span class="p">(</span><span class="n">chains</span><span class="p">),</span><span class="n">alphabet_list</span><span class="p">,</span><span class="n">pymol_color_list</span><span class="p">):</span>
       <span class="n">view</span><span class="o">.</span><span class="n">setStyle</span><span class="p">({</span><span class="s1">&#39;chain&#39;</span><span class="p">:</span><span class="n">chain</span><span class="p">},{</span><span class="s1">&#39;cartoon&#39;</span><span class="p">:</span> <span class="p">{</span><span class="s1">&#39;color&#39;</span><span class="p">:</span><span class="n">color</span><span class="p">}})</span>

  <span class="k">if</span> <span class="n">show_sidechains</span><span class="p">:</span>
    <span class="n">BB</span> <span class="o">=</span> <span class="p">[</span><span class="s1">&#39;C&#39;</span><span class="p">,</span><span class="s1">&#39;O&#39;</span><span class="p">,</span><span class="s1">&#39;N&#39;</span><span class="p">]</span>
    <span class="n">view</span><span class="o">.</span><span class="n">addStyle</span><span class="p">({</span><span class="s1">&#39;and&#39;</span><span class="p">:[{</span><span class="s1">&#39;resn&#39;</span><span class="p">:[</span><span class="s2">&quot;GLY&quot;</span><span class="p">,</span><span class="s2">&quot;PRO&quot;</span><span class="p">],</span><span class="s1">&#39;invert&#39;</span><span class="p">:</span><span class="kc">True</span><span class="p">},{</span><span class="s1">&#39;atom&#39;</span><span class="p">:</span><span class="n">BB</span><span class="p">,</span><span class="s1">&#39;invert&#39;</span><span class="p">:</span><span class="kc">True</span><span class="p">}]},</span>
                        <span class="p">{</span><span class="s1">&#39;stick&#39;</span><span class="p">:{</span><span class="s1">&#39;colorscheme&#39;</span><span class="p">:</span><span class="sa">f</span><span class="s2">&quot;WhiteCarbon&quot;</span><span class="p">,</span><span class="s1">&#39;radius&#39;</span><span class="p">:</span><span class="mf">0.3</span><span class="p">}})</span>
    <span class="n">view</span><span class="o">.</span><span class="n">addStyle</span><span class="p">({</span><span class="s1">&#39;and&#39;</span><span class="p">:[{</span><span class="s1">&#39;resn&#39;</span><span class="p">:</span><span class="s2">&quot;GLY&quot;</span><span class="p">},{</span><span class="s1">&#39;atom&#39;</span><span class="p">:</span><span class="s1">&#39;CA&#39;</span><span class="p">}]},</span>
                        <span class="p">{</span><span class="s1">&#39;sphere&#39;</span><span class="p">:{</span><span class="s1">&#39;colorscheme&#39;</span><span class="p">:</span><span class="sa">f</span><span class="s2">&quot;WhiteCarbon&quot;</span><span class="p">,</span><span class="s1">&#39;radius&#39;</span><span class="p">:</span><span class="mf">0.3</span><span class="p">}})</span>
    <span class="n">view</span><span class="o">.</span><span class="n">addStyle</span><span class="p">({</span><span class="s1">&#39;and&#39;</span><span class="p">:[{</span><span class="s1">&#39;resn&#39;</span><span class="p">:</span><span class="s2">&quot;PRO&quot;</span><span class="p">},{</span><span class="s1">&#39;atom&#39;</span><span class="p">:[</span><span class="s1">&#39;C&#39;</span><span class="p">,</span><span class="s1">&#39;O&#39;</span><span class="p">],</span><span class="s1">&#39;invert&#39;</span><span class="p">:</span><span class="kc">True</span><span class="p">}]},</span>
                        <span class="p">{</span><span class="s1">&#39;stick&#39;</span><span class="p">:{</span><span class="s1">&#39;colorscheme&#39;</span><span class="p">:</span><span class="sa">f</span><span class="s2">&quot;WhiteCarbon&quot;</span><span class="p">,</span><span class="s1">&#39;radius&#39;</span><span class="p">:</span><span class="mf">0.3</span><span class="p">}})</span>
  <span class="k">if</span> <span class="n">show_mainchains</span><span class="p">:</span>
    <span class="n">BB</span> <span class="o">=</span> <span class="p">[</span><span class="s1">&#39;C&#39;</span><span class="p">,</span><span class="s1">&#39;O&#39;</span><span class="p">,</span><span class="s1">&#39;N&#39;</span><span class="p">,</span><span class="s1">&#39;CA&#39;</span><span class="p">]</span>
    <span class="n">view</span><span class="o">.</span><span class="n">addStyle</span><span class="p">({</span><span class="s1">&#39;atom&#39;</span><span class="p">:</span><span class="n">BB</span><span class="p">},{</span><span class="s1">&#39;stick&#39;</span><span class="p">:{</span><span class="s1">&#39;colorscheme&#39;</span><span class="p">:</span><span class="sa">f</span><span class="s2">&quot;WhiteCarbon&quot;</span><span class="p">,</span><span class="s1">&#39;radius&#39;</span><span class="p">:</span><span class="mf">0.3</span><span class="p">}},</span> <span class="n">viewer</span><span class="o">=</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">0</span><span class="p">))</span>

  <span class="n">view</span><span class="o">.</span><span class="n">zoomTo</span><span class="p">()</span>
  <span class="k">return</span> <span class="n">view</span>

<span class="nb">print</span><span class="p">(</span><span class="s1">&#39;Prediction: Rank </span><span class="si">%d</span><span class="s1">&#39;</span> <span class="o">%</span> <span class="n">rank_num</span><span class="p">)</span>
<span class="n">show_pdb</span><span class="p">(</span><span class="n">rank_num</span><span class="p">,</span> <span class="n">show_sidechains</span><span class="p">,</span> <span class="n">show_mainchains</span><span class="p">,</span> <span class="n">color</span><span class="p">)</span><span class="o">.</span><span class="n">show</span><span class="p">()</span>
<span class="k">if</span> <span class="n">color</span> <span class="o">==</span> <span class="s2">&quot;lDDT&quot;</span><span class="p">:</span>
  <span class="n">plot_plddt_legend</span><span class="p">()</span><span class="o">.</span><span class="n">show</span><span class="p">()</span>


<span class="c1">#@markdown \</span>
</pre></div>
</div>
</div>
</div>
<div class="cell docutils container">
<div class="cell_input docutils container">
<div class="highlight-ipython3 notranslate"><div class="highlight"><pre><span></span><span class="c1">#@title **8. Package and Download**</span>

<span class="c1">#@markdown If you are having issues downloading the result archive, try disabling your adblocker and run this cell again. If that fails click on the little folder icon to the left, navigate to file: `jobname.result.zip`, right-click and select \&quot;Download\&quot; (see [screenshot](https://pbs.twimg.com/media/E6wRW2lWUAEOuoe?format=jpg&amp;name=small)).</span>

<span class="k">if</span> <span class="n">msa_mode</span> <span class="o">==</span> <span class="s2">&quot;custom&quot;</span><span class="p">:</span>
  <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;Don&#39;t forget to cite your custom MSA generation method.&quot;</span><span class="p">)</span>

<span class="n">files</span><span class="o">.</span><span class="n">download</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.result.zip&quot;</span><span class="p">)</span>

<span class="k">if</span> <span class="n">save_to_google_drive</span> <span class="o">==</span> <span class="kc">True</span> <span class="ow">and</span> <span class="n">drive</span><span class="p">:</span>
  <span class="n">uploaded</span> <span class="o">=</span> <span class="n">drive</span><span class="o">.</span><span class="n">CreateFile</span><span class="p">({</span><span class="s1">&#39;title&#39;</span><span class="p">:</span> <span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.result.zip&quot;</span><span class="p">})</span>
  <span class="n">uploaded</span><span class="o">.</span><span class="n">SetContentFile</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.result.zip&quot;</span><span class="p">)</span>
  <span class="n">uploaded</span><span class="o">.</span><span class="n">Upload</span><span class="p">()</span>
  <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;Uploaded </span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.result.zip to Google Drive with ID </span><span class="si">{</span><span class="n">uploaded</span><span class="o">.</span><span class="n">get</span><span class="p">(</span><span class="s1">&#39;id&#39;</span><span class="p">)</span><span class="si">}</span><span class="s2">&quot;</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
<p>Colab notebooks are adapted from:</p>
<ul class="simple">
<li><p><a class="reference external" href="https://colab.research.google.com/github/sokrypton/ColabFold/blob/main/AlphaFold2.ipynb">AlphaFold2 Colab</a></p></li>
<li><p><a class="reference external" href="https://www.nature.com/articles/s41592-022-01488-1">Mirdita M, SchÃ¼tze K, Moriwaki Y, Heo L, Ovchinnikov S, Steinegger M. ColabFold: Making protein folding accessible to all.
<em>Nature Methods</em>, 2022</a></p></li>
</ul>
<p><em>From the original notebook:</em></p>
<blockquote>
<div><a class="reference internal image-reference" href="https://raw.githubusercontent.com/sokrypton/ColabFold/main/.github/ColabFold_Marv_Logo_Small.png"><img alt="https://raw.githubusercontent.com/sokrypton/ColabFold/main/.github/ColabFold_Marv_Logo_Small.png" class="align-right" src="https://raw.githubusercontent.com/sokrypton/ColabFold/main/.github/ColabFold_Marv_Logo_Small.png" style="height: 200px;" /></a>
</div></blockquote>
<section id="colabfold-v1-5-3-alphafold2-using-mmseqs2">
<h2>ColabFold v1.5.3: AlphaFold2 using MMseqs2<a class="headerlink" href="#colabfold-v1-5-3-alphafold2-using-mmseqs2" title="Permalink to this heading">#</a></h2>
<p>Easy to use protein structure and complex prediction using <a class="reference external" href="https://www.nature.com/articles/s41586-021-03819-2">AlphaFold2</a> and <a class="reference external" href="https://www.biorxiv.org/content/10.1101/2021.10.04.463034v1">Alphafold2-multimer</a>. Sequence alignments/templates are generated through <span class="xref myst">MMseqs2</span> and <a class="reference external" href="https://github.com/soedinglab/hh-suite">HHsearch</a>. For more details, see <a href="#Instructions">bottom</a> of the notebook, checkout the <a class="reference external" href="https://github.com/sokrypton/ColabFold">ColabFold GitHub</a> and read our manuscript.</p>
<p>Old versions: <a class="reference external" href="https://colab.research.google.com/github/sokrypton/ColabFold/blob/v1.4.0/AlphaFold2.ipynb">v1.4</a>, <a class="reference external" href="https://colab.research.google.com/github/sokrypton/ColabFold/blob/v1.5.1/AlphaFold2.ipynb">v1.5.1</a>, <a class="reference external" href="https://colab.research.google.com/github/sokrypton/ColabFold/blob/v1.5.2/AlphaFold2.ipynb">v1.5.2</a></p>
<p><a class="reference external" href="https://www.nature.com/articles/s41592-022-01488-1">Mirdita M, SchÃ¼tze K, Moriwaki Y, Heo L, Ovchinnikov S, Steinegger M. ColabFold: Making protein folding accessible to all.
<em>Nature Methods</em>, 2022</a></p>
<hr class="docutils" />
<section id="news">
<h3>News<a class="headerlink" href="#news" title="Permalink to this heading">#</a></h3>
<ul class="simple">
<li><p><b><font color='green'>2023/07/31: The ColabFold MSA server is back to normal. It was using older DB (UniRef30 2202/PDB70 220313) from 27th ~8:30 AM CEST to 31st ~11:10 AM CEST.</font></b></p></li>
<li><p><b><font color='green'>2023/06/12: New databases! UniRef30 updated to 2023_02 and PDB to 230517. We now use PDB100 instead of PDB70 (see <span class="xref myst">notes</span>).</font></b></p></li>
<li><p><b><font color='green'>2023/06/12: We introduced a new default pairing strategy: Previously, for multimer predictions with more than 2 chains, we only pair if all sequences taxonomically match (â€œcompleteâ€ pairing). The new default â€œgreedyâ€ strategy pairs any taxonomically matching subsets.</font></b></p></li>
</ul>
</section>
</section>
</section>
<section class="tex2jax_ignore mathjax_ignore" id="instructions-a-name-instructions-a">
<h1>Instructions <a name="Instructions"></a><a class="headerlink" href="#instructions-a-name-instructions-a" title="Permalink to this heading">#</a></h1>
<p><strong>Quick start</strong></p>
<ol class="arabic simple">
<li><p>Paste your protein sequence(s) in the input field.</p></li>
<li><p>Press â€œRuntimeâ€ -&gt; â€œRun allâ€.</p></li>
<li><p>The pipeline consists of 5 steps. The currently running step is indicated by a circle with a stop sign next to it.</p></li>
</ol>
<p><strong>Result zip file contents</strong></p>
<ol class="arabic simple">
<li><p>PDB formatted structures sorted by avg. pLDDT and complexes are sorted by pTMscore. (unrelaxed and relaxed if <code class="docutils literal notranslate"><span class="pre">use_amber</span></code> is enabled).</p></li>
<li><p>Plots of the model quality.</p></li>
<li><p>Plots of the MSA coverage.</p></li>
<li><p>Parameter log file.</p></li>
<li><p>A3M formatted input MSA.</p></li>
<li><p>A <code class="docutils literal notranslate"><span class="pre">predicted_aligned_error_v1.json</span></code> using <a class="reference external" href="https://alphafold.ebi.ac.uk/faq#faq-7">AlphaFold-DBâ€™s format</a> and a <code class="docutils literal notranslate"><span class="pre">scores.json</span></code> for each model which contains  an array (list of lists) for PAE, a list with the average pLDDT and the pTMscore.</p></li>
<li><p>BibTeX file with citations for all used tools and databases.</p></li>
</ol>
<p>At the end of the job a download modal box will pop up with a <code class="docutils literal notranslate"><span class="pre">jobname.result.zip</span></code> file. Additionally, if the <code class="docutils literal notranslate"><span class="pre">save_to_google_drive</span></code> option was selected, the <code class="docutils literal notranslate"><span class="pre">jobname.result.zip</span></code> will be uploaded to your Google Drive.</p>
<p><strong>License</strong>
<a class="reference external" href="http://githubusercontent.com/sokrypton/ColabFold/main/LICENSE">githubusercontent.com/sokrypton/ColabFold/main/LICENSE</a>). Additionally, this &gt; notebook uses the AlphaFold2 source code and its parameters licensed under <a class="reference external" href="https://raw.githubusercontent.com/deepmind/alphafold/main/LICENSE">Apache 2.0</a> &gt; and <a class="reference external" href="https://creativecommons.org/licenses/by-sa/4.0/">CC BY 4.0</a> respectively. Read more about the AlphaFold license [here](<a class="github reference external" href="https://github.com/deepmind/">deepmind/</a>&gt; alphafold).</p>
<p><strong>Acknowledgments</strong></p>
<ul class="simple">
<li><p>We thank the AlphaFold team for developing an excellent model and open &gt; sourcing the software.</p></li>
<li><p><a class="reference external" href="https://kobic.re.kr">KOBIC</a> and <a class="reference external" href="https://www.mpinat.mpg.de/soeding">SÃ¶ding Lab</a> for providing the computational resources for the MMseqs2 MSA server.</p></li>
<li><p>Richard Evans for helping to benchmark the ColabFoldâ€™s Alphafold-multimer support.</p></li>
<li><p><a class="reference external" href="https://github.com/dkoes">David Koes</a> for his awesome <a class="reference external" href="https://3dmol.csb.pitt.edu/">py3Dmol</a> plugin, without whom these notebooks would be quite boring!</p></li>
<li><p>Do-Yoon Kim for creating the ColabFold logo.</p></li>
<li><p>A colab by Sergey Ovchinnikov (<a class="reference external" href="https://twitter.com/sokrypton">&#64;sokrypton</a>), Milot Mirdita (<a class="reference external" href="https://twitter.com/milot_mirdita">&#64;milot_mirdita</a>) and Martin Steinegger (<a class="reference external" href="https://twitter.com/thesteinegger">&#64;thesteinegger</a>).</p></li>
</ul>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./ColabFold"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

                </article>
              

              
              
                <footer class="bd-footer-article">
                  
<div class="footer-article-items footer-article__inner">
  
    <div class="footer-article-item"><!-- Previous / next buttons -->
<div class="prev-next-area">
    <a class="left-prev"
       href="Tutorial-Monomer.html"
       title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
        <p class="prev-next-subtitle">previous</p>
        <p class="prev-next-title">1. <strong>ColabFold Tutorial:</strong> Predicting the structure of Bax protein</p>
      </div>
    </a>
</div></div>
  
</div>

                </footer>
              
            </div>
            
            
              
                <div class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">

  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h1 nav-item toc-entry"><a class="reference internal nav-link" href="#">1. <strong>ColabFold Tutorial:</strong> Predicting Protein-Protein Complexes of TA Systems</a><ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#colabfold-v1-5-3-alphafold2-using-mmseqs2">ColabFold v1.5.3: AlphaFold2 using MMseqs2</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#news">News</a></li>
</ul>
</li>
</ul>
</li>
<li class="toc-h1 nav-item toc-entry"><a class="reference internal nav-link" href="#instructions-a-name-instructions-a">Instructions <a name="Instructions"></a></a></li>
</ul>

  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By CCATS Group
</p>

  </div>
  
  <div class="footer-item">
    
  <p class="copyright">
    
      Â© Copyright 2023.
      <br/>
    
  </p>

  </div>
  
  <div class="footer-item">
    
  </div>
  
  <div class="footer-item">
    
  </div>
  
</div>
          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../_static/scripts/bootstrap.js?digest=e353d410970836974a52"></script>
<script src="../_static/scripts/pydata-sphinx-theme.js?digest=e353d410970836974a52"></script>

  <footer class="bd-footer">
  </footer>
  </body>
</html>