

<!DOCTYPE html>


<html lang="en" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.19: https://docutils.sourceforge.io/" />

    <title>1. ColabFold Tutorial: Predicting the structure of Bax protein &#8212; Tutorials</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "light";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="../_static/styles/theme.css?digest=e353d410970836974a52" rel="stylesheet" />
<link href="../_static/styles/bootstrap.css?digest=e353d410970836974a52" rel="stylesheet" />
<link href="../_static/styles/pydata-sphinx-theme.css?digest=e353d410970836974a52" rel="stylesheet" />

  
  <link href="../_static/vendor/fontawesome/6.1.2/css/all.min.css?digest=e353d410970836974a52" rel="stylesheet" />
  <link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.1.2/webfonts/fa-solid-900.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.1.2/webfonts/fa-brands-400.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.1.2/webfonts/fa-regular-400.woff2" />

    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <link rel="stylesheet" href="../_static/styles/sphinx-book-theme.css?digest=14f4ca6b54d191a8c7657f6c759bf11a5fb86285" type="text/css" />
    <link rel="stylesheet" type="text/css" href="../_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/mystnb.4510f1fc1dee50b3e5859aac5469c37c29e427902b24a333a5f9fcb2f0b3ac41.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="../_static/design-style.4045f2051d55cab465a707391d5b2007.min.css" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../_static/scripts/bootstrap.js?digest=e353d410970836974a52" />
<link rel="preload" as="script" href="../_static/scripts/pydata-sphinx-theme.js?digest=e353d410970836974a52" />

    <script data-url_root="../" id="documentation_options" src="../_static/documentation_options.js"></script>
    <script src="../_static/jquery.js"></script>
    <script src="../_static/underscore.js"></script>
    <script src="../_static/_sphinx_javascript_frameworks_compat.js"></script>
    <script src="../_static/doctools.js"></script>
    <script src="../_static/sphinx_highlight.js"></script>
    <script src="../_static/clipboard.min.js"></script>
    <script src="../_static/copybutton.js"></script>
    <script src="../_static/scripts/sphinx-book-theme.js?digest=5a5c038af52cf7bc1a1ec88eea08e6366ee68824"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../_static/togglebutton.js"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="../_static/design-tabs.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="../_static/sphinx-thebe.js"></script>
    <script>window.MathJax = {"options": {"processHtmlClass": "tex2jax_process|mathjax_process|math|output_area"}}</script>
    <script defer="defer" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'ColabFold/Tutorial-Monomer';</script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="1. ColabFold Tutorial: Predicting Protein-Protein Complexes of TA Systems" href="Tutorial-Multimer.html" />
    <link rel="prev" title="*Original ColabFold Notebook" href="AlphaFold2.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <a class="skip-link" href="#main-content">Skip to main content</a>
  
  <input type="checkbox"
          class="sidebar-toggle"
          name="__primary"
          id="__primary"/>
  <label class="overlay overlay-primary" for="__primary"></label>
  
  <input type="checkbox"
          class="sidebar-toggle"
          name="__secondary"
          id="__secondary"/>
  <label class="overlay overlay-secondary" for="__secondary"></label>
  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
<form class="bd-search d-flex align-items-center"
      action="../search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         id="search-input"
         placeholder="Search this book..."
         aria-label="Search this book..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form></div>
  </div>
  
    <nav class="bd-header navbar navbar-expand-lg bd-navbar">
    </nav>
  
  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      <div class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">
  

<a class="navbar-brand logo" href="../index.html">
  
  
  
  
    
    
      
    
    
    <img src="../_static/schrodinger_cat_white_bbg.png" class="logo__image only-light" alt="Logo image"/>
    <script>document.write(`<img src="../_static/schrodinger_cat_white_bbg.png" class="logo__image only-dark" alt="Logo image"/>`);</script>
  
  
</a></div>
        <div class="sidebar-primary-item"><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../index.html">
                    Tutorials
                </a>
            </li>
        </ul>
        <ul class="current nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="AlphaFold2.html">*Original ColabFold Notebook</a></li>

<li class="toctree-l1 current active"><a class="current reference internal" href="#">1. <strong>ColabFold Tutorial:</strong> Predicting the structure of Bax protein</a></li>

<li class="toctree-l1"><a class="reference internal" href="Tutorial-Multimer.html">1. <strong>ColabFold Tutorial:</strong> Predicting Protein-Protein Complexes of TA Systems</a></li>

</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
  </div>
  
  <div id="rtd-footer-container"></div>


      </div>
      
      <main id="main-content" class="bd-main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><label class="sidebar-toggle primary-toggle btn btn-sm" for="__primary" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</label></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">





<div class="dropdown dropdown-launch-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Launch interactive content">
    <i class="fas fa-rocket"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://colab.research.google.com/github/cc-ats/Tutorials/blob/master/./ColabFold/Tutorial-Monomer.ipynb" target="_blank"
   class="btn btn-sm dropdown-item"
   title="Launch onColab"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  
    <img src="../_static/images/logo_colab.png">
  </span>
<span class="btn__text-container">Colab</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-source-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Source repositories">
    <i class="fab fa-github"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://github.com/cc-ats/Tutorials" target="_blank"
   class="btn btn-sm btn-source-repository-button dropdown-item"
   title="Source repository"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="btn__text-container">Repository</span>
</a>
</li>
      
      
      
      
      <li><a href="https://github.com/cc-ats/Tutorials/issues/new?title=Issue%20on%20page%20%2FColabFold/Tutorial-Monomer.html&body=Your%20issue%20content%20here." target="_blank"
   class="btn btn-sm btn-source-issues-button dropdown-item"
   title="Open an issue"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="btn__text-container">Open issue</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="../_sources/ColabFold/Tutorial-Monomer.ipynb" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.ipynb</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>


<script>
document.write(`
  <button class="theme-switch-button btn btn-sm btn-outline-primary navbar-btn rounded-circle" title="light/dark" aria-label="light/dark" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="theme-switch" data-mode="light"><i class="fa-solid fa-sun"></i></span>
    <span class="theme-switch" data-mode="dark"><i class="fa-solid fa-moon"></i></span>
    <span class="theme-switch" data-mode="auto"><i class="fa-solid fa-circle-half-stroke"></i></span>
  </button>
`);
</script>

<script>
document.write(`
  <button class="btn btn-sm navbar-btn search-button search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass"></i>
  </button>
`);
</script>
<label class="sidebar-toggle secondary-toggle btn btn-sm" for="__secondary"title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</label>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>1. ColabFold Tutorial: Predicting the structure of Bax protein</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h1 nav-item toc-entry"><a class="reference internal nav-link" href="#">1. <strong>ColabFold Tutorial:</strong> Predicting the structure of Bax protein</a><ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#colabfold-v1-5-3-alphafold2-using-mmseqs2">ColabFold v1.5.3: AlphaFold2 using MMseqs2</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#news">News</a></li>
</ul>
</li>
</ul>
</li>
<li class="toc-h1 nav-item toc-entry"><a class="reference internal nav-link" href="#instructions-a-name-instructions-a">Instructions <a name="Instructions"></a></a></li>
</ul>

            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article" role="main">
                  
  <p><a href="https://colab.research.google.com/github/cc-ats/Tutorials/blob/main/ColabFold/Tutorial-Monomer.ipynb" target="_parent"><img alt="Open In Colab" src="https://colab.research.google.com/assets/colab-badge.svg" /></a></p>
<div class="cell docutils container">
<div class="cell_input docutils container">
<div class="highlight-ipython3 notranslate"><div class="highlight"><pre><span></span><span class="o">%%time</span>

<span class="c1"># @title **Install Python Packages** { run: &quot;auto&quot;, display-mode: &quot;form&quot; }</span>

<span class="kn">import</span> <span class="nn">os</span>
<span class="kn">from</span> <span class="nn">sys</span> <span class="kn">import</span> <span class="n">version_info</span>
<span class="n">PYTHON_VERSION</span> <span class="o">=</span> <span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">version_info</span><span class="o">.</span><span class="n">major</span><span class="si">}</span><span class="s2">.</span><span class="si">{</span><span class="n">version_info</span><span class="o">.</span><span class="n">minor</span><span class="si">}</span><span class="s2">&quot;</span>

<span class="k">if</span> <span class="ow">not</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">isfile</span><span class="p">(</span><span class="s2">&quot;COLABFOLD_READY&quot;</span><span class="p">):</span>
  <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;installing colabfold...&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;pip install -q --no-warn-conflicts &#39;colabfold[alphafold-minus-jax] @ git+https://github.com/sokrypton/ColabFold&#39;&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;pip install --upgrade dm-haiku&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;ln -s /usr/local/lib/python3.*/dist-packages/colabfold colabfold&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;ln -s /usr/local/lib/python3.*/dist-packages/alphafold alphafold&quot;</span><span class="p">)</span>
  <span class="c1"># patch for jax &gt; 0.3.25</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;sed -i &#39;s/weights = jax.nn.softmax(logits)/logits=jnp.clip(logits,-1e8,1e8);weights=jax.nn.softmax(logits)/g&#39; alphafold/model/modules.py&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;touch COLABFOLD_READY&quot;</span><span class="p">)</span>


<span class="k">if</span> <span class="ow">not</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">isfile</span><span class="p">(</span><span class="s2">&quot;CONDA_READY&quot;</span><span class="p">):</span>
  <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;installing conda...&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;wget -qnc https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;bash Miniforge3-Linux-x86_64.sh -bfp /usr/local&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;conda config --set auto_update_conda false&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;touch CONDA_READY&quot;</span><span class="p">)</span>


<span class="k">if</span> <span class="ow">not</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">isfile</span><span class="p">(</span><span class="s2">&quot;CONDA_READY&quot;</span><span class="p">):</span>
  <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;installing conda...&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;wget -qnc https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;bash Mambaforge-Linux-x86_64.sh -bfp /usr/local&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;mamba config --set auto_update_conda false&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;touch CONDA_READY&quot;</span><span class="p">)</span>


<span class="k">if</span> <span class="ow">not</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">isfile</span><span class="p">(</span><span class="s2">&quot;HH_READY&quot;</span><span class="p">)</span> <span class="ow">and</span> <span class="ow">not</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">isfile</span><span class="p">(</span><span class="s2">&quot;AMBER_READY&quot;</span><span class="p">):</span>
  <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;installing hhsuite and amber...&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;mamba install -y -c conda-forge -c bioconda kalign2=2.04 hhsuite=3.3.0 openmm=7.7.0 python=&#39;</span><span class="si">{</span><span class="n">PYTHON_VERSION</span><span class="si">}</span><span class="s2">&#39; pdbfixer&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;touch HH_READY&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;touch AMBER_READY&quot;</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
  <span class="k">if</span> <span class="ow">not</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">isfile</span><span class="p">(</span><span class="s2">&quot;HH_READY&quot;</span><span class="p">):</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;installing hhsuite...&quot;</span><span class="p">)</span>
    <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;mamba install -y -c conda-forge -c bioconda kalign2=2.04 hhsuite=3.3.0 python=&#39;</span><span class="si">{</span><span class="n">PYTHON_VERSION</span><span class="si">}</span><span class="s2">&#39;&quot;</span><span class="p">)</span>
    <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;touch HH_READY&quot;</span><span class="p">)</span>
  <span class="k">if</span> <span class="ow">not</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">isfile</span><span class="p">(</span><span class="s2">&quot;AMBER_READY&quot;</span><span class="p">):</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;installing amber...&quot;</span><span class="p">)</span>
    <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;mamba install -y -c conda-forge openmm=7.7.0 python=&#39;</span><span class="si">{</span><span class="n">PYTHON_VERSION</span><span class="si">}</span><span class="s2">&#39; pdbfixer&quot;</span><span class="p">)</span>
    <span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;touch AMBER_READY&quot;</span><span class="p">)</span>
</pre></div>
</div>
</div>
<div class="cell_output docutils container">
<div class="output stream highlight-myst-ansi notranslate"><div class="highlight"><pre><span></span>installing colabfold...
installing conda...
installing hhsuite and amber...
CPU times: user 269 ms, sys: 52.3 ms, total: 321 ms
Wall time: 1min 24s
</pre></div>
</div>
</div>
</div>
<section class="tex2jax_ignore mathjax_ignore" id="colabfold-tutorial-predicting-the-structure-of-bax-protein">
<h1>1. <strong>ColabFold Tutorial:</strong> Predicting the structure of Bax protein<a class="headerlink" href="#colabfold-tutorial-predicting-the-structure-of-bax-protein" title="Permalink to this heading">#</a></h1>
<p>Let’s get familar with the workflow for predicting a monomer structure. This example looks at Bax, an apoptosis regulating protein.</p>
<p>We will only use the default options set by Steinegger <em>et al.</em> which excludes Amber minimization and structure templates.</p>
<p><strong>a. Background</strong></p>
<ul class="simple">
<li><p>Encoded by the BAX gene in the Bcl-2 family</p></li>
<li><p>Highly conserved DNA sequence and structural motifs</p></li>
<li><p>Different regulatory function in anti-apoptosis or pro-apoptosis</p></li>
<li><p>Permeation of the mitochondrial outer membrane</p></li>
</ul>
<p><strong>b. Pro-death signals triggers conformational shift</strong></p>
<ul class="simple">
<li><p>Bax adopts a globular state when inactive</p></li>
<li><p>Apoptotic signals displaces N-terminal <span class="math notranslate nohighlight">\(\alpha\)</span>-helicies</p></li>
<li><p>Becomes membrane-associated</p></li>
<li><p>Oligomers form, resulting in a pore and cell death</p></li>
</ul>
<p>Dysregulation of Bax is implicated in cancer progression, and suggests some contributions towards in Alzheimer’s and Parkinson’s disease.</p>
<p>Experiments have shown point-mutations near opposite of the binding site region, has been shown to affect activity. This is also near the loop region between <span class="math notranslate nohighlight">\(\alpha\)</span>-helix1 and <span class="math notranslate nohighlight">\(\alpha\)</span>-helicies2, which has been difficult to structurally resolve. Does this loop play a role in membrane insertion mechanism?</p>
<div class="cell docutils container">
<div class="cell_input docutils container">
<div class="highlight-ipython3 notranslate"><div class="highlight"><pre><span></span><span class="c1">#@title # **2. Notebook Setup** { display-mode: &quot;form&quot; }</span>

<span class="kn">from</span> <span class="nn">google.colab</span> <span class="kn">import</span> <span class="n">files</span>
<span class="kn">import</span> <span class="nn">os</span>
<span class="kn">import</span> <span class="nn">re</span>
<span class="kn">import</span> <span class="nn">hashlib</span>
<span class="kn">import</span> <span class="nn">random</span>

<span class="kn">from</span> <span class="nn">sys</span> <span class="kn">import</span> <span class="n">version_info</span>
<span class="n">python_version</span> <span class="o">=</span> <span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">version_info</span><span class="o">.</span><span class="n">major</span><span class="si">}</span><span class="s2">.</span><span class="si">{</span><span class="n">version_info</span><span class="o">.</span><span class="n">minor</span><span class="si">}</span><span class="s2">&quot;</span>

<span class="k">def</span> <span class="nf">add_hash</span><span class="p">(</span><span class="n">x</span><span class="p">,</span><span class="n">y</span><span class="p">):</span>
  <span class="k">return</span> <span class="n">x</span><span class="o">+</span><span class="s2">&quot;_&quot;</span><span class="o">+</span><span class="n">hashlib</span><span class="o">.</span><span class="n">sha1</span><span class="p">(</span><span class="n">y</span><span class="o">.</span><span class="n">encode</span><span class="p">())</span><span class="o">.</span><span class="n">hexdigest</span><span class="p">()[:</span><span class="mi">5</span><span class="p">]</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown ##a. Protein Sequence for Structure Prediction</span>
<span class="c1">#@markdown For bax (human), the amino acid sequence was obtained from</span>
<span class="c1">#@markdown UniProt ([Q07812](https://www.uniprot.org/uniprotkb/Q07812/entry))</span>
<span class="n">query_sequence</span> <span class="o">=</span> <span class="s1">&#39;MDGSGEQPRGGGPTSSEQIMKTGALLLQGFIQDRAGRMGGEAPELALDPVPQDASTKKLSECLKRIGDELDSNMELQRMIAAVDTDSPREVFFRVAADMFSDGNFNWGRVVALFYFASKLVLKALCTKVPELIRTIMGWTLDFLRERLLGWIQDQGGWDGLLSYFGTPTWQTVTIFVAGVLTASLTIWKKMG&#39;</span> <span class="c1">#@param {type:&quot;string&quot;}</span>
<span class="c1">#@markdown **Note:** For modeling complexes, use `:` to specify inter-protein chainbreaks</span>
<span class="c1">#@markdown (supports homo- and hetro-oligomers).</span>
<span class="c1">#@markdown For example, **PI...SK:PI...SK** for a homodimer.</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown ## b. Job Name (also name of saving directory)</span>
<span class="n">jobname</span> <span class="o">=</span> <span class="s1">&#39;colabfold-bax-1f16&#39;</span> <span class="c1">#@param {type:&quot;string&quot;}</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown  ## c. Number of predicted models to relax using Amber</span>
<span class="n">num_relax</span> <span class="o">=</span> <span class="mi">0</span> <span class="c1">#@param [0, 1, 5] {type:&quot;raw&quot;}</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown ## **d. Option for Structure Template**</span>
<span class="c1">#@markdown  Search for PDB structure (xyz-coordinates)</span>
<span class="n">template_mode</span> <span class="o">=</span> <span class="s2">&quot;none&quot;</span> <span class="c1">#@param [&quot;none&quot;, &quot;pdb100&quot;,&quot;custom&quot;]</span>
<span class="c1">#@markdown `none` = no template information is used. \</span>
<span class="c1">#@markdown `pdb100` = detect templates in pdb100 (see [notes](#pdb100)). \</span>
<span class="c1">#@markdown `custom` - upload and search own templates (PDB or mmCIF format, see [notes](#custom_templates))</span>

<span class="n">use_amber</span> <span class="o">=</span> <span class="n">num_relax</span> <span class="o">&gt;</span> <span class="mi">0</span>

<span class="n">query_sequence</span> <span class="o">=</span> <span class="s2">&quot;&quot;</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">query_sequence</span><span class="o">.</span><span class="n">split</span><span class="p">())</span> <span class="c1"># remove whitespaces</span>

<span class="n">basejobname</span> <span class="o">=</span> <span class="s2">&quot;&quot;</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">jobname</span><span class="o">.</span><span class="n">split</span><span class="p">())</span>
<span class="n">basejobname</span> <span class="o">=</span> <span class="n">re</span><span class="o">.</span><span class="n">sub</span><span class="p">(</span><span class="sa">r</span><span class="s1">&#39;\W+&#39;</span><span class="p">,</span> <span class="s1">&#39;&#39;</span><span class="p">,</span> <span class="n">basejobname</span><span class="p">)</span>
<span class="n">jobname</span> <span class="o">=</span> <span class="n">add_hash</span><span class="p">(</span><span class="n">basejobname</span><span class="p">,</span> <span class="n">query_sequence</span><span class="p">)</span>

<span class="c1"># check if directory with jobname exists</span>
<span class="k">def</span> <span class="nf">check</span><span class="p">(</span><span class="n">folder</span><span class="p">):</span>
  <span class="k">if</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">folder</span><span class="p">):</span>
    <span class="k">return</span> <span class="kc">False</span>
  <span class="k">else</span><span class="p">:</span>
    <span class="k">return</span> <span class="kc">True</span>
<span class="k">if</span> <span class="ow">not</span> <span class="n">check</span><span class="p">(</span><span class="n">jobname</span><span class="p">):</span>
  <span class="n">n</span> <span class="o">=</span> <span class="mi">0</span>
  <span class="k">while</span> <span class="ow">not</span> <span class="n">check</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">_</span><span class="si">{</span><span class="n">n</span><span class="si">}</span><span class="s2">&quot;</span><span class="p">):</span> <span class="n">n</span> <span class="o">+=</span> <span class="mi">1</span>
  <span class="n">jobname</span> <span class="o">=</span> <span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">_</span><span class="si">{</span><span class="n">n</span><span class="si">}</span><span class="s2">&quot;</span>

<span class="c1"># make directory to save results</span>
<span class="n">os</span><span class="o">.</span><span class="n">makedirs</span><span class="p">(</span><span class="n">jobname</span><span class="p">,</span> <span class="n">exist_ok</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>

<span class="c1"># save queries</span>
<span class="n">queries_path</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">jobname</span><span class="p">,</span> <span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.csv&quot;</span><span class="p">)</span>
<span class="k">with</span> <span class="nb">open</span><span class="p">(</span><span class="n">queries_path</span><span class="p">,</span> <span class="s2">&quot;w&quot;</span><span class="p">)</span> <span class="k">as</span> <span class="n">text_file</span><span class="p">:</span>
  <span class="n">text_file</span><span class="o">.</span><span class="n">write</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;id,sequence</span><span class="se">\n</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">,</span><span class="si">{</span><span class="n">query_sequence</span><span class="si">}</span><span class="s2">&quot;</span><span class="p">)</span>

<span class="k">if</span> <span class="n">template_mode</span> <span class="o">==</span> <span class="s2">&quot;pdb100&quot;</span><span class="p">:</span>
  <span class="n">use_templates</span> <span class="o">=</span> <span class="kc">True</span>
  <span class="n">custom_template_path</span> <span class="o">=</span> <span class="kc">None</span>
<span class="k">elif</span> <span class="n">template_mode</span> <span class="o">==</span> <span class="s2">&quot;custom&quot;</span><span class="p">:</span>
  <span class="n">custom_template_path</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">jobname</span><span class="p">,</span><span class="sa">f</span><span class="s2">&quot;template&quot;</span><span class="p">)</span>
  <span class="n">os</span><span class="o">.</span><span class="n">makedirs</span><span class="p">(</span><span class="n">custom_template_path</span><span class="p">,</span> <span class="n">exist_ok</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
  <span class="n">uploaded</span> <span class="o">=</span> <span class="n">files</span><span class="o">.</span><span class="n">upload</span><span class="p">()</span>
  <span class="n">use_templates</span> <span class="o">=</span> <span class="kc">True</span>
  <span class="k">for</span> <span class="n">fn</span> <span class="ow">in</span> <span class="n">uploaded</span><span class="o">.</span><span class="n">keys</span><span class="p">():</span>
    <span class="n">os</span><span class="o">.</span><span class="n">rename</span><span class="p">(</span><span class="n">fn</span><span class="p">,</span><span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">custom_template_path</span><span class="p">,</span><span class="n">fn</span><span class="p">))</span>
<span class="k">else</span><span class="p">:</span>
  <span class="n">custom_template_path</span> <span class="o">=</span> <span class="kc">None</span>
  <span class="n">use_templates</span> <span class="o">=</span> <span class="kc">False</span>
</pre></div>
</div>
</div>
</div>
<div class="cell docutils container">
<div class="cell_input docutils container">
<div class="highlight-ipython3 notranslate"><div class="highlight"><pre><span></span><span class="c1">#@title # **3. Multiple Sequence Alignment (MSA)**</span>

<span class="c1">#@markdown ## a. MSA options</span>
<span class="n">msa_mode</span> <span class="o">=</span> <span class="s2">&quot;mmseqs2_uniref&quot;</span> <span class="c1">#@param [&quot;mmseqs2_uniref_env&quot;, &quot;mmseqs2_uniref&quot;,&quot;single_sequence&quot;,&quot;custom&quot;]</span>
<span class="c1">#@markdown - pairing mode</span>
<span class="c1">#@markdown - single sequence</span>
<span class="c1">#@markdown - custom MSA upload</span>

<span class="c1">#@markdown \</span>
<span class="n">pair_mode</span> <span class="o">=</span> <span class="s2">&quot;unpaired_paired&quot;</span> <span class="c1">#@param [&quot;unpaired_paired&quot;,&quot;unpaired&quot;,&quot;paired&quot;] {type:&quot;string&quot;}</span>
<span class="c1">#@markdown - &quot;unpaired_paired&quot; - pair sequences from same species + unpaired MSA</span>
<span class="c1">#@markdown - &quot;unpaired&quot; - seperate MSA for each chain</span>
<span class="c1">#@markdown - &quot;paired&quot; - only use paired sequences</span>

<span class="c1"># decide which a3m to use</span>
<span class="k">if</span> <span class="s2">&quot;mmseqs2&quot;</span> <span class="ow">in</span> <span class="n">msa_mode</span><span class="p">:</span>
  <span class="n">a3m_file</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">jobname</span><span class="p">,</span><span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.a3m&quot;</span><span class="p">)</span>

<span class="k">elif</span> <span class="n">msa_mode</span> <span class="o">==</span> <span class="s2">&quot;custom&quot;</span><span class="p">:</span>
  <span class="n">a3m_file</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">jobname</span><span class="p">,</span><span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.custom.a3m&quot;</span><span class="p">)</span>
  <span class="k">if</span> <span class="ow">not</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">isfile</span><span class="p">(</span><span class="n">a3m_file</span><span class="p">):</span>
    <span class="n">custom_msa_dict</span> <span class="o">=</span> <span class="n">files</span><span class="o">.</span><span class="n">upload</span><span class="p">()</span>
    <span class="n">custom_msa</span> <span class="o">=</span> <span class="nb">list</span><span class="p">(</span><span class="n">custom_msa_dict</span><span class="o">.</span><span class="n">keys</span><span class="p">())[</span><span class="mi">0</span><span class="p">]</span>
    <span class="n">header</span> <span class="o">=</span> <span class="mi">0</span>
    <span class="kn">import</span> <span class="nn">fileinput</span>
    <span class="k">for</span> <span class="n">line</span> <span class="ow">in</span> <span class="n">fileinput</span><span class="o">.</span><span class="n">FileInput</span><span class="p">(</span><span class="n">custom_msa</span><span class="p">,</span><span class="n">inplace</span><span class="o">=</span><span class="mi">1</span><span class="p">):</span>
      <span class="k">if</span> <span class="n">line</span><span class="o">.</span><span class="n">startswith</span><span class="p">(</span><span class="s2">&quot;&gt;&quot;</span><span class="p">):</span>
         <span class="n">header</span> <span class="o">=</span> <span class="n">header</span> <span class="o">+</span> <span class="mi">1</span>
      <span class="k">if</span> <span class="ow">not</span> <span class="n">line</span><span class="o">.</span><span class="n">rstrip</span><span class="p">():</span>
        <span class="k">continue</span>
      <span class="k">if</span> <span class="n">line</span><span class="o">.</span><span class="n">startswith</span><span class="p">(</span><span class="s2">&quot;&gt;&quot;</span><span class="p">)</span> <span class="o">==</span> <span class="kc">False</span> <span class="ow">and</span> <span class="n">header</span> <span class="o">==</span> <span class="mi">1</span><span class="p">:</span>
         <span class="n">query_sequence</span> <span class="o">=</span> <span class="n">line</span><span class="o">.</span><span class="n">rstrip</span><span class="p">()</span>
      <span class="nb">print</span><span class="p">(</span><span class="n">line</span><span class="p">,</span> <span class="n">end</span><span class="o">=</span><span class="s1">&#39;&#39;</span><span class="p">)</span>

    <span class="n">os</span><span class="o">.</span><span class="n">rename</span><span class="p">(</span><span class="n">custom_msa</span><span class="p">,</span> <span class="n">a3m_file</span><span class="p">)</span>
    <span class="n">queries_path</span><span class="o">=</span><span class="n">a3m_file</span>
    <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;moving </span><span class="si">{</span><span class="n">custom_msa</span><span class="si">}</span><span class="s2"> to </span><span class="si">{</span><span class="n">a3m_file</span><span class="si">}</span><span class="s2">&quot;</span><span class="p">)</span>

<span class="k">else</span><span class="p">:</span>
  <span class="n">a3m_file</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">jobname</span><span class="p">,</span><span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.single_sequence.a3m&quot;</span><span class="p">)</span>
  <span class="k">with</span> <span class="nb">open</span><span class="p">(</span><span class="n">a3m_file</span><span class="p">,</span> <span class="s2">&quot;w&quot;</span><span class="p">)</span> <span class="k">as</span> <span class="n">text_file</span><span class="p">:</span>
    <span class="n">text_file</span><span class="o">.</span><span class="n">write</span><span class="p">(</span><span class="s2">&quot;&gt;1</span><span class="se">\n</span><span class="si">%s</span><span class="s2">&quot;</span> <span class="o">%</span> <span class="n">query_sequence</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
<div class="cell docutils container">
<div class="cell_input docutils container">
<div class="highlight-ipython3 notranslate"><div class="highlight"><pre><span></span><span class="c1"># @title # **4. Advanced Settings** { run: &quot;auto&quot;, display-mode: &quot;form&quot; }</span>

<span class="c1">#@markdown ## a. ML Model</span>
<span class="n">model_type</span> <span class="o">=</span> <span class="s2">&quot;auto&quot;</span> <span class="c1">#@param [&quot;auto&quot;, &quot;alphafold2_ptm&quot;, &quot;alphafold2_multimer_v1&quot;, &quot;alphafold2_multimer_v2&quot;, &quot;alphafold2_multimer_v3&quot;]</span>
<span class="c1">#@markdown **Note:** If `auto` selected, will use `alphafold2_ptm` for monomer prediction</span>
<span class="c1">#@markdown and `alphafold2_multimer_v3` for complex prediction.</span>
<span class="c1">#@markdown Any of the `mode_types` can be used (regardless if input is monomer or complex).</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown ## b. Re-Train with Predicted Structures</span>
<span class="n">num_recycles</span> <span class="o">=</span> <span class="s2">&quot;3&quot;</span> <span class="c1">#@param [&quot;auto&quot;, &quot;0&quot;, &quot;1&quot;, &quot;3&quot;, &quot;6&quot;, &quot;12&quot;, &quot;24&quot;, &quot;48&quot;]</span>
<span class="c1">#@markdown **Note:**</span>
<span class="c1">#@markdown - If `auto` selected, will use `num_recycles=20`,</span>
<span class="c1">#@markdown - If `model_type=alphafold2_multimer_v3`, else `num_recycles=3`</span>

<span class="c1">#@markdown \</span>
<span class="n">recycle_early_stop_tolerance</span> <span class="o">=</span> <span class="s2">&quot;auto&quot;</span> <span class="c1">#@param [&quot;auto&quot;, &quot;0.0&quot;, &quot;0.5&quot;, &quot;1.0&quot;]</span>
<span class="c1">#@markdown - if `auto` selected, will use `tol=0.5`</span>
<span class="c1">#@markdown - If `model_type=alphafold2_multimer_v3`,  else `tol=0.0`.</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown ## c. Amber Relaxation</span>
<span class="n">relax_max_iterations</span> <span class="o">=</span> <span class="mi">200</span> <span class="c1">#@param [0, 200, 2000] {type:&quot;raw&quot;}</span>
<span class="c1">#@markdown - max amber relax iterations</span>
<span class="c1">#@markdown -  `0` = unlimited (AlphaFold2 default, can take very long)</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown ## d. MSA Pairing</span>
<span class="n">pairing_strategy</span> <span class="o">=</span> <span class="s2">&quot;greedy&quot;</span> <span class="c1">#@param [&quot;greedy&quot;, &quot;complete&quot;] {type:&quot;string&quot;}</span>
<span class="c1">#@markdown - `greedy` = pair any taxonomically matching subsets,</span>
<span class="c1">#@markdown - `complete` = all sequences have to match in one line.</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown ## e. Sample settings</span>
<span class="c1">#@markdown -  enable dropouts and increase number of seeds to sample predictions from uncertainty of the model.</span>
<span class="c1">#@markdown -  decrease `max_msa` to increase uncertainity</span>
<span class="n">max_msa</span> <span class="o">=</span> <span class="s2">&quot;auto&quot;</span> <span class="c1">#@param [&quot;auto&quot;, &quot;512:1024&quot;, &quot;256:512&quot;, &quot;64:128&quot;, &quot;32:64&quot;, &quot;16:32&quot;]</span>
<span class="n">num_seeds</span> <span class="o">=</span> <span class="mi">1</span> <span class="c1">#@param [1,2,4,8,16] {type:&quot;raw&quot;}</span>
<span class="n">use_dropout</span> <span class="o">=</span> <span class="kc">False</span> <span class="c1">#@param {type:&quot;boolean&quot;}</span>

<span class="n">num_recycles</span> <span class="o">=</span> <span class="kc">None</span> <span class="k">if</span> <span class="n">num_recycles</span> <span class="o">==</span> <span class="s2">&quot;auto&quot;</span> <span class="k">else</span> <span class="nb">int</span><span class="p">(</span><span class="n">num_recycles</span><span class="p">)</span>
<span class="n">recycle_early_stop_tolerance</span> <span class="o">=</span> <span class="kc">None</span> <span class="k">if</span> <span class="n">recycle_early_stop_tolerance</span> <span class="o">==</span> <span class="s2">&quot;auto&quot;</span> <span class="k">else</span> <span class="nb">float</span><span class="p">(</span><span class="n">recycle_early_stop_tolerance</span><span class="p">)</span>
<span class="k">if</span> <span class="n">max_msa</span> <span class="o">==</span> <span class="s2">&quot;auto&quot;</span><span class="p">:</span> <span class="n">max_msa</span> <span class="o">=</span> <span class="kc">None</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown # **5. Save settings**</span>
<span class="c1">#@markdown This will ask for your credentials, to save the outputs.</span>
<span class="n">save_all</span> <span class="o">=</span> <span class="kc">False</span> <span class="c1">#@param {type:&quot;boolean&quot;}</span>
<span class="n">save_recycles</span> <span class="o">=</span> <span class="kc">False</span> <span class="c1">#@param {type:&quot;boolean&quot;}</span>
<span class="n">save_to_google_drive</span> <span class="o">=</span> <span class="kc">False</span> <span class="c1">#@param {type:&quot;boolean&quot;}</span>
<span class="c1">#@markdown -  if the save_to_google_drive option was selected, the result zip will be uploaded to your Google Drive</span>
<span class="n">dpi</span> <span class="o">=</span> <span class="mi">200</span> <span class="c1">#@param {type:&quot;integer&quot;}</span>
<span class="c1">#@markdown - set dpi for image resolution</span>

<span class="k">if</span> <span class="n">save_to_google_drive</span><span class="p">:</span>
  <span class="kn">from</span> <span class="nn">pydrive.drive</span> <span class="kn">import</span> <span class="n">GoogleDrive</span>
  <span class="kn">from</span> <span class="nn">pydrive.auth</span> <span class="kn">import</span> <span class="n">GoogleAuth</span>
  <span class="kn">from</span> <span class="nn">google.colab</span> <span class="kn">import</span> <span class="n">auth</span>
  <span class="kn">from</span> <span class="nn">oauth2client.client</span> <span class="kn">import</span> <span class="n">GoogleCredentials</span>
  <span class="n">auth</span><span class="o">.</span><span class="n">authenticate_user</span><span class="p">()</span>
  <span class="n">gauth</span> <span class="o">=</span> <span class="n">GoogleAuth</span><span class="p">()</span>
  <span class="n">gauth</span><span class="o">.</span><span class="n">credentials</span> <span class="o">=</span> <span class="n">GoogleCredentials</span><span class="o">.</span><span class="n">get_application_default</span><span class="p">()</span>
  <span class="n">drive</span> <span class="o">=</span> <span class="n">GoogleDrive</span><span class="p">(</span><span class="n">gauth</span><span class="p">)</span>
  <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;You are logged into Google Drive and are good to go!&quot;</span><span class="p">)</span>

<span class="c1">#@markdown Don&#39;t forget to hit `Runtime` -&gt; `Run all` after updating the form.</span>
</pre></div>
</div>
</div>
</div>
<div class="cell docutils container">
<div class="cell_input docutils container">
<div class="highlight-ipython3 notranslate"><div class="highlight"><pre><span></span><span class="c1">#@title # **6. Run Prediction** { display-mode: &quot;form&quot; }</span>

<span class="n">display_images</span> <span class="o">=</span> <span class="kc">True</span> <span class="c1">#@param {type:&quot;boolean&quot;}</span>


<span class="kn">import</span> <span class="nn">sys</span>
<span class="kn">import</span> <span class="nn">warnings</span>
<span class="n">warnings</span><span class="o">.</span><span class="n">simplefilter</span><span class="p">(</span><span class="n">action</span><span class="o">=</span><span class="s1">&#39;ignore&#39;</span><span class="p">,</span> <span class="n">category</span><span class="o">=</span><span class="ne">FutureWarning</span><span class="p">)</span>
<span class="kn">from</span> <span class="nn">Bio</span> <span class="kn">import</span> <span class="n">BiopythonDeprecationWarning</span>
<span class="n">warnings</span><span class="o">.</span><span class="n">simplefilter</span><span class="p">(</span><span class="n">action</span><span class="o">=</span><span class="s1">&#39;ignore&#39;</span><span class="p">,</span> <span class="n">category</span><span class="o">=</span><span class="n">BiopythonDeprecationWarning</span><span class="p">)</span>
<span class="kn">from</span> <span class="nn">pathlib</span> <span class="kn">import</span> <span class="n">Path</span>
<span class="kn">from</span> <span class="nn">colabfold.download</span> <span class="kn">import</span> <span class="n">download_alphafold_params</span><span class="p">,</span> <span class="n">default_data_dir</span>
<span class="kn">from</span> <span class="nn">colabfold.utils</span> <span class="kn">import</span> <span class="n">setup_logging</span>
<span class="kn">from</span> <span class="nn">colabfold.batch</span> <span class="kn">import</span> <span class="n">get_queries</span><span class="p">,</span> <span class="n">run</span><span class="p">,</span> <span class="n">set_model_type</span>
<span class="kn">from</span> <span class="nn">colabfold.plot</span> <span class="kn">import</span> <span class="n">plot_msa_v2</span>

<span class="kn">import</span> <span class="nn">os</span>
<span class="kn">import</span> <span class="nn">numpy</span> <span class="k">as</span> <span class="nn">np</span>
<span class="k">try</span><span class="p">:</span>
  <span class="n">K80_chk</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">popen</span><span class="p">(</span><span class="s1">&#39;nvidia-smi | grep &quot;Tesla K80&quot; | wc -l&#39;</span><span class="p">)</span><span class="o">.</span><span class="n">read</span><span class="p">()</span>
<span class="k">except</span><span class="p">:</span>
  <span class="n">K80_chk</span> <span class="o">=</span> <span class="s2">&quot;0&quot;</span>
  <span class="k">pass</span>
<span class="k">if</span> <span class="s2">&quot;1&quot;</span> <span class="ow">in</span> <span class="n">K80_chk</span><span class="p">:</span>
  <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;WARNING: found GPU Tesla K80: limited to total length &lt; 1000&quot;</span><span class="p">)</span>
  <span class="k">if</span> <span class="s2">&quot;TF_FORCE_UNIFIED_MEMORY&quot;</span> <span class="ow">in</span> <span class="n">os</span><span class="o">.</span><span class="n">environ</span><span class="p">:</span>
    <span class="k">del</span> <span class="n">os</span><span class="o">.</span><span class="n">environ</span><span class="p">[</span><span class="s2">&quot;TF_FORCE_UNIFIED_MEMORY&quot;</span><span class="p">]</span>
  <span class="k">if</span> <span class="s2">&quot;XLA_PYTHON_CLIENT_MEM_FRACTION&quot;</span> <span class="ow">in</span> <span class="n">os</span><span class="o">.</span><span class="n">environ</span><span class="p">:</span>
    <span class="k">del</span> <span class="n">os</span><span class="o">.</span><span class="n">environ</span><span class="p">[</span><span class="s2">&quot;XLA_PYTHON_CLIENT_MEM_FRACTION&quot;</span><span class="p">]</span>

<span class="kn">from</span> <span class="nn">colabfold.colabfold</span> <span class="kn">import</span> <span class="n">plot_protein</span>
<span class="kn">from</span> <span class="nn">pathlib</span> <span class="kn">import</span> <span class="n">Path</span>
<span class="kn">import</span> <span class="nn">matplotlib.pyplot</span> <span class="k">as</span> <span class="nn">plt</span>

<span class="c1"># For some reason we need that to get pdbfixer to import</span>
<span class="k">if</span> <span class="n">use_amber</span> <span class="ow">and</span> <span class="sa">f</span><span class="s2">&quot;/usr/local/lib/python</span><span class="si">{</span><span class="n">python_version</span><span class="si">}</span><span class="s2">/site-packages/&quot;</span> <span class="ow">not</span> <span class="ow">in</span> <span class="n">sys</span><span class="o">.</span><span class="n">path</span><span class="p">:</span>
    <span class="n">sys</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">insert</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span> <span class="sa">f</span><span class="s2">&quot;/usr/local/lib/python</span><span class="si">{</span><span class="n">python_version</span><span class="si">}</span><span class="s2">/site-packages/&quot;</span><span class="p">)</span>

<span class="k">def</span> <span class="nf">input_features_callback</span><span class="p">(</span><span class="n">input_features</span><span class="p">):</span>
  <span class="k">if</span> <span class="n">display_images</span><span class="p">:</span>
    <span class="n">plot_msa_v2</span><span class="p">(</span><span class="n">input_features</span><span class="p">)</span>
    <span class="n">plt</span><span class="o">.</span><span class="n">show</span><span class="p">()</span>
    <span class="n">plt</span><span class="o">.</span><span class="n">close</span><span class="p">()</span>

<span class="k">def</span> <span class="nf">prediction_callback</span><span class="p">(</span><span class="n">protein_obj</span><span class="p">,</span> <span class="n">length</span><span class="p">,</span>
                        <span class="n">prediction_result</span><span class="p">,</span> <span class="n">input_features</span><span class="p">,</span> <span class="n">mode</span><span class="p">):</span>
  <span class="n">model_name</span><span class="p">,</span> <span class="n">relaxed</span> <span class="o">=</span> <span class="n">mode</span>
  <span class="k">if</span> <span class="ow">not</span> <span class="n">relaxed</span><span class="p">:</span>
    <span class="k">if</span> <span class="n">display_images</span><span class="p">:</span>
      <span class="n">fig</span> <span class="o">=</span> <span class="n">plot_protein</span><span class="p">(</span><span class="n">protein_obj</span><span class="p">,</span> <span class="n">Ls</span><span class="o">=</span><span class="n">length</span><span class="p">,</span> <span class="n">dpi</span><span class="o">=</span><span class="mi">150</span><span class="p">)</span>
      <span class="n">plt</span><span class="o">.</span><span class="n">show</span><span class="p">()</span>
      <span class="n">plt</span><span class="o">.</span><span class="n">close</span><span class="p">()</span>

<span class="n">result_dir</span> <span class="o">=</span> <span class="n">jobname</span>
<span class="n">log_filename</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">jobname</span><span class="p">,</span><span class="s2">&quot;log.txt&quot;</span><span class="p">)</span>
<span class="n">setup_logging</span><span class="p">(</span><span class="n">Path</span><span class="p">(</span><span class="n">log_filename</span><span class="p">))</span>

<span class="n">queries</span><span class="p">,</span> <span class="n">is_complex</span> <span class="o">=</span> <span class="n">get_queries</span><span class="p">(</span><span class="n">queries_path</span><span class="p">)</span>
<span class="n">model_type</span> <span class="o">=</span> <span class="n">set_model_type</span><span class="p">(</span><span class="n">is_complex</span><span class="p">,</span> <span class="n">model_type</span><span class="p">)</span>

<span class="k">if</span> <span class="s2">&quot;multimer&quot;</span> <span class="ow">in</span> <span class="n">model_type</span> <span class="ow">and</span> <span class="n">max_msa</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span><span class="p">:</span>
  <span class="n">use_cluster_profile</span> <span class="o">=</span> <span class="kc">False</span>
<span class="k">else</span><span class="p">:</span>
  <span class="n">use_cluster_profile</span> <span class="o">=</span> <span class="kc">True</span>

<span class="n">download_alphafold_params</span><span class="p">(</span><span class="n">model_type</span><span class="p">,</span> <span class="n">Path</span><span class="p">(</span><span class="s2">&quot;.&quot;</span><span class="p">))</span>
<span class="n">results</span> <span class="o">=</span> <span class="n">run</span><span class="p">(</span>
    <span class="n">queries</span><span class="o">=</span><span class="n">queries</span><span class="p">,</span>
    <span class="n">result_dir</span><span class="o">=</span><span class="n">result_dir</span><span class="p">,</span>
    <span class="n">use_templates</span><span class="o">=</span><span class="n">use_templates</span><span class="p">,</span>
    <span class="n">custom_template_path</span><span class="o">=</span><span class="n">custom_template_path</span><span class="p">,</span>
    <span class="n">num_relax</span><span class="o">=</span><span class="n">num_relax</span><span class="p">,</span>
    <span class="n">msa_mode</span><span class="o">=</span><span class="n">msa_mode</span><span class="p">,</span>
    <span class="n">model_type</span><span class="o">=</span><span class="n">model_type</span><span class="p">,</span>
    <span class="n">num_models</span><span class="o">=</span><span class="mi">5</span><span class="p">,</span>
    <span class="n">num_recycles</span><span class="o">=</span><span class="n">num_recycles</span><span class="p">,</span>
    <span class="n">relax_max_iterations</span><span class="o">=</span><span class="n">relax_max_iterations</span><span class="p">,</span>
    <span class="n">recycle_early_stop_tolerance</span><span class="o">=</span><span class="n">recycle_early_stop_tolerance</span><span class="p">,</span>
    <span class="n">num_seeds</span><span class="o">=</span><span class="n">num_seeds</span><span class="p">,</span>
    <span class="n">use_dropout</span><span class="o">=</span><span class="n">use_dropout</span><span class="p">,</span>
    <span class="n">model_order</span><span class="o">=</span><span class="p">[</span><span class="mi">1</span><span class="p">,</span><span class="mi">2</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span><span class="mi">4</span><span class="p">,</span><span class="mi">5</span><span class="p">],</span>
    <span class="n">is_complex</span><span class="o">=</span><span class="n">is_complex</span><span class="p">,</span>
    <span class="n">data_dir</span><span class="o">=</span><span class="n">Path</span><span class="p">(</span><span class="s2">&quot;.&quot;</span><span class="p">),</span>
    <span class="n">keep_existing_results</span><span class="o">=</span><span class="kc">False</span><span class="p">,</span>
    <span class="n">rank_by</span><span class="o">=</span><span class="s2">&quot;auto&quot;</span><span class="p">,</span>
    <span class="n">pair_mode</span><span class="o">=</span><span class="n">pair_mode</span><span class="p">,</span>
    <span class="n">pairing_strategy</span><span class="o">=</span><span class="n">pairing_strategy</span><span class="p">,</span>
    <span class="n">stop_at_score</span><span class="o">=</span><span class="nb">float</span><span class="p">(</span><span class="mi">100</span><span class="p">),</span>
    <span class="n">prediction_callback</span><span class="o">=</span><span class="n">prediction_callback</span><span class="p">,</span>
    <span class="n">dpi</span><span class="o">=</span><span class="n">dpi</span><span class="p">,</span>
    <span class="n">zip_results</span><span class="o">=</span><span class="kc">False</span><span class="p">,</span>
    <span class="n">save_all</span><span class="o">=</span><span class="n">save_all</span><span class="p">,</span>
    <span class="n">max_msa</span><span class="o">=</span><span class="n">max_msa</span><span class="p">,</span>
    <span class="n">use_cluster_profile</span><span class="o">=</span><span class="n">use_cluster_profile</span><span class="p">,</span>
    <span class="n">input_features_callback</span><span class="o">=</span><span class="n">input_features_callback</span><span class="p">,</span>
    <span class="n">save_recycles</span><span class="o">=</span><span class="n">save_recycles</span><span class="p">,</span>
    <span class="n">user_agent</span><span class="o">=</span><span class="s2">&quot;colabfold/google-colab-main&quot;</span><span class="p">,</span>
<span class="p">)</span>
<span class="n">results_zip</span> <span class="o">=</span> <span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.result.zip&quot;</span>
<span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;zip -r </span><span class="si">{</span><span class="n">results_zip</span><span class="si">}</span><span class="s2"> </span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">&quot;</span><span class="p">)</span>
</pre></div>
</div>
</div>
<div class="cell_output docutils container">
<div class="output stderr highlight-myst-ansi notranslate"><div class="highlight"><pre><span></span>Downloading alphafold2 weights to .: 100%|██████████| 3.47G/3.47G [01:33&lt;00:00, 39.9MB/s]
</pre></div>
</div>
<div class="output stream highlight-myst-ansi notranslate"><div class="highlight"><pre><span></span>2023-12-08 01:03:26,095 Unable to initialize backend &#39;rocm&#39;: NOT_FOUND: Could not find registered platform with name: &quot;rocm&quot;. Available platform names are: CUDA
2023-12-08 01:03:26,097 Unable to initialize backend &#39;tpu&#39;: INTERNAL: Failed to open libtpu.so: libtpu.so: cannot open shared object file: No such file or directory
2023-12-08 01:03:28,068 Running on GPU
2023-12-08 01:03:28,251 Found 3 citations for tools or databases
2023-12-08 01:03:28,252 Query 1/1: colabfoldbax1f16_f85cf (length 192)
</pre></div>
</div>
<div class="output stderr highlight-myst-ansi notranslate"><div class="highlight"><pre><span></span>COMPLETE: 100%|██████████| 150/150 [elapsed: 00:02 remaining: 00:00]
</pre></div>
</div>
<img alt="../_images/51afc86871184df45c2b11a63d853044193ea9104d12bf9adf0d571de06bceb2.png" src="../_images/51afc86871184df45c2b11a63d853044193ea9104d12bf9adf0d571de06bceb2.png" />
<div class="output stream highlight-myst-ansi notranslate"><div class="highlight"><pre><span></span>2023-12-08 01:03:32,044 Setting max_seq=512, max_extra_seq=1703
2023-12-08 01:04:07,724 alphafold2_ptm_model_1_seed_000 recycle=0 pLDDT=70.2 pTM=0.694
2023-12-08 01:04:13,490 alphafold2_ptm_model_1_seed_000 recycle=1 pLDDT=80.9 pTM=0.793 tol=1.07
2023-12-08 01:04:19,257 alphafold2_ptm_model_1_seed_000 recycle=2 pLDDT=78.4 pTM=0.775 tol=0.281
2023-12-08 01:04:25,019 alphafold2_ptm_model_1_seed_000 recycle=3 pLDDT=81.1 pTM=0.798 tol=0.165
2023-12-08 01:04:25,021 alphafold2_ptm_model_1_seed_000 took 39.0s (3 recycles)
</pre></div>
</div>
<img alt="../_images/f65c685391c095fc6944b7e34723544d79938dec21f0d4d5f9f930b0c1266b97.png" src="../_images/f65c685391c095fc6944b7e34723544d79938dec21f0d4d5f9f930b0c1266b97.png" />
<div class="output stream highlight-myst-ansi notranslate"><div class="highlight"><pre><span></span>2023-12-08 01:04:31,023 alphafold2_ptm_model_2_seed_000 recycle=0 pLDDT=68.3 pTM=0.67
2023-12-08 01:04:36,783 alphafold2_ptm_model_2_seed_000 recycle=1 pLDDT=79.3 pTM=0.783 tol=1.21
2023-12-08 01:04:42,548 alphafold2_ptm_model_2_seed_000 recycle=2 pLDDT=78.2 pTM=0.779 tol=0.353
2023-12-08 01:04:48,310 alphafold2_ptm_model_2_seed_000 recycle=3 pLDDT=79.4 pTM=0.789 tol=0.121
2023-12-08 01:04:48,311 alphafold2_ptm_model_2_seed_000 took 23.1s (3 recycles)
</pre></div>
</div>
<img alt="../_images/056fe209adc8b6f3e35c1a5161f46f7a9ad34b5190df977a7ba43c55fded4685.png" src="../_images/056fe209adc8b6f3e35c1a5161f46f7a9ad34b5190df977a7ba43c55fded4685.png" />
<div class="output stream highlight-myst-ansi notranslate"><div class="highlight"><pre><span></span>2023-12-08 01:04:54,270 alphafold2_ptm_model_3_seed_000 recycle=0 pLDDT=74.4 pTM=0.721
2023-12-08 01:05:00,044 alphafold2_ptm_model_3_seed_000 recycle=1 pLDDT=83.6 pTM=0.822 tol=0.98
2023-12-08 01:05:05,823 alphafold2_ptm_model_3_seed_000 recycle=2 pLDDT=84.2 pTM=0.828 tol=0.401
2023-12-08 01:05:11,599 alphafold2_ptm_model_3_seed_000 recycle=3 pLDDT=84 pTM=0.821 tol=0.224
2023-12-08 01:05:11,600 alphafold2_ptm_model_3_seed_000 took 23.1s (3 recycles)
</pre></div>
</div>
<img alt="../_images/90772f57f1ec3fe79cbbc8c18bdf73cc975282b5b48acff8eab5f014fec562e8.png" src="../_images/90772f57f1ec3fe79cbbc8c18bdf73cc975282b5b48acff8eab5f014fec562e8.png" />
<div class="output stream highlight-myst-ansi notranslate"><div class="highlight"><pre><span></span>2023-12-08 01:05:17,555 alphafold2_ptm_model_4_seed_000 recycle=0 pLDDT=71 pTM=0.702
2023-12-08 01:05:23,318 alphafold2_ptm_model_4_seed_000 recycle=1 pLDDT=78.8 pTM=0.778 tol=0.867
2023-12-08 01:05:29,082 alphafold2_ptm_model_4_seed_000 recycle=2 pLDDT=78 pTM=0.763 tol=0.439
2023-12-08 01:05:34,846 alphafold2_ptm_model_4_seed_000 recycle=3 pLDDT=76.6 pTM=0.736 tol=0.257
2023-12-08 01:05:34,847 alphafold2_ptm_model_4_seed_000 took 23.1s (3 recycles)
</pre></div>
</div>
<img alt="../_images/1044d6c4f48649fb7224c0e3b6dbfdc327df4ebf04fde37b437647f254b50063.png" src="../_images/1044d6c4f48649fb7224c0e3b6dbfdc327df4ebf04fde37b437647f254b50063.png" />
<div class="output stream highlight-myst-ansi notranslate"><div class="highlight"><pre><span></span>2023-12-08 01:05:40,809 alphafold2_ptm_model_5_seed_000 recycle=0 pLDDT=75.6 pTM=0.776
2023-12-08 01:05:46,584 alphafold2_ptm_model_5_seed_000 recycle=1 pLDDT=81.9 pTM=0.828 tol=1.01
2023-12-08 01:05:52,360 alphafold2_ptm_model_5_seed_000 recycle=2 pLDDT=81.6 pTM=0.823 tol=0.38
2023-12-08 01:05:58,134 alphafold2_ptm_model_5_seed_000 recycle=3 pLDDT=81.8 pTM=0.822 tol=0.216
2023-12-08 01:05:58,135 alphafold2_ptm_model_5_seed_000 took 23.1s (3 recycles)
</pre></div>
</div>
<img alt="../_images/45c21eb98d5e1c04f68d6357455e20d8e855a9242c8d18230861eb03a1f8edb3.png" src="../_images/45c21eb98d5e1c04f68d6357455e20d8e855a9242c8d18230861eb03a1f8edb3.png" />
<div class="output stream highlight-myst-ansi notranslate"><div class="highlight"><pre><span></span>2023-12-08 01:05:58,341 reranking models by &#39;plddt&#39; metric
2023-12-08 01:05:58,341 rank_001_alphafold2_ptm_model_3_seed_000 pLDDT=84 pTM=0.821
2023-12-08 01:05:58,342 rank_002_alphafold2_ptm_model_5_seed_000 pLDDT=81.8 pTM=0.822
2023-12-08 01:05:58,343 rank_003_alphafold2_ptm_model_1_seed_000 pLDDT=81.1 pTM=0.798
2023-12-08 01:05:58,343 rank_004_alphafold2_ptm_model_2_seed_000 pLDDT=79.4 pTM=0.789
2023-12-08 01:05:58,344 rank_005_alphafold2_ptm_model_4_seed_000 pLDDT=76.6 pTM=0.736
2023-12-08 01:06:00,018 Done
</pre></div>
</div>
<div class="output text_plain highlight-myst-ansi notranslate"><div class="highlight"><pre><span></span>0
</pre></div>
</div>
</div>
</div>
<div class="cell docutils container">
<div class="cell_input docutils container">
<div class="highlight-ipython3 notranslate"><div class="highlight"><pre><span></span><span class="c1"># @title # **7. Visualize 3D Structure of ColabFold Prediction** { run: &quot;auto&quot;, display-mode: &quot;form&quot; }</span>

<span class="c1">#@markdown Rank number/score for the **monomer model is the average pLDDT**,</span>
<span class="c1">#@markdown where a ranking of 1 correlates better local structure confidence.</span>
<span class="c1">#@markdown An acceptable model prediction should average $\ge 85$.</span>
<span class="c1">#@markdown The number of ranked models was set in step `6. Run Prediction` (step 6, line 75.).</span>

<span class="c1">#@markdown \</span>
<span class="c1">#@markdown Here, you can change the value for `rank_num` to visualize the 5 predictions.</span>

<span class="kn">import</span> <span class="nn">py3Dmol</span>
<span class="kn">import</span> <span class="nn">glob</span>
<span class="kn">import</span> <span class="nn">matplotlib.pyplot</span> <span class="k">as</span> <span class="nn">plt</span>
<span class="kn">from</span> <span class="nn">colabfold.colabfold</span> <span class="kn">import</span> <span class="n">plot_plddt_legend</span>
<span class="kn">from</span> <span class="nn">colabfold.colabfold</span> <span class="kn">import</span> <span class="n">pymol_color_list</span><span class="p">,</span> <span class="n">alphabet_list</span>

<span class="n">rank_num</span> <span class="o">=</span> <span class="mi">1</span> <span class="c1">#@param [&quot;1&quot;, &quot;2&quot;, &quot;3&quot;, &quot;4&quot;, &quot;5&quot;] {type:&quot;raw&quot;}</span>
<span class="n">color</span> <span class="o">=</span> <span class="s2">&quot;rainbow&quot;</span> <span class="c1">#@param [&quot;chain&quot;, &quot;lDDT&quot;, &quot;rainbow&quot;]</span>
<span class="n">show_sidechains</span> <span class="o">=</span> <span class="kc">False</span> <span class="c1">#@param {type:&quot;boolean&quot;}</span>
<span class="n">show_mainchains</span> <span class="o">=</span> <span class="kc">False</span> <span class="c1">#@param {type:&quot;boolean&quot;}</span>

<span class="n">tag</span> <span class="o">=</span> <span class="n">results</span><span class="p">[</span><span class="s2">&quot;rank&quot;</span><span class="p">][</span><span class="mi">0</span><span class="p">][</span><span class="n">rank_num</span> <span class="o">-</span> <span class="mi">1</span><span class="p">]</span>
<span class="n">jobname_prefix</span> <span class="o">=</span> <span class="s2">&quot;.custom&quot;</span> <span class="k">if</span> <span class="n">msa_mode</span> <span class="o">==</span> <span class="s2">&quot;custom&quot;</span> <span class="k">else</span> <span class="s2">&quot;&quot;</span>
<span class="n">pdb_filename</span> <span class="o">=</span> <span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">/</span><span class="si">{</span><span class="n">jobname</span><span class="si">}{</span><span class="n">jobname_prefix</span><span class="si">}</span><span class="s2">_unrelaxed_</span><span class="si">{</span><span class="n">tag</span><span class="si">}</span><span class="s2">.pdb&quot;</span>
<span class="n">pdb_file</span> <span class="o">=</span> <span class="n">glob</span><span class="o">.</span><span class="n">glob</span><span class="p">(</span><span class="n">pdb_filename</span><span class="p">)</span>

<span class="k">def</span> <span class="nf">show_pdb</span><span class="p">(</span><span class="n">rank_num</span><span class="o">=</span><span class="mi">1</span><span class="p">,</span> <span class="n">show_sidechains</span><span class="o">=</span><span class="kc">False</span><span class="p">,</span> <span class="n">show_mainchains</span><span class="o">=</span><span class="kc">False</span><span class="p">,</span> <span class="n">color</span><span class="o">=</span><span class="s2">&quot;lDDT&quot;</span><span class="p">):</span>
  <span class="n">model_name</span> <span class="o">=</span> <span class="sa">f</span><span class="s2">&quot;rank_</span><span class="si">{</span><span class="n">rank_num</span><span class="si">}</span><span class="s2">&quot;</span>
  <span class="n">view</span> <span class="o">=</span> <span class="n">py3Dmol</span><span class="o">.</span><span class="n">view</span><span class="p">(</span><span class="n">js</span><span class="o">=</span><span class="s1">&#39;https://3dmol.org/build/3Dmol.js&#39;</span><span class="p">,)</span>
  <span class="n">view</span><span class="o">.</span><span class="n">addModel</span><span class="p">(</span><span class="nb">open</span><span class="p">(</span><span class="n">pdb_file</span><span class="p">[</span><span class="mi">0</span><span class="p">],</span><span class="s1">&#39;r&#39;</span><span class="p">)</span><span class="o">.</span><span class="n">read</span><span class="p">(),</span><span class="s1">&#39;pdb&#39;</span><span class="p">)</span>


  <span class="k">if</span> <span class="n">color</span> <span class="o">==</span> <span class="s2">&quot;lDDT&quot;</span><span class="p">:</span>
    <span class="n">view</span><span class="o">.</span><span class="n">setStyle</span><span class="p">({</span><span class="s1">&#39;cartoon&#39;</span><span class="p">:</span> <span class="p">{</span><span class="s1">&#39;colorscheme&#39;</span><span class="p">:</span> <span class="p">{</span><span class="s1">&#39;prop&#39;</span><span class="p">:</span><span class="s1">&#39;b&#39;</span><span class="p">,</span><span class="s1">&#39;gradient&#39;</span><span class="p">:</span> <span class="s1">&#39;roygb&#39;</span><span class="p">,</span><span class="s1">&#39;min&#39;</span><span class="p">:</span><span class="mi">50</span><span class="p">,</span><span class="s1">&#39;max&#39;</span><span class="p">:</span><span class="mi">90</span><span class="p">}}})</span>
  <span class="k">elif</span> <span class="n">color</span> <span class="o">==</span> <span class="s2">&quot;rainbow&quot;</span><span class="p">:</span>
    <span class="n">view</span><span class="o">.</span><span class="n">setStyle</span><span class="p">({</span><span class="s1">&#39;cartoon&#39;</span><span class="p">:</span> <span class="p">{</span><span class="s1">&#39;color&#39;</span><span class="p">:</span><span class="s1">&#39;spectrum&#39;</span><span class="p">}})</span>
  <span class="k">elif</span> <span class="n">color</span> <span class="o">==</span> <span class="s2">&quot;chain&quot;</span><span class="p">:</span>
    <span class="n">chains</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">queries</span><span class="p">[</span><span class="mi">0</span><span class="p">][</span><span class="mi">1</span><span class="p">])</span> <span class="o">+</span> <span class="mi">1</span> <span class="k">if</span> <span class="n">is_complex</span> <span class="k">else</span> <span class="mi">1</span>
    <span class="k">for</span> <span class="n">n</span><span class="p">,</span><span class="n">chain</span><span class="p">,</span><span class="n">color</span> <span class="ow">in</span> <span class="nb">zip</span><span class="p">(</span><span class="nb">range</span><span class="p">(</span><span class="n">chains</span><span class="p">),</span><span class="n">alphabet_list</span><span class="p">,</span><span class="n">pymol_color_list</span><span class="p">):</span>
       <span class="n">view</span><span class="o">.</span><span class="n">setStyle</span><span class="p">({</span><span class="s1">&#39;chain&#39;</span><span class="p">:</span><span class="n">chain</span><span class="p">},{</span><span class="s1">&#39;cartoon&#39;</span><span class="p">:</span> <span class="p">{</span><span class="s1">&#39;color&#39;</span><span class="p">:</span><span class="n">color</span><span class="p">}})</span>

  <span class="k">if</span> <span class="n">show_sidechains</span><span class="p">:</span>
    <span class="n">BB</span> <span class="o">=</span> <span class="p">[</span><span class="s1">&#39;C&#39;</span><span class="p">,</span><span class="s1">&#39;O&#39;</span><span class="p">,</span><span class="s1">&#39;N&#39;</span><span class="p">]</span>
    <span class="n">view</span><span class="o">.</span><span class="n">addStyle</span><span class="p">({</span><span class="s1">&#39;and&#39;</span><span class="p">:[{</span><span class="s1">&#39;resn&#39;</span><span class="p">:[</span><span class="s2">&quot;GLY&quot;</span><span class="p">,</span><span class="s2">&quot;PRO&quot;</span><span class="p">],</span><span class="s1">&#39;invert&#39;</span><span class="p">:</span><span class="kc">True</span><span class="p">},{</span><span class="s1">&#39;atom&#39;</span><span class="p">:</span><span class="n">BB</span><span class="p">,</span><span class="s1">&#39;invert&#39;</span><span class="p">:</span><span class="kc">True</span><span class="p">}]},</span>
                        <span class="p">{</span><span class="s1">&#39;stick&#39;</span><span class="p">:{</span><span class="s1">&#39;colorscheme&#39;</span><span class="p">:</span><span class="sa">f</span><span class="s2">&quot;WhiteCarbon&quot;</span><span class="p">,</span><span class="s1">&#39;radius&#39;</span><span class="p">:</span><span class="mf">0.3</span><span class="p">}})</span>
    <span class="n">view</span><span class="o">.</span><span class="n">addStyle</span><span class="p">({</span><span class="s1">&#39;and&#39;</span><span class="p">:[{</span><span class="s1">&#39;resn&#39;</span><span class="p">:</span><span class="s2">&quot;GLY&quot;</span><span class="p">},{</span><span class="s1">&#39;atom&#39;</span><span class="p">:</span><span class="s1">&#39;CA&#39;</span><span class="p">}]},</span>
                        <span class="p">{</span><span class="s1">&#39;sphere&#39;</span><span class="p">:{</span><span class="s1">&#39;colorscheme&#39;</span><span class="p">:</span><span class="sa">f</span><span class="s2">&quot;WhiteCarbon&quot;</span><span class="p">,</span><span class="s1">&#39;radius&#39;</span><span class="p">:</span><span class="mf">0.3</span><span class="p">}})</span>
    <span class="n">view</span><span class="o">.</span><span class="n">addStyle</span><span class="p">({</span><span class="s1">&#39;and&#39;</span><span class="p">:[{</span><span class="s1">&#39;resn&#39;</span><span class="p">:</span><span class="s2">&quot;PRO&quot;</span><span class="p">},{</span><span class="s1">&#39;atom&#39;</span><span class="p">:[</span><span class="s1">&#39;C&#39;</span><span class="p">,</span><span class="s1">&#39;O&#39;</span><span class="p">],</span><span class="s1">&#39;invert&#39;</span><span class="p">:</span><span class="kc">True</span><span class="p">}]},</span>
                        <span class="p">{</span><span class="s1">&#39;stick&#39;</span><span class="p">:{</span><span class="s1">&#39;colorscheme&#39;</span><span class="p">:</span><span class="sa">f</span><span class="s2">&quot;WhiteCarbon&quot;</span><span class="p">,</span><span class="s1">&#39;radius&#39;</span><span class="p">:</span><span class="mf">0.3</span><span class="p">}})</span>
  <span class="k">if</span> <span class="n">show_mainchains</span><span class="p">:</span>
    <span class="n">BB</span> <span class="o">=</span> <span class="p">[</span><span class="s1">&#39;C&#39;</span><span class="p">,</span><span class="s1">&#39;O&#39;</span><span class="p">,</span><span class="s1">&#39;N&#39;</span><span class="p">,</span><span class="s1">&#39;CA&#39;</span><span class="p">]</span>
    <span class="n">view</span><span class="o">.</span><span class="n">addStyle</span><span class="p">({</span><span class="s1">&#39;atom&#39;</span><span class="p">:</span><span class="n">BB</span><span class="p">},{</span><span class="s1">&#39;stick&#39;</span><span class="p">:{</span><span class="s1">&#39;colorscheme&#39;</span><span class="p">:</span><span class="sa">f</span><span class="s2">&quot;WhiteCarbon&quot;</span><span class="p">,</span><span class="s1">&#39;radius&#39;</span><span class="p">:</span><span class="mf">0.3</span><span class="p">}},</span> <span class="n">viewer</span><span class="o">=</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">0</span><span class="p">))</span>

  <span class="n">view</span><span class="o">.</span><span class="n">zoomTo</span><span class="p">()</span>
  <span class="k">return</span> <span class="n">view</span>

<span class="nb">print</span><span class="p">(</span><span class="s1">&#39;Prediction: Rank </span><span class="si">%d</span><span class="s1">&#39;</span> <span class="o">%</span> <span class="n">rank_num</span><span class="p">)</span>
<span class="n">show_pdb</span><span class="p">(</span><span class="n">rank_num</span><span class="p">,</span> <span class="n">show_sidechains</span><span class="p">,</span> <span class="n">show_mainchains</span><span class="p">,</span> <span class="n">color</span><span class="p">)</span><span class="o">.</span><span class="n">show</span><span class="p">()</span>
<span class="k">if</span> <span class="n">color</span> <span class="o">==</span> <span class="s2">&quot;lDDT&quot;</span><span class="p">:</span>
  <span class="n">plot_plddt_legend</span><span class="p">()</span><span class="o">.</span><span class="n">show</span><span class="p">()</span>


<span class="c1">#@markdown \</span>
</pre></div>
</div>
</div>
<div class="cell_output docutils container">
<div class="output stream highlight-myst-ansi notranslate"><div class="highlight"><pre><span></span>Prediction: Rank 1
</pre></div>
</div>
<div class="output text_html"><div id="3dmolviewer_17019975603601787"  style="position: relative; width: 640px; height: 480px;">
        <p id="3dmolwarning_17019975603601787" style="background-color:#ffcccc;color:black">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>
        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>
        </div>
<script>

var loadScriptAsync = function(uri){
  return new Promise((resolve, reject) => {
    //this is to ignore the existence of requirejs amd
    var savedexports, savedmodule;
    if (typeof exports !== 'undefined') savedexports = exports;
    else exports = {}
    if (typeof module !== 'undefined') savedmodule = module;
    else module = {}

    var tag = document.createElement('script');
    tag.src = uri;
    tag.async = true;
    tag.onload = () => {
        exports = savedexports;
        module = savedmodule;
        resolve();
    };
  var firstScriptTag = document.getElementsByTagName('script')[0];
  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
});
};

if(typeof $3Dmolpromise === 'undefined') {
$3Dmolpromise = null;
  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');
}

var viewer_17019975603601787 = null;
var warn = document.getElementById("3dmolwarning_17019975603601787");
if(warn) {
    warn.parentNode.removeChild(warn);
}
$3Dmolpromise.then(function() {
viewer_17019975603601787 = $3Dmol.createViewer(document.getElementById("3dmolviewer_17019975603601787"),{backgroundColor:"white"});
viewer_17019975603601787.zoomTo();
	viewer_17019975603601787.addModel("MODEL     1                                                                     \nATOM      1  N   MET A   1      41.656 -40.344   1.231  1.00 50.78           N  \nATOM      2  CA  MET A   1      41.250 -38.938   1.264  1.00 50.78           C  \nATOM      3  C   MET A   1      40.625 -38.531  -0.065  1.00 50.78           C  \nATOM      4  CB  MET A   1      42.438 -38.031   1.587  1.00 50.78           C  \nATOM      5  O   MET A   1      41.219 -38.688  -1.122  1.00 50.78           O  \nATOM      6  CG  MET A   1      42.688 -37.875   3.076  1.00 50.78           C  \nATOM      7  SD  MET A   1      44.156 -36.812   3.424  1.00 50.78           S  \nATOM      8  CE  MET A   1      43.906 -36.531   5.203  1.00 50.78           C  \nATOM      9  N   ASP A   2      39.250 -38.625  -0.198  1.00 55.22           N  \nATOM     10  CA  ASP A   2      38.562 -37.844  -1.242  1.00 55.22           C  \nATOM     11  C   ASP A   2      37.094 -37.562  -0.849  1.00 55.22           C  \nATOM     12  CB  ASP A   2      38.594 -38.625  -2.570  1.00 55.22           C  \nATOM     13  O   ASP A   2      36.312 -38.469  -0.610  1.00 55.22           O  \nATOM     14  CG  ASP A   2      37.531 -39.688  -2.664  1.00 55.22           C  \nATOM     15  OD1 ASP A   2      36.812 -39.938  -1.662  1.00 55.22           O  \nATOM     16  OD2 ASP A   2      37.406 -40.312  -3.750  1.00 55.22           O  \nATOM     17  N   GLY A   3      36.844 -36.375  -0.252  1.00 46.88           N  \nATOM     18  CA  GLY A   3      36.250 -35.125  -0.689  1.00 46.88           C  \nATOM     19  C   GLY A   3      34.781 -34.969  -0.270  1.00 46.88           C  \nATOM     20  O   GLY A   3      33.938 -35.719  -0.718  1.00 46.88           O  \nATOM     21  N   SER A   4      34.562 -34.594   0.961  1.00 52.16           N  \nATOM     22  CA  SER A   4      33.562 -33.719   1.560  1.00 52.16           C  \nATOM     23  C   SER A   4      33.156 -32.625   0.577  1.00 52.16           C  \nATOM     24  CB  SER A   4      34.125 -33.062   2.828  1.00 52.16           C  \nATOM     25  O   SER A   4      33.938 -31.750   0.211  1.00 52.16           O  \nATOM     26  OG  SER A   4      34.906 -34.000   3.562  1.00 52.16           O  \nATOM     27  N   GLY A   5      32.375 -32.938  -0.415  1.00 46.44           N  \nATOM     28  CA  GLY A   5      31.672 -31.922  -1.189  1.00 46.44           C  \nATOM     29  C   GLY A   5      30.406 -31.422  -0.502  1.00 46.44           C  \nATOM     30  O   GLY A   5      29.547 -32.219  -0.132  1.00 46.44           O  \nATOM     31  N   GLU A   6      30.516 -30.688   0.589  1.00 48.91           N  \nATOM     32  CA  GLU A   6      29.703 -29.578   1.072  1.00 48.91           C  \nATOM     33  C   GLU A   6      28.828 -29.016  -0.036  1.00 48.91           C  \nATOM     34  CB  GLU A   6      30.609 -28.469   1.639  1.00 48.91           C  \nATOM     35  O   GLU A   6      29.312 -28.609  -1.092  1.00 48.91           O  \nATOM     36  CG  GLU A   6      30.656 -28.438   3.160  1.00 48.91           C  \nATOM     37  CD  GLU A   6      31.656 -27.438   3.705  1.00 48.91           C  \nATOM     38  OE1 GLU A   6      31.250 -26.297   4.027  1.00 48.91           O  \nATOM     39  OE2 GLU A   6      32.844 -27.781   3.811  1.00 48.91           O  \nATOM     40  N   GLN A   7      27.797 -29.656  -0.375  1.00 47.62           N  \nATOM     41  CA  GLN A   7      26.641 -29.062  -1.036  1.00 47.62           C  \nATOM     42  C   GLN A   7      26.266 -27.719  -0.405  1.00 47.62           C  \nATOM     43  CB  GLN A   7      25.438 -30.016  -0.981  1.00 47.62           C  \nATOM     44  O   GLN A   7      26.188 -27.609   0.819  1.00 47.62           O  \nATOM     45  CG  GLN A   7      25.047 -30.594  -2.334  1.00 47.62           C  \nATOM     46  CD  GLN A   7      24.016 -31.688  -2.225  1.00 47.62           C  \nATOM     47  NE2 GLN A   7      22.766 -31.359  -2.543  1.00 47.62           N  \nATOM     48  OE1 GLN A   7      24.328 -32.844  -1.860  1.00 47.62           O  \nATOM     49  N   PRO A   8      26.547 -26.562  -1.010  1.00 45.00           N  \nATOM     50  CA  PRO A   8      26.078 -25.250  -0.534  1.00 45.00           C  \nATOM     51  C   PRO A   8      24.578 -25.203  -0.337  1.00 45.00           C  \nATOM     52  CB  PRO A   8      26.516 -24.297  -1.646  1.00 45.00           C  \nATOM     53  O   PRO A   8      23.828 -25.766  -1.138  1.00 45.00           O  \nATOM     54  CG  PRO A   8      26.859 -25.172  -2.805  1.00 45.00           C  \nATOM     55  CD  PRO A   8      26.688 -26.609  -2.377  1.00 45.00           C  \nATOM     56  N   ARG A   9      23.969 -25.609   0.843  1.00 46.38           N  \nATOM     57  CA  ARG A   9      22.672 -25.172   1.353  1.00 46.38           C  \nATOM     58  C   ARG A   9      22.203 -23.906   0.643  1.00 46.38           C  \nATOM     59  CB  ARG A   9      22.750 -24.922   2.861  1.00 46.38           C  \nATOM     60  O   ARG A   9      22.750 -22.828   0.861  1.00 46.38           O  \nATOM     61  CG  ARG A   9      22.297 -26.125   3.697  1.00 46.38           C  \nATOM     62  CD  ARG A   9      22.234 -25.781   5.180  1.00 46.38           C  \nATOM     63  NE  ARG A   9      21.500 -26.781   5.934  1.00 46.38           N  \nATOM     64  NH1 ARG A   9      21.875 -25.812   7.996  1.00 46.38           N  \nATOM     65  NH2 ARG A   9      20.656 -27.750   7.840  1.00 46.38           N  \nATOM     66  CZ  ARG A   9      21.344 -26.781   7.254  1.00 46.38           C  \nATOM     67  N   GLY A  10      22.297 -23.906  -0.730  1.00 41.62           N  \nATOM     68  CA  GLY A  10      21.703 -22.953  -1.654  1.00 41.62           C  \nATOM     69  C   GLY A  10      20.250 -22.672  -1.379  1.00 41.62           C  \nATOM     70  O   GLY A  10      19.453 -23.609  -1.222  1.00 41.62           O  \nATOM     71  N   GLY A  11      19.859 -21.984  -0.265  1.00 47.22           N  \nATOM     72  CA  GLY A  11      18.719 -21.094  -0.112  1.00 47.22           C  \nATOM     73  C   GLY A  11      17.656 -21.312  -1.160  1.00 47.22           C  \nATOM     74  O   GLY A  11      17.844 -21.000  -2.334  1.00 47.22           O  \nATOM     75  N   GLY A  12      17.062 -22.500  -1.331  1.00 52.38           N  \nATOM     76  CA  GLY A  12      15.875 -22.859  -2.100  1.00 52.38           C  \nATOM     77  C   GLY A  12      14.992 -21.672  -2.418  1.00 52.38           C  \nATOM     78  O   GLY A  12      15.133 -20.609  -1.805  1.00 52.38           O  \nATOM     79  N   PRO A  13      14.531 -21.500  -3.699  1.00 58.81           N  \nATOM     80  CA  PRO A  13      13.523 -20.516  -4.078  1.00 58.81           C  \nATOM     81  C   PRO A  13      12.531 -20.219  -2.957  1.00 58.81           C  \nATOM     82  CB  PRO A  13      12.820 -21.156  -5.273  1.00 58.81           C  \nATOM     83  O   PRO A  13      12.219 -21.109  -2.160  1.00 58.81           O  \nATOM     84  CG  PRO A  13      13.406 -22.531  -5.359  1.00 58.81           C  \nATOM     85  CD  PRO A  13      14.477 -22.656  -4.309  1.00 58.81           C  \nATOM     86  N   THR A  14      12.688 -19.078  -2.326  1.00 71.31           N  \nATOM     87  CA  THR A  14      11.648 -18.672  -1.386  1.00 71.31           C  \nATOM     88  C   THR A  14      10.281 -19.172  -1.847  1.00 71.31           C  \nATOM     89  CB  THR A  14      11.609 -17.141  -1.215  1.00 71.31           C  \nATOM     90  O   THR A  14       9.891 -18.953  -3.000  1.00 71.31           O  \nATOM     91  CG2 THR A  14      10.711 -16.734  -0.048  1.00 71.31           C  \nATOM     92  OG1 THR A  14      12.938 -16.656  -0.969  1.00 71.31           O  \nATOM     93  N   SER A  15       9.672 -20.078  -1.165  1.00 87.69           N  \nATOM     94  CA  SER A  15       8.367 -20.688  -1.419  1.00 87.69           C  \nATOM     95  C   SER A  15       7.285 -19.625  -1.562  1.00 87.69           C  \nATOM     96  CB  SER A  15       7.996 -21.656  -0.295  1.00 87.69           C  \nATOM     97  O   SER A  15       7.312 -18.609  -0.863  1.00 87.69           O  \nATOM     98  OG  SER A  15       6.676 -22.141  -0.468  1.00 87.69           O  \nATOM     99  N   SER A  16       6.555 -19.625  -2.615  1.00 91.69           N  \nATOM    100  CA  SER A  16       5.414 -18.750  -2.859  1.00 91.69           C  \nATOM    101  C   SER A  16       4.590 -18.547  -1.592  1.00 91.69           C  \nATOM    102  CB  SER A  16       4.527 -19.312  -3.967  1.00 91.69           C  \nATOM    103  O   SER A  16       4.156 -17.422  -1.300  1.00 91.69           O  \nATOM    104  OG  SER A  16       5.211 -19.312  -5.207  1.00 91.69           O  \nATOM    105  N   GLU A  17       4.426 -19.594  -0.853  1.00 91.62           N  \nATOM    106  CA  GLU A  17       3.643 -19.547   0.377  1.00 91.62           C  \nATOM    107  C   GLU A  17       4.305 -18.641   1.414  1.00 91.62           C  \nATOM    108  CB  GLU A  17       3.439 -20.938   0.952  1.00 91.62           C  \nATOM    109  O   GLU A  17       3.631 -17.875   2.102  1.00 91.62           O  \nATOM    110  CG  GLU A  17       2.453 -21.797   0.166  1.00 91.62           C  \nATOM    111  CD  GLU A  17       2.268 -23.188   0.746  1.00 91.62           C  \nATOM    112  OE1 GLU A  17       1.416 -23.953   0.233  1.00 91.62           O  \nATOM    113  OE2 GLU A  17       2.979 -23.516   1.721  1.00 91.62           O  \nATOM    114  N   GLN A  18       5.605 -18.766   1.507  1.00 91.38           N  \nATOM    115  CA  GLN A  18       6.355 -17.922   2.438  1.00 91.38           C  \nATOM    116  C   GLN A  18       6.281 -16.453   2.039  1.00 91.38           C  \nATOM    117  CB  GLN A  18       7.816 -18.375   2.510  1.00 91.38           C  \nATOM    118  O   GLN A  18       6.148 -15.578   2.895  1.00 91.38           O  \nATOM    119  CG  GLN A  18       8.023 -19.672   3.289  1.00 91.38           C  \nATOM    120  CD  GLN A  18       9.469 -20.141   3.271  1.00 91.38           C  \nATOM    121  NE2 GLN A  18       9.836 -20.953   4.258  1.00 91.38           N  \nATOM    122  OE1 GLN A  18      10.242 -19.781   2.377  1.00 91.38           O  \nATOM    123  N   ILE A  19       6.371 -16.203   0.749  1.00 93.50           N  \nATOM    124  CA  ILE A  19       6.297 -14.836   0.243  1.00 93.50           C  \nATOM    125  C   ILE A  19       4.938 -14.234   0.582  1.00 93.50           C  \nATOM    126  CB  ILE A  19       6.543 -14.781  -1.282  1.00 93.50           C  \nATOM    127  O   ILE A  19       4.852 -13.094   1.037  1.00 93.50           O  \nATOM    128  CG1 ILE A  19       7.996 -15.148  -1.603  1.00 93.50           C  \nATOM    129  CG2 ILE A  19       6.188 -13.406  -1.840  1.00 93.50           C  \nATOM    130  CD1 ILE A  19       8.289 -15.273  -3.092  1.00 93.50           C  \nATOM    131  N   MET A  20       3.922 -15.023   0.380  1.00 95.50           N  \nATOM    132  CA  MET A  20       2.570 -14.555   0.668  1.00 95.50           C  \nATOM    133  C   MET A  20       2.379 -14.336   2.164  1.00 95.50           C  \nATOM    134  CB  MET A  20       1.532 -15.547   0.146  1.00 95.50           C  \nATOM    135  O   MET A  20       1.766 -13.352   2.578  1.00 95.50           O  \nATOM    136  CG  MET A  20       1.376 -15.531  -1.366  1.00 95.50           C  \nATOM    137  SD  MET A  20       0.961 -13.867  -2.016  1.00 95.50           S  \nATOM    138  CE  MET A  20      -0.699 -13.656  -1.315  1.00 95.50           C  \nATOM    139  N   LYS A  21       2.889 -15.273   2.914  1.00 92.94           N  \nATOM    140  CA  LYS A  21       2.775 -15.148   4.363  1.00 92.94           C  \nATOM    141  C   LYS A  21       3.473 -13.883   4.855  1.00 92.94           C  \nATOM    142  CB  LYS A  21       3.365 -16.375   5.059  1.00 92.94           C  \nATOM    143  O   LYS A  21       2.920 -13.141   5.676  1.00 92.94           O  \nATOM    144  CG  LYS A  21       3.133 -16.406   6.562  1.00 92.94           C  \nATOM    145  CD  LYS A  21       3.684 -17.672   7.191  1.00 92.94           C  \nATOM    146  CE  LYS A  21       3.520 -17.672   8.703  1.00 92.94           C  \nATOM    147  NZ  LYS A  21       4.047 -18.922   9.320  1.00 92.94           N  \nATOM    148  N   THR A  22       4.676 -13.656   4.363  1.00 93.12           N  \nATOM    149  CA  THR A  22       5.418 -12.453   4.723  1.00 93.12           C  \nATOM    150  C   THR A  22       4.672 -11.203   4.273  1.00 93.12           C  \nATOM    151  CB  THR A  22       6.828 -12.461   4.109  1.00 93.12           C  \nATOM    152  O   THR A  22       4.578 -10.227   5.020  1.00 93.12           O  \nATOM    153  CG2 THR A  22       7.609 -11.219   4.512  1.00 93.12           C  \nATOM    154  OG1 THR A  22       7.535 -13.625   4.570  1.00 93.12           O  \nATOM    155  N   GLY A  23       4.160 -11.227   3.047  1.00 94.50           N  \nATOM    156  CA  GLY A  23       3.365 -10.109   2.555  1.00 94.50           C  \nATOM    157  C   GLY A  23       2.176  -9.789   3.439  1.00 94.50           C  \nATOM    158  O   GLY A  23       1.900  -8.625   3.721  1.00 94.50           O  \nATOM    159  N   ALA A  24       1.489 -10.812   3.875  1.00 94.06           N  \nATOM    160  CA  ALA A  24       0.335 -10.641   4.750  1.00 94.06           C  \nATOM    161  C   ALA A  24       0.748 -10.016   6.082  1.00 94.06           C  \nATOM    162  CB  ALA A  24      -0.365 -11.977   4.984  1.00 94.06           C  \nATOM    163  O   ALA A  24       0.075  -9.117   6.590  1.00 94.06           O  \nATOM    164  N   LEU A  25       1.843 -10.453   6.602  1.00 92.19           N  \nATOM    165  CA  LEU A  25       2.350  -9.938   7.867  1.00 92.19           C  \nATOM    166  C   LEU A  25       2.746  -8.469   7.734  1.00 92.19           C  \nATOM    167  CB  LEU A  25       3.549 -10.758   8.344  1.00 92.19           C  \nATOM    168  O   LEU A  25       2.453  -7.660   8.617  1.00 92.19           O  \nATOM    169  CG  LEU A  25       3.238 -12.156   8.883  1.00 92.19           C  \nATOM    170  CD1 LEU A  25       4.531 -12.922   9.156  1.00 92.19           C  \nATOM    171  CD2 LEU A  25       2.391 -12.062  10.148  1.00 92.19           C  \nATOM    172  N   LEU A  26       3.371  -8.195   6.617  1.00 94.00           N  \nATOM    173  CA  LEU A  26       3.812  -6.820   6.383  1.00 94.00           C  \nATOM    174  C   LEU A  26       2.619  -5.883   6.242  1.00 94.00           C  \nATOM    175  CB  LEU A  26       4.688  -6.750   5.129  1.00 94.00           C  \nATOM    176  O   LEU A  26       2.617  -4.785   6.801  1.00 94.00           O  \nATOM    177  CG  LEU A  26       6.059  -7.418   5.219  1.00 94.00           C  \nATOM    178  CD1 LEU A  26       6.777  -7.336   3.875  1.00 94.00           C  \nATOM    179  CD2 LEU A  26       6.898  -6.773   6.320  1.00 94.00           C  \nATOM    180  N   LEU A  27       1.660  -6.270   5.445  1.00 94.56           N  \nATOM    181  CA  LEU A  27       0.470  -5.453   5.234  1.00 94.56           C  \nATOM    182  C   LEU A  27      -0.262  -5.215   6.551  1.00 94.56           C  \nATOM    183  CB  LEU A  27      -0.471  -6.121   4.230  1.00 94.56           C  \nATOM    184  O   LEU A  27      -0.620  -4.078   6.871  1.00 94.56           O  \nATOM    185  CG  LEU A  27      -1.763  -5.371   3.906  1.00 94.56           C  \nATOM    186  CD1 LEU A  27      -1.444  -3.996   3.324  1.00 94.56           C  \nATOM    187  CD2 LEU A  27      -2.625  -6.180   2.943  1.00 94.56           C  \nATOM    188  N   GLN A  28      -0.491  -6.258   7.230  1.00 91.62           N  \nATOM    189  CA  GLN A  28      -1.188  -6.160   8.508  1.00 91.62           C  \nATOM    190  C   GLN A  28      -0.437  -5.250   9.477  1.00 91.62           C  \nATOM    191  CB  GLN A  28      -1.370  -7.547   9.125  1.00 91.62           C  \nATOM    192  O   GLN A  28      -1.042  -4.406  10.141  1.00 91.62           O  \nATOM    193  CG  GLN A  28      -2.215  -7.551  10.398  1.00 91.62           C  \nATOM    194  CD  GLN A  28      -2.514  -8.953  10.898  1.00 91.62           C  \nATOM    195  NE2 GLN A  28      -3.270  -9.039  11.984  1.00 91.62           N  \nATOM    196  OE1 GLN A  28      -2.072  -9.945  10.312  1.00 91.62           O  \nATOM    197  N   GLY A  29       0.897  -5.520   9.648  1.00 90.75           N  \nATOM    198  CA  GLY A  29       1.728  -4.676  10.500  1.00 90.75           C  \nATOM    199  C   GLY A  29       1.714  -3.217  10.086  1.00 90.75           C  \nATOM    200  O   GLY A  29       1.664  -2.328  10.938  1.00 90.75           O  \nATOM    201  N   PHE A  30       1.760  -3.037   8.812  1.00 91.75           N  \nATOM    202  CA  PHE A  30       1.729  -1.690   8.258  1.00 91.75           C  \nATOM    203  C   PHE A  30       0.439  -0.976   8.633  1.00 91.75           C  \nATOM    204  CB  PHE A  30       1.875  -1.737   6.730  1.00 91.75           C  \nATOM    205  O   PHE A  30       0.467   0.176   9.078  1.00 91.75           O  \nATOM    206  CG  PHE A  30       2.154  -0.397   6.105  1.00 91.75           C  \nATOM    207  CD1 PHE A  30       1.462   0.016   4.977  1.00 91.75           C  \nATOM    208  CD2 PHE A  30       3.111   0.450   6.652  1.00 91.75           C  \nATOM    209  CE1 PHE A  30       1.719   1.256   4.395  1.00 91.75           C  \nATOM    210  CE2 PHE A  30       3.373   1.690   6.078  1.00 91.75           C  \nATOM    211  CZ  PHE A  30       2.678   2.092   4.949  1.00 91.75           C  \nATOM    212  N   ILE A  31      -0.705  -1.519   8.539  1.00 91.19           N  \nATOM    213  CA  ILE A  31      -2.020  -0.959   8.836  1.00 91.19           C  \nATOM    214  C   ILE A  31      -2.158  -0.735  10.336  1.00 91.19           C  \nATOM    215  CB  ILE A  31      -3.152  -1.875   8.320  1.00 91.19           C  \nATOM    216  O   ILE A  31      -2.658   0.304  10.773  1.00 91.19           O  \nATOM    217  CG1 ILE A  31      -3.146  -1.924   6.789  1.00 91.19           C  \nATOM    218  CG2 ILE A  31      -4.508  -1.402   8.852  1.00 91.19           C  \nATOM    219  CD1 ILE A  31      -4.078  -2.975   6.199  1.00 91.19           C  \nATOM    220  N   GLN A  32      -1.719  -1.665  11.125  1.00 88.69           N  \nATOM    221  CA  GLN A  32      -1.802  -1.565  12.578  1.00 88.69           C  \nATOM    222  C   GLN A  32      -1.021  -0.358  13.094  1.00 88.69           C  \nATOM    223  CB  GLN A  32      -1.281  -2.846  13.234  1.00 88.69           C  \nATOM    224  O   GLN A  32      -1.479   0.343  14.000  1.00 88.69           O  \nATOM    225  CG  GLN A  32      -1.549  -2.920  14.734  1.00 88.69           C  \nATOM    226  CD  GLN A  32      -1.140  -4.250  15.336  1.00 88.69           C  \nATOM    227  NE2 GLN A  32      -1.183  -4.340  16.656  1.00 88.69           N  \nATOM    228  OE1 GLN A  32      -0.788  -5.191  14.617  1.00 88.69           O  \nATOM    229  N   ASP A  33       0.200  -0.296  12.625  1.00 89.00           N  \nATOM    230  CA  ASP A  33       1.037   0.833  13.016  1.00 89.00           C  \nATOM    231  C   ASP A  33       0.313   2.158  12.789  1.00 89.00           C  \nATOM    232  CB  ASP A  33       2.357   0.816  12.242  1.00 89.00           C  \nATOM    233  O   ASP A  33       0.302   3.025  13.664  1.00 89.00           O  \nATOM    234  CG  ASP A  33       3.260   1.990  12.586  1.00 89.00           C  \nATOM    235  OD1 ASP A  33       3.832   2.016  13.695  1.00 89.00           O  \nATOM    236  OD2 ASP A  33       3.395   2.898  11.734  1.00 89.00           O  \nATOM    237  N   ARG A  34      -0.350   2.316  11.734  1.00 87.88           N  \nATOM    238  CA  ARG A  34      -1.006   3.566  11.359  1.00 87.88           C  \nATOM    239  C   ARG A  34      -2.344   3.715  12.078  1.00 87.88           C  \nATOM    240  CB  ARG A  34      -1.211   3.637   9.844  1.00 87.88           C  \nATOM    241  O   ARG A  34      -2.754   4.828  12.414  1.00 87.88           O  \nATOM    242  CG  ARG A  34       0.052   3.971   9.070  1.00 87.88           C  \nATOM    243  CD  ARG A  34      -0.151   3.811   7.570  1.00 87.88           C  \nATOM    244  NE  ARG A  34       0.980   4.340   6.812  1.00 87.88           N  \nATOM    245  NH1 ARG A  34       2.379   2.555   7.258  1.00 87.88           N  \nATOM    246  NH2 ARG A  34       3.117   4.301   5.973  1.00 87.88           N  \nATOM    247  CZ  ARG A  34       2.156   3.730   6.684  1.00 87.88           C  \nATOM    248  N   ALA A  35      -3.062   2.602  12.250  1.00 86.75           N  \nATOM    249  CA  ALA A  35      -4.309   2.637  13.008  1.00 86.75           C  \nATOM    250  C   ALA A  35      -4.055   3.025  14.461  1.00 86.75           C  \nATOM    251  CB  ALA A  35      -5.012   1.284  12.938  1.00 86.75           C  \nATOM    252  O   ALA A  35      -4.867   3.719  15.078  1.00 86.75           O  \nATOM    253  N   GLY A  36      -2.943   2.496  15.070  1.00 83.94           N  \nATOM    254  CA  GLY A  36      -2.561   2.834  16.422  1.00 83.94           C  \nATOM    255  C   GLY A  36      -2.303   4.316  16.625  1.00 83.94           C  \nATOM    256  O   GLY A  36      -2.557   4.855  17.703  1.00 83.94           O  \nATOM    257  N   ARG A  37      -1.934   4.977  15.633  1.00 79.62           N  \nATOM    258  CA  ARG A  37      -1.652   6.410  15.695  1.00 79.62           C  \nATOM    259  C   ARG A  37      -2.943   7.219  15.750  1.00 79.62           C  \nATOM    260  CB  ARG A  37      -0.806   6.848  14.500  1.00 79.62           C  \nATOM    261  O   ARG A  37      -2.957   8.344  16.250  1.00 79.62           O  \nATOM    262  CG  ARG A  37       0.616   6.309  14.523  1.00 79.62           C  \nATOM    263  CD  ARG A  37       1.424   6.805  13.328  1.00 79.62           C  \nATOM    264  NE  ARG A  37       2.754   6.199  13.289  1.00 79.62           N  \nATOM    265  NH1 ARG A  37       3.889   8.211  13.195  1.00 79.62           N  \nATOM    266  NH2 ARG A  37       5.047   6.230  13.203  1.00 79.62           N  \nATOM    267  CZ  ARG A  37       3.893   6.883  13.227  1.00 79.62           C  \nATOM    268  N   MET A  38      -4.020   6.613  15.211  1.00 76.88           N  \nATOM    269  CA  MET A  38      -5.324   7.273  15.211  1.00 76.88           C  \nATOM    270  C   MET A  38      -6.035   7.062  16.547  1.00 76.88           C  \nATOM    271  CB  MET A  38      -6.191   6.754  14.070  1.00 76.88           C  \nATOM    272  O   MET A  38      -7.059   7.695  16.812  1.00 76.88           O  \nATOM    273  CG  MET A  38      -7.395   7.629  13.766  1.00 76.88           C  \nATOM    274  SD  MET A  38      -8.508   6.891  12.508  1.00 76.88           S  \nATOM    275  CE  MET A  38      -7.461   7.031  11.031  1.00 76.88           C  \nATOM    276  N   GLY A  39      -5.418   6.375  17.500  1.00 70.69           N  \nATOM    277  CA  GLY A  39      -6.023   6.148  18.797  1.00 70.69           C  \nATOM    278  C   GLY A  39      -7.059   5.043  18.797  1.00 70.69           C  \nATOM    279  O   GLY A  39      -7.777   4.848  19.781  1.00 70.69           O  \nATOM    280  N   GLY A  40      -7.215   4.262  17.625  1.00 64.44           N  \nATOM    281  CA  GLY A  40      -8.258   3.250  17.516  1.00 64.44           C  \nATOM    282  C   GLY A  40      -7.730   1.836  17.656  1.00 64.44           C  \nATOM    283  O   GLY A  40      -6.609   1.537  17.250  1.00 64.44           O  \nATOM    284  N   GLU A  41      -7.762   1.247  18.875  1.00 60.88           N  \nATOM    285  CA  GLU A  41      -7.477  -0.176  19.031  1.00 60.88           C  \nATOM    286  C   GLU A  41      -8.336  -1.013  18.078  1.00 60.88           C  \nATOM    287  CB  GLU A  41      -7.703  -0.622  20.469  1.00 60.88           C  \nATOM    288  O   GLU A  41      -9.555  -0.836  18.016  1.00 60.88           O  \nATOM    289  CG  GLU A  41      -6.602  -0.188  21.438  1.00 60.88           C  \nATOM    290  CD  GLU A  41      -6.840  -0.635  22.859  1.00 60.88           C  \nATOM    291  OE1 GLU A  41      -5.961  -0.401  23.734  1.00 60.88           O  \nATOM    292  OE2 GLU A  41      -7.910  -1.224  23.125  1.00 60.88           O  \nATOM    293  N   ALA A  42      -7.770  -1.328  16.891  1.00 65.38           N  \nATOM    294  CA  ALA A  42      -8.484  -2.248  16.016  1.00 65.38           C  \nATOM    295  C   ALA A  42      -8.188  -3.699  16.375  1.00 65.38           C  \nATOM    296  CB  ALA A  42      -8.117  -1.985  14.555  1.00 65.38           C  \nATOM    297  O   ALA A  42      -7.137  -4.234  16.016  1.00 65.38           O  \nATOM    298  N   PRO A  43      -9.016  -4.203  17.344  1.00 68.81           N  \nATOM    299  CA  PRO A  43      -8.844  -5.613  17.703  1.00 68.81           C  \nATOM    300  C   PRO A  43      -8.719  -6.520  16.469  1.00 68.81           C  \nATOM    301  CB  PRO A  43     -10.117  -5.938  18.484  1.00 68.81           C  \nATOM    302  O   PRO A  43      -8.086  -7.574  16.547  1.00 68.81           O  \nATOM    303  CG  PRO A  43     -10.688  -4.605  18.859  1.00 68.81           C  \nATOM    304  CD  PRO A  43     -10.125  -3.574  17.922  1.00 68.81           C  \nATOM    305  N   GLU A  44      -9.336  -6.023  15.344  1.00 70.12           N  \nATOM    306  CA  GLU A  44      -9.352  -6.824  14.125  1.00 70.12           C  \nATOM    307  C   GLU A  44      -7.945  -6.969  13.547  1.00 70.12           C  \nATOM    308  CB  GLU A  44     -10.289  -6.207  13.086  1.00 70.12           C  \nATOM    309  O   GLU A  44      -7.664  -7.926  12.820  1.00 70.12           O  \nATOM    310  CG  GLU A  44     -11.750  -6.223  13.492  1.00 70.12           C  \nATOM    311  CD  GLU A  44     -12.141  -5.051  14.375  1.00 70.12           C  \nATOM    312  OE1 GLU A  44     -13.336  -4.930  14.734  1.00 70.12           O  \nATOM    313  OE2 GLU A  44     -11.250  -4.246  14.719  1.00 70.12           O  \nATOM    314  N   LEU A  45      -7.059  -6.055  13.969  1.00 68.44           N  \nATOM    315  CA  LEU A  45      -5.703  -6.074  13.430  1.00 68.44           C  \nATOM    316  C   LEU A  45      -4.730  -6.680  14.445  1.00 68.44           C  \nATOM    317  CB  LEU A  45      -5.254  -4.660  13.055  1.00 68.44           C  \nATOM    318  O   LEU A  45      -3.521  -6.707  14.203  1.00 68.44           O  \nATOM    319  CG  LEU A  45      -5.961  -4.020  11.859  1.00 68.44           C  \nATOM    320  CD1 LEU A  45      -5.535  -2.564  11.711  1.00 68.44           C  \nATOM    321  CD2 LEU A  45      -5.676  -4.805  10.586  1.00 68.44           C  \nATOM    322  N   ALA A  46      -5.227  -7.059  15.586  1.00 68.75           N  \nATOM    323  CA  ALA A  46      -4.359  -7.625  16.609  1.00 68.75           C  \nATOM    324  C   ALA A  46      -3.695  -8.906  16.125  1.00 68.75           C  \nATOM    325  CB  ALA A  46      -5.152  -7.887  17.891  1.00 68.75           C  \nATOM    326  O   ALA A  46      -4.359  -9.781  15.570  1.00 68.75           O  \nATOM    327  N   LEU A  47      -2.318  -8.812  15.914  1.00 62.34           N  \nATOM    328  CA  LEU A  47      -1.540  -9.992  15.562  1.00 62.34           C  \nATOM    329  C   LEU A  47      -1.271 -10.852  16.781  1.00 62.34           C  \nATOM    330  CB  LEU A  47      -0.216  -9.586  14.906  1.00 62.34           C  \nATOM    331  O   LEU A  47      -1.119 -10.336  17.891  1.00 62.34           O  \nATOM    332  CG  LEU A  47      -0.313  -8.938  13.523  1.00 62.34           C  \nATOM    333  CD1 LEU A  47       1.062  -8.469  13.062  1.00 62.34           C  \nATOM    334  CD2 LEU A  47      -0.914  -9.922  12.516  1.00 62.34           C  \nATOM    335  N   ASP A  48      -1.601 -12.039  16.625  1.00 61.00           N  \nATOM    336  CA  ASP A  48      -0.934 -12.945  17.547  1.00 61.00           C  \nATOM    337  C   ASP A  48       0.584 -12.820  17.438  1.00 61.00           C  \nATOM    338  CB  ASP A  48      -1.366 -14.391  17.297  1.00 61.00           C  \nATOM    339  O   ASP A  48       1.120 -12.617  16.344  1.00 61.00           O  \nATOM    340  CG  ASP A  48      -2.730 -14.711  17.875  1.00 61.00           C  \nATOM    341  OD1 ASP A  48      -3.096 -14.141  18.938  1.00 61.00           O  \nATOM    342  OD2 ASP A  48      -3.445 -15.547  17.281  1.00 61.00           O  \nATOM    343  N   PRO A  49       1.260 -12.328  18.484  1.00 56.16           N  \nATOM    344  CA  PRO A  49       2.721 -12.227  18.469  1.00 56.16           C  \nATOM    345  C   PRO A  49       3.375 -13.344  17.656  1.00 56.16           C  \nATOM    346  CB  PRO A  49       3.102 -12.336  19.953  1.00 56.16           C  \nATOM    347  O   PRO A  49       3.148 -14.523  17.938  1.00 56.16           O  \nATOM    348  CG  PRO A  49       1.799 -12.445  20.672  1.00 56.16           C  \nATOM    349  CD  PRO A  49       0.699 -12.562  19.656  1.00 56.16           C  \nATOM    350  N   VAL A  50       3.285 -13.336  16.375  1.00 59.81           N  \nATOM    351  CA  VAL A  50       4.039 -14.312  15.594  1.00 59.81           C  \nATOM    352  C   VAL A  50       5.523 -13.977  15.633  1.00 59.81           C  \nATOM    353  CB  VAL A  50       3.545 -14.375  14.125  1.00 59.81           C  \nATOM    354  O   VAL A  50       5.902 -12.805  15.555  1.00 59.81           O  \nATOM    355  CG1 VAL A  50       4.379 -15.367  13.320  1.00 59.81           C  \nATOM    356  CG2 VAL A  50       2.064 -14.758  14.078  1.00 59.81           C  \nATOM    357  N   PRO A  51       6.348 -14.906  16.266  1.00 59.00           N  \nATOM    358  CA  PRO A  51       7.789 -14.664  16.172  1.00 59.00           C  \nATOM    359  C   PRO A  51       8.219 -14.227  14.766  1.00 59.00           C  \nATOM    360  CB  PRO A  51       8.406 -16.016  16.531  1.00 59.00           C  \nATOM    361  O   PRO A  51       7.898 -14.898  13.781  1.00 59.00           O  \nATOM    362  CG  PRO A  51       7.258 -16.828  17.047  1.00 59.00           C  \nATOM    363  CD  PRO A  51       5.977 -16.188  16.594  1.00 59.00           C  \nATOM    364  N   GLN A  52       8.359 -12.930  14.664  1.00 70.31           N  \nATOM    365  CA  GLN A  52       8.852 -12.406  13.398  1.00 70.31           C  \nATOM    366  C   GLN A  52      10.359 -12.562  13.281  1.00 70.31           C  \nATOM    367  CB  GLN A  52       8.461 -10.930  13.250  1.00 70.31           C  \nATOM    368  O   GLN A  52      11.086 -12.352  14.258  1.00 70.31           O  \nATOM    369  CG  GLN A  52       6.961 -10.695  13.156  1.00 70.31           C  \nATOM    370  CD  GLN A  52       6.613  -9.289  12.703  1.00 70.31           C  \nATOM    371  NE2 GLN A  52       5.699  -9.180  11.750  1.00 70.31           N  \nATOM    372  OE1 GLN A  52       7.164  -8.305  13.211  1.00 70.31           O  \nATOM    373  N   ASP A  53      10.852 -13.320  12.273  1.00 81.31           N  \nATOM    374  CA  ASP A  53      12.289 -13.398  12.031  1.00 81.31           C  \nATOM    375  C   ASP A  53      12.891 -12.016  11.828  1.00 81.31           C  \nATOM    376  CB  ASP A  53      12.578 -14.281  10.812  1.00 81.31           C  \nATOM    377  O   ASP A  53      12.164 -11.031  11.672  1.00 81.31           O  \nATOM    378  CG  ASP A  53      12.039 -13.703   9.523  1.00 81.31           C  \nATOM    379  OD1 ASP A  53      12.047 -12.469   9.352  1.00 81.31           O  \nATOM    380  OD2 ASP A  53      11.594 -14.500   8.656  1.00 81.31           O  \nATOM    381  N   ALA A  54      14.102 -11.758  12.188  1.00 86.06           N  \nATOM    382  CA  ALA A  54      14.859 -10.508  12.133  1.00 86.06           C  \nATOM    383  C   ALA A  54      14.617  -9.781  10.812  1.00 86.06           C  \nATOM    384  CB  ALA A  54      16.359 -10.781  12.312  1.00 86.06           C  \nATOM    385  O   ALA A  54      14.461  -8.555  10.789  1.00 86.06           O  \nATOM    386  N   SER A  55      14.484 -10.484   9.719  1.00 87.25           N  \nATOM    387  CA  SER A  55      14.297  -9.883   8.398  1.00 87.25           C  \nATOM    388  C   SER A  55      12.914  -9.242   8.281  1.00 87.25           C  \nATOM    389  CB  SER A  55      14.469 -10.938   7.305  1.00 87.25           C  \nATOM    390  O   SER A  55      12.789  -8.141   7.738  1.00 87.25           O  \nATOM    391  OG  SER A  55      13.469 -11.938   7.398  1.00 87.25           O  \nATOM    392  N   THR A  56      11.883  -9.906   8.742  1.00 88.94           N  \nATOM    393  CA  THR A  56      10.523  -9.383   8.695  1.00 88.94           C  \nATOM    394  C   THR A  56      10.398  -8.109   9.523  1.00 88.94           C  \nATOM    395  CB  THR A  56       9.500 -10.422   9.188  1.00 88.94           C  \nATOM    396  O   THR A  56       9.703  -7.168   9.133  1.00 88.94           O  \nATOM    397  CG2 THR A  56       8.078  -9.898   9.070  1.00 88.94           C  \nATOM    398  OG1 THR A  56       9.633 -11.617   8.406  1.00 88.94           O  \nATOM    399  N   LYS A  57      11.102  -8.062  10.602  1.00 90.62           N  \nATOM    400  CA  LYS A  57      11.109  -6.867  11.445  1.00 90.62           C  \nATOM    401  C   LYS A  57      11.734  -5.684  10.719  1.00 90.62           C  \nATOM    402  CB  LYS A  57      11.852  -7.137  12.758  1.00 90.62           C  \nATOM    403  O   LYS A  57      11.219  -4.566  10.781  1.00 90.62           O  \nATOM    404  CG  LYS A  57      11.078  -7.992  13.750  1.00 90.62           C  \nATOM    405  CD  LYS A  57      11.844  -8.164  15.055  1.00 90.62           C  \nATOM    406  CE  LYS A  57      11.094  -9.062  16.031  1.00 90.62           C  \nATOM    407  NZ  LYS A  57      11.891  -9.320  17.266  1.00 90.62           N  \nATOM    408  N   LYS A  58      12.844  -5.922  10.070  1.00 92.25           N  \nATOM    409  CA  LYS A  58      13.508  -4.875   9.297  1.00 92.25           C  \nATOM    410  C   LYS A  58      12.617  -4.383   8.164  1.00 92.25           C  \nATOM    411  CB  LYS A  58      14.844  -5.383   8.734  1.00 92.25           C  \nATOM    412  O   LYS A  58      12.555  -3.180   7.891  1.00 92.25           O  \nATOM    413  CG  LYS A  58      15.938  -5.531   9.781  1.00 92.25           C  \nATOM    414  CD  LYS A  58      17.234  -6.012   9.164  1.00 92.25           C  \nATOM    415  CE  LYS A  58      18.312  -6.234  10.219  1.00 92.25           C  \nATOM    416  NZ  LYS A  58      19.578  -6.773   9.617  1.00 92.25           N  \nATOM    417  N   LEU A  59      11.984  -5.301   7.570  1.00 93.38           N  \nATOM    418  CA  LEU A  59      11.062  -4.945   6.500  1.00 93.38           C  \nATOM    419  C   LEU A  59       9.922  -4.078   7.027  1.00 93.38           C  \nATOM    420  CB  LEU A  59      10.492  -6.207   5.844  1.00 93.38           C  \nATOM    421  O   LEU A  59       9.562  -3.074   6.406  1.00 93.38           O  \nATOM    422  CG  LEU A  59      11.438  -6.977   4.918  1.00 93.38           C  \nATOM    423  CD1 LEU A  59      10.891  -8.383   4.660  1.00 93.38           C  \nATOM    424  CD2 LEU A  59      11.633  -6.223   3.609  1.00 93.38           C  \nATOM    425  N   SER A  60       9.406  -4.477   8.125  1.00 92.69           N  \nATOM    426  CA  SER A  60       8.312  -3.721   8.734  1.00 92.69           C  \nATOM    427  C   SER A  60       8.750  -2.301   9.078  1.00 92.69           C  \nATOM    428  CB  SER A  60       7.809  -4.430   9.984  1.00 92.69           C  \nATOM    429  O   SER A  60       8.008  -1.345   8.844  1.00 92.69           O  \nATOM    430  OG  SER A  60       7.172  -5.652   9.656  1.00 92.69           O  \nATOM    431  N   GLU A  61       9.898  -2.207   9.641  1.00 92.06           N  \nATOM    432  CA  GLU A  61      10.438  -0.896   9.992  1.00 92.06           C  \nATOM    433  C   GLU A  61      10.641  -0.033   8.750  1.00 92.06           C  \nATOM    434  CB  GLU A  61      11.758  -1.040  10.750  1.00 92.06           C  \nATOM    435  O   GLU A  61      10.359   1.168   8.766  1.00 92.06           O  \nATOM    436  CG  GLU A  61      11.594  -1.549  12.180  1.00 92.06           C  \nATOM    437  CD  GLU A  61      12.914  -1.817  12.875  1.00 92.06           C  \nATOM    438  OE1 GLU A  61      12.906  -2.211  14.062  1.00 92.06           O  \nATOM    439  OE2 GLU A  61      13.969  -1.633  12.227  1.00 92.06           O  \nATOM    440  N   CYS A  62      11.148  -0.692   7.754  1.00 92.38           N  \nATOM    441  CA  CYS A  62      11.352   0.015   6.492  1.00 92.38           C  \nATOM    442  C   CYS A  62      10.031   0.541   5.945  1.00 92.38           C  \nATOM    443  CB  CYS A  62      12.016  -0.900   5.465  1.00 92.38           C  \nATOM    444  O   CYS A  62       9.930   1.707   5.562  1.00 92.38           O  \nATOM    445  SG  CYS A  62      12.359  -0.091   3.889  1.00 92.38           S  \nATOM    446  N   LEU A  63       9.000  -0.228   5.895  1.00 93.06           N  \nATOM    447  CA  LEU A  63       7.680   0.164   5.414  1.00 93.06           C  \nATOM    448  C   LEU A  63       7.102   1.288   6.270  1.00 93.06           C  \nATOM    449  CB  LEU A  63       6.730  -1.036   5.422  1.00 93.06           C  \nATOM    450  O   LEU A  63       6.461   2.203   5.746  1.00 93.06           O  \nATOM    451  CG  LEU A  63       6.863  -2.012   4.250  1.00 93.06           C  \nATOM    452  CD1 LEU A  63       5.980  -3.234   4.477  1.00 93.06           C  \nATOM    453  CD2 LEU A  63       6.508  -1.325   2.938  1.00 93.06           C  \nATOM    454  N   LYS A  64       7.383   1.171   7.543  1.00 92.56           N  \nATOM    455  CA  LYS A  64       6.914   2.215   8.453  1.00 92.56           C  \nATOM    456  C   LYS A  64       7.539   3.564   8.102  1.00 92.56           C  \nATOM    457  CB  LYS A  64       7.227   1.849   9.898  1.00 92.56           C  \nATOM    458  O   LYS A  64       6.844   4.582   8.062  1.00 92.56           O  \nATOM    459  CG  LYS A  64       6.371   0.717  10.453  1.00 92.56           C  \nATOM    460  CD  LYS A  64       6.730   0.398  11.898  1.00 92.56           C  \nATOM    461  CE  LYS A  64       5.836  -0.694  12.469  1.00 92.56           C  \nATOM    462  NZ  LYS A  64       6.379  -1.237  13.750  1.00 92.56           N  \nATOM    463  N   ARG A  65       8.766   3.576   7.879  1.00 91.69           N  \nATOM    464  CA  ARG A  65       9.484   4.801   7.535  1.00 91.69           C  \nATOM    465  C   ARG A  65       8.961   5.395   6.234  1.00 91.69           C  \nATOM    466  CB  ARG A  65      10.984   4.527   7.426  1.00 91.69           C  \nATOM    467  O   ARG A  65       8.719   6.602   6.145  1.00 91.69           O  \nATOM    468  CG  ARG A  65      11.828   5.785   7.277  1.00 91.69           C  \nATOM    469  CD  ARG A  65      13.312   5.465   7.230  1.00 91.69           C  \nATOM    470  NE  ARG A  65      14.117   6.652   6.957  1.00 91.69           N  \nATOM    471  NH1 ARG A  65      16.156   5.582   7.141  1.00 91.69           N  \nATOM    472  NH2 ARG A  65      16.062   7.820   6.660  1.00 91.69           N  \nATOM    473  CZ  ARG A  65      15.445   6.684   6.922  1.00 91.69           C  \nATOM    474  N   ILE A  66       8.781   4.539   5.281  1.00 91.06           N  \nATOM    475  CA  ILE A  66       8.266   4.980   3.990  1.00 91.06           C  \nATOM    476  C   ILE A  66       6.859   5.543   4.156  1.00 91.06           C  \nATOM    477  CB  ILE A  66       8.273   3.832   2.957  1.00 91.06           C  \nATOM    478  O   ILE A  66       6.531   6.594   3.596  1.00 91.06           O  \nATOM    479  CG1 ILE A  66       9.703   3.393   2.650  1.00 91.06           C  \nATOM    480  CG2 ILE A  66       7.543   4.254   1.679  1.00 91.06           C  \nATOM    481  CD1 ILE A  66       9.797   2.104   1.844  1.00 91.06           C  \nATOM    482  N   GLY A  67       6.102   4.824   4.848  1.00 92.25           N  \nATOM    483  CA  GLY A  67       4.754   5.285   5.133  1.00 92.25           C  \nATOM    484  C   GLY A  67       4.715   6.656   5.777  1.00 92.25           C  \nATOM    485  O   GLY A  67       3.883   7.492   5.418  1.00 92.25           O  \nATOM    486  N   ASP A  68       5.629   6.914   6.754  1.00 92.19           N  \nATOM    487  CA  ASP A  68       5.711   8.211   7.422  1.00 92.19           C  \nATOM    488  C   ASP A  68       6.035   9.320   6.426  1.00 92.19           C  \nATOM    489  CB  ASP A  68       6.762   8.180   8.531  1.00 92.19           C  \nATOM    490  O   ASP A  68       5.465  10.414   6.500  1.00 92.19           O  \nATOM    491  CG  ASP A  68       6.336   7.336   9.727  1.00 92.19           C  \nATOM    492  OD1 ASP A  68       5.137   7.004   9.844  1.00 92.19           O  \nATOM    493  OD2 ASP A  68       7.211   7.008  10.562  1.00 92.19           O  \nATOM    494  N   GLU A  69       6.863   8.992   5.516  1.00 91.06           N  \nATOM    495  CA  GLU A  69       7.230   9.961   4.488  1.00 91.06           C  \nATOM    496  C   GLU A  69       6.055  10.258   3.564  1.00 91.06           C  \nATOM    497  CB  GLU A  69       8.422   9.461   3.676  1.00 91.06           C  \nATOM    498  O   GLU A  69       5.805  11.414   3.217  1.00 91.06           O  \nATOM    499  CG  GLU A  69       9.750   9.508   4.430  1.00 91.06           C  \nATOM    500  CD  GLU A  69      10.922   8.977   3.619  1.00 91.06           C  \nATOM    501  OE1 GLU A  69      12.039   8.859   4.172  1.00 91.06           O  \nATOM    502  OE2 GLU A  69      10.719   8.672   2.422  1.00 91.06           O  \nATOM    503  N   LEU A  70       5.395   9.258   3.156  1.00 91.50           N  \nATOM    504  CA  LEU A  70       4.238   9.430   2.283  1.00 91.50           C  \nATOM    505  C   LEU A  70       3.133  10.211   2.990  1.00 91.50           C  \nATOM    506  CB  LEU A  70       3.705   8.070   1.826  1.00 91.50           C  \nATOM    507  O   LEU A  70       2.453  11.031   2.371  1.00 91.50           O  \nATOM    508  CG  LEU A  70       4.582   7.297   0.838  1.00 91.50           C  \nATOM    509  CD1 LEU A  70       4.047   5.883   0.651  1.00 91.50           C  \nATOM    510  CD2 LEU A  70       4.652   8.031  -0.499  1.00 91.50           C  \nATOM    511  N   ASP A  71       2.971   9.938   4.289  1.00 92.00           N  \nATOM    512  CA  ASP A  71       1.959  10.633   5.078  1.00 92.00           C  \nATOM    513  C   ASP A  71       2.264  12.125   5.168  1.00 92.00           C  \nATOM    514  CB  ASP A  71       1.863  10.031   6.480  1.00 92.00           C  \nATOM    515  O   ASP A  71       1.361  12.938   5.387  1.00 92.00           O  \nATOM    516  CG  ASP A  71       1.146   8.695   6.504  1.00 92.00           C  \nATOM    517  OD1 ASP A  71       0.423   8.367   5.539  1.00 92.00           O  \nATOM    518  OD2 ASP A  71       1.303   7.953   7.504  1.00 92.00           O  \nATOM    519  N   SER A  72       3.541  12.523   5.020  1.00 91.88           N  \nATOM    520  CA  SER A  72       3.936  13.922   5.117  1.00 91.88           C  \nATOM    521  C   SER A  72       3.621  14.680   3.828  1.00 91.88           C  \nATOM    522  CB  SER A  72       5.426  14.039   5.430  1.00 91.88           C  \nATOM    523  O   SER A  72       3.684  15.906   3.791  1.00 91.88           O  \nATOM    524  OG  SER A  72       6.211  13.672   4.309  1.00 91.88           O  \nATOM    525  N   ASN A  73       3.295  13.891   2.758  1.00 89.81           N  \nATOM    526  CA  ASN A  73       2.920  14.492   1.482  1.00 89.81           C  \nATOM    527  C   ASN A  73       1.475  14.977   1.494  1.00 89.81           C  \nATOM    528  CB  ASN A  73       3.137  13.500   0.337  1.00 89.81           C  \nATOM    529  O   ASN A  73       0.543  14.172   1.432  1.00 89.81           O  \nATOM    530  CG  ASN A  73       3.025  14.148  -1.027  1.00 89.81           C  \nATOM    531  ND2 ASN A  73       3.996  13.883  -1.895  1.00 89.81           N  \nATOM    532  OD1 ASN A  73       2.076  14.891  -1.301  1.00 89.81           O  \nATOM    533  N   MET A  74       1.227  16.312   1.522  1.00 90.69           N  \nATOM    534  CA  MET A  74      -0.090  16.922   1.700  1.00 90.69           C  \nATOM    535  C   MET A  74      -0.975  16.656   0.485  1.00 90.69           C  \nATOM    536  CB  MET A  74       0.036  18.422   1.946  1.00 90.69           C  \nATOM    537  O   MET A  74      -2.178  16.422   0.625  1.00 90.69           O  \nATOM    538  CG  MET A  74       0.703  18.766   3.266  1.00 90.69           C  \nATOM    539  SD  MET A  74      -0.245  18.172   4.715  1.00 90.69           S  \nATOM    540  CE  MET A  74      -1.672  19.281   4.656  1.00 90.69           C  \nATOM    541  N   GLU A  75      -0.338  16.719  -0.656  1.00 88.75           N  \nATOM    542  CA  GLU A  75      -1.079  16.453  -1.887  1.00 88.75           C  \nATOM    543  C   GLU A  75      -1.620  15.031  -1.912  1.00 88.75           C  \nATOM    544  CB  GLU A  75      -0.194  16.703  -3.111  1.00 88.75           C  \nATOM    545  O   GLU A  75      -2.770  14.805  -2.295  1.00 88.75           O  \nATOM    546  CG  GLU A  75      -0.902  16.469  -4.438  1.00 88.75           C  \nATOM    547  CD  GLU A  75      -2.074  17.422  -4.664  1.00 88.75           C  \nATOM    548  OE1 GLU A  75      -2.846  17.203  -5.625  1.00 88.75           O  \nATOM    549  OE2 GLU A  75      -2.221  18.375  -3.873  1.00 88.75           O  \nATOM    550  N   LEU A  76      -0.773  14.125  -1.563  1.00 88.19           N  \nATOM    551  CA  LEU A  76      -1.197  12.734  -1.490  1.00 88.19           C  \nATOM    552  C   LEU A  76      -2.340  12.562  -0.494  1.00 88.19           C  \nATOM    553  CB  LEU A  76      -0.022  11.836  -1.095  1.00 88.19           C  \nATOM    554  O   LEU A  76      -3.338  11.906  -0.794  1.00 88.19           O  \nATOM    555  CG  LEU A  76      -0.324  10.344  -0.958  1.00 88.19           C  \nATOM    556  CD1 LEU A  76      -0.846   9.789  -2.277  1.00 88.19           C  \nATOM    557  CD2 LEU A  76       0.918   9.586  -0.503  1.00 88.19           C  \nATOM    558  N   GLN A  77      -2.193  13.242   0.630  1.00 90.38           N  \nATOM    559  CA  GLN A  77      -3.236  13.156   1.648  1.00 90.38           C  \nATOM    560  C   GLN A  77      -4.555  13.727   1.135  1.00 90.38           C  \nATOM    561  CB  GLN A  77      -2.805  13.891   2.920  1.00 90.38           C  \nATOM    562  O   GLN A  77      -5.621  13.164   1.390  1.00 90.38           O  \nATOM    563  CG  GLN A  77      -1.596  13.273   3.607  1.00 90.38           C  \nATOM    564  CD  GLN A  77      -1.799  11.812   3.947  1.00 90.38           C  \nATOM    565  NE2 GLN A  77      -0.854  10.969   3.539  1.00 90.38           N  \nATOM    566  OE1 GLN A  77      -2.797  11.438   4.574  1.00 90.38           O  \nATOM    567  N   ARG A  78      -4.484  14.773   0.477  1.00 91.12           N  \nATOM    568  CA  ARG A  78      -5.664  15.406  -0.100  1.00 91.12           C  \nATOM    569  C   ARG A  78      -6.352  14.484  -1.097  1.00 91.12           C  \nATOM    570  CB  ARG A  78      -5.285  16.719  -0.782  1.00 91.12           C  \nATOM    571  O   ARG A  78      -7.578  14.359  -1.096  1.00 91.12           O  \nATOM    572  CG  ARG A  78      -6.480  17.531  -1.267  1.00 91.12           C  \nATOM    573  CD  ARG A  78      -6.051  18.781  -2.018  1.00 91.12           C  \nATOM    574  NE  ARG A  78      -5.133  18.469  -3.109  1.00 91.12           N  \nATOM    575  NH1 ARG A  78      -5.422  20.484  -4.207  1.00 91.12           N  \nATOM    576  NH2 ARG A  78      -4.008  18.891  -5.066  1.00 91.12           N  \nATOM    577  CZ  ARG A  78      -4.855  19.281  -4.125  1.00 91.12           C  \nATOM    578  N   MET A  79      -5.590  13.820  -1.927  1.00 86.56           N  \nATOM    579  CA  MET A  79      -6.125  12.914  -2.938  1.00 86.56           C  \nATOM    580  C   MET A  79      -6.789  11.703  -2.285  1.00 86.56           C  \nATOM    581  CB  MET A  79      -5.023  12.453  -3.891  1.00 86.56           C  \nATOM    582  O   MET A  79      -7.848  11.250  -2.727  1.00 86.56           O  \nATOM    583  CG  MET A  79      -4.512  13.555  -4.809  1.00 86.56           C  \nATOM    584  SD  MET A  79      -5.773  14.102  -6.016  1.00 86.56           S  \nATOM    585  CE  MET A  79      -5.977  12.586  -6.984  1.00 86.56           C  \nATOM    586  N   ILE A  80      -6.148  11.203  -1.310  1.00 88.50           N  \nATOM    587  CA  ILE A  80      -6.699  10.055  -0.588  1.00 88.50           C  \nATOM    588  C   ILE A  80      -8.031  10.445   0.052  1.00 88.50           C  \nATOM    589  CB  ILE A  80      -5.719   9.539   0.489  1.00 88.50           C  \nATOM    590  O   ILE A  80      -8.992   9.664   0.027  1.00 88.50           O  \nATOM    591  CG1 ILE A  80      -4.492   8.898  -0.171  1.00 88.50           C  \nATOM    592  CG2 ILE A  80      -6.418   8.555   1.429  1.00 88.50           C  \nATOM    593  CD1 ILE A  80      -4.805   7.625  -0.948  1.00 88.50           C  \nATOM    594  N   ALA A  81      -8.102  11.633   0.621  1.00 88.50           N  \nATOM    595  CA  ALA A  81      -9.312  12.117   1.281  1.00 88.50           C  \nATOM    596  C   ALA A  81     -10.461  12.242   0.288  1.00 88.50           C  \nATOM    597  CB  ALA A  81      -9.047  13.453   1.962  1.00 88.50           C  \nATOM    598  O   ALA A  81     -11.625  12.109   0.662  1.00 88.50           O  \nATOM    599  N   ALA A  82     -10.141  12.430  -0.942  1.00 87.25           N  \nATOM    600  CA  ALA A  82     -11.148  12.664  -1.973  1.00 87.25           C  \nATOM    601  C   ALA A  82     -11.672  11.344  -2.537  1.00 87.25           C  \nATOM    602  CB  ALA A  82     -10.570  13.523  -3.092  1.00 87.25           C  \nATOM    603  O   ALA A  82     -12.617  11.336  -3.332  1.00 87.25           O  \nATOM    604  N   VAL A  83     -11.000  10.258  -2.229  1.00 84.75           N  \nATOM    605  CA  VAL A  83     -11.383   8.945  -2.740  1.00 84.75           C  \nATOM    606  C   VAL A  83     -12.758   8.562  -2.197  1.00 84.75           C  \nATOM    607  CB  VAL A  83     -10.344   7.863  -2.369  1.00 84.75           C  \nATOM    608  O   VAL A  83     -13.000   8.641  -0.991  1.00 84.75           O  \nATOM    609  CG1 VAL A  83     -10.859   6.477  -2.742  1.00 84.75           C  \nATOM    610  CG2 VAL A  83      -9.008   8.148  -3.059  1.00 84.75           C  \nATOM    611  N   ASP A  84     -13.688   8.281  -3.129  1.00 79.75           N  \nATOM    612  CA  ASP A  84     -15.047   7.852  -2.812  1.00 79.75           C  \nATOM    613  C   ASP A  84     -15.047   6.465  -2.166  1.00 79.75           C  \nATOM    614  CB  ASP A  84     -15.914   7.844  -4.070  1.00 79.75           C  \nATOM    615  O   ASP A  84     -14.523   5.508  -2.736  1.00 79.75           O  \nATOM    616  CG  ASP A  84     -17.391   7.586  -3.775  1.00 79.75           C  \nATOM    617  OD1 ASP A  84     -17.766   7.480  -2.588  1.00 79.75           O  \nATOM    618  OD2 ASP A  84     -18.172   7.480  -4.742  1.00 79.75           O  \nATOM    619  N   THR A  85     -15.734   6.383  -1.012  1.00 75.62           N  \nATOM    620  CA  THR A  85     -15.734   5.125  -0.271  1.00 75.62           C  \nATOM    621  C   THR A  85     -17.047   4.367  -0.498  1.00 75.62           C  \nATOM    622  CB  THR A  85     -15.531   5.367   1.236  1.00 75.62           C  \nATOM    623  O   THR A  85     -17.297   3.352   0.152  1.00 75.62           O  \nATOM    624  CG2 THR A  85     -14.141   5.910   1.524  1.00 75.62           C  \nATOM    625  OG1 THR A  85     -16.500   6.309   1.699  1.00 75.62           O  \nATOM    626  N   ASP A  86     -17.891   4.895  -1.399  1.00 80.19           N  \nATOM    627  CA  ASP A  86     -19.156   4.227  -1.671  1.00 80.19           C  \nATOM    628  C   ASP A  86     -18.953   2.949  -2.482  1.00 80.19           C  \nATOM    629  CB  ASP A  86     -20.109   5.164  -2.410  1.00 80.19           C  \nATOM    630  O   ASP A  86     -19.734   2.012  -2.395  1.00 80.19           O  \nATOM    631  CG  ASP A  86     -20.656   6.273  -1.528  1.00 80.19           C  \nATOM    632  OD1 ASP A  86     -20.688   6.113  -0.289  1.00 80.19           O  \nATOM    633  OD2 ASP A  86     -21.078   7.320  -2.078  1.00 80.19           O  \nATOM    634  N   SER A  87     -17.844   2.922  -3.178  1.00 84.81           N  \nATOM    635  CA  SER A  87     -17.500   1.737  -3.953  1.00 84.81           C  \nATOM    636  C   SER A  87     -16.016   1.389  -3.785  1.00 84.81           C  \nATOM    637  CB  SER A  87     -17.812   1.950  -5.434  1.00 84.81           C  \nATOM    638  O   SER A  87     -15.227   1.549  -4.719  1.00 84.81           O  \nATOM    639  OG  SER A  87     -17.250   3.164  -5.898  1.00 84.81           O  \nATOM    640  N   PRO A  88     -15.602   0.967  -2.574  1.00 85.31           N  \nATOM    641  CA  PRO A  88     -14.195   0.760  -2.232  1.00 85.31           C  \nATOM    642  C   PRO A  88     -13.531  -0.313  -3.092  1.00 85.31           C  \nATOM    643  CB  PRO A  88     -14.250   0.328  -0.765  1.00 85.31           C  \nATOM    644  O   PRO A  88     -12.359  -0.181  -3.455  1.00 85.31           O  \nATOM    645  CG  PRO A  88     -15.648  -0.138  -0.554  1.00 85.31           C  \nATOM    646  CD  PRO A  88     -16.547   0.629  -1.479  1.00 85.31           C  \nATOM    647  N   ARG A  89     -14.328  -1.303  -3.441  1.00 89.12           N  \nATOM    648  CA  ARG A  89     -13.773  -2.383  -4.250  1.00 89.12           C  \nATOM    649  C   ARG A  89     -13.375  -1.879  -5.633  1.00 89.12           C  \nATOM    650  CB  ARG A  89     -14.781  -3.527  -4.383  1.00 89.12           C  \nATOM    651  O   ARG A  89     -12.266  -2.146  -6.102  1.00 89.12           O  \nATOM    652  CG  ARG A  89     -14.250  -4.727  -5.156  1.00 89.12           C  \nATOM    653  CD  ARG A  89     -15.250  -5.875  -5.168  1.00 89.12           C  \nATOM    654  NE  ARG A  89     -14.719  -7.043  -5.863  1.00 89.12           N  \nATOM    655  NH1 ARG A  89     -16.781  -7.902  -6.430  1.00 89.12           N  \nATOM    656  NH2 ARG A  89     -14.875  -9.000  -7.055  1.00 89.12           N  \nATOM    657  CZ  ARG A  89     -15.461  -7.980  -6.449  1.00 89.12           C  \nATOM    658  N   GLU A  90     -14.297  -1.269  -6.277  1.00 89.38           N  \nATOM    659  CA  GLU A  90     -14.039  -0.740  -7.613  1.00 89.38           C  \nATOM    660  C   GLU A  90     -12.867   0.244  -7.598  1.00 89.38           C  \nATOM    661  CB  GLU A  90     -15.289  -0.065  -8.180  1.00 89.38           C  \nATOM    662  O   GLU A  90     -11.992   0.189  -8.469  1.00 89.38           O  \nATOM    663  CG  GLU A  90     -15.141   0.397  -9.625  1.00 89.38           C  \nATOM    664  CD  GLU A  90     -16.391   1.073 -10.164  1.00 89.38           C  \nATOM    665  OE1 GLU A  90     -16.328   1.657 -11.273  1.00 89.38           O  \nATOM    666  OE2 GLU A  90     -17.438   1.021  -9.484  1.00 89.38           O  \nATOM    667  N   VAL A  91     -12.883   1.057  -6.590  1.00 89.50           N  \nATOM    668  CA  VAL A  91     -11.836   2.064  -6.488  1.00 89.50           C  \nATOM    669  C   VAL A  91     -10.477   1.384  -6.301  1.00 89.50           C  \nATOM    670  CB  VAL A  91     -12.102   3.051  -5.332  1.00 89.50           C  \nATOM    671  O   VAL A  91      -9.508   1.721  -6.980  1.00 89.50           O  \nATOM    672  CG1 VAL A  91     -10.898   3.969  -5.113  1.00 89.50           C  \nATOM    673  CG2 VAL A  91     -13.359   3.871  -5.609  1.00 89.50           C  \nATOM    674  N   PHE A  92     -10.453   0.424  -5.418  1.00 92.50           N  \nATOM    675  CA  PHE A  92      -9.195  -0.266  -5.141  1.00 92.50           C  \nATOM    676  C   PHE A  92      -8.625  -0.886  -6.410  1.00 92.50           C  \nATOM    677  CB  PHE A  92      -9.398  -1.346  -4.074  1.00 92.50           C  \nATOM    678  O   PHE A  92      -7.469  -0.646  -6.762  1.00 92.50           O  \nATOM    679  CG  PHE A  92      -8.172  -2.168  -3.801  1.00 92.50           C  \nATOM    680  CD1 PHE A  92      -7.938  -3.354  -4.488  1.00 92.50           C  \nATOM    681  CD2 PHE A  92      -7.238  -1.754  -2.854  1.00 92.50           C  \nATOM    682  CE1 PHE A  92      -6.801  -4.113  -4.234  1.00 92.50           C  \nATOM    683  CE2 PHE A  92      -6.102  -2.510  -2.596  1.00 92.50           C  \nATOM    684  CZ  PHE A  92      -5.883  -3.691  -3.287  1.00 92.50           C  \nATOM    685  N   PHE A  93      -9.383  -1.708  -7.129  1.00 92.44           N  \nATOM    686  CA  PHE A  93      -8.875  -2.475  -8.258  1.00 92.44           C  \nATOM    687  C   PHE A  93      -8.570  -1.560  -9.438  1.00 92.44           C  \nATOM    688  CB  PHE A  93      -9.883  -3.549  -8.680  1.00 92.44           C  \nATOM    689  O   PHE A  93      -7.633  -1.813 -10.203  1.00 92.44           O  \nATOM    690  CG  PHE A  93      -9.836  -4.793  -7.832  1.00 92.44           C  \nATOM    691  CD1 PHE A  93      -8.797  -5.711  -7.984  1.00 92.44           C  \nATOM    692  CD2 PHE A  93     -10.820  -5.043  -6.887  1.00 92.44           C  \nATOM    693  CE1 PHE A  93      -8.750  -6.863  -7.203  1.00 92.44           C  \nATOM    694  CE2 PHE A  93     -10.773  -6.195  -6.102  1.00 92.44           C  \nATOM    695  CZ  PHE A  93      -9.742  -7.102  -6.262  1.00 92.44           C  \nATOM    696  N   ARG A  94      -9.281  -0.499  -9.539  1.00 90.75           N  \nATOM    697  CA  ARG A  94      -9.000   0.448 -10.609  1.00 90.75           C  \nATOM    698  C   ARG A  94      -7.664   1.148 -10.375  1.00 90.75           C  \nATOM    699  CB  ARG A  94     -10.117   1.481 -10.734  1.00 90.75           C  \nATOM    700  O   ARG A  94      -6.848   1.260 -11.297  1.00 90.75           O  \nATOM    701  CG  ARG A  94     -10.039   2.330 -11.992  1.00 90.75           C  \nATOM    702  CD  ARG A  94     -11.250   3.238 -12.133  1.00 90.75           C  \nATOM    703  NE  ARG A  94     -10.867   4.617 -12.414  1.00 90.75           N  \nATOM    704  NH1 ARG A  94     -13.023   5.418 -12.562  1.00 90.75           N  \nATOM    705  NH2 ARG A  94     -11.250   6.836 -12.859  1.00 90.75           N  \nATOM    706  CZ  ARG A  94     -11.719   5.621 -12.617  1.00 90.75           C  \nATOM    707  N   VAL A  95      -7.512   1.634  -9.195  1.00 91.75           N  \nATOM    708  CA  VAL A  95      -6.262   2.316  -8.867  1.00 91.75           C  \nATOM    709  C   VAL A  95      -5.094   1.343  -8.984  1.00 91.75           C  \nATOM    710  CB  VAL A  95      -6.309   2.926  -7.449  1.00 91.75           C  \nATOM    711  O   VAL A  95      -4.059   1.674  -9.570  1.00 91.75           O  \nATOM    712  CG1 VAL A  95      -4.926   3.422  -7.031  1.00 91.75           C  \nATOM    713  CG2 VAL A  95      -7.328   4.062  -7.391  1.00 91.75           C  \nATOM    714  N   ALA A  96      -5.234   0.112  -8.453  1.00 93.00           N  \nATOM    715  CA  ALA A  96      -4.188  -0.901  -8.516  1.00 93.00           C  \nATOM    716  C   ALA A  96      -3.852  -1.248  -9.969  1.00 93.00           C  \nATOM    717  CB  ALA A  96      -4.609  -2.156  -7.758  1.00 93.00           C  \nATOM    718  O   ALA A  96      -2.678  -1.357 -10.328  1.00 93.00           O  \nATOM    719  N   ALA A  97      -4.887  -1.420 -10.805  1.00 92.25           N  \nATOM    720  CA  ALA A  97      -4.684  -1.743 -12.219  1.00 92.25           C  \nATOM    721  C   ALA A  97      -3.932  -0.625 -12.938  1.00 92.25           C  \nATOM    722  CB  ALA A  97      -6.027  -2.000 -12.898  1.00 92.25           C  \nATOM    723  O   ALA A  97      -3.047  -0.889 -13.750  1.00 92.25           O  \nATOM    724  N   ASP A  98      -4.285   0.556 -12.578  1.00 91.44           N  \nATOM    725  CA  ASP A  98      -3.641   1.705 -13.211  1.00 91.44           C  \nATOM    726  C   ASP A  98      -2.172   1.805 -12.805  1.00 91.44           C  \nATOM    727  CB  ASP A  98      -4.375   2.996 -12.852  1.00 91.44           C  \nATOM    728  O   ASP A  98      -1.318   2.156 -13.625  1.00 91.44           O  \nATOM    729  CG  ASP A  98      -3.842   4.211 -13.594  1.00 91.44           C  \nATOM    730  OD1 ASP A  98      -3.980   4.277 -14.836  1.00 91.44           O  \nATOM    731  OD2 ASP A  98      -3.277   5.105 -12.930  1.00 91.44           O  \nATOM    732  N   MET A  99      -1.854   1.529 -11.586  1.00 91.88           N  \nATOM    733  CA  MET A  99      -0.485   1.602 -11.078  1.00 91.88           C  \nATOM    734  C   MET A  99       0.438   0.688 -11.883  1.00 91.88           C  \nATOM    735  CB  MET A  99      -0.436   1.225  -9.602  1.00 91.88           C  \nATOM    736  O   MET A  99       1.628   0.974 -12.023  1.00 91.88           O  \nATOM    737  CG  MET A  99      -0.969   2.307  -8.672  1.00 91.88           C  \nATOM    738  SD  MET A  99      -0.933   1.802  -6.910  1.00 91.88           S  \nATOM    739  CE  MET A  99       0.629   2.566  -6.391  1.00 91.88           C  \nATOM    740  N   PHE A 100      -0.190  -0.355 -12.375  1.00 92.38           N  \nATOM    741  CA  PHE A 100       0.638  -1.370 -13.016  1.00 92.38           C  \nATOM    742  C   PHE A 100       0.260  -1.527 -14.484  1.00 92.38           C  \nATOM    743  CB  PHE A 100       0.502  -2.713 -12.289  1.00 92.38           C  \nATOM    744  O   PHE A 100       0.500  -2.578 -15.086  1.00 92.38           O  \nATOM    745  CG  PHE A 100       1.017  -2.695 -10.875  1.00 92.38           C  \nATOM    746  CD1 PHE A 100       2.344  -3.002 -10.602  1.00 92.38           C  \nATOM    747  CD2 PHE A 100       0.172  -2.369  -9.820  1.00 92.38           C  \nATOM    748  CE1 PHE A 100       2.820  -2.986  -9.289  1.00 92.38           C  \nATOM    749  CE2 PHE A 100       0.643  -2.352  -8.516  1.00 92.38           C  \nATOM    750  CZ  PHE A 100       1.968  -2.660  -8.250  1.00 92.38           C  \nATOM    751  N   SER A 101      -0.362  -0.566 -15.094  1.00 90.69           N  \nATOM    752  CA  SER A 101      -0.933  -0.653 -16.438  1.00 90.69           C  \nATOM    753  C   SER A 101       0.160  -0.705 -17.500  1.00 90.69           C  \nATOM    754  CB  SER A 101      -1.859   0.534 -16.703  1.00 90.69           C  \nATOM    755  O   SER A 101      -0.052  -1.242 -18.594  1.00 90.69           O  \nATOM    756  OG  SER A 101      -1.145   1.756 -16.625  1.00 90.69           O  \nATOM    757  N   ASP A 102       1.374  -0.160 -17.125  1.00 90.31           N  \nATOM    758  CA  ASP A 102       2.434  -0.095 -18.125  1.00 90.31           C  \nATOM    759  C   ASP A 102       3.250  -1.386 -18.141  1.00 90.31           C  \nATOM    760  CB  ASP A 102       3.350   1.103 -17.859  1.00 90.31           C  \nATOM    761  O   ASP A 102       4.180  -1.524 -18.938  1.00 90.31           O  \nATOM    762  CG  ASP A 102       4.016   1.051 -16.500  1.00 90.31           C  \nATOM    763  OD1 ASP A 102       3.855   0.039 -15.773  1.00 90.31           O  \nATOM    764  OD2 ASP A 102       4.699   2.031 -16.125  1.00 90.31           O  \nATOM    765  N   GLY A 103       2.867  -2.307 -17.281  1.00 88.00           N  \nATOM    766  CA  GLY A 103       3.523  -3.605 -17.281  1.00 88.00           C  \nATOM    767  C   GLY A 103       4.824  -3.611 -16.500  1.00 88.00           C  \nATOM    768  O   GLY A 103       5.516  -4.633 -16.453  1.00 88.00           O  \nATOM    769  N   ASN A 104       5.199  -2.496 -15.922  1.00 90.62           N  \nATOM    770  CA  ASN A 104       6.418  -2.402 -15.125  1.00 90.62           C  \nATOM    771  C   ASN A 104       6.145  -2.668 -13.648  1.00 90.62           C  \nATOM    772  CB  ASN A 104       7.074  -1.031 -15.305  1.00 90.62           C  \nATOM    773  O   ASN A 104       5.152  -2.184 -13.102  1.00 90.62           O  \nATOM    774  CG  ASN A 104       7.613  -0.820 -16.703  1.00 90.62           C  \nATOM    775  ND2 ASN A 104       7.629   0.430 -17.156  1.00 90.62           N  \nATOM    776  OD1 ASN A 104       8.008  -1.773 -17.391  1.00 90.62           O  \nATOM    777  N   PHE A 105       7.023  -3.547 -13.109  1.00 92.31           N  \nATOM    778  CA  PHE A 105       6.930  -3.863 -11.688  1.00 92.31           C  \nATOM    779  C   PHE A 105       8.164  -3.377 -10.938  1.00 92.31           C  \nATOM    780  CB  PHE A 105       6.750  -5.371 -11.484  1.00 92.31           C  \nATOM    781  O   PHE A 105       9.297  -3.623 -11.375  1.00 92.31           O  \nATOM    782  CG  PHE A 105       5.539  -5.938 -12.172  1.00 92.31           C  \nATOM    783  CD1 PHE A 105       4.320  -6.020 -11.508  1.00 92.31           C  \nATOM    784  CD2 PHE A 105       5.617  -6.387 -13.484  1.00 92.31           C  \nATOM    785  CE1 PHE A 105       3.195  -6.543 -12.141  1.00 92.31           C  \nATOM    786  CE2 PHE A 105       4.500  -6.914 -14.125  1.00 92.31           C  \nATOM    787  CZ  PHE A 105       3.289  -6.988 -13.453  1.00 92.31           C  \nATOM    788  N   ASN A 106       7.980  -2.627  -9.852  1.00 93.00           N  \nATOM    789  CA  ASN A 106       9.062  -2.211  -8.961  1.00 93.00           C  \nATOM    790  C   ASN A 106       8.570  -2.064  -7.520  1.00 93.00           C  \nATOM    791  CB  ASN A 106       9.680  -0.901  -9.445  1.00 93.00           C  \nATOM    792  O   ASN A 106       7.367  -2.035  -7.270  1.00 93.00           O  \nATOM    793  CG  ASN A 106       8.695   0.250  -9.445  1.00 93.00           C  \nATOM    794  ND2 ASN A 106       8.391   0.770 -10.633  1.00 93.00           N  \nATOM    795  OD1 ASN A 106       8.211   0.668  -8.398  1.00 93.00           O  \nATOM    796  N   TRP A 107       9.477  -2.072  -6.617  1.00 93.56           N  \nATOM    797  CA  TRP A 107       9.141  -2.084  -5.199  1.00 93.56           C  \nATOM    798  C   TRP A 107       8.383  -0.817  -4.809  1.00 93.56           C  \nATOM    799  CB  TRP A 107      10.398  -2.227  -4.344  1.00 93.56           C  \nATOM    800  O   TRP A 107       7.539  -0.841  -3.912  1.00 93.56           O  \nATOM    801  CG  TRP A 107      10.977  -3.611  -4.336  1.00 93.56           C  \nATOM    802  CD1 TRP A 107      12.195  -4.000  -4.828  1.00 93.56           C  \nATOM    803  CD2 TRP A 107      10.359  -4.793  -3.816  1.00 93.56           C  \nATOM    804  CE2 TRP A 107      11.258  -5.859  -4.023  1.00 93.56           C  \nATOM    805  CE3 TRP A 107       9.125  -5.047  -3.193  1.00 93.56           C  \nATOM    806  NE1 TRP A 107      12.367  -5.352  -4.641  1.00 93.56           N  \nATOM    807  CH2 TRP A 107       9.758  -7.395  -3.023  1.00 93.56           C  \nATOM    808  CZ2 TRP A 107      10.961  -7.168  -3.631  1.00 93.56           C  \nATOM    809  CZ3 TRP A 107       8.836  -6.352  -2.803  1.00 93.56           C  \nATOM    810  N   GLY A 108       8.766   0.331  -5.488  1.00 92.50           N  \nATOM    811  CA  GLY A 108       8.023   1.562  -5.250  1.00 92.50           C  \nATOM    812  C   GLY A 108       6.531   1.418  -5.480  1.00 92.50           C  \nATOM    813  O   GLY A 108       5.727   1.876  -4.664  1.00 92.50           O  \nATOM    814  N   ARG A 109       6.113   0.758  -6.445  1.00 94.06           N  \nATOM    815  CA  ARG A 109       4.703   0.559  -6.770  1.00 94.06           C  \nATOM    816  C   ARG A 109       4.051  -0.422  -5.801  1.00 94.06           C  \nATOM    817  CB  ARG A 109       4.547   0.058  -8.203  1.00 94.06           C  \nATOM    818  O   ARG A 109       2.893  -0.249  -5.418  1.00 94.06           O  \nATOM    819  CG  ARG A 109       4.863   1.106  -9.258  1.00 94.06           C  \nATOM    820  CD  ARG A 109       4.727   0.548 -10.672  1.00 94.06           C  \nATOM    821  NE  ARG A 109       4.992   1.565 -11.680  1.00 94.06           N  \nATOM    822  NH1 ARG A 109       4.719   0.144 -13.484  1.00 94.06           N  \nATOM    823  NH2 ARG A 109       5.238   2.350 -13.828  1.00 94.06           N  \nATOM    824  CZ  ARG A 109       4.984   1.351 -12.992  1.00 94.06           C  \nATOM    825  N   VAL A 110       4.824  -1.469  -5.469  1.00 95.56           N  \nATOM    826  CA  VAL A 110       4.328  -2.414  -4.473  1.00 95.56           C  \nATOM    827  C   VAL A 110       4.020  -1.682  -3.170  1.00 95.56           C  \nATOM    828  CB  VAL A 110       5.340  -3.555  -4.219  1.00 95.56           C  \nATOM    829  O   VAL A 110       2.955  -1.872  -2.580  1.00 95.56           O  \nATOM    830  CG1 VAL A 110       4.859  -4.461  -3.090  1.00 95.56           C  \nATOM    831  CG2 VAL A 110       5.570  -4.359  -5.496  1.00 95.56           C  \nATOM    832  N   VAL A 111       4.945  -0.811  -2.758  1.00 94.75           N  \nATOM    833  CA  VAL A 111       4.770  -0.032  -1.538  1.00 94.75           C  \nATOM    834  C   VAL A 111       3.578   0.909  -1.692  1.00 94.75           C  \nATOM    835  CB  VAL A 111       6.043   0.773  -1.190  1.00 94.75           C  \nATOM    836  O   VAL A 111       2.797   1.092  -0.755  1.00 94.75           O  \nATOM    837  CG1 VAL A 111       5.746   1.814  -0.112  1.00 94.75           C  \nATOM    838  CG2 VAL A 111       7.160  -0.165  -0.738  1.00 94.75           C  \nATOM    839  N   ALA A 112       3.445   1.528  -2.906  1.00 94.69           N  \nATOM    840  CA  ALA A 112       2.312   2.408  -3.172  1.00 94.69           C  \nATOM    841  C   ALA A 112       0.990   1.664  -3.012  1.00 94.69           C  \nATOM    842  CB  ALA A 112       2.418   3.002  -4.574  1.00 94.69           C  \nATOM    843  O   ALA A 112       0.033   2.199  -2.445  1.00 94.69           O  \nATOM    844  N   LEU A 113       0.937   0.431  -3.502  1.00 95.88           N  \nATOM    845  CA  LEU A 113      -0.256  -0.398  -3.361  1.00 95.88           C  \nATOM    846  C   LEU A 113      -0.566  -0.660  -1.892  1.00 95.88           C  \nATOM    847  CB  LEU A 113      -0.076  -1.726  -4.102  1.00 95.88           C  \nATOM    848  O   LEU A 113      -1.711  -0.512  -1.459  1.00 95.88           O  \nATOM    849  CG  LEU A 113      -1.250  -2.703  -4.035  1.00 95.88           C  \nATOM    850  CD1 LEU A 113      -2.455  -2.137  -4.781  1.00 95.88           C  \nATOM    851  CD2 LEU A 113      -0.851  -4.059  -4.609  1.00 95.88           C  \nATOM    852  N   PHE A 114       0.525  -1.097  -1.079  1.00 96.00           N  \nATOM    853  CA  PHE A 114       0.345  -1.322   0.350  1.00 96.00           C  \nATOM    854  C   PHE A 114      -0.174  -0.064   1.036  1.00 96.00           C  \nATOM    855  CB  PHE A 114       1.662  -1.767   0.995  1.00 96.00           C  \nATOM    856  O   PHE A 114      -1.073  -0.135   1.876  1.00 96.00           O  \nATOM    857  CG  PHE A 114       1.917  -3.246   0.896  1.00 96.00           C  \nATOM    858  CD1 PHE A 114       1.427  -3.980  -0.178  1.00 96.00           C  \nATOM    859  CD2 PHE A 114       2.648  -3.904   1.876  1.00 96.00           C  \nATOM    860  CE1 PHE A 114       1.661  -5.352  -0.272  1.00 96.00           C  \nATOM    861  CE2 PHE A 114       2.887  -5.273   1.789  1.00 96.00           C  \nATOM    862  CZ  PHE A 114       2.393  -5.992   0.714  1.00 96.00           C  \nATOM    863  N   TYR A 115       0.382   1.058   0.651  1.00 95.38           N  \nATOM    864  CA  TYR A 115       0.012   2.330   1.261  1.00 95.38           C  \nATOM    865  C   TYR A 115      -1.438   2.684   0.949  1.00 95.38           C  \nATOM    866  CB  TYR A 115       0.938   3.449   0.775  1.00 95.38           C  \nATOM    867  O   TYR A 115      -2.193   3.070   1.842  1.00 95.38           O  \nATOM    868  CG  TYR A 115       0.651   4.789   1.402  1.00 95.38           C  \nATOM    869  CD1 TYR A 115      -0.002   5.793   0.685  1.00 95.38           C  \nATOM    870  CD2 TYR A 115       1.031   5.062   2.713  1.00 95.38           C  \nATOM    871  CE1 TYR A 115      -0.267   7.031   1.259  1.00 95.38           C  \nATOM    872  CE2 TYR A 115       0.771   6.297   3.295  1.00 95.38           C  \nATOM    873  OH  TYR A 115      -0.139   8.500   3.135  1.00 95.38           O  \nATOM    874  CZ  TYR A 115       0.121   7.273   2.562  1.00 95.38           C  \nATOM    875  N   PHE A 116      -1.801   2.584  -0.280  1.00 94.69           N  \nATOM    876  CA  PHE A 116      -3.168   2.873  -0.696  1.00 94.69           C  \nATOM    877  C   PHE A 116      -4.160   1.981   0.044  1.00 94.69           C  \nATOM    878  CB  PHE A 116      -3.322   2.686  -2.209  1.00 94.69           C  \nATOM    879  O   PHE A 116      -5.172   2.461   0.552  1.00 94.69           O  \nATOM    880  CG  PHE A 116      -4.707   2.980  -2.719  1.00 94.69           C  \nATOM    881  CD1 PHE A 116      -5.449   1.998  -3.363  1.00 94.69           C  \nATOM    882  CD2 PHE A 116      -5.266   4.242  -2.555  1.00 94.69           C  \nATOM    883  CE1 PHE A 116      -6.730   2.270  -3.838  1.00 94.69           C  \nATOM    884  CE2 PHE A 116      -6.547   4.520  -3.025  1.00 94.69           C  \nATOM    885  CZ  PHE A 116      -7.277   3.531  -3.668  1.00 94.69           C  \nATOM    886  N   ALA A 117      -3.893   0.680   0.124  1.00 94.69           N  \nATOM    887  CA  ALA A 117      -4.742  -0.261   0.850  1.00 94.69           C  \nATOM    888  C   ALA A 117      -4.871   0.137   2.316  1.00 94.69           C  \nATOM    889  CB  ALA A 117      -4.188  -1.679   0.734  1.00 94.69           C  \nATOM    890  O   ALA A 117      -5.969   0.094   2.883  1.00 94.69           O  \nATOM    891  N   SER A 118      -3.766   0.533   2.912  1.00 93.75           N  \nATOM    892  CA  SER A 118      -3.756   0.956   4.309  1.00 93.75           C  \nATOM    893  C   SER A 118      -4.641   2.178   4.523  1.00 93.75           C  \nATOM    894  CB  SER A 118      -2.330   1.263   4.766  1.00 93.75           C  \nATOM    895  O   SER A 118      -5.406   2.234   5.492  1.00 93.75           O  \nATOM    896  OG  SER A 118      -1.837   2.430   4.129  1.00 93.75           O  \nATOM    897  N   LYS A 119      -4.633   3.127   3.602  1.00 92.31           N  \nATOM    898  CA  LYS A 119      -5.434   4.344   3.703  1.00 92.31           C  \nATOM    899  C   LYS A 119      -6.918   4.035   3.557  1.00 92.31           C  \nATOM    900  CB  LYS A 119      -5.000   5.359   2.646  1.00 92.31           C  \nATOM    901  O   LYS A 119      -7.754   4.633   4.242  1.00 92.31           O  \nATOM    902  CG  LYS A 119      -3.654   6.012   2.928  1.00 92.31           C  \nATOM    903  CD  LYS A 119      -3.680   6.812   4.223  1.00 92.31           C  \nATOM    904  CE  LYS A 119      -2.486   7.754   4.320  1.00 92.31           C  \nATOM    905  NZ  LYS A 119      -2.488   8.516   5.605  1.00 92.31           N  \nATOM    906  N   LEU A 120      -7.203   3.082   2.711  1.00 91.50           N  \nATOM    907  CA  LEU A 120      -8.602   2.682   2.555  1.00 91.50           C  \nATOM    908  C   LEU A 120      -9.117   2.010   3.820  1.00 91.50           C  \nATOM    909  CB  LEU A 120      -8.758   1.736   1.360  1.00 91.50           C  \nATOM    910  O   LEU A 120     -10.266   2.232   4.223  1.00 91.50           O  \nATOM    911  CG  LEU A 120      -8.695   2.379  -0.026  1.00 91.50           C  \nATOM    912  CD1 LEU A 120      -8.852   1.317  -1.109  1.00 91.50           C  \nATOM    913  CD2 LEU A 120      -9.766   3.453  -0.165  1.00 91.50           C  \nATOM    914  N   VAL A 121      -8.234   1.214   4.383  1.00 90.88           N  \nATOM    915  CA  VAL A 121      -8.602   0.550   5.629  1.00 90.88           C  \nATOM    916  C   VAL A 121      -8.820   1.592   6.723  1.00 90.88           C  \nATOM    917  CB  VAL A 121      -7.520  -0.463   6.070  1.00 90.88           C  \nATOM    918  O   VAL A 121      -9.789   1.506   7.484  1.00 90.88           O  \nATOM    919  CG1 VAL A 121      -7.773  -0.935   7.504  1.00 90.88           C  \nATOM    920  CG2 VAL A 121      -7.480  -1.651   5.113  1.00 90.88           C  \nATOM    921  N   LEU A 122      -7.980   2.525   6.840  1.00 89.62           N  \nATOM    922  CA  LEU A 122      -8.102   3.572   7.844  1.00 89.62           C  \nATOM    923  C   LEU A 122      -9.375   4.387   7.633  1.00 89.62           C  \nATOM    924  CB  LEU A 122      -6.879   4.496   7.809  1.00 89.62           C  \nATOM    925  O   LEU A 122     -10.039   4.766   8.594  1.00 89.62           O  \nATOM    926  CG  LEU A 122      -5.555   3.885   8.273  1.00 89.62           C  \nATOM    927  CD1 LEU A 122      -4.418   4.879   8.086  1.00 89.62           C  \nATOM    928  CD2 LEU A 122      -5.652   3.451   9.734  1.00 89.62           C  \nATOM    929  N   LYS A 123      -9.703   4.625   6.414  1.00 87.56           N  \nATOM    930  CA  LYS A 123     -10.961   5.309   6.117  1.00 87.56           C  \nATOM    931  C   LYS A 123     -12.156   4.473   6.559  1.00 87.56           C  \nATOM    932  CB  LYS A 123     -11.062   5.625   4.625  1.00 87.56           C  \nATOM    933  O   LYS A 123     -13.141   5.008   7.070  1.00 87.56           O  \nATOM    934  CG  LYS A 123     -11.992   6.785   4.301  1.00 87.56           C  \nATOM    935  CD  LYS A 123     -11.867   7.211   2.844  1.00 87.56           C  \nATOM    936  CE  LYS A 123     -13.094   7.984   2.379  1.00 87.56           C  \nATOM    937  NZ  LYS A 123     -12.781   9.422   2.129  1.00 87.56           N  \nATOM    938  N   ALA A 124     -12.047   3.154   6.312  1.00 85.94           N  \nATOM    939  CA  ALA A 124     -13.109   2.240   6.723  1.00 85.94           C  \nATOM    940  C   ALA A 124     -13.266   2.230   8.242  1.00 85.94           C  \nATOM    941  CB  ALA A 124     -12.828   0.830   6.211  1.00 85.94           C  \nATOM    942  O   ALA A 124     -14.391   2.152   8.758  1.00 85.94           O  \nATOM    943  N   LEU A 125     -12.125   2.297   8.922  1.00 84.12           N  \nATOM    944  CA  LEU A 125     -12.172   2.334  10.375  1.00 84.12           C  \nATOM    945  C   LEU A 125     -12.859   3.602  10.867  1.00 84.12           C  \nATOM    946  CB  LEU A 125     -10.758   2.252  10.961  1.00 84.12           C  \nATOM    947  O   LEU A 125     -13.641   3.561  11.828  1.00 84.12           O  \nATOM    948  CG  LEU A 125     -10.078   0.881  10.906  1.00 84.12           C  \nATOM    949  CD1 LEU A 125      -8.617   0.993  11.336  1.00 84.12           C  \nATOM    950  CD2 LEU A 125     -10.828  -0.113  11.797  1.00 84.12           C  \nATOM    951  N   CYS A 126     -12.609   4.668  10.211  1.00 83.06           N  \nATOM    952  CA  CYS A 126     -13.180   5.957  10.594  1.00 83.06           C  \nATOM    953  C   CYS A 126     -14.680   5.992  10.312  1.00 83.06           C  \nATOM    954  CB  CYS A 126     -12.484   7.090   9.836  1.00 83.06           C  \nATOM    955  O   CYS A 126     -15.430   6.637  11.047  1.00 83.06           O  \nATOM    956  SG  CYS A 126     -10.797   7.410  10.398  1.00 83.06           S  \nATOM    957  N   THR A 127     -15.094   5.141   9.367  1.00 82.56           N  \nATOM    958  CA  THR A 127     -16.516   5.109   9.023  1.00 82.56           C  \nATOM    959  C   THR A 127     -17.219   3.967   9.750  1.00 82.56           C  \nATOM    960  CB  THR A 127     -16.719   4.957   7.504  1.00 82.56           C  \nATOM    961  O   THR A 127     -18.391   3.684   9.484  1.00 82.56           O  \nATOM    962  CG2 THR A 127     -16.109   6.133   6.750  1.00 82.56           C  \nATOM    963  OG1 THR A 127     -16.094   3.744   7.062  1.00 82.56           O  \nATOM    964  N   LYS A 128     -16.484   3.266  10.711  1.00 78.38           N  \nATOM    965  CA  LYS A 128     -16.984   2.232  11.609  1.00 78.38           C  \nATOM    966  C   LYS A 128     -17.531   1.042  10.836  1.00 78.38           C  \nATOM    967  CB  LYS A 128     -18.062   2.799  12.539  1.00 78.38           C  \nATOM    968  O   LYS A 128     -18.594   0.509  11.180  1.00 78.38           O  \nATOM    969  CG  LYS A 128     -17.562   3.891  13.477  1.00 78.38           C  \nATOM    970  CD  LYS A 128     -18.641   4.328  14.453  1.00 78.38           C  \nATOM    971  CE  LYS A 128     -18.141   5.426  15.383  1.00 78.38           C  \nATOM    972  NZ  LYS A 128     -19.203   5.871  16.328  1.00 78.38           N  \nATOM    973  N   VAL A 129     -16.875   0.651   9.758  1.00 78.44           N  \nATOM    974  CA  VAL A 129     -17.172  -0.545   8.984  1.00 78.44           C  \nATOM    975  C   VAL A 129     -16.016  -1.523   9.062  1.00 78.44           C  \nATOM    976  CB  VAL A 129     -17.484  -0.200   7.512  1.00 78.44           C  \nATOM    977  O   VAL A 129     -15.234  -1.652   8.109  1.00 78.44           O  \nATOM    978  CG1 VAL A 129     -18.000  -1.433   6.762  1.00 78.44           C  \nATOM    979  CG2 VAL A 129     -18.516   0.933   7.434  1.00 78.44           C  \nATOM    980  N   PRO A 130     -15.805  -2.166  10.195  1.00 73.81           N  \nATOM    981  CA  PRO A 130     -14.633  -3.010  10.430  1.00 73.81           C  \nATOM    982  C   PRO A 130     -14.555  -4.199   9.477  1.00 73.81           C  \nATOM    983  CB  PRO A 130     -14.820  -3.480  11.875  1.00 73.81           C  \nATOM    984  O   PRO A 130     -13.461  -4.707   9.203  1.00 73.81           O  \nATOM    985  CG  PRO A 130     -16.281  -3.281  12.156  1.00 73.81           C  \nATOM    986  CD  PRO A 130     -16.766  -2.148  11.297  1.00 73.81           C  \nATOM    987  N   GLU A 131     -15.773  -4.613   8.969  1.00 81.25           N  \nATOM    988  CA  GLU A 131     -15.805  -5.754   8.062  1.00 81.25           C  \nATOM    989  C   GLU A 131     -15.023  -5.461   6.785  1.00 81.25           C  \nATOM    990  CB  GLU A 131     -17.250  -6.125   7.715  1.00 81.25           C  \nATOM    991  O   GLU A 131     -14.547  -6.383   6.121  1.00 81.25           O  \nATOM    992  CG  GLU A 131     -18.078  -6.582   8.914  1.00 81.25           C  \nATOM    993  CD  GLU A 131     -19.484  -7.008   8.547  1.00 81.25           C  \nATOM    994  OE1 GLU A 131     -20.156  -7.668   9.375  1.00 81.25           O  \nATOM    995  OE2 GLU A 131     -19.922  -6.684   7.418  1.00 81.25           O  \nATOM    996  N   LEU A 132     -14.836  -4.172   6.555  1.00 83.19           N  \nATOM    997  CA  LEU A 132     -14.156  -3.760   5.332  1.00 83.19           C  \nATOM    998  C   LEU A 132     -12.648  -3.926   5.465  1.00 83.19           C  \nATOM    999  CB  LEU A 132     -14.492  -2.303   4.996  1.00 83.19           C  \nATOM   1000  O   LEU A 132     -11.945  -4.090   4.465  1.00 83.19           O  \nATOM   1001  CG  LEU A 132     -15.844  -2.059   4.320  1.00 83.19           C  \nATOM   1002  CD1 LEU A 132     -16.047  -0.568   4.066  1.00 83.19           C  \nATOM   1003  CD2 LEU A 132     -15.938  -2.846   3.018  1.00 83.19           C  \nATOM   1004  N   ILE A 133     -12.242  -4.031   6.688  1.00 85.44           N  \nATOM   1005  CA  ILE A 133     -10.805  -4.180   6.914  1.00 85.44           C  \nATOM   1006  C   ILE A 133     -10.320  -5.484   6.293  1.00 85.44           C  \nATOM   1007  CB  ILE A 133     -10.469  -4.145   8.422  1.00 85.44           C  \nATOM   1008  O   ILE A 133      -9.359  -5.488   5.516  1.00 85.44           O  \nATOM   1009  CG1 ILE A 133     -10.844  -2.781   9.023  1.00 85.44           C  \nATOM   1010  CG2 ILE A 133      -8.992  -4.457   8.648  1.00 85.44           C  \nATOM   1011  CD1 ILE A 133     -10.695  -2.707  10.531  1.00 85.44           C  \nATOM   1012  N   ARG A 134     -10.992  -6.609   6.652  1.00 87.00           N  \nATOM   1013  CA  ARG A 134     -10.609  -7.922   6.148  1.00 87.00           C  \nATOM   1014  C   ARG A 134     -10.773  -8.000   4.637  1.00 87.00           C  \nATOM   1015  CB  ARG A 134     -11.430  -9.023   6.824  1.00 87.00           C  \nATOM   1016  O   ARG A 134      -9.953  -8.609   3.945  1.00 87.00           O  \nATOM   1017  CG  ARG A 134     -11.086  -9.234   8.289  1.00 87.00           C  \nATOM   1018  CD  ARG A 134     -11.844 -10.414   8.883  1.00 87.00           C  \nATOM   1019  NE  ARG A 134     -11.602 -10.555  10.312  1.00 87.00           N  \nATOM   1020  NH1 ARG A 134     -13.578  -9.570  10.961  1.00 87.00           N  \nATOM   1021  NH2 ARG A 134     -12.102 -10.328  12.539  1.00 87.00           N  \nATOM   1022  CZ  ARG A 134     -12.430 -10.148  11.266  1.00 87.00           C  \nATOM   1023  N   THR A 135     -11.781  -7.301   4.203  1.00 91.44           N  \nATOM   1024  CA  THR A 135     -12.070  -7.316   2.773  1.00 91.44           C  \nATOM   1025  C   THR A 135     -10.969  -6.617   1.985  1.00 91.44           C  \nATOM   1026  CB  THR A 135     -13.430  -6.648   2.477  1.00 91.44           C  \nATOM   1027  O   THR A 135     -10.484  -7.145   0.987  1.00 91.44           O  \nATOM   1028  CG2 THR A 135     -13.750  -6.703   0.987  1.00 91.44           C  \nATOM   1029  OG1 THR A 135     -14.461  -7.324   3.205  1.00 91.44           O  \nATOM   1030  N   ILE A 136     -10.547  -5.461   2.393  1.00 92.50           N  \nATOM   1031  CA  ILE A 136      -9.523  -4.684   1.704  1.00 92.50           C  \nATOM   1032  C   ILE A 136      -8.188  -5.422   1.773  1.00 92.50           C  \nATOM   1033  CB  ILE A 136      -9.383  -3.270   2.307  1.00 92.50           C  \nATOM   1034  O   ILE A 136      -7.469  -5.512   0.775  1.00 92.50           O  \nATOM   1035  CG1 ILE A 136     -10.664  -2.457   2.061  1.00 92.50           C  \nATOM   1036  CG2 ILE A 136      -8.156  -2.553   1.727  1.00 92.50           C  \nATOM   1037  CD1 ILE A 136     -10.727  -1.152   2.844  1.00 92.50           C  \nATOM   1038  N   MET A 137      -7.914  -6.008   2.947  1.00 92.88           N  \nATOM   1039  CA  MET A 137      -6.695  -6.805   3.072  1.00 92.88           C  \nATOM   1040  C   MET A 137      -6.730  -8.000   2.129  1.00 92.88           C  \nATOM   1041  CB  MET A 137      -6.512  -7.277   4.516  1.00 92.88           C  \nATOM   1042  O   MET A 137      -5.727  -8.320   1.492  1.00 92.88           O  \nATOM   1043  CG  MET A 137      -6.168  -6.164   5.488  1.00 92.88           C  \nATOM   1044  SD  MET A 137      -6.059  -6.754   7.223  1.00 92.88           S  \nATOM   1045  CE  MET A 137      -4.602  -7.828   7.102  1.00 92.88           C  \nATOM   1046  N   GLY A 138      -7.906  -8.625   2.064  1.00 94.44           N  \nATOM   1047  CA  GLY A 138      -8.078  -9.719   1.122  1.00 94.44           C  \nATOM   1048  C   GLY A 138      -7.836  -9.312  -0.318  1.00 94.44           C  \nATOM   1049  O   GLY A 138      -7.172 -10.023  -1.070  1.00 94.44           O  \nATOM   1050  N   TRP A 139      -8.320  -8.156  -0.694  1.00 95.31           N  \nATOM   1051  CA  TRP A 139      -8.117  -7.645  -2.045  1.00 95.31           C  \nATOM   1052  C   TRP A 139      -6.633  -7.434  -2.330  1.00 95.31           C  \nATOM   1053  CB  TRP A 139      -8.883  -6.332  -2.244  1.00 95.31           C  \nATOM   1054  O   TRP A 139      -6.145  -7.793  -3.404  1.00 95.31           O  \nATOM   1055  CG  TRP A 139     -10.367  -6.473  -2.150  1.00 95.31           C  \nATOM   1056  CD1 TRP A 139     -11.102  -7.609  -2.354  1.00 95.31           C  \nATOM   1057  CD2 TRP A 139     -11.305  -5.441  -1.831  1.00 95.31           C  \nATOM   1058  CE2 TRP A 139     -12.594  -6.020  -1.860  1.00 95.31           C  \nATOM   1059  CE3 TRP A 139     -11.180  -4.078  -1.523  1.00 95.31           C  \nATOM   1060  NE1 TRP A 139     -12.445  -7.344  -2.182  1.00 95.31           N  \nATOM   1061  CH2 TRP A 139     -13.602  -3.955  -1.294  1.00 95.31           C  \nATOM   1062  CZ2 TRP A 139     -13.750  -5.285  -1.593  1.00 95.31           C  \nATOM   1063  CZ3 TRP A 139     -12.336  -3.348  -1.258  1.00 95.31           C  \nATOM   1064  N   THR A 140      -6.043  -6.879  -1.413  1.00 96.44           N  \nATOM   1065  CA  THR A 140      -4.625  -6.582  -1.579  1.00 96.44           C  \nATOM   1066  C   THR A 140      -3.816  -7.867  -1.746  1.00 96.44           C  \nATOM   1067  CB  THR A 140      -4.078  -5.781  -0.383  1.00 96.44           C  \nATOM   1068  O   THR A 140      -2.979  -7.969  -2.645  1.00 96.44           O  \nATOM   1069  CG2 THR A 140      -2.623  -5.379  -0.608  1.00 96.44           C  \nATOM   1070  OG1 THR A 140      -4.867  -4.598  -0.205  1.00 96.44           O  \nATOM   1071  N   LEU A 141      -4.105  -8.852  -0.929  1.00 96.19           N  \nATOM   1072  CA  LEU A 141      -3.387 -10.117  -0.970  1.00 96.19           C  \nATOM   1073  C   LEU A 141      -3.734 -10.906  -2.232  1.00 96.19           C  \nATOM   1074  CB  LEU A 141      -3.707 -10.961   0.270  1.00 96.19           C  \nATOM   1075  O   LEU A 141      -2.883 -11.594  -2.793  1.00 96.19           O  \nATOM   1076  CG  LEU A 141      -3.170 -10.430   1.602  1.00 96.19           C  \nATOM   1077  CD1 LEU A 141      -3.615 -11.336   2.748  1.00 96.19           C  \nATOM   1078  CD2 LEU A 141      -1.650 -10.320   1.561  1.00 96.19           C  \nATOM   1079  N   ASP A 142      -4.969 -10.781  -2.631  1.00 96.12           N  \nATOM   1080  CA  ASP A 142      -5.355 -11.398  -3.898  1.00 96.12           C  \nATOM   1081  C   ASP A 142      -4.570 -10.797  -5.062  1.00 96.12           C  \nATOM   1082  CB  ASP A 142      -6.859 -11.234  -4.137  1.00 96.12           C  \nATOM   1083  O   ASP A 142      -4.102 -11.523  -5.941  1.00 96.12           O  \nATOM   1084  CG  ASP A 142      -7.699 -12.148  -3.268  1.00 96.12           C  \nATOM   1085  OD1 ASP A 142      -7.148 -13.102  -2.670  1.00 96.12           O  \nATOM   1086  OD2 ASP A 142      -8.930 -11.922  -3.182  1.00 96.12           O  \nATOM   1087  N   PHE A 143      -4.484  -9.492  -5.031  1.00 95.38           N  \nATOM   1088  CA  PHE A 143      -3.730  -8.820  -6.082  1.00 95.38           C  \nATOM   1089  C   PHE A 143      -2.262  -9.227  -6.043  1.00 95.38           C  \nATOM   1090  CB  PHE A 143      -3.852  -7.297  -5.945  1.00 95.38           C  \nATOM   1091  O   PHE A 143      -1.658  -9.492  -7.082  1.00 95.38           O  \nATOM   1092  CG  PHE A 143      -3.340  -6.539  -7.141  1.00 95.38           C  \nATOM   1093  CD1 PHE A 143      -2.004  -6.168  -7.223  1.00 95.38           C  \nATOM   1094  CD2 PHE A 143      -4.195  -6.195  -8.180  1.00 95.38           C  \nATOM   1095  CE1 PHE A 143      -1.526  -5.465  -8.328  1.00 95.38           C  \nATOM   1096  CE2 PHE A 143      -3.725  -5.492  -9.281  1.00 95.38           C  \nATOM   1097  CZ  PHE A 143      -2.391  -5.129  -9.359  1.00 95.38           C  \nATOM   1098  N   LEU A 144      -1.703  -9.258  -4.898  1.00 95.94           N  \nATOM   1099  CA  LEU A 144      -0.323  -9.688  -4.707  1.00 95.94           C  \nATOM   1100  C   LEU A 144      -0.107 -11.086  -5.281  1.00 95.94           C  \nATOM   1101  CB  LEU A 144       0.044  -9.672  -3.221  1.00 95.94           C  \nATOM   1102  O   LEU A 144       0.857 -11.320  -6.016  1.00 95.94           O  \nATOM   1103  CG  LEU A 144       1.501  -9.992  -2.881  1.00 95.94           C  \nATOM   1104  CD1 LEU A 144       2.420  -8.898  -3.420  1.00 95.94           C  \nATOM   1105  CD2 LEU A 144       1.676 -10.148  -1.373  1.00 95.94           C  \nATOM   1106  N   ARG A 145      -1.022 -11.977  -4.980  1.00 96.44           N  \nATOM   1107  CA  ARG A 145      -0.920 -13.375  -5.406  1.00 96.44           C  \nATOM   1108  C   ARG A 145      -1.020 -13.492  -6.926  1.00 96.44           C  \nATOM   1109  CB  ARG A 145      -2.010 -14.219  -4.742  1.00 96.44           C  \nATOM   1110  O   ARG A 145      -0.221 -14.188  -7.551  1.00 96.44           O  \nATOM   1111  CG  ARG A 145      -1.936 -15.695  -5.090  1.00 96.44           C  \nATOM   1112  CD  ARG A 145      -2.996 -16.500  -4.352  1.00 96.44           C  \nATOM   1113  NE  ARG A 145      -2.697 -16.609  -2.924  1.00 96.44           N  \nATOM   1114  NH1 ARG A 145      -4.762 -15.844  -2.227  1.00 96.44           N  \nATOM   1115  NH2 ARG A 145      -3.168 -16.422  -0.684  1.00 96.44           N  \nATOM   1116  CZ  ARG A 145      -3.543 -16.297  -1.948  1.00 96.44           C  \nATOM   1117  N   GLU A 146      -1.889 -12.742  -7.500  1.00 94.94           N  \nATOM   1118  CA  GLU A 146      -2.238 -12.938  -8.906  1.00 94.94           C  \nATOM   1119  C   GLU A 146      -1.280 -12.180  -9.820  1.00 94.94           C  \nATOM   1120  CB  GLU A 146      -3.680 -12.484  -9.164  1.00 94.94           C  \nATOM   1121  O   GLU A 146      -0.918 -12.672 -10.891  1.00 94.94           O  \nATOM   1122  CG  GLU A 146      -4.727 -13.391  -8.531  1.00 94.94           C  \nATOM   1123  CD  GLU A 146      -6.148 -12.898  -8.742  1.00 94.94           C  \nATOM   1124  OE1 GLU A 146      -7.098 -13.547  -8.234  1.00 94.94           O  \nATOM   1125  OE2 GLU A 146      -6.320 -11.859  -9.414  1.00 94.94           O  \nATOM   1126  N   ARG A 147      -0.790 -11.062  -9.305  1.00 93.69           N  \nATOM   1127  CA  ARG A 147      -0.094 -10.211 -10.266  1.00 93.69           C  \nATOM   1128  C   ARG A 147       1.359 -10.000  -9.859  1.00 93.69           C  \nATOM   1129  CB  ARG A 147      -0.803  -8.859 -10.406  1.00 93.69           C  \nATOM   1130  O   ARG A 147       2.229  -9.805 -10.711  1.00 93.69           O  \nATOM   1131  CG  ARG A 147      -2.244  -8.969 -10.875  1.00 93.69           C  \nATOM   1132  CD  ARG A 147      -2.328  -9.461 -12.312  1.00 93.69           C  \nATOM   1133  NE  ARG A 147      -1.521  -8.648 -13.219  1.00 93.69           N  \nATOM   1134  NH1 ARG A 147      -3.068  -6.938 -13.383  1.00 93.69           N  \nATOM   1135  NH2 ARG A 147      -1.078  -6.805 -14.516  1.00 93.69           N  \nATOM   1136  CZ  ARG A 147      -1.891  -7.465 -13.703  1.00 93.69           C  \nATOM   1137  N   LEU A 148       1.692 -10.062  -8.523  1.00 95.31           N  \nATOM   1138  CA  LEU A 148       2.988  -9.578  -8.062  1.00 95.31           C  \nATOM   1139  C   LEU A 148       3.857 -10.727  -7.566  1.00 95.31           C  \nATOM   1140  CB  LEU A 148       2.809  -8.539  -6.953  1.00 95.31           C  \nATOM   1141  O   LEU A 148       5.082 -10.602  -7.496  1.00 95.31           O  \nATOM   1142  CG  LEU A 148       2.107  -7.242  -7.348  1.00 95.31           C  \nATOM   1143  CD1 LEU A 148       1.918  -6.348  -6.125  1.00 95.31           C  \nATOM   1144  CD2 LEU A 148       2.898  -6.512  -8.430  1.00 95.31           C  \nATOM   1145  N   LEU A 149       3.207 -11.859  -7.262  1.00 95.56           N  \nATOM   1146  CA  LEU A 149       3.914 -12.977  -6.645  1.00 95.56           C  \nATOM   1147  C   LEU A 149       5.070 -13.438  -7.523  1.00 95.56           C  \nATOM   1148  CB  LEU A 149       2.955 -14.141  -6.391  1.00 95.56           C  \nATOM   1149  O   LEU A 149       6.191 -13.625  -7.035  1.00 95.56           O  \nATOM   1150  CG  LEU A 149       3.570 -15.406  -5.793  1.00 95.56           C  \nATOM   1151  CD1 LEU A 149       4.258 -15.086  -4.469  1.00 95.56           C  \nATOM   1152  CD2 LEU A 149       2.504 -16.484  -5.602  1.00 95.56           C  \nATOM   1153  N   GLY A 150       4.809 -13.633  -8.797  1.00 95.25           N  \nATOM   1154  CA  GLY A 150       5.852 -14.047  -9.719  1.00 95.25           C  \nATOM   1155  C   GLY A 150       7.035 -13.102  -9.742  1.00 95.25           C  \nATOM   1156  O   GLY A 150       8.188 -13.539  -9.711  1.00 95.25           O  \nATOM   1157  N   TRP A 151       6.742 -11.867  -9.750  1.00 95.19           N  \nATOM   1158  CA  TRP A 151       7.785 -10.852  -9.805  1.00 95.19           C  \nATOM   1159  C   TRP A 151       8.594 -10.828  -8.508  1.00 95.19           C  \nATOM   1160  CB  TRP A 151       7.172  -9.469 -10.062  1.00 95.19           C  \nATOM   1161  O   TRP A 151       9.820 -10.742  -8.539  1.00 95.19           O  \nATOM   1162  CG  TRP A 151       8.172  -8.352 -10.094  1.00 95.19           C  \nATOM   1163  CD1 TRP A 151       8.961  -7.984 -11.141  1.00 95.19           C  \nATOM   1164  CD2 TRP A 151       8.492  -7.465  -9.016  1.00 95.19           C  \nATOM   1165  CE2 TRP A 151       9.492  -6.582  -9.484  1.00 95.19           C  \nATOM   1166  CE3 TRP A 151       8.039  -7.332  -7.699  1.00 95.19           C  \nATOM   1167  NE1 TRP A 151       9.758  -6.918 -10.789  1.00 95.19           N  \nATOM   1168  CH2 TRP A 151       9.570  -5.465  -7.402  1.00 95.19           C  \nATOM   1169  CZ2 TRP A 151      10.031  -5.574  -8.688  1.00 95.19           C  \nATOM   1170  CZ3 TRP A 151       8.578  -6.332  -6.902  1.00 95.19           C  \nATOM   1171  N   ILE A 152       7.926 -10.828  -7.395  1.00 95.38           N  \nATOM   1172  CA  ILE A 152       8.586 -10.805  -6.094  1.00 95.38           C  \nATOM   1173  C   ILE A 152       9.500 -12.023  -5.957  1.00 95.38           C  \nATOM   1174  CB  ILE A 152       7.562 -10.773  -4.938  1.00 95.38           C  \nATOM   1175  O   ILE A 152      10.617 -11.906  -5.449  1.00 95.38           O  \nATOM   1176  CG1 ILE A 152       6.789  -9.453  -4.945  1.00 95.38           C  \nATOM   1177  CG2 ILE A 152       8.258 -11.000  -3.592  1.00 95.38           C  \nATOM   1178  CD1 ILE A 152       5.629  -9.406  -3.959  1.00 95.38           C  \nATOM   1179  N   GLN A 153       9.023 -13.102  -6.445  1.00 94.56           N  \nATOM   1180  CA  GLN A 153       9.836 -14.320  -6.438  1.00 94.56           C  \nATOM   1181  C   GLN A 153      11.102 -14.141  -7.273  1.00 94.56           C  \nATOM   1182  CB  GLN A 153       9.031 -15.508  -6.957  1.00 94.56           C  \nATOM   1183  O   GLN A 153      12.188 -14.562  -6.863  1.00 94.56           O  \nATOM   1184  CG  GLN A 153       8.055 -16.078  -5.938  1.00 94.56           C  \nATOM   1185  CD  GLN A 153       7.246 -17.250  -6.484  1.00 94.56           C  \nATOM   1186  NE2 GLN A 153       6.734 -18.078  -5.594  1.00 94.56           N  \nATOM   1187  OE1 GLN A 153       7.086 -17.391  -7.703  1.00 94.56           O  \nATOM   1188  N   ASP A 154      10.930 -13.461  -8.383  1.00 93.69           N  \nATOM   1189  CA  ASP A 154      12.055 -13.195  -9.273  1.00 93.69           C  \nATOM   1190  C   ASP A 154      13.070 -12.258  -8.617  1.00 93.69           C  \nATOM   1191  CB  ASP A 154      11.570 -12.602 -10.594  1.00 93.69           C  \nATOM   1192  O   ASP A 154      14.242 -12.242  -8.992  1.00 93.69           O  \nATOM   1193  CG  ASP A 154      10.859 -13.617 -11.477  1.00 93.69           C  \nATOM   1194  OD1 ASP A 154      10.969 -14.836 -11.211  1.00 93.69           O  \nATOM   1195  OD2 ASP A 154      10.188 -13.203 -12.445  1.00 93.69           O  \nATOM   1196  N   GLN A 155      12.570 -11.508  -7.680  1.00 92.38           N  \nATOM   1197  CA  GLN A 155      13.430 -10.562  -6.969  1.00 92.38           C  \nATOM   1198  C   GLN A 155      14.086 -11.219  -5.758  1.00 92.38           C  \nATOM   1199  CB  GLN A 155      12.633  -9.328  -6.535  1.00 92.38           C  \nATOM   1200  O   GLN A 155      14.797 -10.555  -4.996  1.00 92.38           O  \nATOM   1201  CG  GLN A 155      12.023  -8.555  -7.695  1.00 92.38           C  \nATOM   1202  CD  GLN A 155      13.070  -8.008  -8.648  1.00 92.38           C  \nATOM   1203  NE2 GLN A 155      12.859  -8.211  -9.945  1.00 92.38           N  \nATOM   1204  OE1 GLN A 155      14.062  -7.406  -8.227  1.00 92.38           O  \nATOM   1205  N   GLY A 156      13.859 -12.477  -5.504  1.00 91.31           N  \nATOM   1206  CA  GLY A 156      14.461 -13.219  -4.406  1.00 91.31           C  \nATOM   1207  C   GLY A 156      13.680 -13.102  -3.109  1.00 91.31           C  \nATOM   1208  O   GLY A 156      14.219 -13.344  -2.029  1.00 91.31           O  \nATOM   1209  N   GLY A 157      12.422 -12.516  -3.215  1.00 93.94           N  \nATOM   1210  CA  GLY A 157      11.570 -12.383  -2.037  1.00 93.94           C  \nATOM   1211  C   GLY A 157      11.469 -10.961  -1.527  1.00 93.94           C  \nATOM   1212  O   GLY A 157      11.891 -10.023  -2.209  1.00 93.94           O  \nATOM   1213  N   TRP A 158      11.016 -10.773  -0.332  1.00 94.31           N  \nATOM   1214  CA  TRP A 158      10.703  -9.469   0.242  1.00 94.31           C  \nATOM   1215  C   TRP A 158      11.977  -8.750   0.689  1.00 94.31           C  \nATOM   1216  CB  TRP A 158       9.742  -9.617   1.425  1.00 94.31           C  \nATOM   1217  O   TRP A 158      11.977  -7.535   0.873  1.00 94.31           O  \nATOM   1218  CG  TRP A 158       8.320  -9.891   1.025  1.00 94.31           C  \nATOM   1219  CD1 TRP A 158       7.699 -11.109   0.984  1.00 94.31           C  \nATOM   1220  CD2 TRP A 158       7.352  -8.930   0.606  1.00 94.31           C  \nATOM   1221  CE2 TRP A 158       6.160  -9.633   0.326  1.00 94.31           C  \nATOM   1222  CE3 TRP A 158       7.371  -7.539   0.440  1.00 94.31           C  \nATOM   1223  NE1 TRP A 158       6.398 -10.961   0.564  1.00 94.31           N  \nATOM   1224  CH2 TRP A 158       5.043  -7.633  -0.265  1.00 94.31           C  \nATOM   1225  CZ2 TRP A 158       4.996  -8.992  -0.111  1.00 94.31           C  \nATOM   1226  CZ3 TRP A 158       6.215  -6.902   0.005  1.00 94.31           C  \nATOM   1227  N   ASP A 159      13.062  -9.539   0.769  1.00 92.12           N  \nATOM   1228  CA  ASP A 159      14.312  -8.906   1.154  1.00 92.12           C  \nATOM   1229  C   ASP A 159      14.781  -7.918   0.085  1.00 92.12           C  \nATOM   1230  CB  ASP A 159      15.398  -9.953   1.409  1.00 92.12           C  \nATOM   1231  O   ASP A 159      15.586  -7.027   0.364  1.00 92.12           O  \nATOM   1232  CG  ASP A 159      15.164 -10.742   2.688  1.00 92.12           C  \nATOM   1233  OD1 ASP A 159      14.562 -10.203   3.639  1.00 92.12           O  \nATOM   1234  OD2 ASP A 159      15.594 -11.914   2.748  1.00 92.12           O  \nATOM   1235  N   GLY A 160      14.266  -8.117  -1.141  1.00 91.19           N  \nATOM   1236  CA  GLY A 160      14.539  -7.180  -2.219  1.00 91.19           C  \nATOM   1237  C   GLY A 160      14.094  -5.766  -1.900  1.00 91.19           C  \nATOM   1238  O   GLY A 160      14.711  -4.797  -2.348  1.00 91.19           O  \nATOM   1239  N   LEU A 161      13.047  -5.672  -1.155  1.00 91.38           N  \nATOM   1240  CA  LEU A 161      12.531  -4.371  -0.736  1.00 91.38           C  \nATOM   1241  C   LEU A 161      13.570  -3.621   0.095  1.00 91.38           C  \nATOM   1242  CB  LEU A 161      11.242  -4.543   0.068  1.00 91.38           C  \nATOM   1243  O   LEU A 161      13.781  -2.422  -0.104  1.00 91.38           O  \nATOM   1244  CG  LEU A 161      10.523  -3.254   0.475  1.00 91.38           C  \nATOM   1245  CD1 LEU A 161       9.016  -3.471   0.491  1.00 91.38           C  \nATOM   1246  CD2 LEU A 161      11.016  -2.775   1.837  1.00 91.38           C  \nATOM   1247  N   LEU A 162      14.227  -4.305   0.993  1.00 90.06           N  \nATOM   1248  CA  LEU A 162      15.242  -3.684   1.842  1.00 90.06           C  \nATOM   1249  C   LEU A 162      16.406  -3.180   1.009  1.00 90.06           C  \nATOM   1250  CB  LEU A 162      15.734  -4.676   2.898  1.00 90.06           C  \nATOM   1251  O   LEU A 162      16.938  -2.088   1.257  1.00 90.06           O  \nATOM   1252  CG  LEU A 162      14.742  -5.055   3.992  1.00 90.06           C  \nATOM   1253  CD1 LEU A 162      15.336  -6.117   4.910  1.00 90.06           C  \nATOM   1254  CD2 LEU A 162      14.320  -3.824   4.789  1.00 90.06           C  \nATOM   1255  N   SER A 163      16.797  -3.977   0.064  1.00 88.00           N  \nATOM   1256  CA  SER A 163      17.906  -3.598  -0.809  1.00 88.00           C  \nATOM   1257  C   SER A 163      17.547  -2.381  -1.655  1.00 88.00           C  \nATOM   1258  CB  SER A 163      18.297  -4.762  -1.717  1.00 88.00           C  \nATOM   1259  O   SER A 163      18.375  -1.485  -1.846  1.00 88.00           O  \nATOM   1260  OG  SER A 163      18.719  -5.879  -0.951  1.00 88.00           O  \nATOM   1261  N   TYR A 164      16.359  -2.307  -2.100  1.00 86.19           N  \nATOM   1262  CA  TYR A 164      15.875  -1.240  -2.967  1.00 86.19           C  \nATOM   1263  C   TYR A 164      15.797   0.082  -2.213  1.00 86.19           C  \nATOM   1264  CB  TYR A 164      14.500  -1.599  -3.539  1.00 86.19           C  \nATOM   1265  O   TYR A 164      16.266   1.111  -2.703  1.00 86.19           O  \nATOM   1266  CG  TYR A 164      13.969  -0.579  -4.516  1.00 86.19           C  \nATOM   1267  CD1 TYR A 164      13.039   0.383  -4.109  1.00 86.19           C  \nATOM   1268  CD2 TYR A 164      14.383  -0.577  -5.844  1.00 86.19           C  \nATOM   1269  CE1 TYR A 164      12.547   1.323  -5.008  1.00 86.19           C  \nATOM   1270  CE2 TYR A 164      13.891   0.358  -6.746  1.00 86.19           C  \nATOM   1271  OH  TYR A 164      12.484   2.232  -7.211  1.00 86.19           O  \nATOM   1272  CZ  TYR A 164      12.977   1.304  -6.320  1.00 86.19           C  \nATOM   1273  N   PHE A 165      15.289   0.046  -1.013  1.00 83.75           N  \nATOM   1274  CA  PHE A 165      15.062   1.288  -0.283  1.00 83.75           C  \nATOM   1275  C   PHE A 165      16.250   1.602   0.628  1.00 83.75           C  \nATOM   1276  CB  PHE A 165      13.773   1.206   0.539  1.00 83.75           C  \nATOM   1277  O   PHE A 165      16.297   2.670   1.240  1.00 83.75           O  \nATOM   1278  CG  PHE A 165      12.523   1.414  -0.272  1.00 83.75           C  \nATOM   1279  CD1 PHE A 165      12.117   2.693  -0.633  1.00 83.75           C  \nATOM   1280  CD2 PHE A 165      11.750   0.331  -0.672  1.00 83.75           C  \nATOM   1281  CE1 PHE A 165      10.961   2.889  -1.384  1.00 83.75           C  \nATOM   1282  CE2 PHE A 165      10.594   0.519  -1.423  1.00 83.75           C  \nATOM   1283  CZ  PHE A 165      10.195   1.799  -1.776  1.00 83.75           C  \nATOM   1284  N   GLY A 166      17.141   0.699   0.756  1.00 79.56           N  \nATOM   1285  CA  GLY A 166      18.344   0.928   1.523  1.00 79.56           C  \nATOM   1286  C   GLY A 166      19.391   1.725   0.764  1.00 79.56           C  \nATOM   1287  O   GLY A 166      20.375   2.186   1.349  1.00 79.56           O  \nATOM   1288  N   THR A 167      19.078   2.117  -0.519  1.00 77.88           N  \nATOM   1289  CA  THR A 167      20.016   2.898  -1.315  1.00 77.88           C  \nATOM   1290  C   THR A 167      19.859   4.391  -1.032  1.00 77.88           C  \nATOM   1291  CB  THR A 167      19.828   2.637  -2.820  1.00 77.88           C  \nATOM   1292  O   THR A 167      18.828   4.820  -0.502  1.00 77.88           O  \nATOM   1293  CG2 THR A 167      19.984   1.153  -3.145  1.00 77.88           C  \nATOM   1294  OG1 THR A 167      18.516   3.064  -3.211  1.00 77.88           O  \nATOM   1295  N   PRO A 168      20.938   5.160  -1.053  1.00 71.38           N  \nATOM   1296  CA  PRO A 168      20.922   6.594  -0.749  1.00 71.38           C  \nATOM   1297  C   PRO A 168      19.859   7.359  -1.532  1.00 71.38           C  \nATOM   1298  CB  PRO A 168      22.328   7.039  -1.154  1.00 71.38           C  \nATOM   1299  O   PRO A 168      19.516   8.492  -1.171  1.00 71.38           O  \nATOM   1300  CG  PRO A 168      23.141   5.781  -1.199  1.00 71.38           C  \nATOM   1301  CD  PRO A 168      22.203   4.629  -1.430  1.00 71.38           C  \nATOM   1302  N   THR A 169      19.219   6.746  -2.469  1.00 71.50           N  \nATOM   1303  CA  THR A 169      18.250   7.473  -3.289  1.00 71.50           C  \nATOM   1304  C   THR A 169      16.844   7.266  -2.764  1.00 71.50           C  \nATOM   1305  CB  THR A 169      18.328   7.027  -4.762  1.00 71.50           C  \nATOM   1306  O   THR A 169      15.867   7.496  -3.482  1.00 71.50           O  \nATOM   1307  CG2 THR A 169      19.672   7.422  -5.387  1.00 71.50           C  \nATOM   1308  OG1 THR A 169      18.188   5.605  -4.832  1.00 71.50           O  \nATOM   1309  N   TRP A 170      16.734   6.852  -1.522  1.00 70.88           N  \nATOM   1310  CA  TRP A 170      15.430   6.445  -1.016  1.00 70.88           C  \nATOM   1311  C   TRP A 170      14.469   7.621  -0.994  1.00 70.88           C  \nATOM   1312  CB  TRP A 170      15.562   5.848   0.388  1.00 70.88           C  \nATOM   1313  O   TRP A 170      13.250   7.438  -1.102  1.00 70.88           O  \nATOM   1314  CG  TRP A 170      15.914   6.852   1.446  1.00 70.88           C  \nATOM   1315  CD1 TRP A 170      17.172   7.266   1.806  1.00 70.88           C  \nATOM   1316  CD2 TRP A 170      15.000   7.559   2.285  1.00 70.88           C  \nATOM   1317  CE2 TRP A 170      15.766   8.391   3.133  1.00 70.88           C  \nATOM   1318  CE3 TRP A 170      13.602   7.574   2.404  1.00 70.88           C  \nATOM   1319  NE1 TRP A 170      17.078   8.195   2.818  1.00 70.88           N  \nATOM   1320  CH2 TRP A 170      13.812   9.227   4.180  1.00 70.88           C  \nATOM   1321  CZ2 TRP A 170      15.180   9.227   4.086  1.00 70.88           C  \nATOM   1322  CZ3 TRP A 170      13.023   8.414   3.354  1.00 70.88           C  \nATOM   1323  N   GLN A 171      14.906   8.883  -0.760  1.00 71.62           N  \nATOM   1324  CA  GLN A 171      14.016  10.039  -0.795  1.00 71.62           C  \nATOM   1325  C   GLN A 171      13.375  10.203  -2.172  1.00 71.62           C  \nATOM   1326  CB  GLN A 171      14.781  11.312  -0.420  1.00 71.62           C  \nATOM   1327  O   GLN A 171      12.195  10.523  -2.277  1.00 71.62           O  \nATOM   1328  CG  GLN A 171      15.188  11.375   1.046  1.00 71.62           C  \nATOM   1329  CD  GLN A 171      15.992  12.617   1.378  1.00 71.62           C  \nATOM   1330  NE2 GLN A 171      15.953  13.031   2.639  1.00 71.62           N  \nATOM   1331  OE1 GLN A 171      16.656  13.195   0.507  1.00 71.62           O  \nATOM   1332  N   THR A 172      14.219   9.945  -3.193  1.00 77.25           N  \nATOM   1333  CA  THR A 172      13.695  10.047  -4.551  1.00 77.25           C  \nATOM   1334  C   THR A 172      12.648   8.969  -4.809  1.00 77.25           C  \nATOM   1335  CB  THR A 172      14.820   9.938  -5.594  1.00 77.25           C  \nATOM   1336  O   THR A 172      11.656   9.211  -5.496  1.00 77.25           O  \nATOM   1337  CG2 THR A 172      14.305  10.227  -7.000  1.00 77.25           C  \nATOM   1338  OG1 THR A 172      15.852  10.875  -5.273  1.00 77.25           O  \nATOM   1339  N   VAL A 173      12.812   7.871  -4.148  1.00 78.44           N  \nATOM   1340  CA  VAL A 173      11.906   6.742  -4.328  1.00 78.44           C  \nATOM   1341  C   VAL A 173      10.531   7.086  -3.746  1.00 78.44           C  \nATOM   1342  CB  VAL A 173      12.453   5.461  -3.668  1.00 78.44           C  \nATOM   1343  O   VAL A 173       9.500   6.805  -4.363  1.00 78.44           O  \nATOM   1344  CG1 VAL A 173      11.406   4.352  -3.682  1.00 78.44           C  \nATOM   1345  CG2 VAL A 173      13.727   5.000  -4.375  1.00 78.44           C  \nATOM   1346  N   THR A 174      10.570   7.793  -2.590  1.00 77.19           N  \nATOM   1347  CA  THR A 174       9.312   8.156  -1.945  1.00 77.19           C  \nATOM   1348  C   THR A 174       8.539   9.156  -2.801  1.00 77.19           C  \nATOM   1349  CB  THR A 174       9.555   8.758  -0.547  1.00 77.19           C  \nATOM   1350  O   THR A 174       7.309   9.078  -2.898  1.00 77.19           O  \nATOM   1351  CG2 THR A 174       8.234   9.039   0.167  1.00 77.19           C  \nATOM   1352  OG1 THR A 174      10.320   7.832   0.235  1.00 77.19           O  \nATOM   1353  N   ILE A 175       9.281  10.023  -3.410  1.00 79.56           N  \nATOM   1354  CA  ILE A 175       8.680  11.000  -4.312  1.00 79.56           C  \nATOM   1355  C   ILE A 175       8.023  10.273  -5.488  1.00 79.56           C  \nATOM   1356  CB  ILE A 175       9.719  12.016  -4.824  1.00 79.56           C  \nATOM   1357  O   ILE A 175       6.902  10.609  -5.883  1.00 79.56           O  \nATOM   1358  CG1 ILE A 175      10.219  12.898  -3.672  1.00 79.56           C  \nATOM   1359  CG2 ILE A 175       9.125  12.875  -5.949  1.00 79.56           C  \nATOM   1360  CD1 ILE A 175      11.414  13.766  -4.027  1.00 79.56           C  \nATOM   1361  N   PHE A 176       8.758   9.297  -5.984  1.00 83.88           N  \nATOM   1362  CA  PHE A 176       8.227   8.492  -7.078  1.00 83.88           C  \nATOM   1363  C   PHE A 176       6.945   7.781  -6.652  1.00 83.88           C  \nATOM   1364  CB  PHE A 176       9.258   7.461  -7.547  1.00 83.88           C  \nATOM   1365  O   PHE A 176       5.949   7.793  -7.383  1.00 83.88           O  \nATOM   1366  CG  PHE A 176       8.773   6.574  -8.664  1.00 83.88           C  \nATOM   1367  CD1 PHE A 176       8.375   5.270  -8.406  1.00 83.88           C  \nATOM   1368  CD2 PHE A 176       8.711   7.051  -9.969  1.00 83.88           C  \nATOM   1369  CE1 PHE A 176       7.922   4.449  -9.438  1.00 83.88           C  \nATOM   1370  CE2 PHE A 176       8.266   6.234 -11.000  1.00 83.88           C  \nATOM   1371  CZ  PHE A 176       7.871   4.934 -10.734  1.00 83.88           C  \nATOM   1372  N   VAL A 177       6.941   7.219  -5.477  1.00 85.31           N  \nATOM   1373  CA  VAL A 177       5.781   6.508  -4.953  1.00 85.31           C  \nATOM   1374  C   VAL A 177       4.590   7.461  -4.859  1.00 85.31           C  \nATOM   1375  CB  VAL A 177       6.074   5.879  -3.572  1.00 85.31           C  \nATOM   1376  O   VAL A 177       3.494   7.141  -5.320  1.00 85.31           O  \nATOM   1377  CG1 VAL A 177       4.793   5.328  -2.947  1.00 85.31           C  \nATOM   1378  CG2 VAL A 177       7.129   4.781  -3.697  1.00 85.31           C  \nATOM   1379  N   ALA A 178       4.824   8.602  -4.289  1.00 83.06           N  \nATOM   1380  CA  ALA A 178       3.768   9.594  -4.172  1.00 83.06           C  \nATOM   1381  C   ALA A 178       3.254  10.016  -5.547  1.00 83.06           C  \nATOM   1382  CB  ALA A 178       4.270  10.812  -3.400  1.00 83.06           C  \nATOM   1383  O   ALA A 178       2.049  10.188  -5.738  1.00 83.06           O  \nATOM   1384  N   GLY A 179       4.180  10.086  -6.488  1.00 85.44           N  \nATOM   1385  CA  GLY A 179       3.826  10.422  -7.859  1.00 85.44           C  \nATOM   1386  C   GLY A 179       2.926   9.398  -8.516  1.00 85.44           C  \nATOM   1387  O   GLY A 179       1.928   9.750  -9.148  1.00 85.44           O  \nATOM   1388  N   VAL A 180       3.256   8.141  -8.391  1.00 87.06           N  \nATOM   1389  CA  VAL A 180       2.469   7.051  -8.961  1.00 87.06           C  \nATOM   1390  C   VAL A 180       1.073   7.043  -8.336  1.00 87.06           C  \nATOM   1391  CB  VAL A 180       3.154   5.680  -8.750  1.00 87.06           C  \nATOM   1392  O   VAL A 180       0.070   6.941  -9.047  1.00 87.06           O  \nATOM   1393  CG1 VAL A 180       2.193   4.539  -9.078  1.00 87.06           C  \nATOM   1394  CG2 VAL A 180       4.418   5.582  -9.594  1.00 87.06           C  \nATOM   1395  N   LEU A 181       0.977   7.258  -7.043  1.00 87.38           N  \nATOM   1396  CA  LEU A 181      -0.308   7.254  -6.352  1.00 87.38           C  \nATOM   1397  C   LEU A 181      -1.165   8.438  -6.793  1.00 87.38           C  \nATOM   1398  CB  LEU A 181      -0.102   7.293  -4.836  1.00 87.38           C  \nATOM   1399  O   LEU A 181      -2.357   8.281  -7.062  1.00 87.38           O  \nATOM   1400  CG  LEU A 181       0.305   5.977  -4.172  1.00 87.38           C  \nATOM   1401  CD1 LEU A 181       0.708   6.215  -2.721  1.00 87.38           C  \nATOM   1402  CD2 LEU A 181      -0.830   4.961  -4.258  1.00 87.38           C  \nATOM   1403  N   THR A 182      -0.523   9.547  -6.836  1.00 87.12           N  \nATOM   1404  CA  THR A 182      -1.249  10.750  -7.250  1.00 87.12           C  \nATOM   1405  C   THR A 182      -1.760  10.602  -8.680  1.00 87.12           C  \nATOM   1406  CB  THR A 182      -0.364  12.000  -7.137  1.00 87.12           C  \nATOM   1407  O   THR A 182      -2.908  10.945  -8.969  1.00 87.12           O  \nATOM   1408  CG2 THR A 182      -1.144  13.258  -7.504  1.00 87.12           C  \nATOM   1409  OG1 THR A 182       0.115  12.125  -5.793  1.00 87.12           O  \nATOM   1410  N   ALA A 183      -0.955  10.062  -9.594  1.00 86.00           N  \nATOM   1411  CA  ALA A 183      -1.333   9.875 -10.992  1.00 86.00           C  \nATOM   1412  C   ALA A 183      -2.492   8.891 -11.117  1.00 86.00           C  \nATOM   1413  CB  ALA A 183      -0.137   9.391 -11.805  1.00 86.00           C  \nATOM   1414  O   ALA A 183      -3.453   9.141 -11.852  1.00 86.00           O  \nATOM   1415  N   SER A 184      -2.408   7.828 -10.375  1.00 81.81           N  \nATOM   1416  CA  SER A 184      -3.451   6.809 -10.438  1.00 81.81           C  \nATOM   1417  C   SER A 184      -4.766   7.332  -9.867  1.00 81.81           C  \nATOM   1418  CB  SER A 184      -3.020   5.555  -9.672  1.00 81.81           C  \nATOM   1419  O   SER A 184      -5.836   7.078 -10.422  1.00 81.81           O  \nATOM   1420  OG  SER A 184      -2.703   5.867  -8.328  1.00 81.81           O  \nATOM   1421  N   LEU A 185      -4.715   8.039  -8.781  1.00 83.00           N  \nATOM   1422  CA  LEU A 185      -5.902   8.594  -8.141  1.00 83.00           C  \nATOM   1423  C   LEU A 185      -6.551   9.656  -9.031  1.00 83.00           C  \nATOM   1424  CB  LEU A 185      -5.547   9.203  -6.781  1.00 83.00           C  \nATOM   1425  O   LEU A 185      -7.777   9.758  -9.086  1.00 83.00           O  \nATOM   1426  CG  LEU A 185      -5.258   8.211  -5.652  1.00 83.00           C  \nATOM   1427  CD1 LEU A 185      -4.730   8.945  -4.426  1.00 83.00           C  \nATOM   1428  CD2 LEU A 185      -6.512   7.414  -5.309  1.00 83.00           C  \nATOM   1429  N   THR A 186      -5.738  10.477  -9.719  1.00 84.56           N  \nATOM   1430  CA  THR A 186      -6.230  11.492 -10.641  1.00 84.56           C  \nATOM   1431  C   THR A 186      -6.930  10.844 -11.836  1.00 84.56           C  \nATOM   1432  CB  THR A 186      -5.086  12.391 -11.141  1.00 84.56           C  \nATOM   1433  O   THR A 186      -7.988  11.305 -12.266  1.00 84.56           O  \nATOM   1434  CG2 THR A 186      -5.609  13.477 -12.078  1.00 84.56           C  \nATOM   1435  OG1 THR A 186      -4.441  13.000 -10.023  1.00 84.56           O  \nATOM   1436  N   ILE A 187      -6.285   9.773 -12.383  1.00 81.12           N  \nATOM   1437  CA  ILE A 187      -6.871   9.070 -13.516  1.00 81.12           C  \nATOM   1438  C   ILE A 187      -8.211   8.461 -13.109  1.00 81.12           C  \nATOM   1439  CB  ILE A 187      -5.922   7.977 -14.047  1.00 81.12           C  \nATOM   1440  O   ILE A 187      -9.172   8.500 -13.875  1.00 81.12           O  \nATOM   1441  CG1 ILE A 187      -4.660   8.609 -14.656  1.00 81.12           C  \nATOM   1442  CG2 ILE A 187      -6.637   7.098 -15.078  1.00 81.12           C  \nATOM   1443  CD1 ILE A 187      -3.561   7.609 -14.977  1.00 81.12           C  \nATOM   1444  N   TRP A 188      -8.219   7.891 -11.914  1.00 77.62           N  \nATOM   1445  CA  TRP A 188      -9.477   7.355 -11.391  1.00 77.62           C  \nATOM   1446  C   TRP A 188     -10.539   8.445 -11.312  1.00 77.62           C  \nATOM   1447  CB  TRP A 188      -9.258   6.730 -10.016  1.00 77.62           C  \nATOM   1448  O   TRP A 188     -11.688   8.227 -11.719  1.00 77.62           O  \nATOM   1449  CG  TRP A 188     -10.531   6.348  -9.312  1.00 77.62           C  \nATOM   1450  CD1 TRP A 188     -11.344   5.281  -9.602  1.00 77.62           C  \nATOM   1451  CD2 TRP A 188     -11.148   7.035  -8.219  1.00 77.62           C  \nATOM   1452  CE2 TRP A 188     -12.320   6.328  -7.891  1.00 77.62           C  \nATOM   1453  CE3 TRP A 188     -10.812   8.180  -7.480  1.00 77.62           C  \nATOM   1454  NE1 TRP A 188     -12.422   5.266  -8.750  1.00 77.62           N  \nATOM   1455  CH2 TRP A 188     -12.820   7.852  -6.148  1.00 77.62           C  \nATOM   1456  CZ2 TRP A 188     -13.172   6.727  -6.852  1.00 77.62           C  \nATOM   1457  CZ3 TRP A 188     -11.656   8.578  -6.449  1.00 77.62           C  \nATOM   1458  N   LYS A 189     -10.172   9.586 -10.688  1.00 77.94           N  \nATOM   1459  CA  LYS A 189     -11.109  10.695 -10.516  1.00 77.94           C  \nATOM   1460  C   LYS A 189     -11.609  11.195 -11.867  1.00 77.94           C  \nATOM   1461  CB  LYS A 189     -10.461  11.836  -9.742  1.00 77.94           C  \nATOM   1462  O   LYS A 189     -12.773  11.570 -12.008  1.00 77.94           O  \nATOM   1463  CG  LYS A 189     -11.438  12.914  -9.297  1.00 77.94           C  \nATOM   1464  CD  LYS A 189     -10.750  13.977  -8.445  1.00 77.94           C  \nATOM   1465  CE  LYS A 189     -11.617  15.219  -8.289  1.00 77.94           C  \nATOM   1466  NZ  LYS A 189     -11.109  16.125  -7.211  1.00 77.94           N  \nATOM   1467  N   LYS A 190     -10.773  11.258 -12.875  1.00 74.31           N  \nATOM   1468  CA  LYS A 190     -11.133  11.797 -14.188  1.00 74.31           C  \nATOM   1469  C   LYS A 190     -12.008  10.820 -14.961  1.00 74.31           C  \nATOM   1470  CB  LYS A 190      -9.883  12.133 -14.992  1.00 74.31           C  \nATOM   1471  O   LYS A 190     -12.836  11.227 -15.781  1.00 74.31           O  \nATOM   1472  CG  LYS A 190      -9.203  13.430 -14.578  1.00 74.31           C  \nATOM   1473  CD  LYS A 190      -8.062  13.789 -15.516  1.00 74.31           C  \nATOM   1474  CE  LYS A 190      -7.395  15.102 -15.117  1.00 74.31           C  \nATOM   1475  NZ  LYS A 190      -6.238  15.430 -16.000  1.00 74.31           N  \nATOM   1476  N   MET A 191     -11.859   9.539 -14.625  1.00 64.88           N  \nATOM   1477  CA  MET A 191     -12.625   8.547 -15.375  1.00 64.88           C  \nATOM   1478  C   MET A 191     -13.945   8.242 -14.680  1.00 64.88           C  \nATOM   1479  CB  MET A 191     -11.812   7.258 -15.539  1.00 64.88           C  \nATOM   1480  O   MET A 191     -14.867   7.711 -15.305  1.00 64.88           O  \nATOM   1481  CG  MET A 191     -10.680   7.371 -16.547  1.00 64.88           C  \nATOM   1482  SD  MET A 191      -9.711   5.820 -16.688  1.00 64.88           S  \nATOM   1483  CE  MET A 191     -11.039   4.598 -16.531  1.00 64.88           C  \nATOM   1484  N   GLY A 192     -14.227   8.570 -13.359  1.00 55.59           N  \nATOM   1485  CA  GLY A 192     -15.523   8.477 -12.703  1.00 55.59           C  \nATOM   1486  C   GLY A 192     -16.391   9.688 -12.930  1.00 55.59           C  \nATOM   1487  O   GLY A 192     -17.594   9.562 -13.188  1.00 55.59           O  \nTER    1488      GLY A 192                                                      \nENDMDL                                                                          \nEND                                                                             \n","pdb");
	viewer_17019975603601787.setStyle({"cartoon": {"color": "spectrum"}});
	viewer_17019975603601787.zoomTo();
viewer_17019975603601787.render();
});
</script></div></div>
</div>
<div class="cell docutils container">
<div class="cell_input docutils container">
<div class="highlight-ipython3 notranslate"><div class="highlight"><pre><span></span><span class="c1"># @title # **8. Align PDBs** { run: &quot;auto&quot;, display-mode: &quot;form&quot; }</span>
<span class="c1">#@markdown We can compare the predictions with experiment.</span>
<span class="c1">#@markdown Example: PDB 1F16 was obtained from NMR, so there is an ensemble of structures.</span>

<span class="kn">import</span> <span class="nn">Bio.PDB</span>

<span class="n">pdb_code</span> <span class="o">=</span> <span class="s2">&quot;1f16&quot;</span> <span class="c1">#@param{type:&quot;string&quot;}</span>
<span class="c1">#@markdown - Reference PDB (Experiment)</span>
<span class="n">cpred</span> <span class="o">=</span> <span class="s2">&quot;&quot;</span><span class="c1">#@param{type:&quot;string&quot;}</span>
<span class="c1">#@markdown - Color for Reference PDB (Experiment)</span>
<span class="n">os</span><span class="o">.</span><span class="n">system</span><span class="p">(</span><span class="s2">&quot;wget -qnc https://files.rcsb.org/download/</span><span class="si">%s</span><span class="s2">.pdb.gz; gunzip </span><span class="si">%s</span><span class="s2">.pdb.gz&quot;</span> <span class="o">%</span> <span class="p">(</span><span class="n">pdb_code</span><span class="p">,</span><span class="n">pdb_code</span><span class="p">))</span>

<span class="n">start_id</span> <span class="o">=</span> <span class="mi">1</span>
<span class="n">end_id</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">query_sequence</span><span class="p">)</span>
<span class="n">atoms_to_be_aligned</span> <span class="o">=</span> <span class="nb">range</span><span class="p">(</span><span class="n">start_id</span><span class="p">,</span> <span class="n">end_id</span> <span class="o">+</span> <span class="mi">1</span><span class="p">)</span>

<span class="c1"># Start the parser</span>
<span class="n">pdb_parser</span> <span class="o">=</span> <span class="n">Bio</span><span class="o">.</span><span class="n">PDB</span><span class="o">.</span><span class="n">PDBParser</span><span class="p">(</span><span class="n">QUIET</span> <span class="o">=</span> <span class="kc">True</span><span class="p">)</span>

<span class="c1"># Get the structures</span>
<span class="n">ref_structure</span> <span class="o">=</span> <span class="n">pdb_parser</span><span class="o">.</span><span class="n">get_structure</span><span class="p">(</span><span class="s2">&quot;reference&quot;</span><span class="p">,</span> <span class="s2">&quot;</span><span class="si">%s</span><span class="s2">.pdb&quot;</span> <span class="o">%</span> <span class="n">pdb_code</span><span class="p">)</span>
<span class="n">sample_structure</span> <span class="o">=</span> <span class="n">pdb_parser</span><span class="o">.</span><span class="n">get_structure</span><span class="p">(</span><span class="s2">&quot;sample&quot;</span><span class="p">,</span> <span class="n">pdb_file</span><span class="p">[</span><span class="mi">0</span><span class="p">])</span>

<span class="c1"># Use the first model in the pdb-files for alignment</span>
<span class="c1"># Change the number 0 if you want to align to another structure</span>
<span class="n">ref_model</span>    <span class="o">=</span> <span class="n">ref_structure</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
<span class="n">sample_model</span> <span class="o">=</span> <span class="n">sample_structure</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>

<span class="c1"># Make a list of the atoms (in the structures) you wish to align.</span>
<span class="c1"># In this case we use CA atoms whose index is in the specified range</span>
<span class="n">ref_atoms</span> <span class="o">=</span> <span class="p">[]</span>
<span class="n">sample_atoms</span> <span class="o">=</span> <span class="p">[]</span>

<span class="c1"># Iterate of all chains in the model in order to find all residues</span>
<span class="k">for</span> <span class="n">ref_chain</span> <span class="ow">in</span> <span class="n">ref_model</span><span class="p">:</span>
  <span class="c1"># Iterate of all residues in each model in order to find proper atoms</span>
  <span class="k">for</span> <span class="n">ref_res</span> <span class="ow">in</span> <span class="n">ref_chain</span><span class="p">:</span>
    <span class="c1"># Check if residue number ( .get_id() ) is in the list</span>
    <span class="k">if</span> <span class="n">ref_res</span><span class="o">.</span><span class="n">get_id</span><span class="p">()[</span><span class="mi">1</span><span class="p">]</span> <span class="ow">in</span> <span class="n">atoms_to_be_aligned</span><span class="p">:</span>
      <span class="c1"># Append CA atom to list</span>
      <span class="n">ref_atoms</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">ref_res</span><span class="p">[</span><span class="s1">&#39;CA&#39;</span><span class="p">])</span>

<span class="c1"># Do the same for the sample structure</span>
<span class="k">for</span> <span class="n">sample_chain</span> <span class="ow">in</span> <span class="n">sample_model</span><span class="p">:</span>
  <span class="k">for</span> <span class="n">sample_res</span> <span class="ow">in</span> <span class="n">sample_chain</span><span class="p">:</span>
    <span class="k">if</span> <span class="n">sample_res</span><span class="o">.</span><span class="n">get_id</span><span class="p">()[</span><span class="mi">1</span><span class="p">]</span> <span class="ow">in</span> <span class="n">atoms_to_be_aligned</span><span class="p">:</span>
      <span class="n">sample_atoms</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">sample_res</span><span class="p">[</span><span class="s1">&#39;CA&#39;</span><span class="p">])</span>

<span class="c1"># Now we initiate the superimposer:</span>
<span class="n">super_imposer</span> <span class="o">=</span> <span class="n">Bio</span><span class="o">.</span><span class="n">PDB</span><span class="o">.</span><span class="n">Superimposer</span><span class="p">()</span>
<span class="n">super_imposer</span><span class="o">.</span><span class="n">set_atoms</span><span class="p">(</span><span class="n">ref_atoms</span><span class="p">,</span> <span class="n">sample_atoms</span><span class="p">)</span>
<span class="n">super_imposer</span><span class="o">.</span><span class="n">apply</span><span class="p">(</span><span class="n">sample_model</span><span class="o">.</span><span class="n">get_atoms</span><span class="p">())</span>

<span class="c1"># Print RMSD:</span>
<span class="nb">print</span><span class="p">(</span><span class="s1">&#39;The calculated RMSD is: </span><span class="si">%.3f</span><span class="s1"> Å&#39;</span> <span class="o">%</span> <span class="n">super_imposer</span><span class="o">.</span><span class="n">rms</span><span class="p">)</span>

<span class="c1"># Save the aligned version of one of the chains of 6ANE</span>
<span class="n">io</span> <span class="o">=</span> <span class="n">Bio</span><span class="o">.</span><span class="n">PDB</span><span class="o">.</span><span class="n">PDBIO</span><span class="p">()</span>
<span class="n">io</span><span class="o">.</span><span class="n">set_structure</span><span class="p">(</span><span class="n">sample_structure</span><span class="p">)</span>
<span class="n">io</span><span class="o">.</span><span class="n">save</span><span class="p">(</span><span class="s2">&quot;</span><span class="si">%s</span><span class="s2">-aligned.pdb&quot;</span> <span class="o">%</span> <span class="n">jobname</span><span class="p">)</span>
<span class="nb">print</span><span class="p">(</span><span class="s1">&#39;Saved Aligned PDB.&#39;</span><span class="p">)</span>

<span class="c1"># rank_num = 1 #@param [&quot;1&quot;, &quot;2&quot;, &quot;3&quot;, &quot;4&quot;, &quot;5&quot;] {type:&quot;raw&quot;}</span>
<span class="c1"># #@markdown - Predicted PDB (AF/CF)</span>
<span class="c1"># color = &quot;lDDT&quot; #@param [&quot;chain&quot;, &quot;lDDT&quot;, &quot;rainbow&quot;]</span>
<span class="c1"># #@markdown - Color Predicted PDB (Default: lDDT)</span>
<span class="c1"># tag = results[&quot;rank&quot;][0][rank_num - 1]</span>
<span class="c1"># jobname_prefix = &quot;.custom&quot; if msa_mode == &quot;custom&quot; else &quot;&quot;</span>
<span class="c1"># pdb_filename = f&quot;{jobname}/{jobname}{jobname_prefix}_unrelaxed_{tag}.pdb&quot;</span>
<span class="c1"># pdb_file = glob.glob(pdb_filename)</span>

<span class="c1"># how_sidechains = False #@param {type:&quot;boolean&quot;}</span>
<span class="c1"># show_mainchains = False #@param {type:&quot;boolean&quot;}</span>

<span class="n">view</span><span class="o">=</span><span class="n">py3Dmol</span><span class="o">.</span><span class="n">view</span><span class="p">()</span>
<span class="n">view</span><span class="o">.</span><span class="n">addModel</span><span class="p">(</span><span class="nb">open</span><span class="p">(</span><span class="s2">&quot;</span><span class="si">%s</span><span class="s2">-aligned.pdb&quot;</span> <span class="o">%</span> <span class="n">jobname</span><span class="p">)</span><span class="o">.</span><span class="n">read</span><span class="p">(),</span><span class="s1">&#39;pdb&#39;</span><span class="p">)</span>
<span class="c1"># view.addModel(open(&quot;%s.pdb&quot; % pdb_code).read())</span>
<span class="n">view</span><span class="o">.</span><span class="n">zoomTo</span><span class="p">()</span>
<span class="n">view</span><span class="o">.</span><span class="n">setBackgroundColor</span><span class="p">(</span><span class="s1">&#39;white&#39;</span><span class="p">)</span>
<span class="n">view</span><span class="o">.</span><span class="n">setStyle</span><span class="p">({</span><span class="s1">&#39;chain&#39;</span><span class="p">:</span><span class="s1">&#39;A&#39;</span><span class="p">},{</span><span class="s1">&#39;cartoon&#39;</span><span class="p">:</span> <span class="p">{</span><span class="s1">&#39;color&#39;</span><span class="p">:</span> <span class="s1">&#39;red&#39;</span><span class="p">}})</span>
<span class="n">view</span><span class="o">.</span><span class="n">setStyle</span><span class="p">({</span><span class="s1">&#39;chain&#39;</span><span class="p">:</span><span class="s1">&#39;B&#39;</span><span class="p">},{</span><span class="s1">&#39;cartoon&#39;</span><span class="p">:</span> <span class="p">{</span><span class="s1">&#39;color&#39;</span><span class="p">:</span> <span class="s1">&#39;blue&#39;</span><span class="p">}})</span>
<span class="n">view</span><span class="o">.</span><span class="n">show</span><span class="p">()</span>

<span class="c1"># show_pdb(rank_num, show_sidechains, show_mainchains, color).show()</span>
<span class="c1"># if color == &quot;lDDT&quot;:</span>
<span class="c1">#   plot_plddt_legend().show()</span>
</pre></div>
</div>
</div>
<div class="cell_output docutils container">
<div class="output stream highlight-myst-ansi notranslate"><div class="highlight"><pre><span></span>The calculated RMSD is: 5.711 Å
Saved Aligned PDB.
</pre></div>
</div>
<div class="output text_html"><div id="3dmolviewer_17019975623954616"  style="position: relative; width: 640px; height: 480px;">
        <p id="3dmolwarning_17019975623954616" style="background-color:#ffcccc;color:black">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>
        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>
        </div>
<script>

var loadScriptAsync = function(uri){
  return new Promise((resolve, reject) => {
    //this is to ignore the existence of requirejs amd
    var savedexports, savedmodule;
    if (typeof exports !== 'undefined') savedexports = exports;
    else exports = {}
    if (typeof module !== 'undefined') savedmodule = module;
    else module = {}

    var tag = document.createElement('script');
    tag.src = uri;
    tag.async = true;
    tag.onload = () => {
        exports = savedexports;
        module = savedmodule;
        resolve();
    };
  var firstScriptTag = document.getElementsByTagName('script')[0];
  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
});
};

if(typeof $3Dmolpromise === 'undefined') {
$3Dmolpromise = null;
  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.4/3Dmol-min.js');
}

var viewer_17019975623954616 = null;
var warn = document.getElementById("3dmolwarning_17019975623954616");
if(warn) {
    warn.parentNode.removeChild(warn);
}
$3Dmolpromise.then(function() {
viewer_17019975623954616 = $3Dmol.createViewer(document.getElementById("3dmolviewer_17019975623954616"),{backgroundColor:"white"});
viewer_17019975623954616.zoomTo();
	viewer_17019975623954616.addModel("ATOM      1  N   MET A   1      53.686  -6.062  -9.919  1.00 50.78           N  \nATOM      2  CA  MET A   1      52.314  -6.548 -10.077  1.00 50.78           C  \nATOM      3  C   MET A   1      51.727  -6.939  -8.726  1.00 50.78           C  \nATOM      4  CB  MET A   1      52.260  -7.736 -11.038  1.00 50.78           C  \nATOM      5  O   MET A   1      52.290  -7.742  -7.996  1.00 50.78           O  \nATOM      6  CG  MET A   1      52.160  -7.325 -12.496  1.00 50.78           C  \nATOM      7  SD  MET A   1      52.144  -8.784 -13.627  1.00 50.78           S  \nATOM      8  CE  MET A   1      51.625  -7.964 -15.165  1.00 50.78           C  \nATOM      9  N   ASP A   2      51.002  -5.988  -8.027  1.00 55.22           N  \nATOM     10  CA  ASP A   2      50.055  -6.415  -6.982  1.00 55.22           C  \nATOM     11  C   ASP A   2      48.930  -5.372  -6.791  1.00 55.22           C  \nATOM     12  CB  ASP A   2      50.809  -6.601  -5.651  1.00 55.22           C  \nATOM     13  O   ASP A   2      49.178  -4.214  -6.495  1.00 55.22           O  \nATOM     14  CG  ASP A   2      51.044  -5.310  -4.912  1.00 55.22           C  \nATOM     15  OD1 ASP A   2      50.739  -4.218  -5.457  1.00 55.22           O  \nATOM     16  OD2 ASP A   2      51.560  -5.365  -3.765  1.00 55.22           O  \nATOM     17  N   GLY A   3      47.780  -5.591  -7.468  1.00 46.88           N  \nATOM     18  CA  GLY A   3      46.461  -6.071  -7.099  1.00 46.88           C  \nATOM     19  C   GLY A   3      45.435  -4.946  -6.904  1.00 46.88           C  \nATOM     20  O   GLY A   3      45.577  -4.142  -6.006  1.00 46.88           O  \nATOM     21  N   SER A   4      44.904  -4.447  -7.988  1.00 52.16           N  \nATOM     22  CA  SER A   4      43.562  -3.968  -8.295  1.00 52.16           C  \nATOM     23  C   SER A   4      42.524  -4.738  -7.484  1.00 52.16           C  \nATOM     24  CB  SER A   4      43.263  -4.155  -9.789  1.00 52.16           C  \nATOM     25  O   SER A   4      42.313  -5.936  -7.661  1.00 52.16           O  \nATOM     26  OG  SER A   4      44.417  -3.845 -10.564  1.00 52.16           O  \nATOM     27  N   GLY A   5      42.396  -4.467  -6.219  1.00 46.44           N  \nATOM     28  CA  GLY A   5      41.228  -4.893  -5.457  1.00 46.44           C  \nATOM     29  C   GLY A   5      40.024  -3.980  -5.655  1.00 46.44           C  \nATOM     30  O   GLY A   5      40.127  -2.770  -5.467  1.00 46.44           O  \nATOM     31  N   GLU A   6      39.410  -3.974  -6.823  1.00 48.91           N  \nATOM     32  CA  GLU A   6      37.999  -3.808  -7.153  1.00 48.91           C  \nATOM     33  C   GLU A   6      37.122  -4.012  -5.929  1.00 48.91           C  \nATOM     34  CB  GLU A   6      37.596  -4.802  -8.258  1.00 48.91           C  \nATOM     35  O   GLU A   6      37.167  -5.052  -5.273  1.00 48.91           O  \nATOM     36  CG  GLU A   6      37.475  -4.166  -9.636  1.00 48.91           C  \nATOM     37  CD  GLU A   6      37.217  -5.175 -10.737  1.00 48.91           C  \nATOM     38  OE1 GLU A   6      36.034  -5.383 -11.096  1.00 48.91           O  \nATOM     39  OE2 GLU A   6      38.184  -5.764 -11.244  1.00 48.91           O  \nATOM     40  N   GLN A   7      37.056  -3.089  -5.073  1.00 47.62           N  \nATOM     41  CA  GLN A   7      35.951  -2.913  -4.138  1.00 47.62           C  \nATOM     42  C   GLN A   7      34.599  -3.116  -4.827  1.00 47.62           C  \nATOM     43  CB  GLN A   7      36.003  -1.518  -3.497  1.00 47.62           C  \nATOM     44  O   GLN A   7      34.364  -2.571  -5.906  1.00 47.62           O  \nATOM     45  CG  GLN A   7      36.347  -1.534  -2.014  1.00 47.62           C  \nATOM     46  CD  GLN A   7      36.609  -0.156  -1.461  1.00 47.62           C  \nATOM     47  NE2 GLN A   7      35.633   0.388  -0.738  1.00 47.62           N  \nATOM     48  OE1 GLN A   7      37.691   0.436  -1.671  1.00 47.62           O  \nATOM     49  N   PRO A   8      33.885  -4.230  -4.646  1.00 45.00           N  \nATOM     50  CA  PRO A   8      32.515  -4.421  -5.152  1.00 45.00           C  \nATOM     51  C   PRO A   8      31.576  -3.313  -4.725  1.00 45.00           C  \nATOM     52  CB  PRO A   8      32.099  -5.759  -4.541  1.00 45.00           C  \nATOM     53  O   PRO A   8      31.652  -2.837  -3.589  1.00 45.00           O  \nATOM     54  CG  PRO A   8      33.099  -6.033  -3.468  1.00 45.00           C  \nATOM     55  CD  PRO A   8      34.117  -4.919  -3.478  1.00 45.00           C  \nATOM     56  N   ARG A   9      31.447  -2.130  -5.440  1.00 46.38           N  \nATOM     57  CA  ARG A   9      30.289  -1.240  -5.456  1.00 46.38           C  \nATOM     58  C   ARG A   9      29.053  -1.939  -4.898  1.00 46.38           C  \nATOM     59  CB  ARG A   9      30.011  -0.753  -6.880  1.00 46.38           C  \nATOM     60  O   ARG A   9      28.492  -2.823  -5.540  1.00 46.38           O  \nATOM     61  CG  ARG A   9      30.641   0.610  -7.186  1.00 46.38           C  \nATOM     62  CD  ARG A   9      30.207   1.131  -8.551  1.00 46.38           C  \nATOM     63  NE  ARG A   9      30.515   2.541  -8.712  1.00 46.38           N  \nATOM     64  NH1 ARG A   9      29.789   2.669 -10.900  1.00 46.38           N  \nATOM     65  NH2 ARG A   9      30.640   4.529  -9.859  1.00 46.38           N  \nATOM     66  CZ  ARG A   9      30.315   3.245  -9.822  1.00 46.38           C  \nATOM     67  N   GLY A  10      29.221  -2.624  -3.716  1.00 41.62           N  \nATOM     68  CA  GLY A  10      28.181  -3.158  -2.852  1.00 41.62           C  \nATOM     69  C   GLY A  10      27.075  -2.178  -2.562  1.00 41.62           C  \nATOM     70  O   GLY A  10      27.345  -1.030  -2.181  1.00 41.62           O  \nATOM     71  N   GLY A  11      26.201  -1.787  -3.537  1.00 47.22           N  \nATOM     72  CA  GLY A  11      24.801  -1.420  -3.394  1.00 47.22           C  \nATOM     73  C   GLY A  11      24.435  -1.031  -1.983  1.00 47.22           C  \nATOM     74  O   GLY A  11      24.391  -1.865  -1.081  1.00 47.22           O  \nATOM     75  N   GLY A  12      25.052  -0.031  -1.340  1.00 52.38           N  \nATOM     76  CA  GLY A  12      24.703   0.649  -0.097  1.00 52.38           C  \nATOM     77  C   GLY A  12      23.255   0.459   0.297  1.00 52.38           C  \nATOM     78  O   GLY A  12      22.434   0.044  -0.527  1.00 52.38           O  \nATOM     79  N   PRO A  13      22.950   0.107   1.588  1.00 58.81           N  \nATOM     80  CA  PRO A  13      21.596   0.089   2.130  1.00 58.81           C  \nATOM     81  C   PRO A  13      20.680   1.120   1.476  1.00 58.81           C  \nATOM     82  CB  PRO A  13      21.793   0.397   3.613  1.00 58.81           C  \nATOM     83  O   PRO A  13      21.146   2.194   1.081  1.00 58.81           O  \nATOM     84  CG  PRO A  13      23.250   0.717   3.738  1.00 58.81           C  \nATOM     85  CD  PRO A  13      23.896   0.514   2.394  1.00 58.81           C  \nATOM     86  N   THR A  14      19.805   0.659   0.613  1.00 71.31           N  \nATOM     87  CA  THR A  14      18.788   1.582   0.117  1.00 71.31           C  \nATOM     88  C   THR A  14      18.421   2.605   1.189  1.00 71.31           C  \nATOM     89  CB  THR A  14      17.521   0.832  -0.339  1.00 71.31           C  \nATOM     90  O   THR A  14      18.109   2.235   2.327  1.00 71.31           O  \nATOM     91  CG2 THR A  14      16.569   1.757  -1.095  1.00 71.31           C  \nATOM     92  OG1 THR A  14      17.895  -0.253  -1.202  1.00 71.31           O  \nATOM     93  N   SER A  15      18.734   3.842   1.021  1.00 87.69           N  \nATOM     94  CA  SER A  15      18.474   4.977   1.906  1.00 87.69           C  \nATOM     95  C   SER A  15      16.994   5.074   2.252  1.00 87.69           C  \nATOM     96  CB  SER A  15      18.941   6.283   1.261  1.00 87.69           C  \nATOM     97  O   SER A  15      16.137   4.804   1.408  1.00 87.69           O  \nATOM     98  OG  SER A  15      18.566   7.395   2.055  1.00 87.69           O  \nATOM     99  N   SER A  16      16.649   5.108   3.486  1.00 91.69           N  \nATOM    100  CA  SER A  16      15.293   5.305   3.985  1.00 91.69           C  \nATOM    101  C   SER A  16      14.541   6.337   3.153  1.00 91.69           C  \nATOM    102  CB  SER A  16      15.312   5.736   5.450  1.00 91.69           C  \nATOM    103  O   SER A  16      13.357   6.144   2.835  1.00 91.69           O  \nATOM    104  OG  SER A  16      15.817   4.700   6.273  1.00 91.69           O  \nATOM    105  N   GLU A  17      15.224   7.369   2.782  1.00 91.62           N  \nATOM    106  CA  GLU A  17      14.624   8.443   1.999  1.00 91.62           C  \nATOM    107  C   GLU A  17      14.202   7.944   0.618  1.00 91.62           C  \nATOM    108  CB  GLU A  17      15.574   9.620   1.862  1.00 91.62           C  \nATOM    109  O   GLU A  17      13.133   8.297   0.121  1.00 91.62           O  \nATOM    110  CG  GLU A  17      15.747  10.432   3.142  1.00 91.62           C  \nATOM    111  CD  GLU A  17      16.707  11.598   2.993  1.00 91.62           C  \nATOM    112  OE1 GLU A  17      16.861  12.387   3.956  1.00 91.62           O  \nATOM    113  OE2 GLU A  17      17.311  11.725   1.905  1.00 91.62           O  \nATOM    114  N   GLN A  18      15.062   7.150   0.031  1.00 91.38           N  \nATOM    115  CA  GLN A  18      14.750   6.576  -1.279  1.00 91.38           C  \nATOM    116  C   GLN A  18      13.559   5.629  -1.202  1.00 91.38           C  \nATOM    117  CB  GLN A  18      15.970   5.843  -1.844  1.00 91.38           C  \nATOM    118  O   GLN A  18      12.708   5.620  -2.091  1.00 91.38           O  \nATOM    119  CG  GLN A  18      17.070   6.774  -2.350  1.00 91.38           C  \nATOM    120  CD  GLN A  18      18.301   6.020  -2.826  1.00 91.38           C  \nATOM    121  NE2 GLN A  18      19.089   6.662  -3.683  1.00 91.38           N  \nATOM    122  OE1 GLN A  18      18.541   4.877  -2.422  1.00 91.38           O  \nATOM    123  N   ILE A  19      13.517   4.845  -0.144  1.00 93.50           N  \nATOM    124  CA  ILE A  19      12.416   3.906   0.050  1.00 93.50           C  \nATOM    125  C   ILE A  19      11.103   4.670   0.177  1.00 93.50           C  \nATOM    126  CB  ILE A  19      12.642   3.016   1.293  1.00 93.50           C  \nATOM    127  O   ILE A  19      10.100   4.299  -0.430  1.00 93.50           O  \nATOM    128  CG1 ILE A  19      13.820   2.063   1.063  1.00 93.50           C  \nATOM    129  CG2 ILE A  19      11.374   2.245   1.647  1.00 93.50           C  \nATOM    130  CD1 ILE A  19      14.215   1.257   2.293  1.00 93.50           C  \nATOM    131  N   MET A  20      11.154   5.727   0.936  1.00 95.50           N  \nATOM    132  CA  MET A  20       9.957   6.538   1.134  1.00 95.50           C  \nATOM    133  C   MET A  20       9.546   7.223  -0.163  1.00 95.50           C  \nATOM    134  CB  MET A  20      10.184   7.579   2.229  1.00 95.50           C  \nATOM    135  O   MET A  20       8.360   7.289  -0.487  1.00 95.50           O  \nATOM    136  CG  MET A  20      10.203   6.997   3.633  1.00 95.50           C  \nATOM    137  SD  MET A  20       8.675   6.065   4.032  1.00 95.50           S  \nATOM    138  CE  MET A  20       7.469   7.420   4.042  1.00 95.50           C  \nATOM    139  N   LYS A  21      10.538   7.729  -0.842  1.00 92.94           N  \nATOM    140  CA  LYS A  21      10.252   8.392  -2.110  1.00 92.94           C  \nATOM    141  C   LYS A  21       9.608   7.422  -3.096  1.00 92.94           C  \nATOM    142  CB  LYS A  21      11.529   8.979  -2.712  1.00 92.94           C  \nATOM    143  O   LYS A  21       8.618   7.764  -3.755  1.00 92.94           O  \nATOM    144  CG  LYS A  21      11.294   9.836  -3.947  1.00 92.94           C  \nATOM    145  CD  LYS A  21      12.585  10.442  -4.466  1.00 92.94           C  \nATOM    146  CE  LYS A  21      12.364  11.238  -5.743  1.00 92.94           C  \nATOM    147  NZ  LYS A  21      13.629  11.846  -6.245  1.00 92.94           N  \nATOM    148  N   THR A  22      10.175   6.236  -3.197  1.00 93.12           N  \nATOM    149  CA  THR A  22       9.617   5.210  -4.071  1.00 93.12           C  \nATOM    150  C   THR A  22       8.210   4.830  -3.628  1.00 93.12           C  \nATOM    151  CB  THR A  22      10.506   3.955  -4.099  1.00 93.12           C  \nATOM    152  O   THR A  22       7.309   4.688  -4.458  1.00 93.12           O  \nATOM    153  CG2 THR A  22       9.936   2.900  -5.036  1.00 93.12           C  \nATOM    154  OG1 THR A  22      11.820   4.319  -4.554  1.00 93.12           O  \nATOM    155  N   GLY A  23       8.027   4.647  -2.325  1.00 94.50           N  \nATOM    156  CA  GLY A  23       6.700   4.355  -1.797  1.00 94.50           C  \nATOM    157  C   GLY A  23       5.670   5.404  -2.165  1.00 94.50           C  \nATOM    158  O   GLY A  23       4.549   5.075  -2.546  1.00 94.50           O  \nATOM    159  N   ALA A  24       6.050   6.650  -2.058  1.00 94.06           N  \nATOM    160  CA  ALA A  24       5.161   7.753  -2.400  1.00 94.06           C  \nATOM    161  C   ALA A  24       4.793   7.718  -3.883  1.00 94.06           C  \nATOM    162  CB  ALA A  24       5.802   9.092  -2.044  1.00 94.06           C  \nATOM    163  O   ALA A  24       3.633   7.915  -4.248  1.00 94.06           O  \nATOM    164  N   LEU A  25       5.748   7.433  -4.701  1.00 92.19           N  \nATOM    165  CA  LEU A  25       5.530   7.363  -6.140  1.00 92.19           C  \nATOM    166  C   LEU A  25       4.594   6.208  -6.490  1.00 92.19           C  \nATOM    167  CB  LEU A  25       6.857   7.200  -6.881  1.00 92.19           C  \nATOM    168  O   LEU A  25       3.699   6.359  -7.325  1.00 92.19           O  \nATOM    169  CG  LEU A  25       7.752   8.440  -6.941  1.00 92.19           C  \nATOM    170  CD1 LEU A  25       9.108   8.089  -7.551  1.00 92.19           C  \nATOM    171  CD2 LEU A  25       7.073   9.548  -7.739  1.00 92.19           C  \nATOM    172  N   LEU A  26       4.834   5.116  -5.810  1.00 94.00           N  \nATOM    173  CA  LEU A  26       4.009   3.936  -6.069  1.00 94.00           C  \nATOM    174  C   LEU A  26       2.564   4.182  -5.653  1.00 94.00           C  \nATOM    175  CB  LEU A  26       4.572   2.721  -5.327  1.00 94.00           C  \nATOM    176  O   LEU A  26       1.635   3.823  -6.379  1.00 94.00           O  \nATOM    177  CG  LEU A  26       5.910   2.179  -5.826  1.00 94.00           C  \nATOM    178  CD1 LEU A  26       6.378   1.027  -4.942  1.00 94.00           C  \nATOM    179  CD2 LEU A  26       5.797   1.731  -7.282  1.00 94.00           C  \nATOM    180  N   LEU A  27       2.374   4.706  -4.473  1.00 94.56           N  \nATOM    181  CA  LEU A  27       1.033   4.984  -3.970  1.00 94.56           C  \nATOM    182  C   LEU A  27       0.302   5.956  -4.892  1.00 94.56           C  \nATOM    183  CB  LEU A  27       1.097   5.560  -2.555  1.00 94.56           C  \nATOM    184  O   LEU A  27      -0.848   5.716  -5.268  1.00 94.56           O  \nATOM    185  CG  LEU A  27      -0.241   5.896  -1.896  1.00 94.56           C  \nATOM    186  CD1 LEU A  27      -1.110   4.644  -1.797  1.00 94.56           C  \nATOM    187  CD2 LEU A  27      -0.021   6.513  -0.520  1.00 94.56           C  \nATOM    188  N   GLN A  28       0.950   7.003  -5.184  1.00 91.62           N  \nATOM    189  CA  GLN A  28       0.355   8.011  -6.056  1.00 91.62           C  \nATOM    190  C   GLN A  28      -0.012   7.417  -7.414  1.00 91.62           C  \nATOM    191  CB  GLN A  28       1.312   9.189  -6.240  1.00 91.62           C  \nATOM    192  O   GLN A  28      -1.101   7.668  -7.934  1.00 91.62           O  \nATOM    193  CG  GLN A  28       0.712  10.355  -7.026  1.00 91.62           C  \nATOM    194  CD  GLN A  28       1.621  11.570  -7.055  1.00 91.62           C  \nATOM    195  NE2 GLN A  28       1.155  12.635  -7.694  1.00 91.62           N  \nATOM    196  OE1 GLN A  28       2.727  11.551  -6.507  1.00 91.62           O  \nATOM    197  N   GLY A  29       0.978   6.715  -8.054  1.00 90.75           N  \nATOM    198  CA  GLY A  29       0.720   6.050  -9.326  1.00 90.75           C  \nATOM    199  C   GLY A  29      -0.426   5.059  -9.258  1.00 90.75           C  \nATOM    200  O   GLY A  29      -1.239   4.983 -10.180  1.00 90.75           O  \nATOM    201  N   PHE A  30      -0.439   4.352  -8.182  1.00 91.75           N  \nATOM    202  CA  PHE A  30      -1.494   3.373  -7.958  1.00 91.75           C  \nATOM    203  C   PHE A  30      -2.859   4.042  -7.915  1.00 91.75           C  \nATOM    204  CB  PHE A  30      -1.245   2.608  -6.650  1.00 91.75           C  \nATOM    205  O   PHE A  30      -3.805   3.581  -8.563  1.00 91.75           O  \nATOM    206  CG  PHE A  30      -2.106   1.384  -6.489  1.00 91.75           C  \nATOM    207  CD1 PHE A  30      -2.754   1.127  -5.290  1.00 91.75           C  \nATOM    208  CD2 PHE A  30      -2.266   0.492  -7.542  1.00 91.75           C  \nATOM    209  CE1 PHE A  30      -3.551  -0.006  -5.137  1.00 91.75           C  \nATOM    210  CE2 PHE A  30      -3.061  -0.641  -7.398  1.00 91.75           C  \nATOM    211  CZ  PHE A  30      -3.702  -0.890  -6.196  1.00 91.75           C  \nATOM    212  N   ILE A  31      -3.090   5.100  -7.251  1.00 91.19           N  \nATOM    213  CA  ILE A  31      -4.340   5.838  -7.096  1.00 91.19           C  \nATOM    214  C   ILE A  31      -4.724   6.486  -8.419  1.00 91.19           C  \nATOM    215  CB  ILE A  31      -4.230   6.905  -5.984  1.00 91.19           C  \nATOM    216  O   ILE A  31      -5.890   6.452  -8.821  1.00 91.19           O  \nATOM    217  CG1 ILE A  31      -4.062   6.237  -4.615  1.00 91.19           C  \nATOM    218  CG2 ILE A  31      -5.454   7.826  -6.002  1.00 91.19           C  \nATOM    219  CD1 ILE A  31      -3.719   7.207  -3.491  1.00 91.19           C  \nATOM    220  N   GLN A  32      -3.783   7.054  -9.105  1.00 88.69           N  \nATOM    221  CA  GLN A  32      -4.031   7.712 -10.383  1.00 88.69           C  \nATOM    222  C   GLN A  32      -4.582   6.727 -11.413  1.00 88.69           C  \nATOM    223  CB  GLN A  32      -2.748   8.359 -10.911  1.00 88.69           C  \nATOM    224  O   GLN A  32      -5.490   7.064 -12.178  1.00 88.69           O  \nATOM    225  CG  GLN A  32      -2.970   9.264 -12.119  1.00 88.69           C  \nATOM    226  CD  GLN A  32      -1.709   9.992 -12.543  1.00 88.69           C  \nATOM    227  NE2 GLN A  32      -1.770  10.668 -13.680  1.00 88.69           N  \nATOM    228  OE1 GLN A  32      -0.687   9.947 -11.851  1.00 88.69           O  \nATOM    229  N   ASP A  33      -3.873   5.630 -11.507  1.00 89.00           N  \nATOM    230  CA  ASP A  33      -4.318   4.594 -12.432  1.00 89.00           C  \nATOM    231  C   ASP A  33      -5.791   4.257 -12.212  1.00 89.00           C  \nATOM    232  CB  ASP A  33      -3.463   3.335 -12.279  1.00 89.00           C  \nATOM    233  O   ASP A  33      -6.566   4.176 -13.167  1.00 89.00           O  \nATOM    234  CG  ASP A  33      -3.901   2.206 -13.199  1.00 89.00           C  \nATOM    235  OD1 ASP A  33      -3.675   2.293 -14.423  1.00 89.00           O  \nATOM    236  OD2 ASP A  33      -4.481   1.221 -12.687  1.00 89.00           O  \nATOM    237  N   ARG A  34      -6.223   4.155 -11.037  1.00 87.88           N  \nATOM    238  CA  ARG A  34      -7.583   3.746 -10.698  1.00 87.88           C  \nATOM    239  C   ARG A  34      -8.551   4.920 -10.822  1.00 87.88           C  \nATOM    240  CB  ARG A  34      -7.637   3.166  -9.282  1.00 87.88           C  \nATOM    241  O   ARG A  34      -9.715   4.736 -11.186  1.00 87.88           O  \nATOM    242  CG  ARG A  34      -7.097   1.750  -9.180  1.00 87.88           C  \nATOM    243  CD  ARG A  34      -6.966   1.305  -7.730  1.00 87.88           C  \nATOM    244  NE  ARG A  34      -6.662  -0.121  -7.629  1.00 87.88           N  \nATOM    245  NH1 ARG A  34      -4.439   0.101  -8.223  1.00 87.88           N  \nATOM    246  NH2 ARG A  34      -5.301  -1.966  -7.747  1.00 87.88           N  \nATOM    247  CZ  ARG A  34      -5.467  -0.657  -7.867  1.00 87.88           C  \nATOM    248  N   ALA A  35      -8.095   6.118 -10.449  1.00 86.75           N  \nATOM    249  CA  ALA A  35      -8.921   7.309 -10.622  1.00 86.75           C  \nATOM    250  C   ALA A  35      -9.201   7.571 -12.098  1.00 86.75           C  \nATOM    251  CB  ALA A  35      -8.243   8.522  -9.990  1.00 86.75           C  \nATOM    252  O   ALA A  35     -10.288   8.028 -12.460  1.00 86.75           O  \nATOM    253  N   GLY A  36      -8.170   7.366 -12.982  1.00 83.94           N  \nATOM    254  CA  GLY A  36      -8.327   7.521 -14.410  1.00 83.94           C  \nATOM    255  C   GLY A  36      -9.382   6.607 -15.005  1.00 83.94           C  \nATOM    256  O   GLY A  36     -10.053   6.969 -15.973  1.00 83.94           O  \nATOM    257  N   ARG A  37      -9.614   5.534 -14.413  1.00 79.62           N  \nATOM    258  CA  ARG A  37     -10.604   4.566 -14.882  1.00 79.62           C  \nATOM    259  C   ARG A  37     -12.020   5.039 -14.574  1.00 79.62           C  \nATOM    260  CB  ARG A  37     -10.359   3.193 -14.257  1.00 79.62           C  \nATOM    261  O   ARG A  37     -12.973   4.647 -15.248  1.00 79.62           O  \nATOM    262  CG  ARG A  37      -9.089   2.513 -14.745  1.00 79.62           C  \nATOM    263  CD  ARG A  37      -8.913   1.135 -14.116  1.00 79.62           C  \nATOM    264  NE  ARG A  37      -7.638   0.528 -14.497  1.00 79.62           N  \nATOM    265  NH1 ARG A  37      -8.577  -1.427 -15.298  1.00 79.62           N  \nATOM    266  NH2 ARG A  37      -6.303  -1.126 -15.363  1.00 79.62           N  \nATOM    267  CZ  ARG A  37      -7.511  -0.675 -15.050  1.00 79.62           C  \nATOM    268  N   MET A  38     -12.124   5.884 -13.529  1.00 76.88           N  \nATOM    269  CA  MET A  38     -13.424   6.425 -13.135  1.00 76.88           C  \nATOM    270  C   MET A  38     -13.783   7.641 -13.988  1.00 76.88           C  \nATOM    271  CB  MET A  38     -13.425   6.804 -11.658  1.00 76.88           C  \nATOM    272  O   MET A  38     -14.917   8.121 -13.938  1.00 76.88           O  \nATOM    273  CG  MET A  38     -14.813   7.018 -11.080  1.00 76.88           C  \nATOM    274  SD  MET A  38     -14.774   7.637  -9.354  1.00 76.88           S  \nATOM    275  CE  MET A  38     -14.148   6.164  -8.497  1.00 76.88           C  \nATOM    276  N   GLY A  39     -12.945   8.024 -14.943  1.00 70.69           N  \nATOM    277  CA  GLY A  39     -13.226   9.157 -15.801  1.00 70.69           C  \nATOM    278  C   GLY A  39     -12.949  10.495 -15.148  1.00 70.69           C  \nATOM    279  O   GLY A  39     -13.297  11.548 -15.688  1.00 70.69           O  \nATOM    280  N   GLY A  40     -12.318  10.511 -13.879  1.00 64.44           N  \nATOM    281  CA  GLY A  40     -12.112  11.753 -13.145  1.00 64.44           C  \nATOM    282  C   GLY A  40     -10.676  12.236 -13.189  1.00 64.44           C  \nATOM    283  O   GLY A  40      -9.741  11.439 -13.223  1.00 64.44           O  \nATOM    284  N   GLU A  41     -10.321  13.137 -14.136  1.00 60.88           N  \nATOM    285  CA  GLU A  41      -9.023  13.802 -14.094  1.00 60.88           C  \nATOM    286  C   GLU A  41      -8.779  14.438 -12.721  1.00 60.88           C  \nATOM    287  CB  GLU A  41      -8.915  14.857 -15.186  1.00 60.88           C  \nATOM    288  O   GLU A  41      -9.635  15.161 -12.206  1.00 60.88           O  \nATOM    289  CG  GLU A  41      -8.694  14.285 -16.587  1.00 60.88           C  \nATOM    290  CD  GLU A  41      -8.591  15.341 -17.660  1.00 60.88           C  \nATOM    291  OE1 GLU A  41      -8.332  14.993 -18.845  1.00 60.88           O  \nATOM    292  OE2 GLU A  41      -8.771  16.535 -17.336  1.00 60.88           O  \nATOM    293  N   ALA A  42      -8.095  13.684 -11.831  1.00 65.38           N  \nATOM    294  CA  ALA A  42      -7.706  14.300 -10.570  1.00 65.38           C  \nATOM    295  C   ALA A  42      -6.395  15.064 -10.707  1.00 65.38           C  \nATOM    296  CB  ALA A  42      -7.581  13.238  -9.476  1.00 65.38           C  \nATOM    297  O   ALA A  42      -5.316  14.469 -10.704  1.00 65.38           O  \nATOM    298  N   PRO A  43      -6.558  16.360 -11.125  1.00 68.81           N  \nATOM    299  CA  PRO A  43      -5.353  17.188 -11.219  1.00 68.81           C  \nATOM    300  C   PRO A  43      -4.450  17.050  -9.984  1.00 68.81           C  \nATOM    301  CB  PRO A  43      -5.907  18.609 -11.326  1.00 68.81           C  \nATOM    302  O   PRO A  43      -3.236  17.232 -10.091  1.00 68.81           O  \nATOM    303  CG  PRO A  43      -7.345  18.432 -11.705  1.00 68.81           C  \nATOM    304  CD  PRO A  43      -7.765  17.042 -11.317  1.00 68.81           C  \nATOM    305  N   GLU A  44      -5.121  16.695  -8.836  1.00 70.12           N  \nATOM    306  CA  GLU A  44      -4.387  16.603  -7.579  1.00 70.12           C  \nATOM    307  C   GLU A  44      -3.393  15.443  -7.609  1.00 70.12           C  \nATOM    308  CB  GLU A  44      -5.351  16.444  -6.402  1.00 70.12           C  \nATOM    309  O   GLU A  44      -2.399  15.453  -6.877  1.00 70.12           O  \nATOM    310  CG  GLU A  44      -6.233  17.654  -6.163  1.00 70.12           C  \nATOM    311  CD  GLU A  44      -7.477  17.671  -7.034  1.00 70.12           C  \nATOM    312  OE1 GLU A  44      -8.309  18.598  -6.891  1.00 70.12           O  \nATOM    313  OE2 GLU A  44      -7.626  16.756  -7.871  1.00 70.12           O  \nATOM    314  N   LEU A  45      -3.639  14.506  -8.536  1.00 68.44           N  \nATOM    315  CA  LEU A  45      -2.780  13.329  -8.606  1.00 68.44           C  \nATOM    316  C   LEU A  45      -1.802  13.447  -9.779  1.00 68.44           C  \nATOM    317  CB  LEU A  45      -3.620  12.058  -8.752  1.00 68.44           C  \nATOM    318  O   LEU A  45      -1.048  12.510 -10.051  1.00 68.44           O  \nATOM    319  CG  LEU A  45      -4.454  11.655  -7.534  1.00 68.44           C  \nATOM    320  CD1 LEU A  45      -5.360  10.479  -7.881  1.00 68.44           C  \nATOM    321  CD2 LEU A  45      -3.551  11.320  -6.355  1.00 68.44           C  \nATOM    322  N   ALA A  46      -1.884  14.519 -10.511  1.00 68.75           N  \nATOM    323  CA  ALA A  46      -1.001  14.691 -11.656  1.00 68.75           C  \nATOM    324  C   ALA A  46       0.459  14.725 -11.229  1.00 68.75           C  \nATOM    325  CB  ALA A  46      -1.364  15.968 -12.417  1.00 68.75           C  \nATOM    326  O   ALA A  46       0.815  15.425 -10.281  1.00 68.75           O  \nATOM    327  N   LEU A  47       1.213  13.618 -11.622  1.00 62.34           N  \nATOM    328  CA  LEU A  47       2.649  13.575 -11.380  1.00 62.34           C  \nATOM    329  C   LEU A  47       3.398  14.417 -12.394  1.00 62.34           C  \nATOM    330  CB  LEU A  47       3.157  12.130 -11.422  1.00 62.34           C  \nATOM    331  O   LEU A  47       2.983  14.524 -13.550  1.00 62.34           O  \nATOM    332  CG  LEU A  47       2.693  11.212 -10.289  1.00 62.34           C  \nATOM    333  CD1 LEU A  47       3.165   9.785 -10.538  1.00 62.34           C  \nATOM    334  CD2 LEU A  47       3.211  11.726  -8.943  1.00 62.34           C  \nATOM    335  N   ASP A  48       4.170  15.238 -11.872  1.00 61.00           N  \nATOM    336  CA  ASP A  48       5.216  15.695 -12.774  1.00 61.00           C  \nATOM    337  C   ASP A  48       6.020  14.518 -13.323  1.00 61.00           C  \nATOM    338  CB  ASP A  48       6.143  16.690 -12.073  1.00 61.00           C  \nATOM    339  O   ASP A  48       6.263  13.538 -12.612  1.00 61.00           O  \nATOM    340  CG  ASP A  48       5.548  18.079 -11.964  1.00 61.00           C  \nATOM    341  OD1 ASP A  48       4.787  18.496 -12.878  1.00 61.00           O  \nATOM    342  OD2 ASP A  48       5.846  18.776 -10.969  1.00 61.00           O  \nATOM    343  N   PRO A  49       5.938  14.245 -14.631  1.00 56.16           N  \nATOM    344  CA  PRO A  49       6.720  13.163 -15.235  1.00 56.16           C  \nATOM    345  C   PRO A  49       8.070  12.964 -14.545  1.00 56.16           C  \nATOM    346  CB  PRO A  49       6.911  13.628 -16.686  1.00 56.16           C  \nATOM    347  O   PRO A  49       8.860  13.907 -14.457  1.00 56.16           O  \nATOM    348  CG  PRO A  49       6.171  14.921 -16.771  1.00 56.16           C  \nATOM    349  CD  PRO A  49       5.699  15.295 -15.395  1.00 56.16           C  \nATOM    350  N   VAL A  50       8.115  12.444 -13.372  1.00 59.81           N  \nATOM    351  CA  VAL A  50       9.408  12.111 -12.781  1.00 59.81           C  \nATOM    352  C   VAL A  50      10.011  10.908 -13.492  1.00 59.81           C  \nATOM    353  CB  VAL A  50       9.287  11.828 -11.261  1.00 59.81           C  \nATOM    354  O   VAL A  50       9.299   9.955 -13.822  1.00 59.81           O  \nATOM    355  CG1 VAL A  50      10.642  11.437 -10.679  1.00 59.81           C  \nATOM    356  CG2 VAL A  50       8.725  13.052 -10.535  1.00 59.81           C  \nATOM    357  N   PRO A  51      11.192  11.137 -14.196  1.00 59.00           N  \nATOM    358  CA  PRO A  51      11.855   9.955 -14.751  1.00 59.00           C  \nATOM    359  C   PRO A  51      11.873   8.778 -13.769  1.00 59.00           C  \nATOM    360  CB  PRO A  51      13.276  10.441 -15.039  1.00 59.00           C  \nATOM    361  O   PRO A  51      12.303   8.931 -12.622  1.00 59.00           O  \nATOM    362  CG  PRO A  51      13.209  11.927 -14.860  1.00 59.00           C  \nATOM    363  CD  PRO A  51      11.976  12.258 -14.068  1.00 59.00           C  \nATOM    364  N   GLN A  52      10.922   7.911 -14.006  1.00 70.31           N  \nATOM    365  CA  GLN A  52      10.896   6.704 -13.192  1.00 70.31           C  \nATOM    366  C   GLN A  52      11.920   5.688 -13.670  1.00 70.31           C  \nATOM    367  CB  GLN A  52       9.491   6.086 -13.209  1.00 70.31           C  \nATOM    368  O   GLN A  52      12.100   5.505 -14.879  1.00 70.31           O  \nATOM    369  CG  GLN A  52       8.425   6.958 -12.563  1.00 70.31           C  \nATOM    370  CD  GLN A  52       7.128   6.212 -12.312  1.00 70.31           C  \nATOM    371  NE2 GLN A  52       6.579   6.358 -11.115  1.00 70.31           N  \nATOM    372  OE1 GLN A  52       6.621   5.509 -13.194  1.00 70.31           O  \nATOM    373  N   ASP A  53      12.902   5.313 -12.817  1.00 81.31           N  \nATOM    374  CA  ASP A  53      13.832   4.246 -13.172  1.00 81.31           C  \nATOM    375  C   ASP A  53      13.094   2.964 -13.526  1.00 81.31           C  \nATOM    376  CB  ASP A  53      14.811   3.987 -12.021  1.00 81.31           C  \nATOM    377  O   ASP A  53      11.886   2.851 -13.296  1.00 81.31           O  \nATOM    378  CG  ASP A  53      14.135   3.459 -10.777  1.00 81.31           C  \nATOM    379  OD1 ASP A  53      13.162   2.688 -10.886  1.00 81.31           O  \nATOM    380  OD2 ASP A  53      14.583   3.822  -9.658  1.00 81.31           O  \nATOM    381  N   ALA A  54      13.571   2.139 -14.394  1.00 86.06           N  \nATOM    382  CA  ALA A  54      13.018   0.889 -14.915  1.00 86.06           C  \nATOM    383  C   ALA A  54      12.400   0.056 -13.794  1.00 86.06           C  \nATOM    384  CB  ALA A  54      14.108   0.077 -15.629  1.00 86.06           C  \nATOM    385  O   ALA A  54      11.325  -0.529 -13.966  1.00 86.06           O  \nATOM    386  N   SER A  55      12.975   0.048 -12.622  1.00 87.25           N  \nATOM    387  CA  SER A  55      12.490  -0.753 -11.497  1.00 87.25           C  \nATOM    388  C   SER A  55      11.169  -0.199 -10.963  1.00 87.25           C  \nATOM    389  CB  SER A  55      13.529  -0.777 -10.375  1.00 87.25           C  \nATOM    390  O   SER A  55      10.255  -0.971 -10.659  1.00 87.25           O  \nATOM    391  OG  SER A  55      13.734   0.519  -9.841  1.00 87.25           O  \nATOM    392  N   THR A  56      11.056   1.098 -10.813  1.00 88.94           N  \nATOM    393  CA  THR A  56       9.838   1.732 -10.326  1.00 88.94           C  \nATOM    394  C   THR A  56       8.673   1.482 -11.277  1.00 88.94           C  \nATOM    395  CB  THR A  56      10.029   3.247 -10.129  1.00 88.94           C  \nATOM    396  O   THR A  56       7.539   1.266 -10.844  1.00 88.94           O  \nATOM    397  CG2 THR A  56       8.778   3.892  -9.554  1.00 88.94           C  \nATOM    398  OG1 THR A  56      11.131   3.468  -9.239  1.00 88.94           O  \nATOM    399  N   LYS A  57       8.962   1.452 -12.533  1.00 90.62           N  \nATOM    400  CA  LYS A  57       7.937   1.161 -13.535  1.00 90.62           C  \nATOM    401  C   LYS A  57       7.415  -0.261 -13.392  1.00 90.62           C  \nATOM    402  CB  LYS A  57       8.485   1.386 -14.949  1.00 90.62           C  \nATOM    403  O   LYS A  57       6.208  -0.498 -13.471  1.00 90.62           O  \nATOM    404  CG  LYS A  57       8.634   2.850 -15.336  1.00 90.62           C  \nATOM    405  CD  LYS A  57       9.118   3.001 -16.772  1.00 90.62           C  \nATOM    406  CE  LYS A  57       9.317   4.464 -17.146  1.00 90.62           C  \nATOM    407  NZ  LYS A  57       9.893   4.610 -18.514  1.00 90.62           N  \nATOM    408  N   LYS A  58       8.314  -1.195 -13.218  1.00 92.25           N  \nATOM    409  CA  LYS A  58       7.928  -2.590 -13.021  1.00 92.25           C  \nATOM    410  C   LYS A  58       7.100  -2.755 -11.754  1.00 92.25           C  \nATOM    411  CB  LYS A  58       9.171  -3.492 -12.960  1.00 92.25           C  \nATOM    412  O   LYS A  58       6.120  -3.506 -11.737  1.00 92.25           O  \nATOM    413  CG  LYS A  58       9.849  -3.699 -14.305  1.00 92.25           C  \nATOM    414  CD  LYS A  58      11.051  -4.612 -14.185  1.00 92.25           C  \nATOM    415  CE  LYS A  58      11.779  -4.764 -15.516  1.00 92.25           C  \nATOM    416  NZ  LYS A  58      13.008  -5.617 -15.385  1.00 92.25           N  \nATOM    417  N   LEU A  59       7.513  -2.070 -10.776  1.00 93.38           N  \nATOM    418  CA  LEU A  59       6.770  -2.108  -9.523  1.00 93.38           C  \nATOM    419  C   LEU A  59       5.358  -1.560  -9.707  1.00 93.38           C  \nATOM    420  CB  LEU A  59       7.501  -1.303  -8.443  1.00 93.38           C  \nATOM    421  O   LEU A  59       4.390  -2.149  -9.219  1.00 93.38           O  \nATOM    422  CG  LEU A  59       8.754  -1.951  -7.845  1.00 93.38           C  \nATOM    423  CD1 LEU A  59       9.582  -0.902  -7.099  1.00 93.38           C  \nATOM    424  CD2 LEU A  59       8.370  -3.097  -6.919  1.00 93.38           C  \nATOM    425  N   SER A  60       5.285  -0.482 -10.389  1.00 92.69           N  \nATOM    426  CA  SER A  60       3.982   0.133 -10.642  1.00 92.69           C  \nATOM    427  C   SER A  60       3.072  -0.809 -11.424  1.00 92.69           C  \nATOM    428  CB  SER A  60       4.153   1.443 -11.400  1.00 92.69           C  \nATOM    429  O   SER A  60       1.885  -0.931 -11.113  1.00 92.69           O  \nATOM    430  OG  SER A  60       4.785   2.420 -10.593  1.00 92.69           O  \nATOM    431  N   GLU A  61       3.627  -1.407 -12.412  1.00 92.06           N  \nATOM    432  CA  GLU A  61       2.864  -2.356 -13.218  1.00 92.06           C  \nATOM    433  C   GLU A  61       2.392  -3.539 -12.378  1.00 92.06           C  \nATOM    434  CB  GLU A  61       3.696  -2.854 -14.400  1.00 92.06           C  \nATOM    435  O   GLU A  61       1.260  -4.005 -12.528  1.00 92.06           O  \nATOM    436  CG  GLU A  61       3.892  -1.811 -15.498  1.00 92.06           C  \nATOM    437  CD  GLU A  61       4.829  -2.268 -16.598  1.00 92.06           C  \nATOM    438  OE1 GLU A  61       5.043  -1.507 -17.567  1.00 92.06           O  \nATOM    439  OE2 GLU A  61       5.356  -3.398 -16.491  1.00 92.06           O  \nATOM    440  N   CYS A  62       3.302  -3.974 -11.562  1.00 92.38           N  \nATOM    441  CA  CYS A  62       2.958  -5.080 -10.671  1.00 92.38           C  \nATOM    442  C   CYS A  62       1.801  -4.700  -9.756  1.00 92.38           C  \nATOM    443  CB  CYS A  62       4.169  -5.496  -9.838  1.00 92.38           C  \nATOM    444  O   CYS A  62       0.836  -5.452  -9.618  1.00 92.38           O  \nATOM    445  SG  CYS A  62       3.850  -6.897  -8.746  1.00 92.38           S  \nATOM    446  N   LEU A  63       1.814  -3.575  -9.130  1.00 93.06           N  \nATOM    447  CA  LEU A  63       0.760  -3.091  -8.246  1.00 93.06           C  \nATOM    448  C   LEU A  63      -0.554  -2.929  -9.006  1.00 93.06           C  \nATOM    449  CB  LEU A  63       1.163  -1.756  -7.615  1.00 93.06           C  \nATOM    450  O   LEU A  63      -1.624  -3.229  -8.470  1.00 93.06           O  \nATOM    451  CG  LEU A  63       2.121  -1.833  -6.424  1.00 93.06           C  \nATOM    452  CD1 LEU A  63       2.563  -0.434  -6.010  1.00 93.06           C  \nATOM    453  CD2 LEU A  63       1.467  -2.561  -5.256  1.00 93.06           C  \nATOM    454  N   LYS A  64      -0.398  -2.485 -10.228  1.00 92.56           N  \nATOM    455  CA  LYS A  64      -1.588  -2.330 -11.064  1.00 92.56           C  \nATOM    456  C   LYS A  64      -2.274  -3.676 -11.289  1.00 92.56           C  \nATOM    457  CB  LYS A  64      -1.228  -1.692 -12.399  1.00 92.56           C  \nATOM    458  O   LYS A  64      -3.499  -3.778 -11.182  1.00 92.56           O  \nATOM    459  CG  LYS A  64      -0.869  -0.214 -12.306  1.00 92.56           C  \nATOM    460  CD  LYS A  64      -0.519   0.365 -13.670  1.00 92.56           C  \nATOM    461  CE  LYS A  64      -0.216   1.855 -13.585  1.00 92.56           C  \nATOM    462  NZ  LYS A  64       0.436   2.357 -14.831  1.00 92.56           N  \nATOM    463  N   ARG A  65      -1.542  -4.638 -11.594  1.00 91.69           N  \nATOM    464  CA  ARG A  65      -2.074  -5.977 -11.837  1.00 91.69           C  \nATOM    465  C   ARG A  65      -2.753  -6.531 -10.591  1.00 91.69           C  \nATOM    466  CB  ARG A  65      -0.960  -6.919 -12.295  1.00 91.69           C  \nATOM    467  O   ARG A  65      -3.858  -7.075 -10.666  1.00 91.69           O  \nATOM    468  CG  ARG A  65      -1.461  -8.276 -12.770  1.00 91.69           C  \nATOM    469  CD  ARG A  65      -0.324  -9.152 -13.266  1.00 91.69           C  \nATOM    470  NE  ARG A  65      -0.780 -10.498 -13.600  1.00 91.69           N  \nATOM    471  NH1 ARG A  65       1.267 -11.239 -14.372  1.00 91.69           N  \nATOM    472  NH2 ARG A  65      -0.547 -12.639 -14.374  1.00 91.69           N  \nATOM    473  CZ  ARG A  65      -0.020 -11.457 -14.118  1.00 91.69           C  \nATOM    474  N   ILE A  66      -2.094  -6.358  -9.492  1.00 91.06           N  \nATOM    475  CA  ILE A  66      -2.646  -6.827  -8.227  1.00 91.06           C  \nATOM    476  C   ILE A  66      -3.942  -6.086  -7.918  1.00 91.06           C  \nATOM    477  CB  ILE A  66      -1.636  -6.658  -7.072  1.00 91.06           C  \nATOM    478  O   ILE A  66      -4.934  -6.696  -7.506  1.00 91.06           O  \nATOM    479  CG1 ILE A  66      -0.414  -7.548  -7.290  1.00 91.06           C  \nATOM    480  CG2 ILE A  66      -2.301  -6.961  -5.726  1.00 91.06           C  \nATOM    481  CD1 ILE A  66       0.743  -7.258  -6.343  1.00 91.06           C  \nATOM    482  N   GLY A  67      -3.868  -4.845  -8.074  1.00 92.25           N  \nATOM    483  CA  GLY A  67      -5.057  -4.035  -7.874  1.00 92.25           C  \nATOM    484  C   GLY A  67      -6.234  -4.482  -8.717  1.00 92.25           C  \nATOM    485  O   GLY A  67      -7.367  -4.530  -8.233  1.00 92.25           O  \nATOM    486  N   ASP A  68      -5.982  -4.822 -10.012  1.00 92.19           N  \nATOM    487  CA  ASP A  68      -7.030  -5.301 -10.910  1.00 92.19           C  \nATOM    488  C   ASP A  68      -7.648  -6.594 -10.389  1.00 92.19           C  \nATOM    489  CB  ASP A  68      -6.476  -5.516 -12.318  1.00 92.19           C  \nATOM    490  O   ASP A  68      -8.869  -6.774 -10.451  1.00 92.19           O  \nATOM    491  CG  ASP A  68      -6.147  -4.209 -13.031  1.00 92.19           C  \nATOM    492  OD1 ASP A  68      -6.597  -3.135 -12.576  1.00 92.19           O  \nATOM    493  OD2 ASP A  68      -5.436  -4.259 -14.061  1.00 92.19           O  \nATOM    494  N   GLU A  69      -6.822  -7.399  -9.850  1.00 91.06           N  \nATOM    495  CA  GLU A  69      -7.300  -8.659  -9.288  1.00 91.06           C  \nATOM    496  C   GLU A  69      -8.156  -8.422  -8.050  1.00 91.06           C  \nATOM    497  CB  GLU A  69      -6.128  -9.577  -8.948  1.00 91.06           C  \nATOM    498  O   GLU A  69      -9.203  -9.049  -7.881  1.00 91.06           O  \nATOM    499  CG  GLU A  69      -5.444 -10.188 -10.170  1.00 91.06           C  \nATOM    500  CD  GLU A  69      -4.260 -11.074  -9.816  1.00 91.06           C  \nATOM    501  OE1 GLU A  69      -3.551 -11.537 -10.739  1.00 91.06           O  \nATOM    502  OE2 GLU A  69      -4.037 -11.303  -8.606  1.00 91.06           O  \nATOM    503  N   LEU A  70      -7.708  -7.589  -7.209  1.00 91.50           N  \nATOM    504  CA  LEU A  70      -8.458  -7.272  -5.997  1.00 91.50           C  \nATOM    505  C   LEU A  70      -9.795  -6.619  -6.338  1.00 91.50           C  \nATOM    506  CB  LEU A  70      -7.642  -6.348  -5.089  1.00 91.50           C  \nATOM    507  O   LEU A  70     -10.804  -6.882  -5.682  1.00 91.50           O  \nATOM    508  CG  LEU A  70      -6.423  -6.974  -4.407  1.00 91.50           C  \nATOM    509  CD1 LEU A  70      -5.588  -5.897  -3.727  1.00 91.50           C  \nATOM    510  CD2 LEU A  70      -6.862  -8.035  -3.401  1.00 91.50           C  \nATOM    511  N   ASP A  71      -9.777  -5.768  -7.369  1.00 92.00           N  \nATOM    512  CA  ASP A  71     -10.997  -5.094  -7.803  1.00 92.00           C  \nATOM    513  C   ASP A  71     -12.023  -6.098  -8.319  1.00 92.00           C  \nATOM    514  CB  ASP A  71     -10.685  -4.059  -8.884  1.00 92.00           C  \nATOM    515  O   ASP A  71     -13.227  -5.824  -8.315  1.00 92.00           O  \nATOM    516  CG  ASP A  71     -10.037  -2.804  -8.333  1.00 92.00           C  \nATOM    517  OD1 ASP A  71     -10.121  -2.552  -7.112  1.00 92.00           O  \nATOM    518  OD2 ASP A  71      -9.430  -2.048  -9.131  1.00 92.00           O  \nATOM    519  N   SER A  72     -11.577  -7.277  -8.792  1.00 91.88           N  \nATOM    520  CA  SER A  72     -12.476  -8.289  -9.332  1.00 91.88           C  \nATOM    521  C   SER A  72     -13.165  -9.073  -8.216  1.00 91.88           C  \nATOM    522  CB  SER A  72     -11.717  -9.251 -10.242  1.00 91.88           C  \nATOM    523  O   SER A  72     -14.110  -9.817  -8.465  1.00 91.88           O  \nATOM    524  OG  SER A  72     -10.867 -10.099  -9.489  1.00 91.88           O  \nATOM    525  N   ASN A  73     -12.636  -8.889  -6.967  1.00 89.81           N  \nATOM    526  CA  ASN A  73     -13.236  -9.538  -5.806  1.00 89.81           C  \nATOM    527  C   ASN A  73     -14.479  -8.796  -5.328  1.00 89.81           C  \nATOM    528  CB  ASN A  73     -12.217  -9.652  -4.669  1.00 89.81           C  \nATOM    529  O   ASN A  73     -14.377  -7.726  -4.725  1.00 89.81           O  \nATOM    530  CG  ASN A  73     -12.692 -10.551  -3.547  1.00 89.81           C  \nATOM    531  ND2 ASN A  73     -11.836 -11.471  -3.116  1.00 89.81           N  \nATOM    532  OD1 ASN A  73     -13.827 -10.428  -3.072  1.00 89.81           O  \nATOM    533  N   MET A  74     -15.700  -9.355  -5.531  1.00 90.69           N  \nATOM    534  CA  MET A  74     -16.982  -8.697  -5.280  1.00 90.69           C  \nATOM    535  C   MET A  74     -17.191  -8.481  -3.783  1.00 90.69           C  \nATOM    536  CB  MET A  74     -18.132  -9.511  -5.863  1.00 90.69           C  \nATOM    537  O   MET A  74     -17.724  -7.449  -3.368  1.00 90.69           O  \nATOM    538  CG  MET A  74     -18.123  -9.572  -7.381  1.00 90.69           C  \nATOM    539  SD  MET A  74     -18.324  -7.927  -8.157  1.00 90.69           S  \nATOM    540  CE  MET A  74     -20.052  -7.577  -7.755  1.00 90.69           C  \nATOM    541  N   GLU A  75     -16.772  -9.474  -3.042  1.00 88.75           N  \nATOM    542  CA  GLU A  75     -16.895  -9.365  -1.590  1.00 88.75           C  \nATOM    543  C   GLU A  75     -16.070  -8.206  -1.050  1.00 88.75           C  \nATOM    544  CB  GLU A  75     -16.473 -10.673  -0.916  1.00 88.75           C  \nATOM    545  O   GLU A  75     -16.535  -7.451  -0.193  1.00 88.75           O  \nATOM    546  CG  GLU A  75     -16.595 -10.648   0.601  1.00 88.75           C  \nATOM    547  CD  GLU A  75     -18.033 -10.464   1.081  1.00 88.75           C  \nATOM    548  OE1 GLU A  75     -18.234 -10.238   2.296  1.00 88.75           O  \nATOM    549  OE2 GLU A  75     -18.950 -10.537   0.239  1.00 88.75           O  \nATOM    550  N   LEU A  76     -14.871  -8.134  -1.516  1.00 88.19           N  \nATOM    551  CA  LEU A  76     -14.009  -7.029  -1.117  1.00 88.19           C  \nATOM    552  C   LEU A  76     -14.627  -5.688  -1.500  1.00 88.19           C  \nATOM    553  CB  LEU A  76     -12.627  -7.169  -1.760  1.00 88.19           C  \nATOM    554  O   LEU A  76     -14.664  -4.762  -0.689  1.00 88.19           O  \nATOM    555  CG  LEU A  76     -11.618  -6.065  -1.447  1.00 88.19           C  \nATOM    556  CD1 LEU A  76     -11.372  -5.987   0.055  1.00 88.19           C  \nATOM    557  CD2 LEU A  76     -10.314  -6.302  -2.199  1.00 88.19           C  \nATOM    558  N   GLN A  77     -15.179  -5.662  -2.701  1.00 90.38           N  \nATOM    559  CA  GLN A  77     -15.808  -4.429  -3.163  1.00 90.38           C  \nATOM    560  C   GLN A  77     -17.003  -4.059  -2.288  1.00 90.38           C  \nATOM    561  CB  GLN A  77     -16.248  -4.565  -4.623  1.00 90.38           C  \nATOM    562  O   GLN A  77     -17.200  -2.888  -1.962  1.00 90.38           O  \nATOM    563  CG  GLN A  77     -15.095  -4.753  -5.597  1.00 90.38           C  \nATOM    564  CD  GLN A  77     -14.069  -3.643  -5.511  1.00 90.38           C  \nATOM    565  NE2 GLN A  77     -12.801  -4.016  -5.358  1.00 90.38           N  \nATOM    566  OE1 GLN A  77     -14.408  -2.457  -5.583  1.00 90.38           O  \nATOM    567  N   ARG A  78     -17.747  -4.989  -1.951  1.00 91.12           N  \nATOM    568  CA  ARG A  78     -18.905  -4.780  -1.089  1.00 91.12           C  \nATOM    569  C   ARG A  78     -18.489  -4.236   0.270  1.00 91.12           C  \nATOM    570  CB  ARG A  78     -19.680  -6.084  -0.912  1.00 91.12           C  \nATOM    571  O   ARG A  78     -19.111  -3.312   0.796  1.00 91.12           O  \nATOM    572  CG  ARG A  78     -20.998  -5.923  -0.163  1.00 91.12           C  \nATOM    573  CD  ARG A  78     -21.687  -7.258   0.068  1.00 91.12           C  \nATOM    574  NE  ARG A  78     -20.806  -8.215   0.728  1.00 91.12           N  \nATOM    575  NH1 ARG A  78     -22.505  -9.633   1.400  1.00 91.12           N  \nATOM    576  NH2 ARG A  78     -20.321 -10.117   1.920  1.00 91.12           N  \nATOM    577  CZ  ARG A  78     -21.211  -9.320   1.348  1.00 91.12           C  \nATOM    578  N   MET A  79     -17.438  -4.771   0.836  1.00 86.56           N  \nATOM    579  CA  MET A  79     -16.943  -4.349   2.142  1.00 86.56           C  \nATOM    580  C   MET A  79     -16.415  -2.917   2.086  1.00 86.56           C  \nATOM    581  CB  MET A  79     -15.844  -5.290   2.636  1.00 86.56           C  \nATOM    582  O   MET A  79     -16.640  -2.125   3.006  1.00 86.56           O  \nATOM    583  CG  MET A  79     -16.353  -6.674   3.013  1.00 86.56           C  \nATOM    584  SD  MET A  79     -17.438  -6.645   4.486  1.00 86.56           S  \nATOM    585  CE  MET A  79     -16.261  -6.094   5.746  1.00 86.56           C  \nATOM    586  N   ILE A  80     -15.715  -2.645   1.063  1.00 88.50           N  \nATOM    587  CA  ILE A  80     -15.177  -1.295   0.886  1.00 88.50           C  \nATOM    588  C   ILE A  80     -16.329  -0.295   0.779  1.00 88.50           C  \nATOM    589  CB  ILE A  80     -14.273  -1.204  -0.363  1.00 88.50           C  \nATOM    590  O   ILE A  80     -16.266   0.798   1.356  1.00 88.50           O  \nATOM    591  CG1 ILE A  80     -12.980  -1.999  -0.143  1.00 88.50           C  \nATOM    592  CG2 ILE A  80     -13.971   0.256  -0.708  1.00 88.50           C  \nATOM    593  CD1 ILE A  80     -12.078  -1.422   0.941  1.00 88.50           C  \nATOM    594  N   ALA A  81     -17.371  -0.652   0.054  1.00 88.50           N  \nATOM    595  CA  ALA A  81     -18.528   0.219  -0.141  1.00 88.50           C  \nATOM    596  C   ALA A  81     -19.225   0.502   1.185  1.00 88.50           C  \nATOM    597  CB  ALA A  81     -19.501  -0.403  -1.133  1.00 88.50           C  \nATOM    598  O   ALA A  81     -19.835   1.556   1.355  1.00 88.50           O  \nATOM    599  N   ALA A  82     -19.087  -0.381   2.108  1.00 87.25           N  \nATOM    600  CA  ALA A  82     -19.784  -0.274   3.386  1.00 87.25           C  \nATOM    601  C   ALA A  82     -18.987   0.572   4.377  1.00 87.25           C  \nATOM    602  CB  ALA A  82     -20.042  -1.661   3.965  1.00 87.25           C  \nATOM    603  O   ALA A  82     -19.473   0.877   5.471  1.00 87.25           O  \nATOM    604  N   VAL A  83     -17.743   0.849   4.056  1.00 84.75           N  \nATOM    605  CA  VAL A  83     -16.873   1.617   4.941  1.00 84.75           C  \nATOM    606  C   VAL A  83     -17.421   3.034   5.100  1.00 84.75           C  \nATOM    607  CB  VAL A  83     -15.421   1.664   4.414  1.00 84.75           C  \nATOM    608  O   VAL A  83     -17.726   3.703   4.110  1.00 84.75           O  \nATOM    609  CG1 VAL A  83     -14.581   2.628   5.245  1.00 84.75           C  \nATOM    610  CG2 VAL A  83     -14.805   0.263   4.422  1.00 84.75           C  \nATOM    611  N   ASP A  84     -17.667   3.418   6.366  1.00 79.75           N  \nATOM    612  CA  ASP A  84     -18.150   4.748   6.728  1.00 79.75           C  \nATOM    613  C   ASP A  84     -17.089   5.812   6.444  1.00 79.75           C  \nATOM    614  CB  ASP A  84     -18.553   4.789   8.202  1.00 79.75           C  \nATOM    615  O   ASP A  84     -15.965   5.724   6.937  1.00 79.75           O  \nATOM    616  CG  ASP A  84     -19.241   6.095   8.596  1.00 79.75           C  \nATOM    617  OD1 ASP A  84     -19.474   6.951   7.716  1.00 79.75           O  \nATOM    618  OD2 ASP A  84     -19.537   6.262   9.796  1.00 79.75           O  \nATOM    619  N   THR A  85     -17.523   6.857   5.716  1.00 75.62           N  \nATOM    620  CA  THR A  85     -16.573   7.893   5.321  1.00 75.62           C  \nATOM    621  C   THR A  85     -16.720   9.128   6.217  1.00 75.62           C  \nATOM    622  CB  THR A  85     -16.771   8.297   3.848  1.00 75.62           C  \nATOM    623  O   THR A  85     -16.106  10.162   5.953  1.00 75.62           O  \nATOM    624  CG2 THR A  85     -16.411   7.155   2.911  1.00 75.62           C  \nATOM    625  OG1 THR A  85     -18.137   8.655   3.635  1.00 75.62           O  \nATOM    626  N   ASP A  86     -17.569   9.016   7.251  1.00 80.19           N  \nATOM    627  CA  ASP A  86     -17.756  10.147   8.149  1.00 80.19           C  \nATOM    628  C   ASP A  86     -16.543  10.352   9.054  1.00 80.19           C  \nATOM    629  CB  ASP A  86     -19.010   9.955   8.998  1.00 80.19           C  \nATOM    630  O   ASP A  86     -16.258  11.458   9.491  1.00 80.19           O  \nATOM    631  CG  ASP A  86     -20.296  10.116   8.206  1.00 80.19           C  \nATOM    632  OD1 ASP A  86     -20.288  10.786   7.151  1.00 80.19           O  \nATOM    633  OD2 ASP A  86     -21.341   9.578   8.648  1.00 80.19           O  \nATOM    634  N   SER A  87     -15.810   9.281   9.226  1.00 84.81           N  \nATOM    635  CA  SER A  87     -14.592   9.352  10.022  1.00 84.81           C  \nATOM    636  C   SER A  87     -13.451   8.588   9.340  1.00 84.81           C  \nATOM    637  CB  SER A  87     -14.826   8.783  11.421  1.00 84.81           C  \nATOM    638  O   SER A  87     -13.038   7.528   9.815  1.00 84.81           O  \nATOM    639  OG  SER A  87     -15.432   7.505  11.350  1.00 84.81           O  \nATOM    640  N   PRO A  88     -12.967   9.081   8.183  1.00 85.31           N  \nATOM    641  CA  PRO A  88     -11.998   8.371   7.348  1.00 85.31           C  \nATOM    642  C   PRO A  88     -10.674   8.118   8.066  1.00 85.31           C  \nATOM    643  CB  PRO A  88     -11.804   9.312   6.157  1.00 85.31           C  \nATOM    644  O   PRO A  88     -10.059   7.064   7.883  1.00 85.31           O  \nATOM    645  CG  PRO A  88     -12.271  10.641   6.638  1.00 85.31           C  \nATOM    646  CD  PRO A  88     -13.342  10.422   7.666  1.00 85.31           C  \nATOM    647  N   ARG A  89     -10.321   9.068   8.908  1.00 89.12           N  \nATOM    648  CA  ARG A  89      -9.060   8.919   9.627  1.00 89.12           C  \nATOM    649  C   ARG A  89      -9.116   7.737  10.588  1.00 89.12           C  \nATOM    650  CB  ARG A  89      -8.724  10.200  10.395  1.00 89.12           C  \nATOM    651  O   ARG A  89      -8.209   6.902  10.608  1.00 89.12           O  \nATOM    652  CG  ARG A  89      -7.384  10.150  11.117  1.00 89.12           C  \nATOM    653  CD  ARG A  89      -7.051  11.482  11.775  1.00 89.12           C  \nATOM    654  NE  ARG A  89      -5.742  11.450  12.421  1.00 89.12           N  \nATOM    655  NH1 ARG A  89      -6.222  13.110  13.945  1.00 89.12           N  \nATOM    656  NH2 ARG A  89      -4.165  12.112  13.953  1.00 89.12           N  \nATOM    657  CZ  ARG A  89      -5.380  12.225  13.440  1.00 89.12           C  \nATOM    658  N   GLU A  90     -10.098   7.749  11.409  1.00 89.38           N  \nATOM    659  CA  GLU A  90     -10.261   6.671  12.381  1.00 89.38           C  \nATOM    660  C   GLU A  90     -10.361   5.313  11.683  1.00 89.38           C  \nATOM    661  CB  GLU A  90     -11.494   6.910  13.253  1.00 89.38           C  \nATOM    662  O   GLU A  90      -9.730   4.341  12.112  1.00 89.38           O  \nATOM    663  CG  GLU A  90     -11.660   5.898  14.381  1.00 89.38           C  \nATOM    664  CD  GLU A  90     -12.896   6.148  15.228  1.00 89.38           C  \nATOM    665  OE1 GLU A  90     -13.238   5.279  16.066  1.00 89.38           O  \nATOM    666  OE2 GLU A  90     -13.528   7.214  15.062  1.00 89.38           O  \nATOM    667  N   VAL A  91     -11.106   5.325  10.624  1.00 89.50           N  \nATOM    668  CA  VAL A  91     -11.306   4.080   9.896  1.00 89.50           C  \nATOM    669  C   VAL A  91      -9.973   3.597   9.317  1.00 89.50           C  \nATOM    670  CB  VAL A  91     -12.350   4.236   8.771  1.00 89.50           C  \nATOM    671  O   VAL A  91      -9.617   2.427   9.455  1.00 89.50           O  \nATOM    672  CG1 VAL A  91     -12.395   2.984   7.893  1.00 89.50           C  \nATOM    673  CG2 VAL A  91     -13.728   4.530   9.358  1.00 89.50           C  \nATOM    674  N   PHE A  92      -9.260   4.513   8.724  1.00 92.50           N  \nATOM    675  CA  PHE A  92      -7.986   4.147   8.108  1.00 92.50           C  \nATOM    676  C   PHE A  92      -7.048   3.523   9.133  1.00 92.50           C  \nATOM    677  CB  PHE A  92      -7.326   5.372   7.469  1.00 92.50           C  \nATOM    678  O   PHE A  92      -6.530   2.425   8.924  1.00 92.50           O  \nATOM    679  CG  PHE A  92      -5.965   5.099   6.898  1.00 92.50           C  \nATOM    680  CD1 PHE A  92      -4.818   5.287   7.661  1.00 92.50           C  \nATOM    681  CD2 PHE A  92      -5.824   4.644   5.589  1.00 92.50           C  \nATOM    682  CE1 PHE A  92      -3.558   5.033   7.130  1.00 92.50           C  \nATOM    683  CE2 PHE A  92      -4.567   4.391   5.054  1.00 92.50           C  \nATOM    684  CZ  PHE A  92      -3.433   4.585   5.826  1.00 92.50           C  \nATOM    685  N   PHE A  93      -6.776   4.185  10.253  1.00 92.44           N  \nATOM    686  CA  PHE A  93      -5.768   3.750  11.210  1.00 92.44           C  \nATOM    687  C   PHE A  93      -6.227   2.495  11.943  1.00 92.44           C  \nATOM    688  CB  PHE A  93      -5.466   4.861  12.221  1.00 92.44           C  \nATOM    689  O   PHE A  93      -5.408   1.638  12.293  1.00 92.44           O  \nATOM    690  CG  PHE A  93      -4.508   5.904  11.708  1.00 92.44           C  \nATOM    691  CD1 PHE A  93      -3.145   5.624  11.611  1.00 92.44           C  \nATOM    692  CD2 PHE A  93      -4.965   7.155  11.322  1.00 92.44           C  \nATOM    693  CE1 PHE A  93      -2.256   6.586  11.139  1.00 92.44           C  \nATOM    694  CE2 PHE A  93      -4.076   8.119  10.846  1.00 92.44           C  \nATOM    695  CZ  PHE A  93      -2.726   7.834  10.757  1.00 92.44           C  \nATOM    696  N   ARG A  94      -7.490   2.353  12.101  1.00 90.75           N  \nATOM    697  CA  ARG A  94      -7.998   1.146  12.739  1.00 90.75           C  \nATOM    698  C   ARG A  94      -7.791  -0.070  11.840  1.00 90.75           C  \nATOM    699  CB  ARG A  94      -9.476   1.292  13.090  1.00 90.75           C  \nATOM    700  O   ARG A  94      -7.324  -1.116  12.303  1.00 90.75           O  \nATOM    701  CG  ARG A  94     -10.009   0.197  13.999  1.00 90.75           C  \nATOM    702  CD  ARG A  94     -11.442   0.471  14.428  1.00 90.75           C  \nATOM    703  NE  ARG A  94     -12.300  -0.692  14.233  1.00 90.75           N  \nATOM    704  NH1 ARG A  94     -14.204   0.297  15.077  1.00 90.75           N  \nATOM    705  NH2 ARG A  94     -14.272  -1.863  14.321  1.00 90.75           N  \nATOM    706  CZ  ARG A  94     -13.593  -0.750  14.551  1.00 90.75           C  \nATOM    707  N   VAL A  95      -8.188   0.086  10.628  1.00 91.75           N  \nATOM    708  CA  VAL A  95      -8.025  -1.018   9.683  1.00 91.75           C  \nATOM    709  C   VAL A  95      -6.545  -1.342   9.514  1.00 91.75           C  \nATOM    710  CB  VAL A  95      -8.659  -0.685   8.315  1.00 91.75           C  \nATOM    711  O   VAL A  95      -6.153  -2.512   9.544  1.00 91.75           O  \nATOM    712  CG1 VAL A  95      -8.276  -1.737   7.275  1.00 91.75           C  \nATOM    713  CG2 VAL A  95     -10.178  -0.583   8.441  1.00 91.75           C  \nATOM    714  N   ALA A  96      -5.682  -0.318   9.357  1.00 93.00           N  \nATOM    715  CA  ALA A  96      -4.246  -0.510   9.198  1.00 93.00           C  \nATOM    716  C   ALA A  96      -3.651  -1.206  10.425  1.00 93.00           C  \nATOM    717  CB  ALA A  96      -3.549   0.825   8.959  1.00 93.00           C  \nATOM    718  O   ALA A  96      -2.841  -2.126  10.288  1.00 93.00           O  \nATOM    719  N   ALA A  97      -4.055  -0.766  11.626  1.00 92.25           N  \nATOM    720  CA  ALA A  97      -3.560  -1.366  12.867  1.00 92.25           C  \nATOM    721  C   ALA A  97      -3.955  -2.837  12.966  1.00 92.25           C  \nATOM    722  CB  ALA A  97      -4.090  -0.593  14.073  1.00 92.25           C  \nATOM    723  O   ALA A  97      -3.155  -3.673  13.382  1.00 92.25           O  \nATOM    724  N   ASP A  98      -5.142  -3.088  12.543  1.00 91.44           N  \nATOM    725  CA  ASP A  98      -5.633  -4.463  12.603  1.00 91.44           C  \nATOM    726  C   ASP A  98      -4.880  -5.359  11.622  1.00 91.44           C  \nATOM    727  CB  ASP A  98      -7.132  -4.509  12.313  1.00 91.44           C  \nATOM    728  O   ASP A  98      -4.591  -6.520  11.929  1.00 91.44           O  \nATOM    729  CG  ASP A  98      -7.733  -5.893  12.501  1.00 91.44           C  \nATOM    730  OD1 ASP A  98      -7.766  -6.394  13.647  1.00 91.44           O  \nATOM    731  OD2 ASP A  98      -8.172  -6.486  11.493  1.00 91.44           O  \nATOM    732  N   MET A  99      -4.570  -4.877  10.466  1.00 91.88           N  \nATOM    733  CA  MET A  99      -3.863  -5.642   9.441  1.00 91.88           C  \nATOM    734  C   MET A  99      -2.518  -6.139   9.971  1.00 91.88           C  \nATOM    735  CB  MET A  99      -3.652  -4.800   8.188  1.00 91.88           C  \nATOM    736  O   MET A  99      -2.035  -7.191   9.550  1.00 91.88           O  \nATOM    737  CG  MET A  99      -4.912  -4.613   7.353  1.00 91.88           C  \nATOM    738  SD  MET A  99      -4.629  -3.562   5.878  1.00 91.88           S  \nATOM    739  CE  MET A  99      -4.363  -4.842   4.619  1.00 91.88           C  \nATOM    740  N   PHE A 100      -2.011  -5.341  10.883  1.00 92.38           N  \nATOM    741  CA  PHE A 100      -0.654  -5.641  11.327  1.00 92.38           C  \nATOM    742  C   PHE A 100      -0.631  -5.953  12.818  1.00 92.38           C  \nATOM    743  CB  PHE A 100       0.285  -4.470  11.017  1.00 92.38           C  \nATOM    744  O   PHE A 100       0.404  -5.805  13.475  1.00 92.38           O  \nATOM    745  CG  PHE A 100       0.458  -4.201   9.546  1.00 92.38           C  \nATOM    746  CD1 PHE A 100       1.466  -4.831   8.827  1.00 92.38           C  \nATOM    747  CD2 PHE A 100      -0.388  -3.318   8.885  1.00 92.38           C  \nATOM    748  CE1 PHE A 100       1.626  -4.580   7.462  1.00 92.38           C  \nATOM    749  CE2 PHE A 100      -0.231  -3.068   7.531  1.00 92.38           C  \nATOM    750  CZ  PHE A 100       0.777  -3.699   6.818  1.00 92.38           C  \nATOM    751  N   SER A 101      -1.720  -6.330  13.413  1.00 90.69           N  \nATOM    752  CA  SER A 101      -1.877  -6.490  14.859  1.00 90.69           C  \nATOM    753  C   SER A 101      -1.104  -7.702  15.368  1.00 90.69           C  \nATOM    754  CB  SER A 101      -3.355  -6.626  15.224  1.00 90.69           C  \nATOM    755  O   SER A 101      -0.708  -7.748  16.539  1.00 90.69           O  \nATOM    756  OG  SER A 101      -3.922  -7.770  14.608  1.00 90.69           O  \nATOM    757  N   ASP A 102      -0.856  -8.682  14.425  1.00 90.31           N  \nATOM    758  CA  ASP A 102      -0.201  -9.908  14.868  1.00 90.31           C  \nATOM    759  C   ASP A 102       1.319  -9.764  14.819  1.00 90.31           C  \nATOM    760  CB  ASP A 102      -0.645 -11.095  14.007  1.00 90.31           C  \nATOM    761  O   ASP A 102       2.043 -10.695  15.177  1.00 90.31           O  \nATOM    762  CG  ASP A 102      -0.321 -10.917  12.539  1.00 90.31           C  \nATOM    763  OD1 ASP A 102       0.337  -9.912  12.170  1.00 90.31           O  \nATOM    764  OD2 ASP A 102      -0.736 -11.770  11.722  1.00 90.31           O  \nATOM    765  N   GLY A 103       1.762  -8.596  14.404  1.00 88.00           N  \nATOM    766  CA  GLY A 103       3.192  -8.329  14.408  1.00 88.00           C  \nATOM    767  C   GLY A 103       3.900  -8.880  13.184  1.00 88.00           C  \nATOM    768  O   GLY A 103       5.126  -8.771  13.074  1.00 88.00           O  \nATOM    769  N   ASN A 104       3.179  -9.502  12.284  1.00 90.62           N  \nATOM    770  CA  ASN A 104       3.757 -10.045  11.059  1.00 90.62           C  \nATOM    771  C   ASN A 104       3.689  -9.040   9.913  1.00 90.62           C  \nATOM    772  CB  ASN A 104       3.057 -11.348  10.665  1.00 90.62           C  \nATOM    773  O   ASN A 104       2.670  -8.372   9.732  1.00 90.62           O  \nATOM    774  CG  ASN A 104       3.320 -12.471  11.643  1.00 90.62           C  \nATOM    775  ND2 ASN A 104       2.363 -13.384  11.778  1.00 90.62           N  \nATOM    776  OD1 ASN A 104       4.375 -12.526  12.293  1.00 90.62           O  \nATOM    777  N   PHE A 105       4.869  -8.918   9.260  1.00 92.31           N  \nATOM    778  CA  PHE A 105       4.951  -8.036   8.101  1.00 92.31           C  \nATOM    779  C   PHE A 105       5.228  -8.829   6.829  1.00 92.31           C  \nATOM    780  CB  PHE A 105       6.040  -6.978   8.308  1.00 92.31           C  \nATOM    781  O   PHE A 105       6.129  -9.678   6.807  1.00 92.31           O  \nATOM    782  CG  PHE A 105       5.837  -6.128   9.532  1.00 92.31           C  \nATOM    783  CD1 PHE A 105       5.130  -4.934   9.457  1.00 92.31           C  \nATOM    784  CD2 PHE A 105       6.351  -6.524  10.760  1.00 92.31           C  \nATOM    785  CE1 PHE A 105       4.938  -4.144  10.588  1.00 92.31           C  \nATOM    786  CE2 PHE A 105       6.168  -5.741  11.896  1.00 92.31           C  \nATOM    787  CZ  PHE A 105       5.458  -4.553  11.808  1.00 92.31           C  \nATOM    788  N   ASN A 106       4.428  -8.629   5.782  1.00 93.00           N  \nATOM    789  CA  ASN A 106       4.659  -9.213   4.462  1.00 93.00           C  \nATOM    790  C   ASN A 106       4.133  -8.302   3.351  1.00 93.00           C  \nATOM    791  CB  ASN A 106       4.011 -10.593   4.366  1.00 93.00           C  \nATOM    792  O   ASN A 106       3.379  -7.368   3.614  1.00 93.00           O  \nATOM    793  CG  ASN A 106       2.505 -10.548   4.527  1.00 93.00           C  \nATOM    794  ND2 ASN A 106       2.005 -11.162   5.598  1.00 93.00           N  \nATOM    795  OD1 ASN A 106       1.798  -9.972   3.707  1.00 93.00           O  \nATOM    796  N   TRP A 107       4.601  -8.523   2.181  1.00 93.56           N  \nATOM    797  CA  TRP A 107       4.296  -7.642   1.060  1.00 93.56           C  \nATOM    798  C   TRP A 107       2.799  -7.645   0.758  1.00 93.56           C  \nATOM    799  CB  TRP A 107       5.082  -8.053  -0.183  1.00 93.56           C  \nATOM    800  O   TRP A 107       2.247  -6.639   0.310  1.00 93.56           O  \nATOM    801  CG  TRP A 107       6.535  -7.681  -0.137  1.00 93.56           C  \nATOM    802  CD1 TRP A 107       7.606  -8.534  -0.115  1.00 93.56           C  \nATOM    803  CD2 TRP A 107       7.077  -6.356  -0.100  1.00 93.56           C  \nATOM    804  CE2 TRP A 107       8.481  -6.482  -0.060  1.00 93.56           C  \nATOM    805  CE3 TRP A 107       6.503  -5.074  -0.097  1.00 93.56           C  \nATOM    806  NE1 TRP A 107       8.779  -7.816  -0.069  1.00 93.56           N  \nATOM    807  CH2 TRP A 107       8.748  -4.129  -0.015  1.00 93.56           C  \nATOM    808  CZ2 TRP A 107       9.325  -5.368  -0.013  1.00 93.56           C  \nATOM    809  CZ3 TRP A 107       7.348  -3.969  -0.053  1.00 93.56           C  \nATOM    810  N   GLY A 108       2.158  -8.858   0.965  1.00 92.50           N  \nATOM    811  CA  GLY A 108       0.712  -8.920   0.799  1.00 92.50           C  \nATOM    812  C   GLY A 108      -0.034  -7.903   1.642  1.00 92.50           C  \nATOM    813  O   GLY A 108      -0.943  -7.230   1.149  1.00 92.50           O  \nATOM    814  N   ARG A 109       0.329  -7.679   2.808  1.00 94.06           N  \nATOM    815  CA  ARG A 109      -0.317  -6.732   3.714  1.00 94.06           C  \nATOM    816  C   ARG A 109       0.007  -5.294   3.324  1.00 94.06           C  \nATOM    817  CB  ARG A 109       0.111  -6.991   5.156  1.00 94.06           C  \nATOM    818  O   ARG A 109      -0.846  -4.411   3.420  1.00 94.06           O  \nATOM    819  CG  ARG A 109      -0.458  -8.271   5.745  1.00 94.06           C  \nATOM    820  CD  ARG A 109       0.025  -8.503   7.174  1.00 94.06           C  \nATOM    821  NE  ARG A 109      -0.553  -9.710   7.749  1.00 94.06           N  \nATOM    822  NH1 ARG A 109       0.575  -9.541   9.761  1.00 94.06           N  \nATOM    823  NH2 ARG A 109      -0.862 -11.288   9.394  1.00 94.06           N  \nATOM    824  CZ  ARG A 109      -0.278 -10.176   8.963  1.00 94.06           C  \nATOM    825  N   VAL A 110       1.277  -5.099   2.932  1.00 95.56           N  \nATOM    826  CA  VAL A 110       1.662  -3.777   2.447  1.00 95.56           C  \nATOM    827  C   VAL A 110       0.786  -3.383   1.262  1.00 95.56           C  \nATOM    828  CB  VAL A 110       3.155  -3.731   2.047  1.00 95.56           C  \nATOM    829  O   VAL A 110       0.262  -2.269   1.212  1.00 95.56           O  \nATOM    830  CG1 VAL A 110       3.507  -2.382   1.429  1.00 95.56           C  \nATOM    831  CG2 VAL A 110       4.041  -4.019   3.256  1.00 95.56           C  \nATOM    832  N   VAL A 111       0.598  -4.327   0.336  1.00 94.75           N  \nATOM    833  CA  VAL A 111      -0.231  -4.090  -0.840  1.00 94.75           C  \nATOM    834  C   VAL A 111      -1.677  -3.853  -0.413  1.00 94.75           C  \nATOM    835  CB  VAL A 111      -0.155  -5.269  -1.837  1.00 94.75           C  \nATOM    836  O   VAL A 111      -2.361  -2.988  -0.966  1.00 94.75           O  \nATOM    837  CG1 VAL A 111      -1.254  -5.156  -2.891  1.00 94.75           C  \nATOM    838  CG2 VAL A 111       1.221  -5.320  -2.498  1.00 94.75           C  \nATOM    839  N   ALA A 112      -2.153  -4.653   0.590  1.00 94.69           N  \nATOM    840  CA  ALA A 112      -3.507  -4.475   1.105  1.00 94.69           C  \nATOM    841  C   ALA A 112      -3.702  -3.066   1.658  1.00 94.69           C  \nATOM    842  CB  ALA A 112      -3.807  -5.512   2.184  1.00 94.69           C  \nATOM    843  O   ALA A 112      -4.743  -2.442   1.432  1.00 94.69           O  \nATOM    844  N   LEU A 113      -2.703  -2.563   2.373  1.00 95.88           N  \nATOM    845  CA  LEU A 113      -2.752  -1.206   2.907  1.00 95.88           C  \nATOM    846  C   LEU A 113      -2.845  -0.181   1.783  1.00 95.88           C  \nATOM    847  CB  LEU A 113      -1.519  -0.925   3.770  1.00 95.88           C  \nATOM    848  O   LEU A 113      -3.675   0.730   1.834  1.00 95.88           O  \nATOM    849  CG  LEU A 113      -1.432   0.468   4.394  1.00 95.88           C  \nATOM    850  CD1 LEU A 113      -2.537   0.656   5.430  1.00 95.88           C  \nATOM    851  CD2 LEU A 113      -0.061   0.688   5.025  1.00 95.88           C  \nATOM    852  N   PHE A 114      -1.917  -0.331   0.707  1.00 96.00           N  \nATOM    853  CA  PHE A 114      -1.959   0.565  -0.443  1.00 96.00           C  \nATOM    854  C   PHE A 114      -3.332   0.534  -1.104  1.00 96.00           C  \nATOM    855  CB  PHE A 114      -0.878   0.186  -1.460  1.00 96.00           C  \nATOM    856  O   PHE A 114      -3.874   1.579  -1.469  1.00 96.00           O  \nATOM    857  CG  PHE A 114       0.469   0.789  -1.167  1.00 96.00           C  \nATOM    858  CD1 PHE A 114       0.857   1.055   0.141  1.00 96.00           C  \nATOM    859  CD2 PHE A 114       1.349   1.088  -2.199  1.00 96.00           C  \nATOM    860  CE1 PHE A 114       2.105   1.615   0.416  1.00 96.00           C  \nATOM    861  CE2 PHE A 114       2.596   1.646  -1.932  1.00 96.00           C  \nATOM    862  CZ  PHE A 114       2.970   1.906  -0.625  1.00 96.00           C  \nATOM    863  N   TYR A 115      -3.874  -0.652  -1.223  1.00 95.38           N  \nATOM    864  CA  TYR A 115      -5.164  -0.829  -1.880  1.00 95.38           C  \nATOM    865  C   TYR A 115      -6.278  -0.157  -1.085  1.00 95.38           C  \nATOM    866  CB  TYR A 115      -5.477  -2.317  -2.060  1.00 95.38           C  \nATOM    867  O   TYR A 115      -7.107   0.557  -1.652  1.00 95.38           O  \nATOM    868  CG  TYR A 115      -6.774  -2.582  -2.780  1.00 95.38           C  \nATOM    869  CD1 TYR A 115      -7.907  -3.010  -2.087  1.00 95.38           C  \nATOM    870  CD2 TYR A 115      -6.876  -2.408  -4.157  1.00 95.38           C  \nATOM    871  CE1 TYR A 115      -9.105  -3.257  -2.747  1.00 95.38           C  \nATOM    872  CE2 TYR A 115      -8.069  -2.653  -4.827  1.00 95.38           C  \nATOM    873  OH  TYR A 115     -10.363  -3.319  -4.774  1.00 95.38           O  \nATOM    874  CZ  TYR A 115      -9.177  -3.075  -4.114  1.00 95.38           C  \nATOM    875  N   PHE A 116      -6.311  -0.402   0.176  1.00 94.69           N  \nATOM    876  CA  PHE A 116      -7.316   0.201   1.043  1.00 94.69           C  \nATOM    877  C   PHE A 116      -7.245   1.724   0.976  1.00 94.69           C  \nATOM    878  CB  PHE A 116      -7.133  -0.269   2.489  1.00 94.69           C  \nATOM    879  O   PHE A 116      -8.269   2.390   0.837  1.00 94.69           O  \nATOM    880  CG  PHE A 116      -8.144   0.302   3.446  1.00 94.69           C  \nATOM    881  CD1 PHE A 116      -7.741   1.076   4.527  1.00 94.69           C  \nATOM    882  CD2 PHE A 116      -9.501   0.061   3.265  1.00 94.69           C  \nATOM    883  CE1 PHE A 116      -8.676   1.602   5.415  1.00 94.69           C  \nATOM    884  CE2 PHE A 116     -10.442   0.587   4.147  1.00 94.69           C  \nATOM    885  CZ  PHE A 116     -10.026   1.357   5.223  1.00 94.69           C  \nATOM    886  N   ALA A 117      -6.049   2.299   1.068  1.00 94.69           N  \nATOM    887  CA  ALA A 117      -5.854   3.743   0.966  1.00 94.69           C  \nATOM    888  C   ALA A 117      -6.369   4.272  -0.367  1.00 94.69           C  \nATOM    889  CB  ALA A 117      -4.378   4.096   1.140  1.00 94.69           C  \nATOM    890  O   ALA A 117      -7.029   5.317  -0.414  1.00 94.69           O  \nATOM    891  N   SER A 118      -6.084   3.550  -1.430  1.00 93.75           N  \nATOM    892  CA  SER A 118      -6.532   3.938  -2.764  1.00 93.75           C  \nATOM    893  C   SER A 118      -8.054   3.969  -2.849  1.00 93.75           C  \nATOM    894  CB  SER A 118      -5.975   2.979  -3.817  1.00 93.75           C  \nATOM    895  O   SER A 118      -8.629   4.908  -3.410  1.00 93.75           O  \nATOM    896  OG  SER A 118      -6.570   1.697  -3.696  1.00 93.75           O  \nATOM    897  N   LYS A 119      -8.736   3.019  -2.232  1.00 92.31           N  \nATOM    898  CA  LYS A 119     -10.194   2.944  -2.250  1.00 92.31           C  \nATOM    899  C   LYS A 119     -10.810   4.085  -1.450  1.00 92.31           C  \nATOM    900  CB  LYS A 119     -10.667   1.599  -1.700  1.00 92.31           C  \nATOM    901  O   LYS A 119     -11.839   4.643  -1.843  1.00 92.31           O  \nATOM    902  CG  LYS A 119     -10.421   0.424  -2.636  1.00 92.31           C  \nATOM    903  CD  LYS A 119     -11.185   0.580  -3.944  1.00 92.31           C  \nATOM    904  CE  LYS A 119     -11.245  -0.733  -4.714  1.00 92.31           C  \nATOM    905  NZ  LYS A 119     -11.963  -0.577  -6.014  1.00 92.31           N  \nATOM    906  N   LEU A 120     -10.143   4.433  -0.383  1.00 91.50           N  \nATOM    907  CA  LEU A 120     -10.634   5.561   0.410  1.00 91.50           C  \nATOM    908  C   LEU A 120     -10.502   6.865  -0.363  1.00 91.50           C  \nATOM    909  CB  LEU A 120      -9.869   5.658   1.734  1.00 91.50           C  \nATOM    910  O   LEU A 120     -11.391   7.724  -0.299  1.00 91.50           O  \nATOM    911  CG  LEU A 120     -10.235   4.630   2.806  1.00 91.50           C  \nATOM    912  CD1 LEU A 120      -9.386   4.844   4.054  1.00 91.50           C  \nATOM    913  CD2 LEU A 120     -11.718   4.715   3.141  1.00 91.50           C  \nATOM    914  N   VAL A 121      -9.388   6.948  -1.057  1.00 90.88           N  \nATOM    915  CA  VAL A 121      -9.173   8.136  -1.876  1.00 90.88           C  \nATOM    916  C   VAL A 121     -10.228   8.201  -2.977  1.00 90.88           C  \nATOM    917  CB  VAL A 121      -7.757   8.146  -2.497  1.00 90.88           C  \nATOM    918  O   VAL A 121     -10.793   9.267  -3.240  1.00 90.88           O  \nATOM    919  CG1 VAL A 121      -7.645   9.233  -3.570  1.00 90.88           C  \nATOM    920  CG2 VAL A 121      -6.701   8.349  -1.415  1.00 90.88           C  \nATOM    921  N   LEU A 122     -10.492   7.148  -3.619  1.00 89.62           N  \nATOM    922  CA  LEU A 122     -11.487   7.102  -4.680  1.00 89.62           C  \nATOM    923  C   LEU A 122     -12.875   7.439  -4.144  1.00 89.62           C  \nATOM    924  CB  LEU A 122     -11.505   5.718  -5.341  1.00 89.62           C  \nATOM    925  O   LEU A 122     -13.650   8.124  -4.806  1.00 89.62           O  \nATOM    926  CG  LEU A 122     -10.271   5.346  -6.166  1.00 89.62           C  \nATOM    927  CD1 LEU A 122     -10.383   3.915  -6.671  1.00 89.62           C  \nATOM    928  CD2 LEU A 122     -10.099   6.314  -7.334  1.00 89.62           C  \nATOM    929  N   LYS A 123     -13.160   6.985  -2.976  1.00 87.56           N  \nATOM    930  CA  LYS A 123     -14.428   7.346  -2.342  1.00 87.56           C  \nATOM    931  C   LYS A 123     -14.497   8.844  -2.073  1.00 87.56           C  \nATOM    932  CB  LYS A 123     -14.619   6.567  -1.041  1.00 87.56           C  \nATOM    933  O   LYS A 123     -15.550   9.463  -2.237  1.00 87.56           O  \nATOM    934  CG  LYS A 123     -16.073   6.422  -0.616  1.00 87.56           C  \nATOM    935  CD  LYS A 123     -16.222   5.440   0.539  1.00 87.56           C  \nATOM    936  CE  LYS A 123     -17.529   5.655   1.291  1.00 87.56           C  \nATOM    937  NZ  LYS A 123     -18.479   4.522   1.085  1.00 87.56           N  \nATOM    938  N   ALA A 124     -13.353   9.393  -1.622  1.00 85.94           N  \nATOM    939  CA  ALA A 124     -13.279  10.828  -1.363  1.00 85.94           C  \nATOM    940  C   ALA A 124     -13.488  11.628  -2.647  1.00 85.94           C  \nATOM    941  CB  ALA A 124     -11.939  11.187  -0.728  1.00 85.94           C  \nATOM    942  O   ALA A 124     -14.131  12.688  -2.630  1.00 85.94           O  \nATOM    943  N   LEU A 125     -12.924  11.102  -3.730  1.00 84.12           N  \nATOM    944  CA  LEU A 125     -13.101  11.770  -5.010  1.00 84.12           C  \nATOM    945  C   LEU A 125     -14.565  11.763  -5.432  1.00 84.12           C  \nATOM    946  CB  LEU A 125     -12.249  11.094  -6.091  1.00 84.12           C  \nATOM    947  O   LEU A 125     -15.072  12.764  -5.959  1.00 84.12           O  \nATOM    948  CG  LEU A 125     -10.742  11.360  -6.033  1.00 84.12           C  \nATOM    949  CD1 LEU A 125     -10.005  10.474  -7.034  1.00 84.12           C  \nATOM    950  CD2 LEU A 125     -10.458  12.839  -6.311  1.00 84.12           C  \nATOM    951  N   CYS A 126     -15.220  10.699  -5.174  1.00 83.06           N  \nATOM    952  CA  CYS A 126     -16.624  10.550  -5.551  1.00 83.06           C  \nATOM    953  C   CYS A 126     -17.513  11.449  -4.696  1.00 83.06           C  \nATOM    954  CB  CYS A 126     -17.061   9.092  -5.399  1.00 83.06           C  \nATOM    955  O   CYS A 126     -18.534  11.943  -5.178  1.00 83.06           O  \nATOM    956  SG  CYS A 126     -16.369   7.991  -6.653  1.00 83.06           S  \nATOM    957  N   THR A 127     -16.997  11.786  -3.509  1.00 82.56           N  \nATOM    958  CA  THR A 127     -17.782  12.639  -2.617  1.00 82.56           C  \nATOM    959  C   THR A 127     -17.339  14.094  -2.737  1.00 82.56           C  \nATOM    960  CB  THR A 127     -17.655  12.180  -1.152  1.00 82.56           C  \nATOM    961  O   THR A 127     -17.781  14.948  -1.964  1.00 82.56           O  \nATOM    962  CG2 THR A 127     -18.178  10.759  -0.977  1.00 82.56           C  \nATOM    963  OG1 THR A 127     -16.276  12.222  -0.761  1.00 82.56           O  \nATOM    964  N   LYS A 128     -16.421  14.406  -3.745  1.00 78.38           N  \nATOM    965  CA  LYS A 128     -15.958  15.736  -4.123  1.00 78.38           C  \nATOM    966  C   LYS A 128     -15.262  16.432  -2.964  1.00 78.38           C  \nATOM    967  CB  LYS A 128     -17.126  16.591  -4.628  1.00 78.38           C  \nATOM    968  O   LYS A 128     -15.489  17.625  -2.724  1.00 78.38           O  \nATOM    969  CG  LYS A 128     -17.785  16.056  -5.894  1.00 78.38           C  \nATOM    970  CD  LYS A 128     -18.852  17.005  -6.413  1.00 78.38           C  \nATOM    971  CE  LYS A 128     -19.515  16.463  -7.673  1.00 78.38           C  \nATOM    972  NZ  LYS A 128     -20.577  17.382  -8.172  1.00 78.38           N  \nATOM    973  N   VAL A 129     -14.473  15.707  -2.192  1.00 78.44           N  \nATOM    974  CA  VAL A 129     -13.624  16.232  -1.132  1.00 78.44           C  \nATOM    975  C   VAL A 129     -12.163  16.007  -1.469  1.00 78.44           C  \nATOM    976  CB  VAL A 129     -13.965  15.593   0.231  1.00 78.44           C  \nATOM    977  O   VAL A 129     -11.520  15.104  -0.914  1.00 78.44           O  \nATOM    978  CG1 VAL A 129     -13.217  16.302   1.366  1.00 78.44           C  \nATOM    979  CG2 VAL A 129     -15.477  15.645   0.484  1.00 78.44           C  \nATOM    980  N   PRO A 130     -11.614  16.730  -2.427  1.00 73.81           N  \nATOM    981  CA  PRO A 130     -10.264  16.490  -2.938  1.00 73.81           C  \nATOM    982  C   PRO A 130      -9.185  16.668  -1.874  1.00 73.81           C  \nATOM    983  CB  PRO A 130     -10.115  17.534  -4.048  1.00 73.81           C  \nATOM    984  O   PRO A 130      -8.109  16.066  -1.971  1.00 73.81           O  \nATOM    985  CG  PRO A 130     -11.160  18.568  -3.743  1.00 73.81           C  \nATOM    986  CD  PRO A 130     -12.286  17.886  -3.018  1.00 73.81           C  \nATOM    987  N   GLU A 131      -9.530  17.519  -0.839  1.00 81.25           N  \nATOM    988  CA  GLU A 131      -8.558  17.768   0.219  1.00 81.25           C  \nATOM    989  C   GLU A 131      -8.227  16.484   0.973  1.00 81.25           C  \nATOM    990  CB  GLU A 131      -9.084  18.826   1.194  1.00 81.25           C  \nATOM    991  O   GLU A 131      -7.151  16.367   1.562  1.00 81.25           O  \nATOM    992  CG  GLU A 131      -9.303  20.198   0.562  1.00 81.25           C  \nATOM    993  CD  GLU A 131      -9.760  21.250   1.551  1.00 81.25           C  \nATOM    994  OE1 GLU A 131      -9.694  22.460   1.227  1.00 81.25           O  \nATOM    995  OE2 GLU A 131     -10.187  20.865   2.665  1.00 81.25           O  \nATOM    996  N   LEU A 132      -9.134  15.531   0.832  1.00 83.19           N  \nATOM    997  CA  LEU A 132      -8.963  14.276   1.555  1.00 83.19           C  \nATOM    998  C   LEU A 132      -7.951  13.378   0.856  1.00 83.19           C  \nATOM    999  CB  LEU A 132     -10.304  13.546   1.686  1.00 83.19           C  \nATOM   1000  O   LEU A 132      -7.323  12.528   1.493  1.00 83.19           O  \nATOM   1001  CG  LEU A 132     -11.243  14.047   2.787  1.00 83.19           C  \nATOM   1002  CD1 LEU A 132     -12.539  13.242   2.784  1.00 83.19           C  \nATOM   1003  CD2 LEU A 132     -10.559  13.964   4.148  1.00 83.19           C  \nATOM   1004  N   ILE A 133      -7.727  13.705  -0.374  1.00 85.44           N  \nATOM   1005  CA  ILE A 133      -6.778  12.889  -1.130  1.00 85.44           C  \nATOM   1006  C   ILE A 133      -5.394  12.998  -0.503  1.00 85.44           C  \nATOM   1007  CB  ILE A 133      -6.732  13.315  -2.615  1.00 85.44           C  \nATOM   1008  O   ILE A 133      -4.760  11.980  -0.204  1.00 85.44           O  \nATOM   1009  CG1 ILE A 133      -8.098  13.087  -3.281  1.00 85.44           C  \nATOM   1010  CG2 ILE A 133      -5.628  12.562  -3.353  1.00 85.44           C  \nATOM   1011  CD1 ILE A 133      -8.193  13.622  -4.698  1.00 85.44           C  \nATOM   1012  N   ARG A 134      -4.916  14.256  -0.313  1.00 87.00           N  \nATOM   1013  CA  ARG A 134      -3.594  14.494   0.253  1.00 87.00           C  \nATOM   1014  C   ARG A 134      -3.505  13.970   1.679  1.00 87.00           C  \nATOM   1015  CB  ARG A 134      -3.249  15.985   0.218  1.00 87.00           C  \nATOM   1016  O   ARG A 134      -2.475  13.426   2.087  1.00 87.00           O  \nATOM   1017  CG  ARG A 134      -2.997  16.523  -1.181  1.00 87.00           C  \nATOM   1018  CD  ARG A 134      -2.545  17.978  -1.153  1.00 87.00           C  \nATOM   1019  NE  ARG A 134      -2.406  18.532  -2.492  1.00 87.00           N  \nATOM   1020  NH1 ARG A 134      -4.416  19.652  -2.469  1.00 87.00           N  \nATOM   1021  NH2 ARG A 134      -3.065  19.758  -4.315  1.00 87.00           N  \nATOM   1022  CZ  ARG A 134      -3.299  19.312  -3.087  1.00 87.00           C  \nATOM   1023  N   THR A 135      -4.625  14.087   2.330  1.00 91.44           N  \nATOM   1024  CA  THR A 135      -4.667  13.651   3.722  1.00 91.44           C  \nATOM   1025  C   THR A 135      -4.514  12.139   3.827  1.00 91.44           C  \nATOM   1026  CB  THR A 135      -5.982  14.093   4.400  1.00 91.44           C  \nATOM   1027  O   THR A 135      -3.722  11.645   4.626  1.00 91.44           O  \nATOM   1028  CG2 THR A 135      -6.004  13.674   5.866  1.00 91.44           C  \nATOM   1029  OG1 THR A 135      -6.106  15.517   4.315  1.00 91.44           O  \nATOM   1030  N   ILE A 136      -5.227  11.384   3.051  1.00 92.50           N  \nATOM   1031  CA  ILE A 136      -5.191   9.927   3.083  1.00 92.50           C  \nATOM   1032  C   ILE A 136      -3.816   9.439   2.631  1.00 92.50           C  \nATOM   1033  CB  ILE A 136      -6.294   9.312   2.194  1.00 92.50           C  \nATOM   1034  O   ILE A 136      -3.238   8.538   3.243  1.00 92.50           O  \nATOM   1035  CG1 ILE A 136      -7.682   9.641   2.765  1.00 92.50           C  \nATOM   1036  CG2 ILE A 136      -6.098   7.796   2.059  1.00 92.50           C  \nATOM   1037  CD1 ILE A 136      -8.832   9.310   1.822  1.00 92.50           C  \nATOM   1038  N   MET A 137      -3.280  10.104   1.598  1.00 92.88           N  \nATOM   1039  CA  MET A 137      -1.931   9.756   1.157  1.00 92.88           C  \nATOM   1040  C   MET A 137      -0.914  10.021   2.259  1.00 92.88           C  \nATOM   1041  CB  MET A 137      -1.562  10.543  -0.103  1.00 92.88           C  \nATOM   1042  O   MET A 137      -0.013   9.214   2.483  1.00 92.88           O  \nATOM   1043  CG  MET A 137      -2.332  10.121  -1.340  1.00 92.88           C  \nATOM   1044  SD  MET A 137      -1.936  11.157  -2.803  1.00 92.88           S  \nATOM   1045  CE  MET A 137      -0.204  10.686  -3.065  1.00 92.88           C  \nATOM   1046  N   GLY A 138      -1.101  11.160   2.927  1.00 94.44           N  \nATOM   1047  CA  GLY A 138      -0.246  11.465   4.063  1.00 94.44           C  \nATOM   1048  C   GLY A 138      -0.313  10.420   5.159  1.00 94.44           C  \nATOM   1049  O   GLY A 138       0.712  10.015   5.705  1.00 94.44           O  \nATOM   1050  N   TRP A 139      -1.496   9.943   5.449  1.00 95.31           N  \nATOM   1051  CA  TRP A 139      -1.676   8.907   6.460  1.00 95.31           C  \nATOM   1052  C   TRP A 139      -0.947   7.627   6.064  1.00 95.31           C  \nATOM   1053  CB  TRP A 139      -3.167   8.619   6.675  1.00 95.31           C  \nATOM   1054  O   TRP A 139      -0.283   7.003   6.894  1.00 95.31           O  \nATOM   1055  CG  TRP A 139      -3.937   9.774   7.226  1.00 95.31           C  \nATOM   1056  CD1 TRP A 139      -3.441  10.828   7.945  1.00 95.31           C  \nATOM   1057  CD2 TRP A 139      -5.345   9.996   7.110  1.00 95.31           C  \nATOM   1058  CE2 TRP A 139      -5.638  11.204   7.782  1.00 95.31           C  \nATOM   1059  CE3 TRP A 139      -6.391   9.288   6.500  1.00 95.31           C  \nATOM   1060  NE1 TRP A 139      -4.460  11.693   8.285  1.00 95.31           N  \nATOM   1061  CH2 TRP A 139      -7.938  11.010   7.259  1.00 95.31           C  \nATOM   1062  CZ2 TRP A 139      -6.932  11.720   7.863  1.00 95.31           C  \nATOM   1063  CZ3 TRP A 139      -7.681   9.806   6.583  1.00 95.31           C  \nATOM   1064  N   THR A 140      -1.120   7.320   4.892  1.00 96.44           N  \nATOM   1065  CA  THR A 140      -0.508   6.091   4.399  1.00 96.44           C  \nATOM   1066  C   THR A 140       1.015   6.169   4.486  1.00 96.44           C  \nATOM   1067  CB  THR A 140      -0.927   5.803   2.946  1.00 96.44           C  \nATOM   1068  O   THR A 140       1.664   5.237   4.965  1.00 96.44           O  \nATOM   1069  CG2 THR A 140      -0.373   4.463   2.468  1.00 96.44           C  \nATOM   1070  OG1 THR A 140      -2.357   5.773   2.862  1.00 96.44           O  \nATOM   1071  N   LEU A 141       1.569   7.288   4.084  1.00 96.19           N  \nATOM   1072  CA  LEU A 141       3.012   7.475   4.092  1.00 96.19           C  \nATOM   1073  C   LEU A 141       3.544   7.587   5.520  1.00 96.19           C  \nATOM   1074  CB  LEU A 141       3.400   8.728   3.297  1.00 96.19           C  \nATOM   1075  O   LEU A 141       4.645   7.125   5.815  1.00 96.19           O  \nATOM   1076  CG  LEU A 141       3.181   8.658   1.783  1.00 96.19           C  \nATOM   1077  CD1 LEU A 141       3.552   9.990   1.136  1.00 96.19           C  \nATOM   1078  CD2 LEU A 141       3.993   7.519   1.176  1.00 96.19           C  \nATOM   1079  N   ASP A 142       2.748   8.198   6.352  1.00 96.12           N  \nATOM   1080  CA  ASP A 142       3.119   8.240   7.765  1.00 96.12           C  \nATOM   1081  C   ASP A 142       3.195   6.833   8.353  1.00 96.12           C  \nATOM   1082  CB  ASP A 142       2.120   9.088   8.557  1.00 96.12           C  \nATOM   1083  O   ASP A 142       4.126   6.515   9.096  1.00 96.12           O  \nATOM   1084  CG  ASP A 142       2.288  10.575   8.318  1.00 96.12           C  \nATOM   1085  OD1 ASP A 142       3.330  10.993   7.761  1.00 96.12           O  \nATOM   1086  OD2 ASP A 142       1.373  11.346   8.697  1.00 96.12           O  \nATOM   1087  N   PHE A 143       2.195   6.059   8.018  1.00 95.38           N  \nATOM   1088  CA  PHE A 143       2.186   4.685   8.504  1.00 95.38           C  \nATOM   1089  C   PHE A 143       3.376   3.906   7.955  1.00 95.38           C  \nATOM   1090  CB  PHE A 143       0.879   3.982   8.113  1.00 95.38           C  \nATOM   1091  O   PHE A 143       4.030   3.165   8.688  1.00 95.38           O  \nATOM   1092  CG  PHE A 143       0.670   2.663   8.809  1.00 95.38           C  \nATOM   1093  CD1 PHE A 143       1.167   1.488   8.259  1.00 95.38           C  \nATOM   1094  CD2 PHE A 143      -0.025   2.598  10.009  1.00 95.38           C  \nATOM   1095  CE1 PHE A 143       0.975   0.265   8.899  1.00 95.38           C  \nATOM   1096  CE2 PHE A 143      -0.223   1.382  10.649  1.00 95.38           C  \nATOM   1097  CZ  PHE A 143       0.279   0.215  10.098  1.00 95.38           C  \nATOM   1098  N   LEU A 144       3.637   4.050   6.716  1.00 95.94           N  \nATOM   1099  CA  LEU A 144       4.781   3.415   6.073  1.00 95.94           C  \nATOM   1100  C   LEU A 144       6.078   3.785   6.788  1.00 95.94           C  \nATOM   1101  CB  LEU A 144       4.863   3.820   4.600  1.00 95.94           C  \nATOM   1102  O   LEU A 144       6.895   2.912   7.097  1.00 95.94           O  \nATOM   1103  CG  LEU A 144       5.951   3.137   3.770  1.00 95.94           C  \nATOM   1104  CD1 LEU A 144       5.659   1.643   3.645  1.00 95.94           C  \nATOM   1105  CD2 LEU A 144       6.057   3.782   2.391  1.00 95.94           C  \nATOM   1106  N   ARG A 145       6.229   5.055   7.080  1.00 96.44           N  \nATOM   1107  CA  ARG A 145       7.447   5.572   7.709  1.00 96.44           C  \nATOM   1108  C   ARG A 145       7.607   5.021   9.125  1.00 96.44           C  \nATOM   1109  CB  ARG A 145       7.428   7.101   7.740  1.00 96.44           C  \nATOM   1110  O   ARG A 145       8.688   4.571   9.500  1.00 96.44           O  \nATOM   1111  CG  ARG A 145       8.686   7.716   8.328  1.00 96.44           C  \nATOM   1112  CD  ARG A 145       8.647   9.236   8.273  1.00 96.44           C  \nATOM   1113  NE  ARG A 145       8.794   9.733   6.904  1.00 96.44           N  \nATOM   1114  NH1 ARG A 145       6.904  11.060   6.968  1.00 96.44           N  \nATOM   1115  NH2 ARG A 145       8.182  10.967   5.067  1.00 96.44           N  \nATOM   1116  CZ  ARG A 145       7.964  10.589   6.317  1.00 96.44           C  \nATOM   1117  N   GLU A 146       6.538   4.949   9.833  1.00 94.94           N  \nATOM   1118  CA  GLU A 146       6.605   4.668  11.266  1.00 94.94           C  \nATOM   1119  C   GLU A 146       6.636   3.165  11.529  1.00 94.94           C  \nATOM   1120  CB  GLU A 146       5.411   5.303  11.989  1.00 94.94           C  \nATOM   1121  O   GLU A 146       7.333   2.704  12.436  1.00 94.94           O  \nATOM   1122  CG  GLU A 146       5.470   6.824  12.043  1.00 94.94           C  \nATOM   1123  CD  GLU A 146       4.253   7.444  12.707  1.00 94.94           C  \nATOM   1124  OE1 GLU A 146       4.172   8.697  12.779  1.00 94.94           O  \nATOM   1125  OE2 GLU A 146       3.372   6.681  13.157  1.00 94.94           O  \nATOM   1126  N   ARG A 147       5.985   2.433  10.638  1.00 93.69           N  \nATOM   1127  CA  ARG A 147       5.791   1.040  11.030  1.00 93.69           C  \nATOM   1128  C   ARG A 147       6.445   0.093  10.031  1.00 93.69           C  \nATOM   1129  CB  ARG A 147       4.298   0.716  11.161  1.00 93.69           C  \nATOM   1130  O   ARG A 147       6.871  -1.006  10.393  1.00 93.69           O  \nATOM   1131  CG  ARG A 147       3.574   1.570  12.189  1.00 93.69           C  \nATOM   1132  CD  ARG A 147       4.038   1.255  13.603  1.00 93.69           C  \nATOM   1133  NE  ARG A 147       3.935  -0.170  13.910  1.00 93.69           N  \nATOM   1134  NH1 ARG A 147       1.662  -0.119  14.329  1.00 93.69           N  \nATOM   1135  NH2 ARG A 147       2.822  -2.091  14.496  1.00 93.69           N  \nATOM   1136  CZ  ARG A 147       2.806  -0.790  14.243  1.00 93.69           C  \nATOM   1137  N   LEU A 148       6.582   0.498   8.721  1.00 95.31           N  \nATOM   1138  CA  LEU A 148       6.920  -0.467   7.681  1.00 95.31           C  \nATOM   1139  C   LEU A 148       8.315  -0.208   7.126  1.00 95.31           C  \nATOM   1140  CB  LEU A 148       5.889  -0.421   6.552  1.00 95.31           C  \nATOM   1141  O   LEU A 148       8.932  -1.100   6.538  1.00 95.31           O  \nATOM   1142  CG  LEU A 148       4.465  -0.834   6.918  1.00 95.31           C  \nATOM   1143  CD1 LEU A 148       3.536  -0.649   5.721  1.00 95.31           C  \nATOM   1144  CD2 LEU A 148       4.434  -2.280   7.404  1.00 95.31           C  \nATOM   1145  N   LEU A 149       8.816   1.013   7.358  1.00 95.56           N  \nATOM   1146  CA  LEU A 149      10.081   1.423   6.755  1.00 95.56           C  \nATOM   1147  C   LEU A 149      11.205   0.478   7.160  1.00 95.56           C  \nATOM   1148  CB  LEU A 149      10.429   2.855   7.164  1.00 95.56           C  \nATOM   1149  O   LEU A 149      11.977   0.021   6.308  1.00 95.56           O  \nATOM   1150  CG  LEU A 149      11.759   3.401   6.646  1.00 95.56           C  \nATOM   1151  CD1 LEU A 149      11.800   3.341   5.122  1.00 95.56           C  \nATOM   1152  CD2 LEU A 149      11.981   4.831   7.137  1.00 95.56           C  \nATOM   1153  N   GLY A 150      11.312   0.194   8.440  1.00 95.25           N  \nATOM   1154  CA  GLY A 150      12.335  -0.718   8.920  1.00 95.25           C  \nATOM   1155  C   GLY A 150      12.276  -2.079   8.260  1.00 95.25           C  \nATOM   1156  O   GLY A 150      13.304  -2.625   7.854  1.00 95.25           O  \nATOM   1157  N   TRP A 151      11.112  -2.568   8.129  1.00 95.19           N  \nATOM   1158  CA  TRP A 151      10.916  -3.885   7.540  1.00 95.19           C  \nATOM   1159  C   TRP A 151      11.268  -3.877   6.052  1.00 95.19           C  \nATOM   1160  CB  TRP A 151       9.464  -4.345   7.729  1.00 95.19           C  \nATOM   1161  O   TRP A 151      11.925  -4.793   5.562  1.00 95.19           O  \nATOM   1162  CG  TRP A 151       9.160  -5.679   7.116  1.00 95.19           C  \nATOM   1163  CD1 TRP A 151       9.415  -6.907   7.648  1.00 95.19           C  \nATOM   1164  CD2 TRP A 151       8.548  -5.911   5.842  1.00 95.19           C  \nATOM   1165  CE2 TRP A 151       8.467  -7.311   5.665  1.00 95.19           C  \nATOM   1166  CE3 TRP A 151       8.066  -5.075   4.829  1.00 95.19           C  \nATOM   1167  NE1 TRP A 151       9.000  -7.898   6.787  1.00 95.19           N  \nATOM   1168  CH2 TRP A 151       7.445  -7.049   3.550  1.00 95.19           C  \nATOM   1169  CZ2 TRP A 151       7.910  -7.890   4.527  1.00 95.19           C  \nATOM   1170  CZ3 TRP A 151       7.515  -5.649   3.691  1.00 95.19           C  \nATOM   1171  N   ILE A 152      10.783  -2.910   5.336  1.00 95.38           N  \nATOM   1172  CA  ILE A 152      11.047  -2.796   3.905  1.00 95.38           C  \nATOM   1173  C   ILE A 152      12.554  -2.691   3.665  1.00 95.38           C  \nATOM   1174  CB  ILE A 152      10.323  -1.579   3.289  1.00 95.38           C  \nATOM   1175  O   ILE A 152      13.077  -3.306   2.734  1.00 95.38           O  \nATOM   1176  CG1 ILE A 152       8.807  -1.779   3.334  1.00 95.38           C  \nATOM   1177  CG2 ILE A 152      10.806  -1.330   1.856  1.00 95.38           C  \nATOM   1178  CD1 ILE A 152       8.004  -0.552   2.922  1.00 95.38           C  \nATOM   1179  N   GLN A 153      13.179  -1.977   4.519  1.00 94.56           N  \nATOM   1180  CA  GLN A 153      14.636  -1.862   4.436  1.00 94.56           C  \nATOM   1181  C   GLN A 153      15.308  -3.220   4.624  1.00 94.56           C  \nATOM   1182  CB  GLN A 153      15.158  -0.872   5.474  1.00 94.56           C  \nATOM   1183  O   GLN A 153      16.253  -3.557   3.905  1.00 94.56           O  \nATOM   1184  CG  GLN A 153      14.957   0.585   5.086  1.00 94.56           C  \nATOM   1185  CD  GLN A 153      15.466   1.556   6.146  1.00 94.56           C  \nATOM   1186  NE2 GLN A 153      15.756   2.776   5.737  1.00 94.56           N  \nATOM   1187  OE1 GLN A 153      15.584   1.197   7.324  1.00 94.56           O  \nATOM   1188  N   ASP A 154      14.751  -3.980   5.539  1.00 93.69           N  \nATOM   1189  CA  ASP A 154      15.274  -5.314   5.815  1.00 93.69           C  \nATOM   1190  C   ASP A 154      15.066  -6.248   4.622  1.00 93.69           C  \nATOM   1191  CB  ASP A 154      14.619  -5.903   7.063  1.00 93.69           C  \nATOM   1192  O   ASP A 154      15.775  -7.243   4.474  1.00 93.69           O  \nATOM   1193  CG  ASP A 154      15.087  -5.239   8.350  1.00 93.69           C  \nATOM   1194  OD1 ASP A 154      16.110  -4.516   8.324  1.00 93.69           O  \nATOM   1195  OD2 ASP A 154      14.438  -5.439   9.397  1.00 93.69           O  \nATOM   1196  N   GLN A 155      14.092  -5.895   3.836  1.00 92.38           N  \nATOM   1197  CA  GLN A 155      13.781  -6.701   2.656  1.00 92.38           C  \nATOM   1198  C   GLN A 155      14.597  -6.246   1.449  1.00 92.38           C  \nATOM   1199  CB  GLN A 155      12.284  -6.638   2.337  1.00 92.38           C  \nATOM   1200  O   GLN A 155      14.421  -6.768   0.343  1.00 92.38           O  \nATOM   1201  CG  GLN A 155      11.398  -7.168   3.456  1.00 92.38           C  \nATOM   1202  CD  GLN A 155      11.655  -8.632   3.761  1.00 92.38           C  \nATOM   1203  NE2 GLN A 155      11.799  -8.957   5.042  1.00 92.38           N  \nATOM   1204  OE1 GLN A 155      11.722  -9.469   2.856  1.00 92.38           O  \nATOM   1205  N   GLY A 156      15.453  -5.271   1.579  1.00 91.31           N  \nATOM   1206  CA  GLY A 156      16.314  -4.782   0.513  1.00 91.31           C  \nATOM   1207  C   GLY A 156      15.653  -3.718  -0.345  1.00 91.31           C  \nATOM   1208  O   GLY A 156      16.077  -3.471  -1.475  1.00 91.31           O  \nATOM   1209  N   GLY A 157      14.449  -3.216   0.139  1.00 93.94           N  \nATOM   1210  CA  GLY A 157      13.743  -2.165  -0.588  1.00 93.94           C  \nATOM   1211  C   GLY A 157      12.500  -2.658  -1.297  1.00 93.94           C  \nATOM   1212  O   GLY A 157      12.051  -3.782  -1.059  1.00 93.94           O  \nATOM   1213  N   TRP A 158      11.984  -1.908  -2.214  1.00 94.31           N  \nATOM   1214  CA  TRP A 158      10.705  -2.158  -2.869  1.00 94.31           C  \nATOM   1215  C   TRP A 158      10.842  -3.245  -3.936  1.00 94.31           C  \nATOM   1216  CB  TRP A 158      10.160  -0.872  -3.497  1.00 94.31           C  \nATOM   1217  O   TRP A 158       9.851  -3.840  -4.354  1.00 94.31           O  \nATOM   1218  CG  TRP A 158       9.574   0.091  -2.504  1.00 94.31           C  \nATOM   1219  CD1 TRP A 158      10.189   1.185  -1.960  1.00 94.31           C  \nATOM   1220  CD2 TRP A 158       8.265   0.041  -1.938  1.00 94.31           C  \nATOM   1221  CE2 TRP A 158       8.149   1.137  -1.056  1.00 94.31           C  \nATOM   1222  CE3 TRP A 158       7.172  -0.823  -2.089  1.00 94.31           C  \nATOM   1223  NE1 TRP A 158       9.337   1.819  -1.087  1.00 94.31           N  \nATOM   1224  CH2 TRP A 158       5.932   0.533  -0.493  1.00 94.31           C  \nATOM   1225  CZ2 TRP A 158       6.983   1.393  -0.327  1.00 94.31           C  \nATOM   1226  CZ3 TRP A 158       6.014  -0.569  -1.364  1.00 94.31           C  \nATOM   1227  N   ASP A 159      12.109  -3.525  -4.285  1.00 92.12           N  \nATOM   1228  CA  ASP A 159      12.307  -4.575  -5.269  1.00 92.12           C  \nATOM   1229  C   ASP A 159      11.877  -5.935  -4.717  1.00 92.12           C  \nATOM   1230  CB  ASP A 159      13.768  -4.633  -5.720  1.00 92.12           C  \nATOM   1231  O   ASP A 159      11.614  -6.867  -5.480  1.00 92.12           O  \nATOM   1232  CG  ASP A 159      14.159  -3.453  -6.596  1.00 92.12           C  \nATOM   1233  OD1 ASP A 159      13.293  -2.905  -7.309  1.00 92.12           O  \nATOM   1234  OD2 ASP A 159      15.349  -3.072  -6.580  1.00 92.12           O  \nATOM   1235  N   GLY A 160      11.834  -6.018  -3.376  1.00 91.19           N  \nATOM   1236  CA  GLY A 160      11.330  -7.217  -2.725  1.00 91.19           C  \nATOM   1237  C   GLY A 160       9.906  -7.552  -3.123  1.00 91.19           C  \nATOM   1238  O   GLY A 160       9.528  -8.724  -3.179  1.00 91.19           O  \nATOM   1239  N   LEU A 161       9.152  -6.539  -3.377  1.00 91.38           N  \nATOM   1240  CA  LEU A 161       7.768  -6.716  -3.811  1.00 91.38           C  \nATOM   1241  C   LEU A 161       7.710  -7.483  -5.131  1.00 91.38           C  \nATOM   1242  CB  LEU A 161       7.080  -5.359  -3.964  1.00 91.38           C  \nATOM   1243  O   LEU A 161       6.888  -8.389  -5.290  1.00 91.38           O  \nATOM   1244  CG  LEU A 161       5.587  -5.393  -4.302  1.00 91.38           C  \nATOM   1245  CD1 LEU A 161       4.871  -4.215  -3.656  1.00 91.38           C  \nATOM   1246  CD2 LEU A 161       5.382  -5.390  -5.814  1.00 91.38           C  \nATOM   1247  N   LEU A 162       8.574  -7.155  -6.054  1.00 90.06           N  \nATOM   1248  CA  LEU A 162       8.604  -7.825  -7.352  1.00 90.06           C  \nATOM   1249  C   LEU A 162       8.954  -9.293  -7.192  1.00 90.06           C  \nATOM   1250  CB  LEU A 162       9.605  -7.139  -8.284  1.00 90.06           C  \nATOM   1251  O   LEU A 162       8.371 -10.161  -7.858  1.00 90.06           O  \nATOM   1252  CG  LEU A 162       9.235  -5.742  -8.772  1.00 90.06           C  \nATOM   1253  CD1 LEU A 162      10.363  -5.150  -9.608  1.00 90.06           C  \nATOM   1254  CD2 LEU A 162       7.932  -5.775  -9.566  1.00 90.06           C  \nATOM   1255  N   SER A 163       9.903  -9.547  -6.345  1.00 88.00           N  \nATOM   1256  CA  SER A 163      10.324 -10.925  -6.101  1.00 88.00           C  \nATOM   1257  C   SER A 163       9.204 -11.735  -5.457  1.00 88.00           C  \nATOM   1258  CB  SER A 163      11.564 -10.958  -5.210  1.00 88.00           C  \nATOM   1259  O   SER A 163       8.988 -12.898  -5.812  1.00 88.00           O  \nATOM   1260  OG  SER A 163      12.647 -10.284  -5.829  1.00 88.00           O  \nATOM   1261  N   TYR A 164       8.480 -11.150  -4.592  1.00 86.19           N  \nATOM   1262  CA  TYR A 164       7.408 -11.798  -3.847  1.00 86.19           C  \nATOM   1263  C   TYR A 164       6.238 -12.141  -4.762  1.00 86.19           C  \nATOM   1264  CB  TYR A 164       6.932 -10.898  -2.703  1.00 86.19           C  \nATOM   1265  O   TYR A 164       5.729 -13.263  -4.733  1.00 86.19           O  \nATOM   1266  CG  TYR A 164       5.879 -11.538  -1.831  1.00 86.19           C  \nATOM   1267  CD1 TYR A 164       4.525 -11.243  -2.015  1.00 86.19           C  \nATOM   1268  CD2 TYR A 164       6.231 -12.426  -0.820  1.00 86.19           C  \nATOM   1269  CE1 TYR A 164       3.553 -11.829  -1.212  1.00 86.19           C  \nATOM   1270  CE2 TYR A 164       5.263 -13.012  -0.014  1.00 86.19           C  \nATOM   1271  OH  TYR A 164       2.966 -13.286   0.582  1.00 86.19           O  \nATOM   1272  CZ  TYR A 164       3.929 -12.710  -0.217  1.00 86.19           C  \nATOM   1273  N   PHE A 165       5.869 -11.226  -5.614  1.00 83.75           N  \nATOM   1274  CA  PHE A 165       4.678 -11.431  -6.430  1.00 83.75           C  \nATOM   1275  C   PHE A 165       5.052 -12.023  -7.790  1.00 83.75           C  \nATOM   1276  CB  PHE A 165       3.916 -10.116  -6.617  1.00 83.75           C  \nATOM   1277  O   PHE A 165       4.172 -12.375  -8.576  1.00 83.75           O  \nATOM   1278  CG  PHE A 165       3.078  -9.727  -5.430  1.00 83.75           C  \nATOM   1279  CD1 PHE A 165       1.841 -10.320  -5.211  1.00 83.75           C  \nATOM   1280  CD2 PHE A 165       3.525  -8.765  -4.533  1.00 83.75           C  \nATOM   1281  CE1 PHE A 165       1.063  -9.963  -4.113  1.00 83.75           C  \nATOM   1282  CE2 PHE A 165       2.753  -8.403  -3.433  1.00 83.75           C  \nATOM   1283  CZ  PHE A 165       1.519  -8.998  -3.224  1.00 83.75           C  \nATOM   1284  N   GLY A 166       6.293 -12.083  -8.078  1.00 79.56           N  \nATOM   1285  CA  GLY A 166       6.756 -12.703  -9.298  1.00 79.56           C  \nATOM   1286  C   GLY A 166       6.795 -14.219  -9.218  1.00 79.56           C  \nATOM   1287  O   GLY A 166       6.958 -14.898 -10.235  1.00 79.56           O  \nATOM   1288  N   THR A 167       6.401 -14.798  -8.032  1.00 77.88           N  \nATOM   1289  CA  THR A 167       6.392 -16.246  -7.870  1.00 77.88           C  \nATOM   1290  C   THR A 167       5.077 -16.842  -8.370  1.00 77.88           C  \nATOM   1291  CB  THR A 167       6.614 -16.649  -6.401  1.00 77.88           C  \nATOM   1292  O   THR A 167       4.081 -16.124  -8.510  1.00 77.88           O  \nATOM   1293  CG2 THR A 167       7.924 -16.076  -5.866  1.00 77.88           C  \nATOM   1294  OG1 THR A 167       5.529 -16.149  -5.608  1.00 77.88           O  \nATOM   1295  N   PRO A 168       5.097 -18.032  -8.952  1.00 71.38           N  \nATOM   1296  CA  PRO A 168       3.911 -18.684  -9.516  1.00 71.38           C  \nATOM   1297  C   PRO A 168       2.734 -18.723  -8.546  1.00 71.38           C  \nATOM   1298  CB  PRO A 168       4.416 -20.094  -9.822  1.00 71.38           C  \nATOM   1299  O   PRO A 168       1.592 -18.953  -8.964  1.00 71.38           O  \nATOM   1300  CG  PRO A 168       5.910 -19.979  -9.851  1.00 71.38           C  \nATOM   1301  CD  PRO A 168       6.301 -18.788  -9.022  1.00 71.38           C  \nATOM   1302  N   THR A 169       2.925 -18.358  -7.324  1.00 71.50           N  \nATOM   1303  CA  THR A 169       1.837 -18.458  -6.352  1.00 71.50           C  \nATOM   1304  C   THR A 169       1.130 -17.126  -6.202  1.00 71.50           C  \nATOM   1305  CB  THR A 169       2.361 -18.928  -4.981  1.00 71.50           C  \nATOM   1306  O   THR A 169       0.428 -16.897  -5.213  1.00 71.50           O  \nATOM   1307  CG2 THR A 169       2.888 -20.366  -5.057  1.00 71.50           C  \nATOM   1308  OG1 THR A 169       3.427 -18.069  -4.565  1.00 71.50           O  \nATOM   1309  N   TRP A 170       1.296 -16.258  -7.174  1.00 70.88           N  \nATOM   1310  CA  TRP A 170       0.813 -14.894  -7.007  1.00 70.88           C  \nATOM   1311  C   TRP A 170      -0.701 -14.870  -6.880  1.00 70.88           C  \nATOM   1312  CB  TRP A 170       1.255 -14.020  -8.184  1.00 70.88           C  \nATOM   1313  O   TRP A 170      -1.264 -13.968  -6.249  1.00 70.88           O  \nATOM   1314  CG  TRP A 170       0.570 -14.348  -9.477  1.00 70.88           C  \nATOM   1315  CD1 TRP A 170       0.950 -15.293 -10.397  1.00 70.88           C  \nATOM   1316  CD2 TRP A 170      -0.606 -13.727  -9.998  1.00 70.88           C  \nATOM   1317  CE2 TRP A 170      -0.892 -14.342 -11.238  1.00 70.88           C  \nATOM   1318  CE3 TRP A 170      -1.452 -12.707  -9.537  1.00 70.88           C  \nATOM   1319  NE1 TRP A 170       0.066 -15.289 -11.452  1.00 70.88           N  \nATOM   1320  CH2 TRP A 170      -2.802 -12.974 -11.546  1.00 70.88           C  \nATOM   1321  CZ2 TRP A 170      -1.987 -13.970 -12.021  1.00 70.88           C  \nATOM   1322  CZ3 TRP A 170      -2.546 -12.344 -10.320  1.00 70.88           C  \nATOM   1323  N   GLN A 171      -1.476 -15.773  -7.530  1.00 71.62           N  \nATOM   1324  CA  GLN A 171      -2.927 -15.813  -7.378  1.00 71.62           C  \nATOM   1325  C   GLN A 171      -3.324 -16.079  -5.927  1.00 71.62           C  \nATOM   1326  CB  GLN A 171      -3.529 -16.887  -8.289  1.00 71.62           C  \nATOM   1327  O   GLN A 171      -4.268 -15.483  -5.419  1.00 71.62           O  \nATOM   1328  CG  GLN A 171      -3.459 -16.545  -9.771  1.00 71.62           C  \nATOM   1329  CD  GLN A 171      -4.009 -17.648 -10.654  1.00 71.62           C  \nATOM   1330  NE2 GLN A 171      -4.468 -17.283 -11.846  1.00 71.62           N  \nATOM   1331  OE1 GLN A 171      -4.011 -18.826 -10.272  1.00 71.62           O  \nATOM   1332  N   THR A 172      -2.535 -16.984  -5.310  1.00 77.25           N  \nATOM   1333  CA  THR A 172      -2.815 -17.288  -3.910  1.00 77.25           C  \nATOM   1334  C   THR A 172      -2.546 -16.074  -3.029  1.00 77.25           C  \nATOM   1335  CB  THR A 172      -1.979 -18.482  -3.420  1.00 77.25           C  \nATOM   1336  O   THR A 172      -3.269 -15.828  -2.064  1.00 77.25           O  \nATOM   1337  CG2 THR A 172      -2.399 -18.919  -2.020  1.00 77.25           C  \nATOM   1338  OG1 THR A 172      -2.149 -19.579  -4.322  1.00 77.25           O  \nATOM   1339  N   VAL A 173      -1.621 -15.278  -3.453  1.00 78.44           N  \nATOM   1340  CA  VAL A 173      -1.234 -14.098  -2.688  1.00 78.44           C  \nATOM   1341  C   VAL A 173      -2.369 -13.069  -2.716  1.00 78.44           C  \nATOM   1342  CB  VAL A 173       0.064 -13.467  -3.230  1.00 78.44           C  \nATOM   1343  O   VAL A 173      -2.701 -12.473  -1.688  1.00 78.44           O  \nATOM   1344  CG1 VAL A 173       0.338 -12.125  -2.558  1.00 78.44           C  \nATOM   1345  CG2 VAL A 173       1.244 -14.417  -3.025  1.00 78.44           C  \nATOM   1346  N   THR A 174      -3.009 -12.969  -3.907  1.00 77.19           N  \nATOM   1347  CA  THR A 174      -4.095 -12.004  -4.043  1.00 77.19           C  \nATOM   1348  C   THR A 174      -5.284 -12.409  -3.176  1.00 77.19           C  \nATOM   1349  CB  THR A 174      -4.550 -11.879  -5.510  1.00 77.19           C  \nATOM   1350  O   THR A 174      -5.939 -11.553  -2.572  1.00 77.19           O  \nATOM   1351  CG2 THR A 174      -5.613 -10.793  -5.665  1.00 77.19           C  \nATOM   1352  OG1 THR A 174      -3.418 -11.543  -6.324  1.00 77.19           O  \nATOM   1353  N   ILE A 175      -5.493 -13.685  -3.119  1.00 79.56           N  \nATOM   1354  CA  ILE A 175      -6.559 -14.218  -2.276  1.00 79.56           C  \nATOM   1355  C   ILE A 175      -6.266 -13.885  -0.811  1.00 79.56           C  \nATOM   1356  CB  ILE A 175      -6.720 -15.739  -2.457  1.00 79.56           C  \nATOM   1357  O   ILE A 175      -7.165 -13.469  -0.073  1.00 79.56           O  \nATOM   1358  CG1 ILE A 175      -7.228 -16.060  -3.870  1.00 79.56           C  \nATOM   1359  CG2 ILE A 175      -7.668 -16.312  -1.395  1.00 79.56           C  \nATOM   1360  CD1 ILE A 175      -7.191 -17.537  -4.223  1.00 79.56           C  \nATOM   1361  N   PHE A 176      -5.008 -14.076  -0.466  1.00 83.88           N  \nATOM   1362  CA  PHE A 176      -4.585 -13.751   0.891  1.00 83.88           C  \nATOM   1363  C   PHE A 176      -4.805 -12.269   1.184  1.00 83.88           C  \nATOM   1364  CB  PHE A 176      -3.110 -14.106   1.103  1.00 83.88           C  \nATOM   1365  O   PHE A 176      -5.343 -11.911   2.238  1.00 83.88           O  \nATOM   1366  CG  PHE A 176      -2.593 -13.777   2.479  1.00 83.88           C  \nATOM   1367  CD1 PHE A 176      -1.801 -12.656   2.685  1.00 83.88           C  \nATOM   1368  CD2 PHE A 176      -2.906 -14.589   3.564  1.00 83.88           C  \nATOM   1369  CE1 PHE A 176      -1.324 -12.348   3.959  1.00 83.88           C  \nATOM   1370  CE2 PHE A 176      -2.427 -14.288   4.833  1.00 83.88           C  \nATOM   1371  CZ  PHE A 176      -1.638 -13.168   5.030  1.00 83.88           C  \nATOM   1372  N   VAL A 177      -4.452 -11.423   0.259  1.00 85.31           N  \nATOM   1373  CA  VAL A 177      -4.608  -9.982   0.415  1.00 85.31           C  \nATOM   1374  C   VAL A 177      -6.084  -9.641   0.618  1.00 85.31           C  \nATOM   1375  CB  VAL A 177      -4.043  -9.212  -0.800  1.00 85.31           C  \nATOM   1376  O   VAL A 177      -6.436  -8.907   1.542  1.00 85.31           O  \nATOM   1377  CG1 VAL A 177      -4.407  -7.730  -0.718  1.00 85.31           C  \nATOM   1378  CG2 VAL A 177      -2.528  -9.391  -0.890  1.00 85.31           C  \nATOM   1379  N   ALA A 178      -6.909 -10.183  -0.222  1.00 83.06           N  \nATOM   1380  CA  ALA A 178      -8.339  -9.948  -0.103  1.00 83.06           C  \nATOM   1381  C   ALA A 178      -8.868 -10.445   1.241  1.00 83.06           C  \nATOM   1382  CB  ALA A 178      -9.084 -10.629  -1.248  1.00 83.06           C  \nATOM   1383  O   ALA A 178      -9.702  -9.788   1.866  1.00 83.06           O  \nATOM   1384  N   GLY A 179      -8.301 -11.554   1.685  1.00 85.44           N  \nATOM   1385  CA  GLY A 179      -8.668 -12.113   2.977  1.00 85.44           C  \nATOM   1386  C   GLY A 179      -8.323 -11.211   4.142  1.00 85.44           C  \nATOM   1387  O   GLY A 179      -9.142 -10.997   5.038  1.00 85.44           O  \nATOM   1388  N   VAL A 180      -7.128 -10.684   4.160  1.00 87.06           N  \nATOM   1389  CA  VAL A 180      -6.670  -9.784   5.215  1.00 87.06           C  \nATOM   1390  C   VAL A 180      -7.539  -8.525   5.231  1.00 87.06           C  \nATOM   1391  CB  VAL A 180      -5.182  -9.402   5.035  1.00 87.06           C  \nATOM   1392  O   VAL A 180      -7.990  -8.090   6.293  1.00 87.06           O  \nATOM   1393  CG1 VAL A 180      -4.806  -8.244   5.957  1.00 87.06           C  \nATOM   1394  CG2 VAL A 180      -4.289 -10.609   5.290  1.00 87.06           C  \nATOM   1395  N   LEU A 181      -7.874  -7.995   4.076  1.00 87.38           N  \nATOM   1396  CA  LEU A 181      -8.685  -6.786   3.987  1.00 87.38           C  \nATOM   1397  C   LEU A 181     -10.106  -7.048   4.481  1.00 87.38           C  \nATOM   1398  CB  LEU A 181      -8.719  -6.267   2.547  1.00 87.38           C  \nATOM   1399  O   LEU A 181     -10.661  -6.251   5.239  1.00 87.38           O  \nATOM   1400  CG  LEU A 181      -7.476  -5.518   2.066  1.00 87.38           C  \nATOM   1401  CD1 LEU A 181      -7.548  -5.277   0.563  1.00 87.38           C  \nATOM   1402  CD2 LEU A 181      -7.323  -4.199   2.818  1.00 87.38           C  \nATOM   1403  N   THR A 182     -10.614  -8.133   4.025  1.00 87.12           N  \nATOM   1404  CA  THR A 182     -11.975  -8.485   4.438  1.00 87.12           C  \nATOM   1405  C   THR A 182     -12.041  -8.691   5.948  1.00 87.12           C  \nATOM   1406  CB  THR A 182     -12.466  -9.748   3.715  1.00 87.12           C  \nATOM   1407  O   THR A 182     -12.970  -8.213   6.602  1.00 87.12           O  \nATOM   1408  CG2 THR A 182     -13.907 -10.071   4.096  1.00 87.12           C  \nATOM   1409  OG1 THR A 182     -12.394  -9.545   2.299  1.00 87.12           O  \nATOM   1410  N   ALA A 183     -11.057  -9.363   6.545  1.00 86.00           N  \nATOM   1411  CA  ALA A 183     -11.016  -9.626   7.981  1.00 86.00           C  \nATOM   1412  C   ALA A 183     -10.899  -8.326   8.770  1.00 86.00           C  \nATOM   1413  CB  ALA A 183      -9.855 -10.556   8.316  1.00 86.00           C  \nATOM   1414  O   ALA A 183     -11.606  -8.128   9.763  1.00 86.00           O  \nATOM   1415  N   SER A 184     -10.056  -7.457   8.299  1.00 81.81           N  \nATOM   1416  CA  SER A 184      -9.848  -6.191   8.993  1.00 81.81           C  \nATOM   1417  C   SER A 184     -11.090  -5.308   8.913  1.00 81.81           C  \nATOM   1418  CB  SER A 184      -8.649  -5.446   8.399  1.00 81.81           C  \nATOM   1419  O   SER A 184     -11.472  -4.672   9.896  1.00 81.81           O  \nATOM   1420  OG  SER A 184      -8.822  -5.235   7.010  1.00 81.81           O  \nATOM   1421  N   LEU A 185     -11.717  -5.248   7.779  1.00 83.00           N  \nATOM   1422  CA  LEU A 185     -12.915  -4.442   7.579  1.00 83.00           C  \nATOM   1423  C   LEU A 185     -14.079  -4.984   8.412  1.00 83.00           C  \nATOM   1424  CB  LEU A 185     -13.306  -4.416   6.098  1.00 83.00           C  \nATOM   1425  O   LEU A 185     -14.879  -4.211   8.940  1.00 83.00           O  \nATOM   1426  CG  LEU A 185     -12.431  -3.555   5.184  1.00 83.00           C  \nATOM   1427  CD1 LEU A 185     -12.810  -3.779   3.725  1.00 83.00           C  \nATOM   1428  CD2 LEU A 185     -12.559  -2.082   5.557  1.00 83.00           C  \nATOM   1429  N   THR A 186     -14.202  -6.318   8.520  1.00 84.56           N  \nATOM   1430  CA  THR A 186     -15.232  -6.957   9.328  1.00 84.56           C  \nATOM   1431  C   THR A 186     -15.027  -6.647  10.811  1.00 84.56           C  \nATOM   1432  CB  THR A 186     -15.239  -8.481   9.118  1.00 84.56           C  \nATOM   1433  O   THR A 186     -15.986  -6.361  11.529  1.00 84.56           O  \nATOM   1434  CG2 THR A 186     -16.343  -9.145   9.937  1.00 84.56           C  \nATOM   1435  OG1 THR A 186     -15.439  -8.766   7.734  1.00 84.56           O  \nATOM   1436  N   ILE A 187     -13.741  -6.746  11.258  1.00 81.12           N  \nATOM   1437  CA  ILE A 187     -13.430  -6.457  12.651  1.00 81.12           C  \nATOM   1438  C   ILE A 187     -13.764  -5.000  12.964  1.00 81.12           C  \nATOM   1439  CB  ILE A 187     -11.948  -6.748  12.965  1.00 81.12           C  \nATOM   1440  O   ILE A 187     -14.300  -4.698  14.028  1.00 81.12           O  \nATOM   1441  CG1 ILE A 187     -11.662  -8.255  12.859  1.00 81.12           C  \nATOM   1442  CG2 ILE A 187     -11.580  -6.225  14.356  1.00 81.12           C  \nATOM   1443  CD1 ILE A 187     -10.184  -8.608  12.905  1.00 81.12           C  \nATOM   1444  N   TRP A 188     -13.409  -4.137  12.024  1.00 77.62           N  \nATOM   1445  CA  TRP A 188     -13.763  -2.726  12.185  1.00 77.62           C  \nATOM   1446  C   TRP A 188     -15.272  -2.558  12.320  1.00 77.62           C  \nATOM   1447  CB  TRP A 188     -13.245  -1.910  11.004  1.00 77.62           C  \nATOM   1448  O   TRP A 188     -15.741  -1.820  13.196  1.00 77.62           O  \nATOM   1449  CG  TRP A 188     -13.746  -0.492  10.977  1.00 77.62           C  \nATOM   1450  CD1 TRP A 188     -13.345   0.538  11.790  1.00 77.62           C  \nATOM   1451  CD2 TRP A 188     -14.752   0.048  10.114  1.00 77.62           C  \nATOM   1452  CE2 TRP A 188     -14.902   1.406  10.449  1.00 77.62           C  \nATOM   1453  CE3 TRP A 188     -15.534  -0.492   9.081  1.00 77.62           C  \nATOM   1454  NE1 TRP A 188     -14.039   1.682  11.476  1.00 77.62           N  \nATOM   1455  CH2 TRP A 188     -16.564   1.691   8.785  1.00 77.62           C  \nATOM   1456  CZ2 TRP A 188     -15.810   2.246   9.789  1.00 77.62           C  \nATOM   1457  CZ3 TRP A 188     -16.436   0.339   8.426  1.00 77.62           C  \nATOM   1458  N   LYS A 189     -16.023  -3.169  11.377  1.00 77.94           N  \nATOM   1459  CA  LYS A 189     -17.481  -3.057  11.374  1.00 77.94           C  \nATOM   1460  C   LYS A 189     -18.067  -3.597  12.674  1.00 77.94           C  \nATOM   1461  CB  LYS A 189     -18.078  -3.796  10.183  1.00 77.94           C  \nATOM   1462  O   LYS A 189     -19.043  -3.058  13.196  1.00 77.94           O  \nATOM   1463  CG  LYS A 189     -19.557  -3.515   9.961  1.00 77.94           C  \nATOM   1464  CD  LYS A 189     -20.075  -4.203   8.701  1.00 77.94           C  \nATOM   1465  CE  LYS A 189     -21.597  -4.221   8.656  1.00 77.94           C  \nATOM   1466  NZ  LYS A 189     -22.113  -4.594   7.302  1.00 77.94           N  \nATOM   1467  N   LYS A 190     -17.542  -4.669  13.216  1.00 74.31           N  \nATOM   1468  CA  LYS A 190     -18.080  -5.311  14.417  1.00 74.31           C  \nATOM   1469  C   LYS A 190     -17.748  -4.505  15.665  1.00 74.31           C  \nATOM   1470  CB  LYS A 190     -17.546  -6.732  14.551  1.00 74.31           C  \nATOM   1471  O   LYS A 190     -18.496  -4.525  16.647  1.00 74.31           O  \nATOM   1472  CG  LYS A 190     -18.221  -7.742  13.635  1.00 74.31           C  \nATOM   1473  CD  LYS A 190     -17.759  -9.161  13.928  1.00 74.31           C  \nATOM   1474  CE  LYS A 190     -18.453 -10.178  13.027  1.00 74.31           C  \nATOM   1475  NZ  LYS A 190     -17.962 -11.565  13.271  1.00 74.31           N  \nATOM   1476  N   MET A 191     -16.658  -3.743  15.574  1.00 64.88           N  \nATOM   1477  CA  MET A 191     -16.252  -2.994  16.760  1.00 64.88           C  \nATOM   1478  C   MET A 191     -16.843  -1.591  16.745  1.00 64.88           C  \nATOM   1479  CB  MET A 191     -14.724  -2.916  16.844  1.00 64.88           C  \nATOM   1480  O   MET A 191     -16.909  -0.934  17.788  1.00 64.88           O  \nATOM   1481  CG  MET A 191     -14.064  -4.222  17.254  1.00 64.88           C  \nATOM   1482  SD  MET A 191     -12.235  -4.096  17.309  1.00 64.88           S  \nATOM   1483  CE  MET A 191     -12.050  -2.419  17.966  1.00 64.88           C  \nATOM   1484  N   GLY A 192     -17.381  -0.982  15.617  1.00 55.59           N  \nATOM   1485  CA  GLY A 192     -18.124   0.269  15.582  1.00 55.59           C  \nATOM   1486  C   GLY A 192     -19.590   0.096  15.885  1.00 55.59           C  \nATOM   1487  O   GLY A 192     -20.178   0.889  16.632  1.00 55.59           O  \nTER    1488      GLY A 192                                                       \nEND   \n","pdb");
	viewer_17019975623954616.zoomTo();
	viewer_17019975623954616.setBackgroundColor("white");
	viewer_17019975623954616.setStyle({"chain": "A"},{"cartoon": {"color": "red"}});
	viewer_17019975623954616.setStyle({"chain": "B"},{"cartoon": {"color": "blue"}});
viewer_17019975623954616.render();
});
</script></div></div>
</div>
<div class="cell docutils container">
<div class="cell_input docutils container">
<div class="highlight-ipython3 notranslate"><div class="highlight"><pre><span></span><span class="c1">#@title **9. Package and Download**</span>

<span class="c1">#@markdown If you are having issues downloading the result archive, try disabling your adblocker and run this cell again. If that fails click on the little folder icon to the left, navigate to file: `jobname.result.zip`, right-click and select \&quot;Download\&quot; (see [screenshot](https://pbs.twimg.com/media/E6wRW2lWUAEOuoe?format=jpg&amp;name=small)).</span>

<span class="k">if</span> <span class="n">msa_mode</span> <span class="o">==</span> <span class="s2">&quot;custom&quot;</span><span class="p">:</span>
  <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;Don&#39;t forget to cite your custom MSA generation method.&quot;</span><span class="p">)</span>

<span class="n">files</span><span class="o">.</span><span class="n">download</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.result.zip&quot;</span><span class="p">)</span>

<span class="k">if</span> <span class="n">save_to_google_drive</span> <span class="o">==</span> <span class="kc">True</span> <span class="ow">and</span> <span class="n">drive</span><span class="p">:</span>
  <span class="n">uploaded</span> <span class="o">=</span> <span class="n">drive</span><span class="o">.</span><span class="n">CreateFile</span><span class="p">({</span><span class="s1">&#39;title&#39;</span><span class="p">:</span> <span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.result.zip&quot;</span><span class="p">})</span>
  <span class="n">uploaded</span><span class="o">.</span><span class="n">SetContentFile</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;</span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.result.zip&quot;</span><span class="p">)</span>
  <span class="n">uploaded</span><span class="o">.</span><span class="n">Upload</span><span class="p">()</span>
  <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">&quot;Uploaded </span><span class="si">{</span><span class="n">jobname</span><span class="si">}</span><span class="s2">.result.zip to Google Drive with ID </span><span class="si">{</span><span class="n">uploaded</span><span class="o">.</span><span class="n">get</span><span class="p">(</span><span class="s1">&#39;id&#39;</span><span class="p">)</span><span class="si">}</span><span class="s2">&quot;</span><span class="p">)</span>
</pre></div>
</div>
</div>
<div class="cell_output docutils container">
<script type="application/javascript">
    async function download(id, filename, size) {
      if (!google.colab.kernel.accessAllowed) {
        return;
      }
      const div = document.createElement('div');
      const label = document.createElement('label');
      label.textContent = `Downloading "${filename}": `;
      div.appendChild(label);
      const progress = document.createElement('progress');
      progress.max = size;
      div.appendChild(progress);
      document.body.appendChild(div);

      const buffers = [];
      let downloaded = 0;

      const channel = await google.colab.kernel.comms.open(id);
      // Send a message to notify the kernel that we're ready.
      channel.send({})

      for await (const message of channel.messages) {
        // Send a message to notify the kernel that we're ready.
        channel.send({})
        if (message.buffers) {
          for (const buffer of message.buffers) {
            buffers.push(buffer);
            downloaded += buffer.byteLength;
            progress.value = downloaded;
          }
        }
      }
      const blob = new Blob(buffers, {type: 'application/binary'});
      const a = document.createElement('a');
      a.href = window.URL.createObjectURL(blob);
      a.download = filename;
      div.appendChild(a);
      a.click();
      div.remove();
    }
  </script><script type="application/javascript">download("download_d2010ba7-b6ad-46a1-a55a-49808bff52f7", "colabfoldbax1f16_f85cf.result.zip", 1846420)</script></div>
</div>
<p>Colab notebooks are adapted from:</p>
<ul class="simple">
<li><p><a class="reference external" href="https://colab.research.google.com/github/sokrypton/ColabFold/blob/main/AlphaFold2.ipynb">AlphaFold2 Colab</a></p></li>
<li><p><a class="reference external" href="https://www.nature.com/articles/s41592-022-01488-1">Mirdita M, Schütze K, Moriwaki Y, Heo L, Ovchinnikov S, Steinegger M. ColabFold: Making protein folding accessible to all.
<em>Nature Methods</em>, 2022</a></p></li>
</ul>
<p><em>From the original notebook:</em></p>
<blockquote>
<div><a class="reference internal image-reference" href="https://raw.githubusercontent.com/sokrypton/ColabFold/main/.github/ColabFold_Marv_Logo_Small.png"><img alt="https://raw.githubusercontent.com/sokrypton/ColabFold/main/.github/ColabFold_Marv_Logo_Small.png" class="align-right" src="https://raw.githubusercontent.com/sokrypton/ColabFold/main/.github/ColabFold_Marv_Logo_Small.png" style="height: 200px;" /></a>
</div></blockquote>
<section id="colabfold-v1-5-3-alphafold2-using-mmseqs2">
<h2>ColabFold v1.5.3: AlphaFold2 using MMseqs2<a class="headerlink" href="#colabfold-v1-5-3-alphafold2-using-mmseqs2" title="Permalink to this heading">#</a></h2>
<p>Easy to use protein structure and complex prediction using <a class="reference external" href="https://www.nature.com/articles/s41586-021-03819-2">AlphaFold2</a> and <a class="reference external" href="https://www.biorxiv.org/content/10.1101/2021.10.04.463034v1">Alphafold2-multimer</a>. Sequence alignments/templates are generated through <span class="xref myst">MMseqs2</span> and <a class="reference external" href="https://github.com/soedinglab/hh-suite">HHsearch</a>. For more details, see <a href="#Instructions">bottom</a> of the notebook, checkout the <a class="reference external" href="https://github.com/sokrypton/ColabFold">ColabFold GitHub</a> and read our manuscript.</p>
<p>Old versions: <a class="reference external" href="https://colab.research.google.com/github/sokrypton/ColabFold/blob/v1.4.0/AlphaFold2.ipynb">v1.4</a>, <a class="reference external" href="https://colab.research.google.com/github/sokrypton/ColabFold/blob/v1.5.1/AlphaFold2.ipynb">v1.5.1</a>, <a class="reference external" href="https://colab.research.google.com/github/sokrypton/ColabFold/blob/v1.5.2/AlphaFold2.ipynb">v1.5.2</a></p>
<p><a class="reference external" href="https://www.nature.com/articles/s41592-022-01488-1">Mirdita M, Schütze K, Moriwaki Y, Heo L, Ovchinnikov S, Steinegger M. ColabFold: Making protein folding accessible to all.
<em>Nature Methods</em>, 2022</a></p>
<hr class="docutils" />
<section id="news">
<h3>News<a class="headerlink" href="#news" title="Permalink to this heading">#</a></h3>
<ul class="simple">
<li><p><b><font color='green'>2023/07/31: The ColabFold MSA server is back to normal. It was using older DB (UniRef30 2202/PDB70 220313) from 27th ~8:30 AM CEST to 31st ~11:10 AM CEST.</font></b></p></li>
<li><p><b><font color='green'>2023/06/12: New databases! UniRef30 updated to 2023_02 and PDB to 230517. We now use PDB100 instead of PDB70 (see <span class="xref myst">notes</span>).</font></b></p></li>
<li><p><b><font color='green'>2023/06/12: We introduced a new default pairing strategy: Previously, for multimer predictions with more than 2 chains, we only pair if all sequences taxonomically match (“complete” pairing). The new default “greedy” strategy pairs any taxonomically matching subsets.</font></b></p></li>
</ul>
</section>
</section>
</section>
<section class="tex2jax_ignore mathjax_ignore" id="instructions-a-name-instructions-a">
<h1>Instructions <a name="Instructions"></a><a class="headerlink" href="#instructions-a-name-instructions-a" title="Permalink to this heading">#</a></h1>
<p><strong>Quick start</strong></p>
<ol class="arabic simple">
<li><p>Paste your protein sequence(s) in the input field.</p></li>
<li><p>Press “Runtime” -&gt; “Run all”.</p></li>
<li><p>The pipeline consists of 5 steps. The currently running step is indicated by a circle with a stop sign next to it.</p></li>
</ol>
<p><strong>Result zip file contents</strong></p>
<ol class="arabic simple">
<li><p>PDB formatted structures sorted by avg. pLDDT and complexes are sorted by pTMscore. (unrelaxed and relaxed if <code class="docutils literal notranslate"><span class="pre">use_amber</span></code> is enabled).</p></li>
<li><p>Plots of the model quality.</p></li>
<li><p>Plots of the MSA coverage.</p></li>
<li><p>Parameter log file.</p></li>
<li><p>A3M formatted input MSA.</p></li>
<li><p>A <code class="docutils literal notranslate"><span class="pre">predicted_aligned_error_v1.json</span></code> using <a class="reference external" href="https://alphafold.ebi.ac.uk/faq#faq-7">AlphaFold-DB’s format</a> and a <code class="docutils literal notranslate"><span class="pre">scores.json</span></code> for each model which contains  an array (list of lists) for PAE, a list with the average pLDDT and the pTMscore.</p></li>
<li><p>BibTeX file with citations for all used tools and databases.</p></li>
</ol>
<p>At the end of the job a download modal box will pop up with a <code class="docutils literal notranslate"><span class="pre">jobname.result.zip</span></code> file. Additionally, if the <code class="docutils literal notranslate"><span class="pre">save_to_google_drive</span></code> option was selected, the <code class="docutils literal notranslate"><span class="pre">jobname.result.zip</span></code> will be uploaded to your Google Drive.</p>
<p><strong>License</strong>
<a class="reference external" href="http://githubusercontent.com/sokrypton/ColabFold/main/LICENSE">githubusercontent.com/sokrypton/ColabFold/main/LICENSE</a>). Additionally, this &gt; notebook uses the AlphaFold2 source code and its parameters licensed under <a class="reference external" href="https://raw.githubusercontent.com/deepmind/alphafold/main/LICENSE">Apache 2.0</a> &gt; and <a class="reference external" href="https://creativecommons.org/licenses/by-sa/4.0/">CC BY 4.0</a> respectively. Read more about the AlphaFold license [here](<a class="github reference external" href="https://github.com/deepmind/">deepmind/</a>&gt; alphafold).</p>
<p><strong>Acknowledgments</strong></p>
<ul class="simple">
<li><p>We thank the AlphaFold team for developing an excellent model and open &gt; sourcing the software.</p></li>
<li><p><a class="reference external" href="https://kobic.re.kr">KOBIC</a> and <a class="reference external" href="https://www.mpinat.mpg.de/soeding">Söding Lab</a> for providing the computational resources for the MMseqs2 MSA server.</p></li>
<li><p>Richard Evans for helping to benchmark the ColabFold’s Alphafold-multimer support.</p></li>
<li><p><a class="reference external" href="https://github.com/dkoes">David Koes</a> for his awesome <a class="reference external" href="https://3dmol.csb.pitt.edu/">py3Dmol</a> plugin, without whom these notebooks would be quite boring!</p></li>
<li><p>Do-Yoon Kim for creating the ColabFold logo.</p></li>
<li><p>A colab by Sergey Ovchinnikov (<a class="reference external" href="https://twitter.com/sokrypton">&#64;sokrypton</a>), Milot Mirdita (<a class="reference external" href="https://twitter.com/milot_mirdita">&#64;milot_mirdita</a>) and Martin Steinegger (<a class="reference external" href="https://twitter.com/thesteinegger">&#64;thesteinegger</a>).</p></li>
</ul>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./ColabFold"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

                </article>
              

              
              
                <footer class="bd-footer-article">
                  
<div class="footer-article-items footer-article__inner">
  
    <div class="footer-article-item"><!-- Previous / next buttons -->
<div class="prev-next-area">
    <a class="left-prev"
       href="AlphaFold2.html"
       title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
        <p class="prev-next-subtitle">previous</p>
        <p class="prev-next-title">*Original ColabFold Notebook</p>
      </div>
    </a>
    <a class="right-next"
       href="Tutorial-Multimer.html"
       title="next page">
      <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">1. <strong>ColabFold Tutorial:</strong> Predicting Protein-Protein Complexes of TA Systems</p>
      </div>
      <i class="fa-solid fa-angle-right"></i>
    </a>
</div></div>
  
</div>

                </footer>
              
            </div>
            
            
              
                <div class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">

  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h1 nav-item toc-entry"><a class="reference internal nav-link" href="#">1. <strong>ColabFold Tutorial:</strong> Predicting the structure of Bax protein</a><ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#colabfold-v1-5-3-alphafold2-using-mmseqs2">ColabFold v1.5.3: AlphaFold2 using MMseqs2</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#news">News</a></li>
</ul>
</li>
</ul>
</li>
<li class="toc-h1 nav-item toc-entry"><a class="reference internal nav-link" href="#instructions-a-name-instructions-a">Instructions <a name="Instructions"></a></a></li>
</ul>

  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By CCATS Group
</p>

  </div>
  
  <div class="footer-item">
    
  <p class="copyright">
    
      © Copyright 2023.
      <br/>
    
  </p>

  </div>
  
  <div class="footer-item">
    
  </div>
  
  <div class="footer-item">
    
  </div>
  
</div>
          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../_static/scripts/bootstrap.js?digest=e353d410970836974a52"></script>
<script src="../_static/scripts/pydata-sphinx-theme.js?digest=e353d410970836974a52"></script>

  <footer class="bd-footer">
  </footer>
  </body>
</html>